{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "6db67447",
   "metadata": {},
   "outputs": [],
   "source": [
    "from Bio import Entrez\n",
    "from Bio import SeqIO"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "fa508205",
   "metadata": {},
   "outputs": [],
   "source": [
    "#setting your email\n",
    "Entrez.email='rakeshgartaula983@gmail.com'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "id": "9bad737d",
   "metadata": {},
   "outputs": [],
   "source": [
    "#loading single fasta file for the given id\n",
    "handle=Entrez.efetch(db='nucleotide',id='186972394',rettype='fasta')\n",
    "record=SeqIO.read(handle,'fasta')\n",
    "outputname='D:/ADSS_matL.fasta'\n",
    "\n",
    "\n",
    "#for multiple fasta files with multiple ids- we need to parse the fasta sequences with SeqIO\n",
    "#record=SeqIO.parse(handle,'fasta')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "id": "be58cbf0",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "1"
      ]
     },
     "execution_count": 34,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "SeqIO.write(record,outputname,'fasta')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "id": "9e35fcfb",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Id 2240894608\n",
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZC010000000</GBSeq_locus>\\n    <GBSeq_length>322</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Candidatus Bipolaricaulota bacterium isolate HOT.Cluster.56, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZC000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZC000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZC000000000.1|JAILZC010000000</GBSeqid>\\n      <GBSeqid>gi|2240894608</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Candidatus Bipolaricaulota bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Candidatus Bipolaricaulota bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Candidatus Bipolaricaulota</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..322</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..322</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Candidatus Bipolaricaulota bacterium whole genome shotgun (WGS) project has the project accession JAILZC000000000. This version of the project (01) has the accession number JAILZC010000000, and consists of sequences JAILZC010000001-JAILZC010000322.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 41.4x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:28:30 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 1,727 ; CDSs (total) :: 1,689 ; Genes (coding) :: 1,663 ; CDSs (with protein) :: 1,663 ; Genes (RNA) :: 38 ; rRNAs :: 2, 1 (16S, 23S) ; partial rRNAs :: 2, 1 (16S, 23S) ; tRNAs :: 32 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 26 ; CDSs (without protein) :: 26 ; Pseudo Genes (ambiguous residues) :: 0 of 26 ; Pseudo Genes (frameshifted) :: 6 of 26 ; Pseudo Genes (incomplete) :: 17 of 26 ; Pseudo Genes (internal stop) :: 4 of 26 ; Pseudo Genes (multiple problems) :: 1 of 26 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..322</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>322</GBInterval_to>\\n            <GBInterval_accession>JAILZC000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Candidatus Bipolaricaulota bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.Cluster.56</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2306012</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZC010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZC010000322</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023700</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Candidatus Bipolaricaulota bacterium isolate HOT.Cluster.56, whole genome shotgun sequencing project', 'Candidatus Bipolaricaulota bacterium (hot springs metagenome)', 'Candidatus Bipolaricaulota bacterium', 'Bacteria; Candidatus Bipolaricaulota']\n",
      "Id 2240894606\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZD010000000</GBSeq_locus>\\n    <GBSeq_length>679</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Dehalococcoidia bacterium isolate HOT.Cluster.58, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZD000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZD000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZD000000000.1|JAILZD010000000</GBSeqid>\\n      <GBSeqid>gi|2240894606</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Dehalococcoidia bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Dehalococcoidia bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Chloroflexi; Dehalococcoidia</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..679</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..679</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Dehalococcoidia bacterium whole genome shotgun (WGS) project has the project accession JAILZD000000000. This version of the project (01) has the accession number JAILZD010000000, and consists of sequences JAILZD010000001-JAILZD010000679.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 23.7x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 13:39:08 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,476 ; CDSs (total) :: 3,420 ; Genes (coding) :: 3,342 ; CDSs (with protein) :: 3,342 ; Genes (RNA) :: 56 ; rRNAs :: 1, 1 (5S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1 (23S) ; tRNAs :: 50 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 78 ; CDSs (without protein) :: 78 ; Pseudo Genes (ambiguous residues) :: 0 of 78 ; Pseudo Genes (frameshifted) :: 4 of 78 ; Pseudo Genes (incomplete) :: 66 of 78 ; Pseudo Genes (internal stop) :: 9 of 78 ; Pseudo Genes (multiple problems) :: 1 of 78 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..679</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>679</GBInterval_to>\\n            <GBInterval_accession>JAILZD000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Dehalococcoidia bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.Cluster.58</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2026734</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZD010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZD010000679</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023701</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Dehalococcoidia bacterium isolate HOT.Cluster.58, whole genome shotgun sequencing project', 'Dehalococcoidia bacterium (hot springs metagenome)', 'Dehalococcoidia bacterium', 'Bacteria; Chloroflexi; Dehalococcoidia']\n",
      "Id 2240894605\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZB010000000</GBSeq_locus>\\n    <GBSeq_length>461</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Chloroflexi bacterium isolate HOT.Cluster.3, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZB000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZB000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZB000000000.1|JAILZB010000000</GBSeqid>\\n      <GBSeqid>gi|2240894605</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Chloroflexi bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Chloroflexi bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Chloroflexi</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..461</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..461</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Chloroflexi bacterium whole genome shotgun (WGS) project has the project accession JAILZB000000000. This version of the project (01) has the accession number JAILZB010000000, and consists of sequences JAILZB010000001-JAILZB010000461.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 13.1x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 13:46:36 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 4,087 ; CDSs (total) :: 4,034 ; Genes (coding) :: 3,968 ; CDSs (with protein) :: 3,968 ; Genes (RNA) :: 53 ; rRNAs :: 1, 1, 2 (5S, 16S, 23S) ; complete rRNAs :: 1, 1 (5S, 16S) ; partial rRNAs :: 2 (23S) ; tRNAs :: 46 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 66 ; CDSs (without protein) :: 66 ; Pseudo Genes (ambiguous residues) :: 0 of 66 ; Pseudo Genes (frameshifted) :: 13 of 66 ; Pseudo Genes (incomplete) :: 44 of 66 ; Pseudo Genes (internal stop) :: 14 of 66 ; Pseudo Genes (multiple problems) :: 5 of 66 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..461</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>461</GBInterval_to>\\n            <GBInterval_accession>JAILZB000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Chloroflexi bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.Cluster.3</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2026724</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZB010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZB010000461</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023699</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Chloroflexi bacterium isolate HOT.Cluster.3, whole genome shotgun sequencing project', 'Chloroflexi bacterium (hot springs metagenome)', 'Chloroflexi bacterium', 'Bacteria; Chloroflexi']\n",
      "Id 2240894604\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZG010000000</GBSeq_locus>\\n    <GBSeq_length>172</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Firmicutes bacterium isolate HOT.Cluster.78, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZG000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZG000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZG000000000.1|JAILZG010000000</GBSeqid>\\n      <GBSeqid>gi|2240894604</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Firmicutes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Firmicutes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..172</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..172</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Firmicutes bacterium whole genome shotgun (WGS) project has the project accession JAILZG000000000. This version of the project (01) has the accession number JAILZG010000000, and consists of sequences JAILZG010000001-JAILZG010000172.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 17.5x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:42:59 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 1,737 ; CDSs (total) :: 1,715 ; Genes (coding) :: 1,686 ; CDSs (with protein) :: 1,686 ; Genes (RNA) :: 22 ; rRNAs :: 1, 1 (5S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1 (23S) ; tRNAs :: 19 ; ncRNAs :: 1 ; Pseudo Genes (total) :: 29 ; CDSs (without protein) :: 29 ; Pseudo Genes (ambiguous residues) :: 0 of 29 ; Pseudo Genes (frameshifted) :: 6 of 29 ; Pseudo Genes (incomplete) :: 20 of 29 ; Pseudo Genes (internal stop) :: 3 of 29 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..172</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>172</GBInterval_to>\\n            <GBInterval_accession>JAILZG000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Firmicutes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.Cluster.78</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1879010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZG010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZG010000172</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023704</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Firmicutes bacterium isolate HOT.Cluster.78, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes']\n",
      "Id 2240894603\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZE010000000</GBSeq_locus>\\n    <GBSeq_length>555</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Candidatus Rokubacteria bacterium isolate HOT.Cluster.5, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZE000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZE000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZE000000000.1|JAILZE010000000</GBSeqid>\\n      <GBSeqid>gi|2240894603</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Candidatus Rokubacteria bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Candidatus Rokubacteria bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Candidatus Rokubacteria; unclassified Candidatus Rokubacteria</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..555</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..555</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Candidatus Rokubacteria bacterium whole genome shotgun (WGS) project has the project accession JAILZE000000000. This version of the project (01) has the accession number JAILZE010000000, and consists of sequences JAILZE010000001-JAILZE010000555.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 14.4x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 11:26:23 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,007 ; CDSs (total) :: 2,000 ; Genes (coding) :: 1,962 ; CDSs (with protein) :: 1,962 ; Genes (RNA) :: 7 ; tRNAs :: 7 ; ncRNAs :: 0 ; Pseudo Genes (total) :: 38 ; CDSs (without protein) :: 38 ; Pseudo Genes (ambiguous residues) :: 0 of 38 ; Pseudo Genes (frameshifted) :: 3 of 38 ; Pseudo Genes (incomplete) :: 24 of 38 ; Pseudo Genes (internal stop) :: 12 of 38 ; Pseudo Genes (multiple problems) :: 1 of 38 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..555</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>555</GBInterval_to>\\n            <GBInterval_accession>JAILZE000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Candidatus Rokubacteria bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.Cluster.5</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2053607</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZE010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZE010000555</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023702</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Candidatus Rokubacteria bacterium isolate HOT.Cluster.5, whole genome shotgun sequencing project', 'Candidatus Rokubacteria bacterium (hot springs metagenome)', 'Candidatus Rokubacteria bacterium', 'Bacteria; Candidatus Rokubacteria; unclassified Candidatus Rokubacteria']\n",
      "Id 2240894602\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZF010000000</GBSeq_locus>\\n    <GBSeq_length>214</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Alicyclobacillus sp. isolate HOT.Cluster.61, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZF000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZF000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZF000000000.1|JAILZF010000000</GBSeqid>\\n      <GBSeqid>gi|2240894602</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Alicyclobacillus sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Alicyclobacillus sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..214</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..214</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Alicyclobacillus sp. whole genome shotgun (WGS) project has the project accession JAILZF000000000. This version of the project (01) has the accession number JAILZF010000000, and consists of sequences JAILZF010000001-JAILZF010000214.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 16.9x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 09:47:40 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 1,773 ; CDSs (total) :: 1,759 ; Genes (coding) :: 1,747 ; CDSs (with protein) :: 1,747 ; Genes (RNA) :: 14 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 9 ; ncRNAs :: 2 ; Pseudo Genes (total) :: 12 ; CDSs (without protein) :: 12 ; Pseudo Genes (ambiguous residues) :: 0 of 12 ; Pseudo Genes (frameshifted) :: 1 of 12 ; Pseudo Genes (incomplete) :: 8 of 12 ; Pseudo Genes (internal stop) :: 3 of 12 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..214</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>214</GBInterval_to>\\n            <GBInterval_accession>JAILZF000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Alicyclobacillus sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.Cluster.61</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:61169</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZF010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZF010000214</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023703</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Alicyclobacillus sp. isolate HOT.Cluster.61, whole genome shotgun sequencing project', 'Alicyclobacillus sp. (hot springs metagenome)', 'Alicyclobacillus sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus']\n",
      "Id 2240894601\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZH010000000</GBSeq_locus>\\n    <GBSeq_length>341</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Peptococcaceae bacterium isolate HOT.Cluster.86, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZH000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZH000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZH000000000.1|JAILZH010000000</GBSeqid>\\n      <GBSeqid>gi|2240894601</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Peptococcaceae bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Peptococcaceae bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Clostridia; Eubacteriales; Peptococcaceae</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..341</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..341</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Peptococcaceae bacterium whole genome shotgun (WGS) project has the project accession JAILZH000000000. This version of the project (01) has the accession number JAILZH010000000, and consists of sequences JAILZH010000001-JAILZH010000341.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 38.1x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 09:49:04 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,231 ; CDSs (total) :: 2,204 ; Genes (coding) :: 2,158 ; CDSs (with protein) :: 2,158 ; Genes (RNA) :: 27 ; rRNAs :: 1, 1 (5S, 16S) ; partial rRNAs :: 1, 1 (5S, 16S) ; tRNAs :: 23 ; ncRNAs :: 2 ; Pseudo Genes (total) :: 46 ; CDSs (without protein) :: 46 ; Pseudo Genes (ambiguous residues) :: 0 of 46 ; Pseudo Genes (frameshifted) :: 12 of 46 ; Pseudo Genes (incomplete) :: 29 of 46 ; Pseudo Genes (internal stop) :: 7 of 46 ; Pseudo Genes (multiple problems) :: 2 of 46 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..341</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>341</GBInterval_to>\\n            <GBInterval_accession>JAILZH000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Peptococcaceae bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.Cluster.86</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2052179</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZH010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZH010000341</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023705</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Peptococcaceae bacterium isolate HOT.Cluster.86, whole genome shotgun sequencing project', 'Peptococcaceae bacterium (hot springs metagenome)', 'Peptococcaceae bacterium', 'Bacteria; Firmicutes; Clostridia; Eubacteriales; Peptococcaceae']\n",
      "Id 2240894593\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZJ010000000</GBSeq_locus>\\n    <GBSeq_length>217</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Alicyclobacillus herbarius isolate HOT.CON.104, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZJ000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZJ000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZJ000000000.1|JAILZJ010000000</GBSeqid>\\n      <GBSeqid>gi|2240894593</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Alicyclobacillus herbarius (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Alicyclobacillus herbarius</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..217</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..217</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Alicyclobacillus herbarius whole genome shotgun (WGS) project has the project accession JAILZJ000000000. This version of the project (01) has the accession number JAILZJ010000000, and consists of sequences JAILZJ010000001-JAILZJ010000217.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 42.2x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:21:23 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,235 ; CDSs (total) :: 3,197 ; Genes (coding) :: 3,155 ; CDSs (with protein) :: 3,155 ; Genes (RNA) :: 38 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 30 ; ncRNAs :: 5 ; Pseudo Genes (total) :: 42 ; CDSs (without protein) :: 42 ; Pseudo Genes (ambiguous residues) :: 0 of 42 ; Pseudo Genes (frameshifted) :: 16 of 42 ; Pseudo Genes (incomplete) :: 26 of 42 ; Pseudo Genes (internal stop) :: 5 of 42 ; Pseudo Genes (multiple problems) :: 5 of 42 ; CRISPR Arrays :: 3 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..217</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>217</GBInterval_to>\\n            <GBInterval_accession>JAILZJ000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Alicyclobacillus herbarius</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.104</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:122960</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZJ010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZJ010000217</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023707</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Alicyclobacillus herbarius isolate HOT.CON.104, whole genome shotgun sequencing project', 'Alicyclobacillus herbarius (hot springs metagenome)', 'Alicyclobacillus herbarius', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus']\n",
      "Id 2240894592\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZR010000000</GBSeq_locus>\\n    <GBSeq_length>729</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Firmicutes bacterium isolate HOT.CON.146, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZR000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZR000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZR000000000.1|JAILZR010000000</GBSeqid>\\n      <GBSeqid>gi|2240894592</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Firmicutes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Firmicutes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..729</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..729</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Firmicutes bacterium whole genome shotgun (WGS) project has the project accession JAILZR000000000. This version of the project (01) has the accession number JAILZR010000000, and consists of sequences JAILZR010000001-JAILZR010000729.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 14.5x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 11:55:08 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,616 ; CDSs (total) :: 2,581 ; Genes (coding) :: 2,519 ; CDSs (with protein) :: 2,519 ; Genes (RNA) :: 35 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 30 ; ncRNAs :: 2 ; Pseudo Genes (total) :: 62 ; CDSs (without protein) :: 62 ; Pseudo Genes (ambiguous residues) :: 0 of 62 ; Pseudo Genes (frameshifted) :: 16 of 62 ; Pseudo Genes (incomplete) :: 40 of 62 ; Pseudo Genes (internal stop) :: 8 of 62 ; Pseudo Genes (multiple problems) :: 2 of 62 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..729</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>729</GBInterval_to>\\n            <GBInterval_accession>JAILZR000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Firmicutes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.146</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1879010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZR010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZR010000729</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023715</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Firmicutes bacterium isolate HOT.CON.146, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes']\n",
      "Id 2240894591\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZK010000000</GBSeq_locus>\\n    <GBSeq_length>104</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Thermomonas hydrothermalis isolate HOT.CON.106, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZK000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZK000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZK000000000.1|JAILZK010000000</GBSeqid>\\n      <GBSeqid>gi|2240894591</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Thermomonas hydrothermalis (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Thermomonas hydrothermalis</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Proteobacteria; Gammaproteobacteria; Xanthomonadales; Xanthomonadaceae; Thermomonas</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..104</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..104</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Thermomonas hydrothermalis whole genome shotgun (WGS) project has the project accession JAILZK000000000. This version of the project (01) has the accession number JAILZK010000000, and consists of sequences JAILZK010000001-JAILZK010000104.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 14.3x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 09:56:04 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,366 ; CDSs (total) :: 2,319 ; Genes (coding) :: 2,301 ; CDSs (with protein) :: 2,301 ; Genes (RNA) :: 47 ; tRNAs :: 43 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 18 ; CDSs (without protein) :: 18 ; Pseudo Genes (ambiguous residues) :: 0 of 18 ; Pseudo Genes (frameshifted) :: 5 of 18 ; Pseudo Genes (incomplete) :: 13 of 18 ; Pseudo Genes (internal stop) :: 1 of 18 ; Pseudo Genes (multiple problems) :: 1 of 18 ; CRISPR Arrays :: 2 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..104</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>104</GBInterval_to>\\n            <GBInterval_accession>JAILZK000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Thermomonas hydrothermalis</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.106</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:213588</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZK010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZK010000104</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023708</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Thermomonas hydrothermalis isolate HOT.CON.106, whole genome shotgun sequencing project', 'Thermomonas hydrothermalis (hot springs metagenome)', 'Thermomonas hydrothermalis', 'Bacteria; Proteobacteria; Gammaproteobacteria; Xanthomonadales; Xanthomonadaceae; Thermomonas']\n",
      "Id 2240894590\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZO010000000</GBSeq_locus>\\n    <GBSeq_length>556</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Alicyclobacillus shizuokensis isolate HOT.CON.119, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZO000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZO000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZO000000000.1|JAILZO010000000</GBSeqid>\\n      <GBSeqid>gi|2240894590</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Alicyclobacillus shizuokensis (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Alicyclobacillus shizuokensis</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..556</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..556</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Alicyclobacillus shizuokensis whole genome shotgun (WGS) project has the project accession JAILZO000000000. This version of the project (01) has the accession number JAILZO010000000, and consists of sequences JAILZO010000001-JAILZO010000556.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 13.2x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:26:26 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,086 ; CDSs (total) :: 3,061 ; Genes (coding) :: 3,014 ; CDSs (with protein) :: 3,014 ; Genes (RNA) :: 25 ; rRNAs :: 1, 1 (5S, 16S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1 (16S) ; tRNAs :: 20 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 47 ; CDSs (without protein) :: 47 ; Pseudo Genes (ambiguous residues) :: 0 of 47 ; Pseudo Genes (frameshifted) :: 20 of 47 ; Pseudo Genes (incomplete) :: 26 of 47 ; Pseudo Genes (internal stop) :: 7 of 47 ; Pseudo Genes (multiple problems) :: 5 of 47 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..556</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>556</GBInterval_to>\\n            <GBInterval_accession>JAILZO000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Alicyclobacillus shizuokensis</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.119</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:392014</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZO010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZO010000556</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023712</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Alicyclobacillus shizuokensis isolate HOT.CON.119, whole genome shotgun sequencing project', 'Alicyclobacillus shizuokensis (hot springs metagenome)', 'Alicyclobacillus shizuokensis', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus']\n",
      "Id 2240894589\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZM010000000</GBSeq_locus>\\n    <GBSeq_length>878</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Anoxybacillus ayderensis isolate HOT.CON.111, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZM000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZM000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZM000000000.1|JAILZM010000000</GBSeqid>\\n      <GBSeqid>gi|2240894589</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Anoxybacillus ayderensis (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Anoxybacillus ayderensis</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Bacillaceae; Anoxybacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..878</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..878</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Anoxybacillus ayderensis whole genome shotgun (WGS) project has the project accession JAILZM000000000. This version of the project (01) has the accession number JAILZM010000000, and consists of sequences JAILZM010000001-JAILZM010000878.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 14x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 09:58:52 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,790 ; CDSs (total) :: 2,776 ; Genes (coding) :: 2,642 ; CDSs (with protein) :: 2,642 ; Genes (RNA) :: 14 ; rRNAs :: 1 (16S) ; partial rRNAs :: 1 (16S) ; tRNAs :: 10 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 134 ; CDSs (without protein) :: 134 ; Pseudo Genes (ambiguous residues) :: 0 of 134 ; Pseudo Genes (frameshifted) :: 13 of 134 ; Pseudo Genes (incomplete) :: 81 of 134 ; Pseudo Genes (internal stop) :: 45 of 134 ; Pseudo Genes (multiple problems) :: 5 of 134 ; CRISPR Arrays :: 2 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..878</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>878</GBInterval_to>\\n            <GBInterval_accession>JAILZM000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Anoxybacillus ayderensis</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.111</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:265546</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZM010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZM010000878</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023710</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Anoxybacillus ayderensis isolate HOT.CON.111, whole genome shotgun sequencing project', 'Anoxybacillus ayderensis (hot springs metagenome)', 'Anoxybacillus ayderensis', 'Bacteria; Firmicutes; Bacilli; Bacillales; Bacillaceae; Anoxybacillus']\n",
      "Id 2240894588\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZN010000000</GBSeq_locus>\\n    <GBSeq_length>1094</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Armatimonadetes bacterium isolate HOT.CON.113, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZN000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZN000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZN000000000.1|JAILZN010000000</GBSeqid>\\n      <GBSeqid>gi|2240894588</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Armatimonadetes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Armatimonadetes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Armatimonadetes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..1094</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..1094</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Armatimonadetes bacterium whole genome shotgun (WGS) project has the project accession JAILZN000000000. This version of the project (01) has the accession number JAILZN010000000, and consists of sequences JAILZN010000001-JAILZN010001094.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 14.4x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 11:48:59 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,086 ; CDSs (total) :: 3,056 ; Genes (coding) :: 3,023 ; CDSs (with protein) :: 3,023 ; Genes (RNA) :: 30 ; rRNAs :: 1, 2 (5S, 23S) ; partial rRNAs :: 1, 2 (5S, 23S) ; tRNAs :: 24 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 33 ; CDSs (without protein) :: 33 ; Pseudo Genes (ambiguous residues) :: 0 of 33 ; Pseudo Genes (frameshifted) :: 2 of 33 ; Pseudo Genes (incomplete) :: 17 of 33 ; Pseudo Genes (internal stop) :: 14 of 33 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..1094</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>1094</GBInterval_to>\\n            <GBInterval_accession>JAILZN000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Armatimonadetes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.113</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2033014</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZN010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZN010001094</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023711</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Armatimonadetes bacterium isolate HOT.CON.113, whole genome shotgun sequencing project', 'Armatimonadetes bacterium (hot springs metagenome)', 'Armatimonadetes bacterium', 'Bacteria; Armatimonadetes']\n",
      "Id 2240894583\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZI010000000</GBSeq_locus>\\n    <GBSeq_length>661</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Fimbriimonadales bacterium isolate HOT.CON.102, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZI000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZI000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZI000000000.1|JAILZI010000000</GBSeqid>\\n      <GBSeqid>gi|2240894583</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fimbriimonadales bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Fimbriimonadales bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Armatimonadetes; Fimbriimonadia; Fimbriimonadales; unclassified Fimbriimonadales</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..661</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..661</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Fimbriimonadales bacterium whole genome shotgun (WGS) project has the project accession JAILZI000000000. This version of the project (01) has the accession number JAILZI010000000, and consists of sequences JAILZI010000001-JAILZI010000661.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 34.8x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 09:37:11 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,016 ; CDSs (total) :: 1,985 ; Genes (coding) :: 1,968 ; CDSs (with protein) :: 1,968 ; Genes (RNA) :: 31 ; rRNAs :: 1, 1 (16S, 23S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 27 ; ncRNAs :: 2 ; Pseudo Genes (total) :: 17 ; CDSs (without protein) :: 17 ; Pseudo Genes (ambiguous residues) :: 0 of 17 ; Pseudo Genes (frameshifted) :: 1 of 17 ; Pseudo Genes (incomplete) :: 16 of 17 ; Pseudo Genes (internal stop) :: 0 of 17 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..661</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>661</GBInterval_to>\\n            <GBInterval_accession>JAILZI000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fimbriimonadales bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.102</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2682145</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZI010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZI010000661</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023706</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Fimbriimonadales bacterium isolate HOT.CON.102, whole genome shotgun sequencing project', 'Fimbriimonadales bacterium (hot springs metagenome)', 'Fimbriimonadales bacterium', 'Bacteria; Armatimonadetes; Fimbriimonadia; Fimbriimonadales; unclassified Fimbriimonadales']\n",
      "Id 2240894582\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZL010000000</GBSeq_locus>\\n    <GBSeq_length>73</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Syntrophobacterales bacterium isolate HOT.CON.10, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZL000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZL000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZL000000000.1|JAILZL010000000</GBSeqid>\\n      <GBSeqid>gi|2240894582</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Syntrophobacterales bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Syntrophobacterales bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Proteobacteria; Deltaproteobacteria; Syntrophobacterales</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..73</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..73</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Syntrophobacterales bacterium whole genome shotgun (WGS) project has the project accession JAILZL000000000. This version of the project (01) has the accession number JAILZL010000000, and consists of sequences JAILZL010000001-JAILZL010000073.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 15x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:12:39 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,581 ; CDSs (total) :: 2,531 ; Genes (coding) :: 2,523 ; CDSs (with protein) :: 2,523 ; Genes (RNA) :: 50 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 44 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 8 ; CDSs (without protein) :: 8 ; Pseudo Genes (ambiguous residues) :: 0 of 8 ; Pseudo Genes (frameshifted) :: 2 of 8 ; Pseudo Genes (incomplete) :: 5 of 8 ; Pseudo Genes (internal stop) :: 2 of 8 ; Pseudo Genes (multiple problems) :: 1 of 8 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..73</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>73</GBInterval_to>\\n            <GBInterval_accession>JAILZL000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Syntrophobacterales bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.10</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2282155</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZL010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZL010000073</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023709</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Syntrophobacterales bacterium isolate HOT.CON.10, whole genome shotgun sequencing project', 'Syntrophobacterales bacterium (hot springs metagenome)', 'Syntrophobacterales bacterium', 'Bacteria; Proteobacteria; Deltaproteobacteria; Syntrophobacterales']\n",
      "Id 2240894581\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZP010000000</GBSeq_locus>\\n    <GBSeq_length>80</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Peptococcaceae bacterium isolate HOT.CON.130, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZP000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZP000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZP000000000.1|JAILZP010000000</GBSeqid>\\n      <GBSeqid>gi|2240894581</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Peptococcaceae bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Peptococcaceae bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Clostridia; Eubacteriales; Peptococcaceae</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..80</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..80</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Peptococcaceae bacterium whole genome shotgun (WGS) project has the project accession JAILZP000000000. This version of the project (01) has the accession number JAILZP010000000, and consists of sequences JAILZP010000001-JAILZP010000080.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 15x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:25:56 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,161 ; CDSs (total) :: 3,119 ; Genes (coding) :: 3,098 ; CDSs (with protein) :: 3,098 ; Genes (RNA) :: 42 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 34 ; ncRNAs :: 5 ; Pseudo Genes (total) :: 21 ; CDSs (without protein) :: 21 ; Pseudo Genes (ambiguous residues) :: 0 of 21 ; Pseudo Genes (frameshifted) :: 7 of 21 ; Pseudo Genes (incomplete) :: 15 of 21 ; Pseudo Genes (internal stop) :: 1 of 21 ; Pseudo Genes (multiple problems) :: 2 of 21 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..80</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>80</GBInterval_to>\\n            <GBInterval_accession>JAILZP000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Peptococcaceae bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.130</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2052179</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZP010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZP010000080</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023713</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Peptococcaceae bacterium isolate HOT.CON.130, whole genome shotgun sequencing project', 'Peptococcaceae bacterium (hot springs metagenome)', 'Peptococcaceae bacterium', 'Bacteria; Firmicutes; Clostridia; Eubacteriales; Peptococcaceae']\n",
      "Id 2240894580\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZQ010000000</GBSeq_locus>\\n    <GBSeq_length>263</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Geminicoccaceae bacterium isolate HOT.CON.139, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZQ000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZQ000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZQ000000000.1|JAILZQ010000000</GBSeqid>\\n      <GBSeqid>gi|2240894580</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Geminicoccaceae bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Geminicoccaceae bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Proteobacteria; Alphaproteobacteria; Rhodospirillales; Geminicoccaceae</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..263</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..263</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Geminicoccaceae bacterium whole genome shotgun (WGS) project has the project accession JAILZQ000000000. This version of the project (01) has the accession number JAILZQ010000000, and consists of sequences JAILZQ010000001-JAILZQ010000263.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 14x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:34:32 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,518 ; CDSs (total) :: 3,478 ; Genes (coding) :: 3,452 ; CDSs (with protein) :: 3,452 ; Genes (RNA) :: 40 ; rRNAs :: 1 (16S) ; partial rRNAs :: 1 (16S) ; tRNAs :: 37 ; ncRNAs :: 2 ; Pseudo Genes (total) :: 26 ; CDSs (without protein) :: 26 ; Pseudo Genes (ambiguous residues) :: 0 of 26 ; Pseudo Genes (frameshifted) :: 5 of 26 ; Pseudo Genes (incomplete) :: 17 of 26 ; Pseudo Genes (internal stop) :: 4 of 26 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..263</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>263</GBInterval_to>\\n            <GBInterval_accession>JAILZQ000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Geminicoccaceae bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.139</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2448052</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZQ010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZQ010000263</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023714</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Geminicoccaceae bacterium isolate HOT.CON.139, whole genome shotgun sequencing project', 'Geminicoccaceae bacterium (hot springs metagenome)', 'Geminicoccaceae bacterium', 'Bacteria; Proteobacteria; Alphaproteobacteria; Rhodospirillales; Geminicoccaceae']\n",
      "Id 2240894574\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZT010000000</GBSeq_locus>\\n    <GBSeq_length>49</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Paenibacillus sp. isolate HOT.CON.154, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZT000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZT000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZT000000000.1|JAILZT010000000</GBSeqid>\\n      <GBSeqid>gi|2240894574</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Paenibacillus sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Paenibacillus sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Paenibacillaceae; Paenibacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..49</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..49</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Paenibacillus sp. whole genome shotgun (WGS) project has the project accession JAILZT000000000. This version of the project (01) has the accession number JAILZT010000000, and consists of sequences JAILZT010000001-JAILZT010000049.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 17x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 11:34:20 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 5,322 ; CDSs (total) :: 5,268 ; Genes (coding) :: 5,214 ; CDSs (with protein) :: 5,214 ; Genes (RNA) :: 54 ; rRNAs :: 2, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 2, 1 (5S, 16S) ; partial rRNAs :: 1, 3 (16S, 23S) ; tRNAs :: 43 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 54 ; CDSs (without protein) :: 54 ; Pseudo Genes (ambiguous residues) :: 0 of 54 ; Pseudo Genes (frameshifted) :: 23 of 54 ; Pseudo Genes (incomplete) :: 30 of 54 ; Pseudo Genes (internal stop) :: 11 of 54 ; Pseudo Genes (multiple problems) :: 8 of 54 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..49</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>49</GBInterval_to>\\n            <GBInterval_accession>JAILZT000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Paenibacillus sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.154</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:58172</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZT010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZT010000049</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023717</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Paenibacillus sp. isolate HOT.CON.154, whole genome shotgun sequencing project', 'Paenibacillus sp. (hot springs metagenome)', 'Paenibacillus sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Paenibacillaceae; Paenibacillus']\n",
      "Id 2240894573\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZW010000000</GBSeq_locus>\\n    <GBSeq_length>349</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Alicyclobacillus macrosporangiidus isolate HOT.CON.25, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZW000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZW000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZW000000000.1|JAILZW010000000</GBSeqid>\\n      <GBSeqid>gi|2240894573</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Alicyclobacillus macrosporangiidus (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Alicyclobacillus macrosporangiidus</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..349</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..349</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Alicyclobacillus macrosporangiidus whole genome shotgun (WGS) project has the project accession JAILZW000000000. This version of the project (01) has the accession number JAILZW010000000, and consists of sequences JAILZW010000001-JAILZW010000349.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 24.9x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:44:22 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 4,249 ; CDSs (total) :: 4,205 ; Genes (coding) :: 4,125 ; CDSs (with protein) :: 4,125 ; Genes (RNA) :: 44 ; rRNAs :: 1, 2, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 1 (16S, 23S) ; tRNAs :: 38 ; ncRNAs :: 2 ; Pseudo Genes (total) :: 80 ; CDSs (without protein) :: 80 ; Pseudo Genes (ambiguous residues) :: 0 of 80 ; Pseudo Genes (frameshifted) :: 31 of 80 ; Pseudo Genes (incomplete) :: 45 of 80 ; Pseudo Genes (internal stop) :: 14 of 80 ; Pseudo Genes (multiple problems) :: 8 of 80 ; CRISPR Arrays :: 2 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..349</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>349</GBInterval_to>\\n            <GBInterval_accession>JAILZW000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Alicyclobacillus macrosporangiidus</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.25</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:392015</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZW010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZW010000349</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023720</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Alicyclobacillus macrosporangiidus isolate HOT.CON.25, whole genome shotgun sequencing project', 'Alicyclobacillus macrosporangiidus (hot springs metagenome)', 'Alicyclobacillus macrosporangiidus', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus']\n",
      "Id 2240894572\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZV010000000</GBSeq_locus>\\n    <GBSeq_length>349</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Anoxybacillus sp. isolate HOT.CON.20, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZV000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZV000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZV000000000.1|JAILZV010000000</GBSeqid>\\n      <GBSeqid>gi|2240894572</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Anoxybacillus sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Anoxybacillus sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Bacillaceae; Anoxybacillus; unclassified Anoxybacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..349</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..349</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Anoxybacillus sp. whole genome shotgun (WGS) project has the project accession JAILZV000000000. This version of the project (01) has the accession number JAILZV010000000, and consists of sequences JAILZV010000001-JAILZV010000349.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 58x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:31:31 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,558 ; CDSs (total) :: 3,499 ; Genes (coding) :: 3,416 ; CDSs (with protein) :: 3,416 ; Genes (RNA) :: 59 ; rRNAs :: 2, 3, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 3, 1 (5S, 16S, 23S) ; tRNAs :: 45 ; ncRNAs :: 8 ; Pseudo Genes (total) :: 83 ; CDSs (without protein) :: 83 ; Pseudo Genes (ambiguous residues) :: 0 of 83 ; Pseudo Genes (frameshifted) :: 24 of 83 ; Pseudo Genes (incomplete) :: 57 of 83 ; Pseudo Genes (internal stop) :: 11 of 83 ; Pseudo Genes (multiple problems) :: 8 of 83 ; CRISPR Arrays :: 3 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..349</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>349</GBInterval_to>\\n            <GBInterval_accession>JAILZV000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Anoxybacillus sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.20</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1872573</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZV010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZV010000349</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023719</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Anoxybacillus sp. isolate HOT.CON.20, whole genome shotgun sequencing project', 'Anoxybacillus sp. (hot springs metagenome)', 'Anoxybacillus sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Bacillaceae; Anoxybacillus; unclassified Anoxybacillus']\n",
      "Id 2240894571\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZS010000000</GBSeq_locus>\\n    <GBSeq_length>104</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Firmicutes bacterium isolate HOT.CON.150, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZS000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZS000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZS000000000.1|JAILZS010000000</GBSeqid>\\n      <GBSeqid>gi|2240894571</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Firmicutes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Firmicutes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..104</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..104</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Firmicutes bacterium whole genome shotgun (WGS) project has the project accession JAILZS000000000. This version of the project (01) has the accession number JAILZS010000000, and consists of sequences JAILZS010000001-JAILZS010000104.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 15.2x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 14:27:22 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,606 ; CDSs (total) :: 2,556 ; Genes (coding) :: 2,531 ; CDSs (with protein) :: 2,531 ; Genes (RNA) :: 50 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 43 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 25 ; CDSs (without protein) :: 25 ; Pseudo Genes (ambiguous residues) :: 0 of 25 ; Pseudo Genes (frameshifted) :: 6 of 25 ; Pseudo Genes (incomplete) :: 17 of 25 ; Pseudo Genes (internal stop) :: 3 of 25 ; Pseudo Genes (multiple problems) :: 1 of 25 ; CRISPR Arrays :: 2 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..104</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>104</GBInterval_to>\\n            <GBInterval_accession>JAILZS000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Firmicutes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.150</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1879010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZS010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZS010000104</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023716</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Firmicutes bacterium isolate HOT.CON.150, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes']\n",
      "Id 2240894570\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZU010000000</GBSeq_locus>\\n    <GBSeq_length>806</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Alicyclobacillus sp. isolate HOT.CON.16, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZU000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZU000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZU000000000.1|JAILZU010000000</GBSeqid>\\n      <GBSeqid>gi|2240894570</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Alicyclobacillus sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Alicyclobacillus sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..806</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..806</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Alicyclobacillus sp. whole genome shotgun (WGS) project has the project accession JAILZU000000000. This version of the project (01) has the accession number JAILZU010000000, and consists of sequences JAILZU010000001-JAILZU010000806.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 29.8x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 09:53:29 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,283 ; CDSs (total) :: 2,269 ; Genes (coding) :: 2,247 ; CDSs (with protein) :: 2,247 ; Genes (RNA) :: 14 ; rRNAs :: 1, 1 (5S, 16S) ; partial rRNAs :: 1, 1 (5S, 16S) ; tRNAs :: 10 ; ncRNAs :: 2 ; Pseudo Genes (total) :: 22 ; CDSs (without protein) :: 22 ; Pseudo Genes (ambiguous residues) :: 0 of 22 ; Pseudo Genes (frameshifted) :: 10 of 22 ; Pseudo Genes (incomplete) :: 8 of 22 ; Pseudo Genes (internal stop) :: 6 of 22 ; Pseudo Genes (multiple problems) :: 2 of 22 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..806</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>806</GBInterval_to>\\n            <GBInterval_accession>JAILZU000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Alicyclobacillus sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.16</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:61169</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZU010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZU010000806</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023718</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Alicyclobacillus sp. isolate HOT.CON.16, whole genome shotgun sequencing project', 'Alicyclobacillus sp. (hot springs metagenome)', 'Alicyclobacillus sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus']\n",
      "Id 2240894569\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZX010000000</GBSeq_locus>\\n    <GBSeq_length>340</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Firmicutes bacterium isolate HOT.CON.49, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZX000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZX000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZX000000000.1|JAILZX010000000</GBSeqid>\\n      <GBSeqid>gi|2240894569</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Firmicutes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Firmicutes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..340</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..340</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Firmicutes bacterium whole genome shotgun (WGS) project has the project accession JAILZX000000000. This version of the project (01) has the accession number JAILZX010000000, and consists of sequences JAILZX010000001-JAILZX010000340.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 14.8x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 12:11:24 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,080 ; CDSs (total) :: 2,041 ; Genes (coding) :: 2,008 ; CDSs (with protein) :: 2,008 ; Genes (RNA) :: 39 ; rRNAs :: 1, 1 (5S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1 (23S) ; tRNAs :: 34 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 33 ; CDSs (without protein) :: 33 ; Pseudo Genes (ambiguous residues) :: 0 of 33 ; Pseudo Genes (frameshifted) :: 6 of 33 ; Pseudo Genes (incomplete) :: 25 of 33 ; Pseudo Genes (internal stop) :: 3 of 33 ; Pseudo Genes (multiple problems) :: 1 of 33 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..340</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>340</GBInterval_to>\\n            <GBInterval_accession>JAILZX000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Firmicutes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.49</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1879010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZX010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZX010000340</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023721</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Firmicutes bacterium isolate HOT.CON.49, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes']\n",
      "Id 2240894568\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAA010000000</GBSeq_locus>\\n    <GBSeq_length>195</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Candidatus Bathyarchaeota archaeon isolate HOT.CON.75, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAA000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAA000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAA000000000.1|JAIMAA010000000</GBSeqid>\\n      <GBSeqid>gi|2240894568</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Candidatus Bathyarchaeota archaeon (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Candidatus Bathyarchaeota archaeon</GBSeq_organism>\\n    <GBSeq_taxonomy>Archaea; Candidatus Bathyarchaeota</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..195</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..195</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Candidatus Bathyarchaeota archaeon whole genome shotgun (WGS) project has the project accession JAIMAA000000000. This version of the project (01) has the accession number JAIMAA010000000, and consists of sequences JAIMAA010000001-JAIMAA010000195.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 15x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:32:13 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,309 ; CDSs (total) :: 2,251 ; Genes (coding) :: 2,235 ; CDSs (with protein) :: 2,235 ; Genes (RNA) :: 58 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1, 2, 1 (5S, 16S, 23S) ; partial rRNAs :: 2 (23S) ; tRNAs :: 50 ; ncRNAs :: 2 ; Pseudo Genes (total) :: 16 ; CDSs (without protein) :: 16 ; Pseudo Genes (ambiguous residues) :: 0 of 16 ; Pseudo Genes (frameshifted) :: 2 of 16 ; Pseudo Genes (incomplete) :: 10 of 16 ; Pseudo Genes (internal stop) :: 4 of 16 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..195</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>195</GBInterval_to>\\n            <GBInterval_accession>JAIMAA000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Candidatus Bathyarchaeota archaeon</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.75</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2026714</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAA010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAA010000195</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023724</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Candidatus Bathyarchaeota archaeon isolate HOT.CON.75, whole genome shotgun sequencing project', 'Candidatus Bathyarchaeota archaeon (hot springs metagenome)', 'Candidatus Bathyarchaeota archaeon', 'Archaea; Candidatus Bathyarchaeota']\n",
      "Id 2240894567\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAB010000000</GBSeq_locus>\\n    <GBSeq_length>173</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Firmicutes bacterium isolate HOT.CON.82, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAB000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAB000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAB000000000.1|JAIMAB010000000</GBSeqid>\\n      <GBSeqid>gi|2240894567</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Firmicutes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Firmicutes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..173</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..173</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Firmicutes bacterium whole genome shotgun (WGS) project has the project accession JAIMAB000000000. This version of the project (01) has the accession number JAIMAB010000000, and consists of sequences JAIMAB010000001-JAIMAB010000173.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 24.4x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 13:25:34 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 4,118 ; CDSs (total) :: 4,063 ; Genes (coding) :: 4,016 ; CDSs (with protein) :: 4,016 ; Genes (RNA) :: 55 ; rRNAs :: 1, 1, 2 (5S, 16S, 23S) ; complete rRNAs :: 1, 1 (5S, 16S) ; partial rRNAs :: 2 (23S) ; tRNAs :: 47 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 47 ; CDSs (without protein) :: 47 ; Pseudo Genes (ambiguous residues) :: 0 of 47 ; Pseudo Genes (frameshifted) :: 17 of 47 ; Pseudo Genes (incomplete) :: 30 of 47 ; Pseudo Genes (internal stop) :: 4 of 47 ; Pseudo Genes (multiple problems) :: 3 of 47 ; CRISPR Arrays :: 9 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..173</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>173</GBInterval_to>\\n            <GBInterval_accession>JAIMAB000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Firmicutes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.82</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1879010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAB010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAB010000173</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023725</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Firmicutes bacterium isolate HOT.CON.82, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes']\n",
      "Id 2240894566\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZY010000000</GBSeq_locus>\\n    <GBSeq_length>172</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: bacterium isolate HOT.CON.50, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZY000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZY000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZY000000000.1|JAILZY010000000</GBSeqid>\\n      <GBSeqid>gi|2240894566</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..172</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..172</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The bacterium whole genome shotgun (WGS) project has the project accession JAILZY000000000. This version of the project (01) has the accession number JAILZY010000000, and consists of sequences JAILZY010000001-JAILZY010000172.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 23.9x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 12:15:36 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,972 ; CDSs (total) :: 2,926 ; Genes (coding) :: 2,900 ; CDSs (with protein) :: 2,900 ; Genes (RNA) :: 46 ; tRNAs :: 42 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 26 ; CDSs (without protein) :: 26 ; Pseudo Genes (ambiguous residues) :: 0 of 26 ; Pseudo Genes (frameshifted) :: 7 of 26 ; Pseudo Genes (incomplete) :: 12 of 26 ; Pseudo Genes (internal stop) :: 9 of 26 ; Pseudo Genes (multiple problems) :: 2 of 26 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..172</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>172</GBInterval_to>\\n            <GBInterval_accession>JAILZY000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.50</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1869227</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZY010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZY010000172</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023722</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: bacterium isolate HOT.CON.50, whole genome shotgun sequencing project', 'bacterium (hot springs metagenome)', 'bacterium', 'Bacteria']\n",
      "Id 2240894565\n",
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZZ010000000</GBSeq_locus>\\n    <GBSeq_length>124</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Kyrpidia sp. isolate HOT.CON.60, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZZ000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZZ000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZZ000000000.1|JAILZZ010000000</GBSeqid>\\n      <GBSeqid>gi|2240894565</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Kyrpidia sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Kyrpidia sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Kyrpidia; unclassified Kyrpidia</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..124</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..124</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Kyrpidia sp. whole genome shotgun (WGS) project has the project accession JAILZZ000000000. This version of the project (01) has the accession number JAILZZ010000000, and consists of sequences JAILZZ010000001-JAILZZ010000124.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 16.4x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:21:48 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,370 ; CDSs (total) :: 3,316 ; Genes (coding) :: 3,245 ; CDSs (with protein) :: 3,245 ; Genes (RNA) :: 54 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 47 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 71 ; CDSs (without protein) :: 71 ; Pseudo Genes (ambiguous residues) :: 0 of 71 ; Pseudo Genes (frameshifted) :: 21 of 71 ; Pseudo Genes (incomplete) :: 50 of 71 ; Pseudo Genes (internal stop) :: 11 of 71 ; Pseudo Genes (multiple problems) :: 11 of 71 ; CRISPR Arrays :: 5 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..124</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>124</GBInterval_to>\\n            <GBInterval_accession>JAILZZ000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Kyrpidia sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.60</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2073077</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZZ010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZZ010000124</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023723</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Kyrpidia sp. isolate HOT.CON.60, whole genome shotgun sequencing project', 'Kyrpidia sp. (hot springs metagenome)', 'Kyrpidia sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Kyrpidia; unclassified Kyrpidia']\n",
      "Id 2240894564\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAD010000000</GBSeq_locus>\\n    <GBSeq_length>638</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Bacillus sp. (in: Bacteria) isolate HOT.CON.97, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAD000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAD000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAD000000000.1|JAIMAD010000000</GBSeqid>\\n      <GBSeqid>gi|2240894564</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Bacillus sp. (in: Bacteria) (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Bacillus sp. (in: Bacteria)</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Bacillaceae; Bacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..638</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..638</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Bacillus sp. (in: Bacteria) whole genome shotgun (WGS) project has the project accession JAIMAD000000000. This version of the project (01) has the accession number JAIMAD010000000, and consists of sequences JAIMAD010000001-JAIMAD010000638.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 257.7x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:52:06 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 4,598 ; CDSs (total) :: 4,564 ; Genes (coding) :: 4,460 ; CDSs (with protein) :: 4,460 ; Genes (RNA) :: 34 ; rRNAs :: 2, 2, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 2, 1 (5S, 16S, 23S) ; tRNAs :: 25 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 104 ; CDSs (without protein) :: 104 ; Pseudo Genes (ambiguous residues) :: 0 of 104 ; Pseudo Genes (frameshifted) :: 32 of 104 ; Pseudo Genes (incomplete) :: 47 of 104 ; Pseudo Genes (internal stop) :: 41 of 104 ; Pseudo Genes (multiple problems) :: 13 of 104 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..638</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>638</GBInterval_to>\\n            <GBInterval_accession>JAIMAD000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Bacillus sp. (in: Bacteria)</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.97</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1409</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAD010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAD010000638</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023727</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Bacillus sp. (in: Bacteria) isolate HOT.CON.97, whole genome shotgun sequencing project', 'Bacillus sp. (in: Bacteria) (hot springs metagenome)', 'Bacillus sp. (in: Bacteria)', 'Bacteria; Firmicutes; Bacilli; Bacillales; Bacillaceae; Bacillus']\n",
      "Id 2240894563\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAJ010000000</GBSeq_locus>\\n    <GBSeq_length>1051</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Firmicutes bacterium isolate HOT.MAX.058, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAJ000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAJ000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAJ000000000.1|JAIMAJ010000000</GBSeqid>\\n      <GBSeqid>gi|2240894563</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Firmicutes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Firmicutes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..1051</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..1051</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Firmicutes bacterium whole genome shotgun (WGS) project has the project accession JAIMAJ000000000. This version of the project (01) has the accession number JAIMAJ010000000, and consists of sequences JAIMAJ010000001-JAIMAJ010001051.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 16.3x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 12:45:59 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 4,363 ; CDSs (total) :: 4,319 ; Genes (coding) :: 4,185 ; CDSs (with protein) :: 4,185 ; Genes (RNA) :: 44 ; rRNAs :: 3, 2 (16S, 23S) ; partial rRNAs :: 3, 2 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 0 ; Pseudo Genes (total) :: 134 ; CDSs (without protein) :: 134 ; Pseudo Genes (ambiguous residues) :: 0 of 134 ; Pseudo Genes (frameshifted) :: 30 of 134 ; Pseudo Genes (incomplete) :: 92 of 134 ; Pseudo Genes (internal stop) :: 23 of 134 ; Pseudo Genes (multiple problems) :: 10 of 134 ; CRISPR Arrays :: 5 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..1051</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>1051</GBInterval_to>\\n            <GBInterval_accession>JAIMAJ000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Firmicutes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MAX.058</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1879010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAJ010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAJ010001051</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023733</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Firmicutes bacterium isolate HOT.MAX.058, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes']\n",
      "Id 2240894562\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAK010000000</GBSeq_locus>\\n    <GBSeq_length>410</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Firmicutes bacterium isolate HOT.MAX.062, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAK000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAK000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAK000000000.1|JAIMAK010000000</GBSeqid>\\n      <GBSeqid>gi|2240894562</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Firmicutes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Firmicutes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..410</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..410</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Firmicutes bacterium whole genome shotgun (WGS) project has the project accession JAIMAK000000000. This version of the project (01) has the accession number JAIMAK010000000, and consists of sequences JAIMAK010000001-JAIMAK010000410.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 14.2x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 15:09:07 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,324 ; CDSs (total) :: 3,267 ; Genes (coding) :: 3,230 ; CDSs (with protein) :: 3,230 ; Genes (RNA) :: 57 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 51 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 37 ; CDSs (without protein) :: 37 ; Pseudo Genes (ambiguous residues) :: 0 of 37 ; Pseudo Genes (frameshifted) :: 3 of 37 ; Pseudo Genes (incomplete) :: 30 of 37 ; Pseudo Genes (internal stop) :: 6 of 37 ; Pseudo Genes (multiple problems) :: 2 of 37 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..410</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>410</GBInterval_to>\\n            <GBInterval_accession>JAIMAK000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Firmicutes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MAX.062</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1879010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAK010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAK010000410</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023734</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Firmicutes bacterium isolate HOT.MAX.062, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes']\n",
      "Id 2240894555\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAH010000000</GBSeq_locus>\\n    <GBSeq_length>376</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Alicyclobacillus sp. isolate HOT.MAX.041, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAH000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAH000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAH000000000.1|JAIMAH010000000</GBSeqid>\\n      <GBSeqid>gi|2240894555</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Alicyclobacillus sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Alicyclobacillus sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..376</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..376</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Alicyclobacillus sp. whole genome shotgun (WGS) project has the project accession JAIMAH000000000. This version of the project (01) has the accession number JAIMAH010000000, and consists of sequences JAIMAH010000001-JAIMAH010000376.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 16.2x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 09:53:16 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,158 ; CDSs (total) :: 2,132 ; Genes (coding) :: 2,110 ; CDSs (with protein) :: 2,110 ; Genes (RNA) :: 26 ; tRNAs :: 23 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 22 ; CDSs (without protein) :: 22 ; Pseudo Genes (ambiguous residues) :: 0 of 22 ; Pseudo Genes (frameshifted) :: 6 of 22 ; Pseudo Genes (incomplete) :: 13 of 22 ; Pseudo Genes (internal stop) :: 5 of 22 ; Pseudo Genes (multiple problems) :: 1 of 22 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..376</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>376</GBInterval_to>\\n            <GBInterval_accession>JAIMAH000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Alicyclobacillus sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MAX.041</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:61169</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAH010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAH010000376</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023731</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Alicyclobacillus sp. isolate HOT.MAX.041, whole genome shotgun sequencing project', 'Alicyclobacillus sp. (hot springs metagenome)', 'Alicyclobacillus sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus']\n",
      "Id 2240894553\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAI010000000</GBSeq_locus>\\n    <GBSeq_length>75</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Acidothermus sp. isolate HOT.MAX.052, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAI000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAI000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAI000000000.1|JAIMAI010000000</GBSeqid>\\n      <GBSeqid>gi|2240894553</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Acidothermus sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Acidothermus sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Actinobacteria; Acidothermales; Acidothermaceae; Acidothermus; unclassified Acidothermus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..75</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..75</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Acidothermus sp. whole genome shotgun (WGS) project has the project accession JAIMAI000000000. This version of the project (01) has the accession number JAIMAI010000000, and consists of sequences JAIMAI010000001-JAIMAI010000075.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 14.7x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 09:48:49 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,075 ; CDSs (total) :: 2,029 ; Genes (coding) :: 1,995 ; CDSs (with protein) :: 1,995 ; Genes (RNA) :: 46 ; tRNAs :: 43 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 34 ; CDSs (without protein) :: 34 ; Pseudo Genes (ambiguous residues) :: 0 of 34 ; Pseudo Genes (frameshifted) :: 2 of 34 ; Pseudo Genes (incomplete) :: 31 of 34 ; Pseudo Genes (internal stop) :: 1 of 34 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..75</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>75</GBInterval_to>\\n            <GBInterval_accession>JAIMAI000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Acidothermus sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MAX.052</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1872121</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAI010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAI010000075</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023732</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Acidothermus sp. isolate HOT.MAX.052, whole genome shotgun sequencing project', 'Acidothermus sp. (hot springs metagenome)', 'Acidothermus sp.', 'Bacteria; Actinobacteria; Acidothermales; Acidothermaceae; Acidothermus; unclassified Acidothermus']\n",
      "Id 2240894552\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAL010000000</GBSeq_locus>\\n    <GBSeq_length>44</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Burkholderiales bacterium isolate HOT.MAX.068, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAL000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAL000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAL000000000.1|JAIMAL010000000</GBSeqid>\\n      <GBSeqid>gi|2240894552</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Burkholderiales bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Burkholderiales bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Proteobacteria; Betaproteobacteria; Burkholderiales</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..44</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..44</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Burkholderiales bacterium whole genome shotgun (WGS) project has the project accession JAIMAL000000000. This version of the project (01) has the accession number JAIMAL010000000, and consists of sequences JAIMAL010000001-JAIMAL010000044.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 14.4x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:13:49 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,525 ; CDSs (total) :: 2,474 ; Genes (coding) :: 2,463 ; CDSs (with protein) :: 2,463 ; Genes (RNA) :: 51 ; rRNAs :: 1, 1, 2 (5S, 16S, 23S) ; complete rRNAs :: 1, 1 (5S, 16S) ; partial rRNAs :: 2 (23S) ; tRNAs :: 43 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 11 ; CDSs (without protein) :: 11 ; Pseudo Genes (ambiguous residues) :: 0 of 11 ; Pseudo Genes (frameshifted) :: 3 of 11 ; Pseudo Genes (incomplete) :: 6 of 11 ; Pseudo Genes (internal stop) :: 5 of 11 ; Pseudo Genes (multiple problems) :: 3 of 11 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..44</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>44</GBInterval_to>\\n            <GBInterval_accession>JAIMAL000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Burkholderiales bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MAX.068</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1891238</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAL010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAL010000044</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023735</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Burkholderiales bacterium isolate HOT.MAX.068, whole genome shotgun sequencing project', 'Burkholderiales bacterium (hot springs metagenome)', 'Burkholderiales bacterium', 'Bacteria; Proteobacteria; Betaproteobacteria; Burkholderiales']\n",
      "Id 2240894551\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAR010000000</GBSeq_locus>\\n    <GBSeq_length>394</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Bryobacteraceae bacterium isolate HOT.MAX.089, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAR000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAR000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAR000000000.1|JAIMAR010000000</GBSeqid>\\n      <GBSeqid>gi|2240894551</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Bryobacteraceae bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Bryobacteraceae bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Acidobacteria; Bryobacterales; Bryobacteraceae</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..394</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..394</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Bryobacteraceae bacterium whole genome shotgun (WGS) project has the project accession JAIMAR000000000. This version of the project (01) has the accession number JAIMAR010000000, and consists of sequences JAIMAR010000001-JAIMAR010000394.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 16.7x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:19:39 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,768 ; CDSs (total) :: 3,710 ; Genes (coding) :: 3,688 ; CDSs (with protein) :: 3,688 ; Genes (RNA) :: 58 ; rRNAs :: 2, 4, 4 (5S, 16S, 23S) ; complete rRNAs :: 2, 1 (5S, 23S) ; partial rRNAs :: 4, 3 (16S, 23S) ; tRNAs :: 45 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 22 ; CDSs (without protein) :: 22 ; Pseudo Genes (ambiguous residues) :: 0 of 22 ; Pseudo Genes (frameshifted) :: 2 of 22 ; Pseudo Genes (incomplete) :: 16 of 22 ; Pseudo Genes (internal stop) :: 4 of 22 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..394</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>394</GBInterval_to>\\n            <GBInterval_accession>JAIMAR000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Bryobacteraceae bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MAX.089</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2212468</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAR010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAR010000394</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023741</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Bryobacteraceae bacterium isolate HOT.MAX.089, whole genome shotgun sequencing project', 'Bryobacteraceae bacterium (hot springs metagenome)', 'Bryobacteraceae bacterium', 'Bacteria; Acidobacteria; Bryobacterales; Bryobacteraceae']\n",
      "Id 2240894550\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAE010000000</GBSeq_locus>\\n    <GBSeq_length>405</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Firmicutes bacterium isolate HOT.CON.99, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAE000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAE000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAE000000000.1|JAIMAE010000000</GBSeqid>\\n      <GBSeqid>gi|2240894550</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Firmicutes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Firmicutes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..405</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..405</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Firmicutes bacterium whole genome shotgun (WGS) project has the project accession JAIMAE000000000. This version of the project (01) has the accession number JAIMAE010000000, and consists of sequences JAIMAE010000001-JAIMAE010000405.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 31x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 13:35:38 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,492 ; CDSs (total) :: 3,447 ; Genes (coding) :: 3,417 ; CDSs (with protein) :: 3,417 ; Genes (RNA) :: 45 ; rRNAs :: 1, 1 (16S, 23S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 30 ; CDSs (without protein) :: 30 ; Pseudo Genes (ambiguous residues) :: 0 of 30 ; Pseudo Genes (frameshifted) :: 9 of 30 ; Pseudo Genes (incomplete) :: 18 of 30 ; Pseudo Genes (internal stop) :: 9 of 30 ; Pseudo Genes (multiple problems) :: 5 of 30 ; CRISPR Arrays :: 2 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..405</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>405</GBInterval_to>\\n            <GBInterval_accession>JAIMAE000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Firmicutes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.99</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1879010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAE010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAE010000405</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023728</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Firmicutes bacterium isolate HOT.CON.99, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes']\n",
      "Id 2240894549\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAC010000000</GBSeq_locus>\\n    <GBSeq_length>1082</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Meiothermus silvanus isolate HOT.CON.95, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAC000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAC000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAC000000000.1|JAIMAC010000000</GBSeqid>\\n      <GBSeqid>gi|2240894549</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Meiothermus silvanus (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Meiothermus silvanus</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Deinococcus-Thermus; Deinococci; Thermales; Thermaceae; Meiothermus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..1082</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..1082</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Meiothermus silvanus whole genome shotgun (WGS) project has the project accession JAIMAC000000000. This version of the project (01) has the accession number JAIMAC010000000, and consists of sequences JAIMAC010000001-JAIMAC010001082.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 17x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 09:57:43 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,913 ; CDSs (total) :: 2,880 ; Genes (coding) :: 2,758 ; CDSs (with protein) :: 2,758 ; Genes (RNA) :: 33 ; rRNAs :: 2 (16S) ; partial rRNAs :: 2 (16S) ; tRNAs :: 30 ; ncRNAs :: 1 ; Pseudo Genes (total) :: 122 ; CDSs (without protein) :: 122 ; Pseudo Genes (ambiguous residues) :: 0 of 122 ; Pseudo Genes (frameshifted) :: 14 of 122 ; Pseudo Genes (incomplete) :: 100 of 122 ; Pseudo Genes (internal stop) :: 17 of 122 ; Pseudo Genes (multiple problems) :: 8 of 122 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..1082</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>1082</GBInterval_to>\\n            <GBInterval_accession>JAIMAC000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Meiothermus silvanus</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.95</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:52022</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAC010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAC010001082</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023726</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Meiothermus silvanus isolate HOT.CON.95, whole genome shotgun sequencing project', 'Meiothermus silvanus (hot springs metagenome)', 'Meiothermus silvanus', 'Bacteria; Deinococcus-Thermus; Deinococci; Thermales; Thermaceae; Meiothermus']\n",
      "Id 2240894548\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAG010000000</GBSeq_locus>\\n    <GBSeq_length>51</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Acidobacteriia bacterium isolate HOT.MAX.039, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAG000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAG000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAG000000000.1|JAIMAG010000000</GBSeqid>\\n      <GBSeqid>gi|2240894548</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Acidobacteriia bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Acidobacteriia bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Acidobacteria</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..51</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..51</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Acidobacteriia bacterium whole genome shotgun (WGS) project has the project accession JAIMAG000000000. This version of the project (01) has the accession number JAIMAG010000000, and consists of sequences JAIMAG010000001-JAIMAG010000051.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 14.9x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 12:11:52 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,569 ; CDSs (total) :: 2,518 ; Genes (coding) :: 2,504 ; CDSs (with protein) :: 2,504 ; Genes (RNA) :: 51 ; rRNAs :: 1, 2, 2 (5S, 16S, 23S) ; complete rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 44 ; ncRNAs :: 2 ; Pseudo Genes (total) :: 14 ; CDSs (without protein) :: 14 ; Pseudo Genes (ambiguous residues) :: 0 of 14 ; Pseudo Genes (frameshifted) :: 5 of 14 ; Pseudo Genes (incomplete) :: 7 of 14 ; Pseudo Genes (internal stop) :: 2 of 14 ; CRISPR Arrays :: 2 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..51</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>51</GBInterval_to>\\n            <GBInterval_accession>JAIMAG000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Acidobacteriia bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MAX.039</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2081523</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAG010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAG010000051</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023730</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Acidobacteriia bacterium isolate HOT.MAX.039, whole genome shotgun sequencing project', 'Acidobacteriia bacterium (hot springs metagenome)', 'Acidobacteriia bacterium', 'Bacteria; Acidobacteria']\n",
      "Id 2240894547\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAM010000000</GBSeq_locus>\\n    <GBSeq_length>140</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Armatimonadetes bacterium isolate HOT.MAX.070, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAM000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAM000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAM000000000.1|JAIMAM010000000</GBSeqid>\\n      <GBSeqid>gi|2240894547</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Armatimonadetes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Armatimonadetes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Armatimonadetes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..140</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..140</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Armatimonadetes bacterium whole genome shotgun (WGS) project has the project accession JAIMAM000000000. This version of the project (01) has the accession number JAIMAM010000000, and consists of sequences JAIMAM010000001-JAIMAM010000140.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 15.9x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 12:21:12 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,854 ; CDSs (total) :: 2,804 ; Genes (coding) :: 2,786 ; CDSs (with protein) :: 2,786 ; Genes (RNA) :: 50 ; rRNAs :: 1 (16S) ; partial rRNAs :: 1 (16S) ; tRNAs :: 46 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 18 ; CDSs (without protein) :: 18 ; Pseudo Genes (ambiguous residues) :: 0 of 18 ; Pseudo Genes (frameshifted) :: 3 of 18 ; Pseudo Genes (incomplete) :: 15 of 18 ; Pseudo Genes (internal stop) :: 0 of 18 ; CRISPR Arrays :: 5 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..140</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>140</GBInterval_to>\\n            <GBInterval_accession>JAIMAM000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Armatimonadetes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MAX.070</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2033014</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAM010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAM010000140</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023736</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Armatimonadetes bacterium isolate HOT.MAX.070, whole genome shotgun sequencing project', 'Armatimonadetes bacterium (hot springs metagenome)', 'Armatimonadetes bacterium', 'Bacteria; Armatimonadetes']\n",
      "Id 2240894546\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAF010000000</GBSeq_locus>\\n    <GBSeq_length>185</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Acidothermus cellulolyticus isolate HOT.MAX.011, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAF000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAF000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAF000000000.1|JAIMAF010000000</GBSeqid>\\n      <GBSeqid>gi|2240894546</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Acidothermus cellulolyticus (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Acidothermus cellulolyticus</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Actinobacteria; Acidothermales; Acidothermaceae; Acidothermus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..185</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..185</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Acidothermus cellulolyticus whole genome shotgun (WGS) project has the project accession JAIMAF000000000. This version of the project (01) has the accession number JAIMAF010000000, and consists of sequences JAIMAF010000001-JAIMAF010000185.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 14.8x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 09:58:50 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,171 ; CDSs (total) :: 2,123 ; Genes (coding) :: 2,112 ; CDSs (with protein) :: 2,112 ; Genes (RNA) :: 48 ; rRNAs :: 1, 1 (5S, 16S) ; partial rRNAs :: 1, 1 (5S, 16S) ; tRNAs :: 43 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 11 ; CDSs (without protein) :: 11 ; Pseudo Genes (ambiguous residues) :: 0 of 11 ; Pseudo Genes (frameshifted) :: 3 of 11 ; Pseudo Genes (incomplete) :: 8 of 11 ; Pseudo Genes (internal stop) :: 1 of 11 ; Pseudo Genes (multiple problems) :: 1 of 11 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..185</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>185</GBInterval_to>\\n            <GBInterval_accession>JAIMAF000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Acidothermus cellulolyticus</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MAX.011</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:28049</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAF010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAF010000185</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023729</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Acidothermus cellulolyticus isolate HOT.MAX.011, whole genome shotgun sequencing project', 'Acidothermus cellulolyticus (hot springs metagenome)', 'Acidothermus cellulolyticus', 'Bacteria; Actinobacteria; Acidothermales; Acidothermaceae; Acidothermus']\n",
      "Id 2240894545\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAO010000000</GBSeq_locus>\\n    <GBSeq_length>206</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Roseiflexus sp. isolate HOT.MAX.081, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAO000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAO000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAO000000000.1|JAIMAO010000000</GBSeqid>\\n      <GBSeqid>gi|2240894545</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Roseiflexus sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Roseiflexus sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Chloroflexi; Chloroflexia; Chloroflexales; Roseiflexineae; Roseiflexaceae; Roseiflexus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..206</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..206</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Roseiflexus sp. whole genome shotgun (WGS) project has the project accession JAIMAO000000000. This version of the project (01) has the accession number JAIMAO010000000, and consists of sequences JAIMAO010000001-JAIMAO010000206.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 27.7x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:50:43 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 4,771 ; CDSs (total) :: 4,715 ; Genes (coding) :: 4,616 ; CDSs (with protein) :: 4,616 ; Genes (RNA) :: 56 ; rRNAs :: 2, 1, 2 (5S, 16S, 23S) ; complete rRNAs :: 2, 1, 1 (5S, 16S, 23S) ; partial rRNAs :: 1 (23S) ; tRNAs :: 48 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 99 ; CDSs (without protein) :: 99 ; Pseudo Genes (ambiguous residues) :: 0 of 99 ; Pseudo Genes (frameshifted) :: 36 of 99 ; Pseudo Genes (incomplete) :: 61 of 99 ; Pseudo Genes (internal stop) :: 14 of 99 ; Pseudo Genes (multiple problems) :: 10 of 99 ; CRISPR Arrays :: 6 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..206</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>206</GBInterval_to>\\n            <GBInterval_accession>JAIMAO000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Roseiflexus sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MAX.081</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2562120</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAO010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAO010000206</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023738</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Roseiflexus sp. isolate HOT.MAX.081, whole genome shotgun sequencing project', 'Roseiflexus sp. (hot springs metagenome)', 'Roseiflexus sp.', 'Bacteria; Chloroflexi; Chloroflexia; Chloroflexales; Roseiflexineae; Roseiflexaceae; Roseiflexus']\n",
      "Id 2240894544\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAN010000000</GBSeq_locus>\\n    <GBSeq_length>122</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Armatimonadetes bacterium isolate HOT.MAX.079, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAN000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAN000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAN000000000.1|JAIMAN010000000</GBSeqid>\\n      <GBSeqid>gi|2240894544</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Armatimonadetes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Armatimonadetes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Armatimonadetes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..122</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..122</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Armatimonadetes bacterium whole genome shotgun (WGS) project has the project accession JAIMAN000000000. This version of the project (01) has the accession number JAIMAN010000000, and consists of sequences JAIMAN010000001-JAIMAN010000122.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 18.2x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 13:16:57 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,610 ; CDSs (total) :: 2,558 ; Genes (coding) :: 2,544 ; CDSs (with protein) :: 2,544 ; Genes (RNA) :: 52 ; rRNAs :: 1, 1 (5S, 16S) ; partial rRNAs :: 1, 1 (5S, 16S) ; tRNAs :: 47 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 14 ; CDSs (without protein) :: 14 ; Pseudo Genes (ambiguous residues) :: 0 of 14 ; Pseudo Genes (frameshifted) :: 2 of 14 ; Pseudo Genes (incomplete) :: 12 of 14 ; Pseudo Genes (internal stop) :: 0 of 14 ; CRISPR Arrays :: 2 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..122</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>122</GBInterval_to>\\n            <GBInterval_accession>JAIMAN000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Armatimonadetes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MAX.079</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2033014</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAN010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAN010000122</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023737</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Armatimonadetes bacterium isolate HOT.MAX.079, whole genome shotgun sequencing project', 'Armatimonadetes bacterium (hot springs metagenome)', 'Armatimonadetes bacterium', 'Bacteria; Armatimonadetes']\n",
      "Id 2240894509\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAP010000000</GBSeq_locus>\\n    <GBSeq_length>64</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Meiothermus ruber isolate HOT.MAX.085, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAP000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAP000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAP000000000.1|JAIMAP010000000</GBSeqid>\\n      <GBSeqid>gi|2240894509</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Meiothermus ruber (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Meiothermus ruber</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Deinococcus-Thermus; Deinococci; Thermales; Thermaceae; Meiothermus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..64</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..64</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Meiothermus ruber whole genome shotgun (WGS) project has the project accession JAIMAP000000000. This version of the project (01) has the accession number JAIMAP010000000, and consists of sequences JAIMAP010000001-JAIMAP010000064.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 21.5x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:27:06 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,247 ; CDSs (total) :: 3,189 ; Genes (coding) :: 3,160 ; CDSs (with protein) :: 3,160 ; Genes (RNA) :: 58 ; rRNAs :: 1, 4, 2 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 4, 2 (16S, 23S) ; tRNAs :: 48 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 29 ; CDSs (without protein) :: 29 ; Pseudo Genes (ambiguous residues) :: 0 of 29 ; Pseudo Genes (frameshifted) :: 7 of 29 ; Pseudo Genes (incomplete) :: 19 of 29 ; Pseudo Genes (internal stop) :: 4 of 29 ; Pseudo Genes (multiple problems) :: 1 of 29 ; CRISPR Arrays :: 6 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..64</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>64</GBInterval_to>\\n            <GBInterval_accession>JAIMAP000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Meiothermus ruber</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MAX.085</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:277</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAP010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAP010000064</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023739</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Meiothermus ruber isolate HOT.MAX.085, whole genome shotgun sequencing project', 'Meiothermus ruber (hot springs metagenome)', 'Meiothermus ruber', 'Bacteria; Deinococcus-Thermus; Deinococci; Thermales; Thermaceae; Meiothermus']\n",
      "Id 2240894508\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAS010000000</GBSeq_locus>\\n    <GBSeq_length>96</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Thermaceae bacterium isolate HOT.MAX.091, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAS000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAS000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAS000000000.1|JAIMAS010000000</GBSeqid>\\n      <GBSeqid>gi|2240894508</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Thermaceae bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Thermaceae bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Deinococcus-Thermus; Deinococci; Thermales; Thermaceae; unclassified Thermaceae</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..96</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..96</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Thermaceae bacterium whole genome shotgun (WGS) project has the project accession JAIMAS000000000. This version of the project (01) has the accession number JAIMAS010000000, and consists of sequences JAIMAS010000001-JAIMAS010000096.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 14.6x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:14:17 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,328 ; CDSs (total) :: 3,270 ; Genes (coding) :: 3,245 ; CDSs (with protein) :: 3,245 ; Genes (RNA) :: 58 ; rRNAs :: 1, 4, 2 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 4, 2 (16S, 23S) ; tRNAs :: 48 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 25 ; CDSs (without protein) :: 25 ; Pseudo Genes (ambiguous residues) :: 0 of 25 ; Pseudo Genes (frameshifted) :: 12 of 25 ; Pseudo Genes (incomplete) :: 13 of 25 ; Pseudo Genes (internal stop) :: 4 of 25 ; Pseudo Genes (multiple problems) :: 4 of 25 ; CRISPR Arrays :: 5 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..96</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>96</GBInterval_to>\\n            <GBInterval_accession>JAIMAS000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Thermaceae bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MAX.091</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2614257</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAS010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAS010000096</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023742</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Thermaceae bacterium isolate HOT.MAX.091, whole genome shotgun sequencing project', 'Thermaceae bacterium (hot springs metagenome)', 'Thermaceae bacterium', 'Bacteria; Deinococcus-Thermus; Deinococci; Thermales; Thermaceae; unclassified Thermaceae']\n",
      "Id 2240894507\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAQ010000000</GBSeq_locus>\\n    <GBSeq_length>35</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Thermoflavifilum sp. isolate HOT.MAX.088, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAQ000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAQ000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAQ000000000.1|JAIMAQ010000000</GBSeqid>\\n      <GBSeqid>gi|2240894507</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Thermoflavifilum sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Thermoflavifilum sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Bacteroidetes; Chitinophagia; Chitinophagales; Chitinophagaceae; Thermoflavifilum</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..35</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..35</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Thermoflavifilum sp. whole genome shotgun (WGS) project has the project accession JAIMAQ000000000. This version of the project (01) has the accession number JAIMAQ010000000, and consists of sequences JAIMAQ010000001-JAIMAQ010000035.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 19.6x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 09:56:32 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,269 ; CDSs (total) :: 2,228 ; Genes (coding) :: 2,223 ; CDSs (with protein) :: 2,223 ; Genes (RNA) :: 41 ; rRNAs :: 1 (16S) ; partial rRNAs :: 1 (16S) ; tRNAs :: 37 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 5 ; CDSs (without protein) :: 5 ; Pseudo Genes (ambiguous residues) :: 0 of 5 ; Pseudo Genes (frameshifted) :: 2 of 5 ; Pseudo Genes (incomplete) :: 2 of 5 ; Pseudo Genes (internal stop) :: 1 of 5 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..35</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>35</GBInterval_to>\\n            <GBInterval_accession>JAIMAQ000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Thermoflavifilum sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MAX.088</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1968839</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAQ010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAQ010000035</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023740</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Thermoflavifilum sp. isolate HOT.MAX.088, whole genome shotgun sequencing project', 'Thermoflavifilum sp. (hot springs metagenome)', 'Thermoflavifilum sp.', 'Bacteria; Bacteroidetes; Chitinophagia; Chitinophagales; Chitinophagaceae; Thermoflavifilum']\n",
      "Id 2240894506\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAU010000000</GBSeq_locus>\\n    <GBSeq_length>35</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Armatimonadetes bacterium isolate HOT.MB2.102, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAU000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAU000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAU000000000.1|JAIMAU010000000</GBSeqid>\\n      <GBSeqid>gi|2240894506</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Armatimonadetes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Armatimonadetes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Armatimonadetes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..35</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..35</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Armatimonadetes bacterium whole genome shotgun (WGS) project has the project accession JAIMAU000000000. This version of the project (01) has the accession number JAIMAU010000000, and consists of sequences JAIMAU010000001-JAIMAU010000035.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 54.7x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 11:45:01 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,611 ; CDSs (total) :: 2,559 ; Genes (coding) :: 2,551 ; CDSs (with protein) :: 2,551 ; Genes (RNA) :: 52 ; rRNAs :: 1, 3, 2 (5S, 16S, 23S) ; complete rRNAs :: 1, 1, 2 (5S, 16S, 23S) ; partial rRNAs :: 2 (16S) ; tRNAs :: 42 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 8 ; CDSs (without protein) :: 8 ; Pseudo Genes (ambiguous residues) :: 0 of 8 ; Pseudo Genes (frameshifted) :: 2 of 8 ; Pseudo Genes (incomplete) :: 3 of 8 ; Pseudo Genes (internal stop) :: 3 of 8 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..35</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>35</GBInterval_to>\\n            <GBInterval_accession>JAIMAU000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Armatimonadetes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.102</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2033014</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAU010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAU010000035</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023744</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Armatimonadetes bacterium isolate HOT.MB2.102, whole genome shotgun sequencing project', 'Armatimonadetes bacterium (hot springs metagenome)', 'Armatimonadetes bacterium', 'Bacteria; Armatimonadetes']\n",
      "Id 2240894505\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAT010000000</GBSeq_locus>\\n    <GBSeq_length>156</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Firmicutes bacterium isolate HOT.MB2.100, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAT000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAT000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAT000000000.1|JAIMAT010000000</GBSeqid>\\n      <GBSeqid>gi|2240894505</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Firmicutes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Firmicutes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..156</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..156</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Firmicutes bacterium whole genome shotgun (WGS) project has the project accession JAIMAT000000000. This version of the project (01) has the accession number JAIMAT010000000, and consists of sequences JAIMAT010000001-JAIMAT010000156.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 16.1x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 13:22:10 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,335 ; CDSs (total) :: 2,290 ; Genes (coding) :: 2,267 ; CDSs (with protein) :: 2,267 ; Genes (RNA) :: 45 ; rRNAs :: 1 (16S) ; partial rRNAs :: 1 (16S) ; tRNAs :: 41 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 23 ; CDSs (without protein) :: 23 ; Pseudo Genes (ambiguous residues) :: 0 of 23 ; Pseudo Genes (frameshifted) :: 6 of 23 ; Pseudo Genes (incomplete) :: 17 of 23 ; Pseudo Genes (internal stop) :: 2 of 23 ; Pseudo Genes (multiple problems) :: 2 of 23 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..156</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>156</GBInterval_to>\\n            <GBInterval_accession>JAIMAT000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Firmicutes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.100</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1879010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAT010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAT010000156</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023743</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Firmicutes bacterium isolate HOT.MB2.100, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes']\n",
      "Id 2240894498\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAV010000000</GBSeq_locus>\\n    <GBSeq_length>165</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Alicyclobacillus sp. isolate HOT.MB2.104, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAV000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAV000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAV000000000.1|JAIMAV010000000</GBSeqid>\\n      <GBSeqid>gi|2240894498</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Alicyclobacillus sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Alicyclobacillus sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..165</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..165</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Alicyclobacillus sp. whole genome shotgun (WGS) project has the project accession JAIMAV000000000. This version of the project (01) has the accession number JAIMAV010000000, and consists of sequences JAIMAV010000001-JAIMAV010000165.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 20.2x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:26:11 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,947 ; CDSs (total) :: 2,915 ; Genes (coding) :: 2,895 ; CDSs (with protein) :: 2,895 ; Genes (RNA) :: 32 ; rRNAs :: 1, 1 (5S, 16S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1 (16S) ; tRNAs :: 25 ; ncRNAs :: 5 ; Pseudo Genes (total) :: 20 ; CDSs (without protein) :: 20 ; Pseudo Genes (ambiguous residues) :: 0 of 20 ; Pseudo Genes (frameshifted) :: 4 of 20 ; Pseudo Genes (incomplete) :: 16 of 20 ; Pseudo Genes (internal stop) :: 1 of 20 ; Pseudo Genes (multiple problems) :: 1 of 20 ; CRISPR Arrays :: 2 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..165</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>165</GBInterval_to>\\n            <GBInterval_accession>JAIMAV000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Alicyclobacillus sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.104</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:61169</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAV010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAV010000165</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023745</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Alicyclobacillus sp. isolate HOT.MB2.104, whole genome shotgun sequencing project', 'Alicyclobacillus sp. (hot springs metagenome)', 'Alicyclobacillus sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus']\n",
      "Id 2240894485\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAY010000000</GBSeq_locus>\\n    <GBSeq_length>76</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Alicyclobacillus sp. isolate HOT.MB2.19, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAY000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAY000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAY000000000.1|JAIMAY010000000</GBSeqid>\\n      <GBSeqid>gi|2240894485</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Alicyclobacillus sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Alicyclobacillus sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..76</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..76</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Alicyclobacillus sp. whole genome shotgun (WGS) project has the project accession JAIMAY000000000. This version of the project (01) has the accession number JAIMAY010000000, and consists of sequences JAIMAY010000001-JAIMAY010000076.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 18.5x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:21:29 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,589 ; CDSs (total) :: 2,551 ; Genes (coding) :: 2,532 ; CDSs (with protein) :: 2,532 ; Genes (RNA) :: 38 ; rRNAs :: 1, 1, 2 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 2 (16S, 23S) ; tRNAs :: 30 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 19 ; CDSs (without protein) :: 19 ; Pseudo Genes (ambiguous residues) :: 0 of 19 ; Pseudo Genes (frameshifted) :: 11 of 19 ; Pseudo Genes (incomplete) :: 6 of 19 ; Pseudo Genes (internal stop) :: 2 of 19 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..76</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>76</GBInterval_to>\\n            <GBInterval_accession>JAIMAY000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Alicyclobacillus sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.19</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:61169</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAY010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAY010000076</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023748</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Alicyclobacillus sp. isolate HOT.MB2.19, whole genome shotgun sequencing project', 'Alicyclobacillus sp. (hot springs metagenome)', 'Alicyclobacillus sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus']\n",
      "Id 2240894483\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBB010000000</GBSeq_locus>\\n    <GBSeq_length>302</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Anaerolineae bacterium isolate HOT.MB2.30, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBB000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBB000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBB000000000.1|JAIMBB010000000</GBSeqid>\\n      <GBSeqid>gi|2240894483</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Anaerolineae bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Anaerolineae bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Chloroflexi; Anaerolineae; unclassified Anaerolineae</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..302</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..302</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Anaerolineae bacterium whole genome shotgun (WGS) project has the project accession JAIMBB000000000. This version of the project (01) has the accession number JAIMBB010000000, and consists of sequences JAIMBB010000001-JAIMBB010000302.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 21.2x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 13:17:40 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,809 ; CDSs (total) :: 3,765 ; Genes (coding) :: 3,721 ; CDSs (with protein) :: 3,721 ; Genes (RNA) :: 44 ; rRNAs :: 1, 1 (5S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1 (23S) ; tRNAs :: 38 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 44 ; CDSs (without protein) :: 44 ; Pseudo Genes (ambiguous residues) :: 0 of 44 ; Pseudo Genes (frameshifted) :: 9 of 44 ; Pseudo Genes (incomplete) :: 30 of 44 ; Pseudo Genes (internal stop) :: 6 of 44 ; Pseudo Genes (multiple problems) :: 1 of 44 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..302</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>302</GBInterval_to>\\n            <GBInterval_accession>JAIMBB000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Anaerolineae bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.30</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2052143</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBB010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBB010000302</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023751</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Anaerolineae bacterium isolate HOT.MB2.30, whole genome shotgun sequencing project', 'Anaerolineae bacterium (hot springs metagenome)', 'Anaerolineae bacterium', 'Bacteria; Chloroflexi; Anaerolineae; unclassified Anaerolineae']\n",
      "Id 2240894482\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBA010000000</GBSeq_locus>\\n    <GBSeq_length>839</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Bryobacteraceae bacterium isolate HOT.MB2.26, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBA000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBA000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBA000000000.1|JAIMBA010000000</GBSeqid>\\n      <GBSeqid>gi|2240894482</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Bryobacteraceae bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Bryobacteraceae bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Acidobacteria; Bryobacterales; Bryobacteraceae</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..839</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..839</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Bryobacteraceae bacterium whole genome shotgun (WGS) project has the project accession JAIMBA000000000. This version of the project (01) has the accession number JAIMBA010000000, and consists of sequences JAIMBA010000001-JAIMBA010000839.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 20.6x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:17:19 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,746 ; CDSs (total) :: 3,703 ; Genes (coding) :: 3,663 ; CDSs (with protein) :: 3,663 ; Genes (RNA) :: 43 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 38 ; ncRNAs :: 2 ; Pseudo Genes (total) :: 40 ; CDSs (without protein) :: 40 ; Pseudo Genes (ambiguous residues) :: 0 of 40 ; Pseudo Genes (frameshifted) :: 4 of 40 ; Pseudo Genes (incomplete) :: 22 of 40 ; Pseudo Genes (internal stop) :: 15 of 40 ; Pseudo Genes (multiple problems) :: 1 of 40 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..839</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>839</GBInterval_to>\\n            <GBInterval_accession>JAIMBA000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Bryobacteraceae bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.26</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2212468</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBA010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBA010000839</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023750</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Bryobacteraceae bacterium isolate HOT.MB2.26, whole genome shotgun sequencing project', 'Bryobacteraceae bacterium (hot springs metagenome)', 'Bryobacteraceae bacterium', 'Bacteria; Acidobacteria; Bryobacterales; Bryobacteraceae']\n",
      "Id 2240894481\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBC010000000</GBSeq_locus>\\n    <GBSeq_length>422</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Pirellulales bacterium isolate HOT.MB2.34, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBC000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBC000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBC000000000.1|JAIMBC010000000</GBSeqid>\\n      <GBSeqid>gi|2240894481</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Pirellulales bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Pirellulales bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Planctomycetes; Planctomycetia; Pirellulales; unclassified Pirellulales</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..422</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..422</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Pirellulales bacterium whole genome shotgun (WGS) project has the project accession JAIMBC000000000. This version of the project (01) has the accession number JAIMBC010000000, and consists of sequences JAIMBC010000001-JAIMBC010000422.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 41.3x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:50:28 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 4,428 ; CDSs (total) :: 4,377 ; Genes (coding) :: 4,355 ; CDSs (with protein) :: 4,355 ; Genes (RNA) :: 51 ; rRNAs :: 1, 1 (16S, 23S) ; complete rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 46 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 22 ; CDSs (without protein) :: 22 ; Pseudo Genes (ambiguous residues) :: 0 of 22 ; Pseudo Genes (frameshifted) :: 4 of 22 ; Pseudo Genes (incomplete) :: 15 of 22 ; Pseudo Genes (internal stop) :: 4 of 22 ; Pseudo Genes (multiple problems) :: 1 of 22 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..422</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>422</GBInterval_to>\\n            <GBInterval_accession>JAIMBC000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Pirellulales bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.34</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2715253</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBC010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBC010000422</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023752</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Pirellulales bacterium isolate HOT.MB2.34, whole genome shotgun sequencing project', 'Pirellulales bacterium (hot springs metagenome)', 'Pirellulales bacterium', 'Bacteria; Planctomycetes; Planctomycetia; Pirellulales; unclassified Pirellulales']\n",
      "Id 2240894480\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAX010000000</GBSeq_locus>\\n    <GBSeq_length>456</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Firmicutes bacterium isolate HOT.MB2.13, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAX000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAX000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAX000000000.1|JAIMAX010000000</GBSeqid>\\n      <GBSeqid>gi|2240894480</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Firmicutes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Firmicutes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..456</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..456</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Firmicutes bacterium whole genome shotgun (WGS) project has the project accession JAIMAX000000000. This version of the project (01) has the accession number JAIMAX010000000, and consists of sequences JAIMAX010000001-JAIMAX010000456.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 31.6x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 12:11:10 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,077 ; CDSs (total) :: 2,045 ; Genes (coding) :: 2,003 ; CDSs (with protein) :: 2,003 ; Genes (RNA) :: 32 ; tRNAs :: 29 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 42 ; CDSs (without protein) :: 42 ; Pseudo Genes (ambiguous residues) :: 0 of 42 ; Pseudo Genes (frameshifted) :: 8 of 42 ; Pseudo Genes (incomplete) :: 26 of 42 ; Pseudo Genes (internal stop) :: 8 of 42 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..456</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>456</GBInterval_to>\\n            <GBInterval_accession>JAIMAX000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Firmicutes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.13</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1879010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAX010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAX010000456</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023747</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Firmicutes bacterium isolate HOT.MB2.13, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes']\n",
      "Id 2240894479\n",
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBT010000000</GBSeq_locus>\\n    <GBSeq_length>42</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Alicyclobacillus sp. isolate HOT.MB2.91, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBT000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBT000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBT000000000.1|JAIMBT010000000</GBSeqid>\\n      <GBSeqid>gi|2240894479</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Alicyclobacillus sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Alicyclobacillus sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..42</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..42</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Alicyclobacillus sp. whole genome shotgun (WGS) project has the project accession JAIMBT000000000. This version of the project (01) has the accession number JAIMBT010000000, and consists of sequences JAIMBT010000001-JAIMBT010000042.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 17x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:42:45 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,270 ; CDSs (total) :: 3,216 ; Genes (coding) :: 3,183 ; CDSs (with protein) :: 3,183 ; Genes (RNA) :: 54 ; rRNAs :: 1, 2, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 1 (16S, 23S) ; tRNAs :: 46 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 33 ; CDSs (without protein) :: 33 ; Pseudo Genes (ambiguous residues) :: 0 of 33 ; Pseudo Genes (frameshifted) :: 8 of 33 ; Pseudo Genes (incomplete) :: 25 of 33 ; Pseudo Genes (internal stop) :: 2 of 33 ; Pseudo Genes (multiple problems) :: 2 of 33 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..42</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>42</GBInterval_to>\\n            <GBInterval_accession>JAIMBT000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Alicyclobacillus sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.91</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:61169</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBT010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBT010000042</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023769</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Alicyclobacillus sp. isolate HOT.MB2.91, whole genome shotgun sequencing project', 'Alicyclobacillus sp. (hot springs metagenome)', 'Alicyclobacillus sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus']\n",
      "Id 2240894478\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBN010000000</GBSeq_locus>\\n    <GBSeq_length>196</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: candidate division WOR-3 bacterium isolate HOT.MB2.69, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBN000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBN000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBN000000000.1|JAIMBN010000000</GBSeqid>\\n      <GBSeqid>gi|2240894478</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>candidate division WOR-3 bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>candidate division WOR-3 bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; candidate division WOR-3</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..196</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..196</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The candidate division WOR-3 bacterium whole genome shotgun (WGS) project has the project accession JAIMBN000000000. This version of the project (01) has the accession number JAIMBN010000000, and consists of sequences JAIMBN010000001-JAIMBN010000196.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 12x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:40:21 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 1,620 ; CDSs (total) :: 1,573 ; Genes (coding) :: 1,570 ; CDSs (with protein) :: 1,570 ; Genes (RNA) :: 47 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; tRNAs :: 41 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 3 ; CDSs (without protein) :: 3 ; Pseudo Genes (ambiguous residues) :: 0 of 3 ; Pseudo Genes (frameshifted) :: 2 of 3 ; Pseudo Genes (incomplete) :: 1 of 3 ; Pseudo Genes (internal stop) :: 0 of 3 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..196</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>196</GBInterval_to>\\n            <GBInterval_accession>JAIMBN000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>candidate division WOR-3 bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.69</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2052148</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBN010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBN010000196</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023763</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: candidate division WOR-3 bacterium isolate HOT.MB2.69, whole genome shotgun sequencing project', 'candidate division WOR-3 bacterium (hot springs metagenome)', 'candidate division WOR-3 bacterium', 'Bacteria; candidate division WOR-3']\n",
      "Id 2240894477\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JADRFK010000000</GBSeq_locus>\\n    <GBSeq_length>1761</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Ralstonia sp. isolate MAG-2, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JADRFK000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JADRFK000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JADRFK000000000.1|JADRFK010000000</GBSeqid>\\n      <GBSeqid>gi|2240894477</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA674475</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Ralstonia sp. (snow metagenome)</GBSeq_source>\\n    <GBSeq_organism>Ralstonia sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Proteobacteria; Betaproteobacteria; Burkholderiales; Burkholderiaceae; Ralstonia</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..1761</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Rahlff,J.</GBAuthor>\\n          <GBAuthor>Bornemann,T.L.V.</GBAuthor>\\n          <GBAuthor>Lopatina,A.</GBAuthor>\\n          <GBAuthor>Severinov,K.</GBAuthor>\\n          <GBAuthor>Probst,A.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Host-Associated Phages Disperse across the Extraterrestrial Analogue Antarctica</GBReference_title>\\n        <GBReference_journal>Appl Environ Microbiol, e0031522 (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1128/aem.00315-22</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35499326</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..1761</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Rahlff,J.</GBAuthor>\\n          <GBAuthor>Bornemann,T.L.V.</GBAuthor>\\n          <GBAuthor>Lopatina,A.</GBAuthor>\\n          <GBAuthor>Severinov,K.</GBAuthor>\\n          <GBAuthor>Probst,A.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (15-NOV-2020) Department of Chemistry, University Duisburg-Essen, Universitaetsstrasse 5, Essen 45147, Germany</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Ralstonia sp. whole genome shotgun (WGS) project has the project accession JADRFK000000000. This version of the project (01) has the accession number JADRFK010000000, and consists of sequences JADRFK010000001-JADRFK010001761.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: SPAdes v. 3.13 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 8.6x ; Sequencing Technology :: Illumina MiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 11/25/2020 14:26:07 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 4.13 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 4,616 ; CDSs (total) :: 4,586 ; Genes (coding) :: 4,493 ; CDSs (with protein) :: 4,493 ; Genes (RNA) :: 30 ; tRNAs :: 27 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 93 ; CDSs (without protein) :: 93 ; Pseudo Genes (ambiguous residues) :: 1 of 93 ; Pseudo Genes (frameshifted) :: 22 of 93 ; Pseudo Genes (incomplete) :: 50 of 93 ; Pseudo Genes (internal stop) :: 26 of 93 ; Pseudo Genes (multiple problems) :: 6 of 93 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..1761</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>1761</GBInterval_to>\\n            <GBInterval_accession>JADRFK000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Ralstonia sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>MAG-2</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>surface snow</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:54061</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Antarctica: Druzhnaja</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>69.44462 S 73.42109 E</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2009-12-17</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>snow metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JADRFK010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JADRFK010001761</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA674475</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN16789484</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Ralstonia sp. isolate MAG-2, whole genome shotgun sequencing project', 'Ralstonia sp. (snow metagenome)', 'Ralstonia sp.', 'Bacteria; Proteobacteria; Betaproteobacteria; Burkholderiales; Burkholderiaceae; Ralstonia']\n",
      "Id 2240894475\n",
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBD010000000</GBSeq_locus>\\n    <GBSeq_length>40</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Firmicutes bacterium isolate HOT.MB2.4, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBD000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBD000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBD000000000.1|JAIMBD010000000</GBSeqid>\\n      <GBSeqid>gi|2240894475</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Firmicutes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Firmicutes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..40</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..40</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Firmicutes bacterium whole genome shotgun (WGS) project has the project accession JAIMBD000000000. This version of the project (01) has the accession number JAIMBD010000000, and consists of sequences JAIMBD010000001-JAIMBD010000040.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 11.8x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 12:37:04 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,473 ; CDSs (total) :: 2,425 ; Genes (coding) :: 2,396 ; CDSs (with protein) :: 2,396 ; Genes (RNA) :: 48 ; rRNAs :: 1, 1 (16S, 23S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 42 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 29 ; CDSs (without protein) :: 29 ; Pseudo Genes (ambiguous residues) :: 0 of 29 ; Pseudo Genes (frameshifted) :: 17 of 29 ; Pseudo Genes (incomplete) :: 19 of 29 ; Pseudo Genes (internal stop) :: 5 of 29 ; Pseudo Genes (multiple problems) :: 9 of 29 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..40</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>40</GBInterval_to>\\n            <GBInterval_accession>JAIMBD000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Firmicutes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.4</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1879010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBD010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBD010000040</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023753</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Firmicutes bacterium isolate HOT.MB2.4, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes']\n",
      "Id 2240894467\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAW010000000</GBSeq_locus>\\n    <GBSeq_length>90</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Candidatus Pacearchaeota archaeon isolate HOT.MB2.108, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAW000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAW000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAW000000000.1|JAIMAW010000000</GBSeqid>\\n      <GBSeqid>gi|2240894467</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Candidatus Pacearchaeota archaeon (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Candidatus Pacearchaeota archaeon</GBSeq_organism>\\n    <GBSeq_taxonomy>Archaea; Candidatus Pacearchaeota</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..90</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..90</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Candidatus Pacearchaeota archaeon whole genome shotgun (WGS) project has the project accession JAIMAW000000000. This version of the project (01) has the accession number JAIMAW010000000, and consists of sequences JAIMAW010000001-JAIMAW010000090.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 13.6x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 09:35:31 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 662 ; CDSs (total) :: 629 ; Genes (coding) :: 622 ; CDSs (with protein) :: 622 ; Genes (RNA) :: 33 ; rRNAs :: 1 (5S) ; complete rRNAs :: 1 (5S) ; tRNAs :: 31 ; ncRNAs :: 1 ; Pseudo Genes (total) :: 7 ; CDSs (without protein) :: 7 ; Pseudo Genes (ambiguous residues) :: 0 of 7 ; Pseudo Genes (frameshifted) :: 0 of 7 ; Pseudo Genes (incomplete) :: 6 of 7 ; Pseudo Genes (internal stop) :: 1 of 7 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..90</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>90</GBInterval_to>\\n            <GBInterval_accession>JAIMAW000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Candidatus Pacearchaeota archaeon</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.108</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2026773</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAW010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAW010000090</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023746</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Candidatus Pacearchaeota archaeon isolate HOT.MB2.108, whole genome shotgun sequencing project', 'Candidatus Pacearchaeota archaeon (hot springs metagenome)', 'Candidatus Pacearchaeota archaeon', 'Archaea; Candidatus Pacearchaeota']\n",
      "Id 2240894466\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JADRFL010000000</GBSeq_locus>\\n    <GBSeq_length>165</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Ralstonia pickettii isolate MAG-3, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JADRFL000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JADRFL000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JADRFL000000000.1|JADRFL010000000</GBSeqid>\\n      <GBSeqid>gi|2240894466</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA674475</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Ralstonia pickettii (snow metagenome)</GBSeq_source>\\n    <GBSeq_organism>Ralstonia pickettii</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Proteobacteria; Betaproteobacteria; Burkholderiales; Burkholderiaceae; Ralstonia</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..165</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Rahlff,J.</GBAuthor>\\n          <GBAuthor>Bornemann,T.L.V.</GBAuthor>\\n          <GBAuthor>Lopatina,A.</GBAuthor>\\n          <GBAuthor>Severinov,K.</GBAuthor>\\n          <GBAuthor>Probst,A.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Host-Associated Phages Disperse across the Extraterrestrial Analogue Antarctica</GBReference_title>\\n        <GBReference_journal>Appl Environ Microbiol, e0031522 (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1128/aem.00315-22</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35499326</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..165</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Rahlff,J.</GBAuthor>\\n          <GBAuthor>Bornemann,T.L.V.</GBAuthor>\\n          <GBAuthor>Lopatina,A.</GBAuthor>\\n          <GBAuthor>Severinov,K.</GBAuthor>\\n          <GBAuthor>Probst,A.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (15-NOV-2020) Department of Chemistry, University Duisburg-Essen, Universitaetsstrasse 5, Essen 45147, Germany</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Ralstonia pickettii whole genome shotgun (WGS) project has the project accession JADRFL000000000. This version of the project (01) has the accession number JADRFL010000000, and consists of sequences JADRFL010000001-JADRFL010000165.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: SPAdes v. 3.13 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 8.7x ; Sequencing Technology :: Illumina MiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 11/25/2020 12:24:30 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 4.13 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 1,230 ; CDSs (total) :: 1,225 ; Genes (coding) :: 1,189 ; CDSs (with protein) :: 1,189 ; Genes (RNA) :: 5 ; tRNAs :: 5 ; ncRNAs :: 0 ; Pseudo Genes (total) :: 36 ; CDSs (without protein) :: 36 ; Pseudo Genes (ambiguous residues) :: 0 of 36 ; Pseudo Genes (frameshifted) :: 7 of 36 ; Pseudo Genes (incomplete) :: 19 of 36 ; Pseudo Genes (internal stop) :: 12 of 36 ; Pseudo Genes (multiple problems) :: 2 of 36 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..165</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>165</GBInterval_to>\\n            <GBInterval_accession>JADRFL000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Ralstonia pickettii</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>MAG-3</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>surface snow</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:329</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Antarctica: Mirnii</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>66.33245 S 93.00296 E</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2009-12-18</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>snow metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JADRFL010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JADRFL010000165</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA674475</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN16789485</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Ralstonia pickettii isolate MAG-3, whole genome shotgun sequencing project', 'Ralstonia pickettii (snow metagenome)', 'Ralstonia pickettii', 'Bacteria; Proteobacteria; Betaproteobacteria; Burkholderiales; Burkholderiaceae; Ralstonia']\n",
      "Id 2240894465\n",
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAZ010000000</GBSeq_locus>\\n    <GBSeq_length>432</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Ignavibacterium sp. isolate HOT.MB2.1, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAZ000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAZ000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAZ000000000.1|JAIMAZ010000000</GBSeqid>\\n      <GBSeqid>gi|2240894465</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Ignavibacterium sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Ignavibacterium sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Ignavibacteriae; Ignavibacteria; Ignavibacteriales; Ignavibacteriaceae; Ignavibacterium; unclassified Ignavibacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..432</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..432</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Ignavibacterium sp. whole genome shotgun (WGS) project has the project accession JAIMAZ000000000. This version of the project (01) has the accession number JAIMAZ010000000, and consists of sequences JAIMAZ010000001-JAIMAZ010000432.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 38.6x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:12:11 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,940 ; CDSs (total) :: 2,900 ; Genes (coding) :: 2,873 ; CDSs (with protein) :: 2,873 ; Genes (RNA) :: 40 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 34 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 27 ; CDSs (without protein) :: 27 ; Pseudo Genes (ambiguous residues) :: 0 of 27 ; Pseudo Genes (frameshifted) :: 1 of 27 ; Pseudo Genes (incomplete) :: 20 of 27 ; Pseudo Genes (internal stop) :: 6 of 27 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..432</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>432</GBInterval_to>\\n            <GBInterval_accession>JAIMAZ000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Ignavibacterium sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2651167</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAZ010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAZ010000432</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023749</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Ignavibacterium sp. isolate HOT.MB2.1, whole genome shotgun sequencing project', 'Ignavibacterium sp. (hot springs metagenome)', 'Ignavibacterium sp.', 'Bacteria; Ignavibacteriae; Ignavibacteria; Ignavibacteriales; Ignavibacteriaceae; Ignavibacterium; unclassified Ignavibacterium']\n",
      "Id 2240894464\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBF010000000</GBSeq_locus>\\n    <GBSeq_length>201</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Acidobacteriia bacterium isolate HOT.MB2.42, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBF000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBF000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBF000000000.1|JAIMBF010000000</GBSeqid>\\n      <GBSeqid>gi|2240894464</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Acidobacteriia bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Acidobacteriia bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Acidobacteria</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..201</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..201</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Acidobacteriia bacterium whole genome shotgun (WGS) project has the project accession JAIMBF000000000. This version of the project (01) has the accession number JAIMBF010000000, and consists of sequences JAIMBF010000001-JAIMBF010000201.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 13.4x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 12:11:22 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,024 ; CDSs (total) :: 2,969 ; Genes (coding) :: 2,951 ; CDSs (with protein) :: 2,951 ; Genes (RNA) :: 55 ; rRNAs :: 2, 2, 1 (5S, 16S, 23S) ; complete rRNAs :: 2, 1 (5S, 23S) ; partial rRNAs :: 2 (16S) ; tRNAs :: 47 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 18 ; CDSs (without protein) :: 18 ; Pseudo Genes (ambiguous residues) :: 0 of 18 ; Pseudo Genes (frameshifted) :: 5 of 18 ; Pseudo Genes (incomplete) :: 16 of 18 ; Pseudo Genes (internal stop) :: 0 of 18 ; Pseudo Genes (multiple problems) :: 3 of 18 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..201</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>201</GBInterval_to>\\n            <GBInterval_accession>JAIMBF000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Acidobacteriia bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.42</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2081523</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBF010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBF010000201</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023755</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Acidobacteriia bacterium isolate HOT.MB2.42, whole genome shotgun sequencing project', 'Acidobacteriia bacterium (hot springs metagenome)', 'Acidobacteriia bacterium', 'Bacteria; Acidobacteria']\n",
      "Id 2240894463\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBL010000000</GBSeq_locus>\\n    <GBSeq_length>383</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Leptolyngbyaceae cyanobacterium HOT.MB2.61, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBL000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBL000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBL000000000.1|JAIMBL010000000</GBSeqid>\\n      <GBSeqid>gi|2240894463</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Leptolyngbyaceae cyanobacterium HOT.MB2.61 (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Leptolyngbyaceae cyanobacterium HOT.MB2.61</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Cyanobacteria; Pseudanabaenales; Leptolyngbyaceae; unclassified Leptolyngbyaceae</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..383</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..383</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Leptolyngbyaceae cyanobacterium HOT.MB2.61 whole genome shotgun (WGS) project has the project accession JAIMBL000000000. This version of the project (01) has the accession number JAIMBL010000000, and consists of sequences JAIMBL010000001-JAIMBL010000383.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 16.3x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 11:07:14 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 4,543 ; CDSs (total) :: 4,496 ; Genes (coding) :: 4,370 ; CDSs (with protein) :: 4,370 ; Genes (RNA) :: 47 ; rRNAs :: 1, 1 (16S, 23S) ; complete rRNAs :: 1 (16S) ; partial rRNAs :: 1 (23S) ; tRNAs :: 41 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 126 ; CDSs (without protein) :: 126 ; Pseudo Genes (ambiguous residues) :: 0 of 126 ; Pseudo Genes (frameshifted) :: 52 of 126 ; Pseudo Genes (incomplete) :: 64 of 126 ; Pseudo Genes (internal stop) :: 34 of 126 ; Pseudo Genes (multiple problems) :: 21 of 126 ; CRISPR Arrays :: 3 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..383</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>383</GBInterval_to>\\n            <GBInterval_accession>JAIMBL000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Leptolyngbyaceae cyanobacterium HOT.MB2.61</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.61</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2855467</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBL010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBL010000383</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023761</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Leptolyngbyaceae cyanobacterium HOT.MB2.61, whole genome shotgun sequencing project', 'Leptolyngbyaceae cyanobacterium HOT.MB2.61 (hot springs metagenome)', 'Leptolyngbyaceae cyanobacterium HOT.MB2.61', 'Bacteria; Cyanobacteria; Pseudanabaenales; Leptolyngbyaceae; unclassified Leptolyngbyaceae']\n",
      "Id 2240894462\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBJ010000000</GBSeq_locus>\\n    <GBSeq_length>73</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Firmicutes bacterium isolate HOT.MB2.53, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBJ000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBJ000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBJ000000000.1|JAIMBJ010000000</GBSeqid>\\n      <GBSeqid>gi|2240894462</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Firmicutes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Firmicutes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..73</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..73</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Firmicutes bacterium whole genome shotgun (WGS) project has the project accession JAIMBJ000000000. This version of the project (01) has the accession number JAIMBJ010000000, and consists of sequences JAIMBJ010000001-JAIMBJ010000073.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 17x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 12:48:30 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,449 ; CDSs (total) :: 3,398 ; Genes (coding) :: 3,358 ; CDSs (with protein) :: 3,358 ; Genes (RNA) :: 51 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 45 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 40 ; CDSs (without protein) :: 40 ; Pseudo Genes (ambiguous residues) :: 0 of 40 ; Pseudo Genes (frameshifted) :: 8 of 40 ; Pseudo Genes (incomplete) :: 25 of 40 ; Pseudo Genes (internal stop) :: 7 of 40 ; CRISPR Arrays :: 2 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..73</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>73</GBInterval_to>\\n            <GBInterval_accession>JAIMBJ000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Firmicutes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.53</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1879010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBJ010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBJ010000073</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023759</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Firmicutes bacterium isolate HOT.MB2.53, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes']\n",
      "Id 2240894461\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBG010000000</GBSeq_locus>\\n    <GBSeq_length>243</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Acetobacteraceae bacterium isolate HOT.MB2.46, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBG000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBG000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBG000000000.1|JAIMBG010000000</GBSeqid>\\n      <GBSeqid>gi|2240894461</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Acetobacteraceae bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Acetobacteraceae bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Proteobacteria; Alphaproteobacteria; Rhodospirillales; Acetobacteraceae; unclassified Acetobacteraceae</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..243</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..243</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Acetobacteraceae bacterium whole genome shotgun (WGS) project has the project accession JAIMBG000000000. This version of the project (01) has the accession number JAIMBG010000000, and consists of sequences JAIMBG010000001-JAIMBG010000243.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 39.7x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:36:40 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,382 ; CDSs (total) :: 2,343 ; Genes (coding) :: 2,324 ; CDSs (with protein) :: 2,324 ; Genes (RNA) :: 39 ; rRNAs :: 1 (16S) ; partial rRNAs :: 1 (16S) ; tRNAs :: 34 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 19 ; CDSs (without protein) :: 19 ; Pseudo Genes (ambiguous residues) :: 0 of 19 ; Pseudo Genes (frameshifted) :: 1 of 19 ; Pseudo Genes (incomplete) :: 17 of 19 ; Pseudo Genes (internal stop) :: 1 of 19 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..243</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>243</GBInterval_to>\\n            <GBInterval_accession>JAIMBG000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Acetobacteraceae bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.46</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1909293</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBG010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBG010000243</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023756</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Acetobacteraceae bacterium isolate HOT.MB2.46, whole genome shotgun sequencing project', 'Acetobacteraceae bacterium (hot springs metagenome)', 'Acetobacteraceae bacterium', 'Bacteria; Proteobacteria; Alphaproteobacteria; Rhodospirillales; Acetobacteraceae; unclassified Acetobacteraceae']\n",
      "Id 2240894460\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBM010000000</GBSeq_locus>\\n    <GBSeq_length>757</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Anaerolineae bacterium isolate HOT.MB2.62, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBM000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBM000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBM000000000.1|JAIMBM010000000</GBSeqid>\\n      <GBSeqid>gi|2240894460</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Anaerolineae bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Anaerolineae bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Chloroflexi; Anaerolineae; unclassified Anaerolineae</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..757</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..757</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Anaerolineae bacterium whole genome shotgun (WGS) project has the project accession JAIMBM000000000. This version of the project (01) has the accession number JAIMBM010000000, and consists of sequences JAIMBM010000001-JAIMBM010000757.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 13x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 14:09:41 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,461 ; CDSs (total) :: 3,415 ; Genes (coding) :: 3,369 ; CDSs (with protein) :: 3,369 ; Genes (RNA) :: 46 ; tRNAs :: 42 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 46 ; CDSs (without protein) :: 46 ; Pseudo Genes (ambiguous residues) :: 0 of 46 ; Pseudo Genes (frameshifted) :: 7 of 46 ; Pseudo Genes (incomplete) :: 38 of 46 ; Pseudo Genes (internal stop) :: 3 of 46 ; Pseudo Genes (multiple problems) :: 2 of 46 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..757</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>757</GBInterval_to>\\n            <GBInterval_accession>JAIMBM000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Anaerolineae bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.62</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2052143</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBM010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBM010000757</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023762</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Anaerolineae bacterium isolate HOT.MB2.62, whole genome shotgun sequencing project', 'Anaerolineae bacterium (hot springs metagenome)', 'Anaerolineae bacterium', 'Bacteria; Chloroflexi; Anaerolineae; unclassified Anaerolineae']\n",
      "Id 2240894459\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBK010000000</GBSeq_locus>\\n    <GBSeq_length>136</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Armatimonadetes bacterium isolate HOT.MB2.58, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBK000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBK000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBK000000000.1|JAIMBK010000000</GBSeqid>\\n      <GBSeqid>gi|2240894459</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Armatimonadetes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Armatimonadetes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Armatimonadetes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..136</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..136</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Armatimonadetes bacterium whole genome shotgun (WGS) project has the project accession JAIMBK000000000. This version of the project (01) has the accession number JAIMBK010000000, and consists of sequences JAIMBK010000001-JAIMBK010000136.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 18.1x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 11:59:15 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,262 ; CDSs (total) :: 3,213 ; Genes (coding) :: 3,135 ; CDSs (with protein) :: 3,135 ; Genes (RNA) :: 49 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 42 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 78 ; CDSs (without protein) :: 78 ; Pseudo Genes (ambiguous residues) :: 0 of 78 ; Pseudo Genes (frameshifted) :: 28 of 78 ; Pseudo Genes (incomplete) :: 52 of 78 ; Pseudo Genes (internal stop) :: 5 of 78 ; Pseudo Genes (multiple problems) :: 6 of 78 ; CRISPR Arrays :: 3 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..136</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>136</GBInterval_to>\\n            <GBInterval_accession>JAIMBK000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Armatimonadetes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.58</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2033014</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBK010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBK010000136</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023760</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Armatimonadetes bacterium isolate HOT.MB2.58, whole genome shotgun sequencing project', 'Armatimonadetes bacterium (hot springs metagenome)', 'Armatimonadetes bacterium', 'Bacteria; Armatimonadetes']\n",
      "Id 2240894458\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBH010000000</GBSeq_locus>\\n    <GBSeq_length>58</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Firmicutes bacterium isolate HOT.MB2.5, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBH000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBH000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBH000000000.1|JAIMBH010000000</GBSeqid>\\n      <GBSeqid>gi|2240894458</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Firmicutes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Firmicutes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..58</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..58</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Firmicutes bacterium whole genome shotgun (WGS) project has the project accession JAIMBH000000000. This version of the project (01) has the accession number JAIMBH010000000, and consists of sequences JAIMBH010000001-JAIMBH010000058.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 24.3x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 11:14:31 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,311 ; CDSs (total) :: 2,257 ; Genes (coding) :: 2,238 ; CDSs (with protein) :: 2,238 ; Genes (RNA) :: 54 ; rRNAs :: 1, 1, 2 (5S, 16S, 23S) ; complete rRNAs :: 1, 1 (5S, 16S) ; partial rRNAs :: 2 (23S) ; tRNAs :: 46 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 19 ; CDSs (without protein) :: 19 ; Pseudo Genes (ambiguous residues) :: 0 of 19 ; Pseudo Genes (frameshifted) :: 5 of 19 ; Pseudo Genes (incomplete) :: 11 of 19 ; Pseudo Genes (internal stop) :: 4 of 19 ; Pseudo Genes (multiple problems) :: 1 of 19 ; CRISPR Arrays :: 2 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..58</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>58</GBInterval_to>\\n            <GBInterval_accession>JAIMBH000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Firmicutes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.5</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1879010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBH010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBH010000058</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023757</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Firmicutes bacterium isolate HOT.MB2.5, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes']\n",
      "Id 2240894457\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBO010000000</GBSeq_locus>\\n    <GBSeq_length>218</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Rubrobacteraceae bacterium isolate HOT.MB2.74, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBO000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBO000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBO000000000.1|JAIMBO010000000</GBSeqid>\\n      <GBSeqid>gi|2240894457</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Rubrobacteraceae bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Rubrobacteraceae bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Actinobacteria; Rubrobacteria; Rubrobacterales; Rubrobacteraceae; unclassified Rubrobacteraceae</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..218</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..218</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Rubrobacteraceae bacterium whole genome shotgun (WGS) project has the project accession JAIMBO000000000. This version of the project (01) has the accession number JAIMBO010000000, and consists of sequences JAIMBO010000001-JAIMBO010000218.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 13x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:07:44 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,950 ; CDSs (total) :: 2,904 ; Genes (coding) :: 2,897 ; CDSs (with protein) :: 2,897 ; Genes (RNA) :: 46 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 40 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 7 ; CDSs (without protein) :: 7 ; Pseudo Genes (ambiguous residues) :: 0 of 7 ; Pseudo Genes (frameshifted) :: 3 of 7 ; Pseudo Genes (incomplete) :: 4 of 7 ; Pseudo Genes (internal stop) :: 2 of 7 ; Pseudo Genes (multiple problems) :: 2 of 7 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..218</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>218</GBInterval_to>\\n            <GBInterval_accession>JAIMBO000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Rubrobacteraceae bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.74</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2740537</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBO010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBO010000218</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023764</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Rubrobacteraceae bacterium isolate HOT.MB2.74, whole genome shotgun sequencing project', 'Rubrobacteraceae bacterium (hot springs metagenome)', 'Rubrobacteraceae bacterium', 'Bacteria; Actinobacteria; Rubrobacteria; Rubrobacterales; Rubrobacteraceae; unclassified Rubrobacteraceae']\n",
      "Id 2240894456\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBP010000000</GBSeq_locus>\\n    <GBSeq_length>239</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Acidobacterium ailaaui isolate HOT.MB2.78, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBP000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBP000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBP000000000.1|JAIMBP010000000</GBSeqid>\\n      <GBSeqid>gi|2240894456</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Acidobacterium ailaaui (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Acidobacterium ailaaui</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Acidobacteria; Acidobacteriales; Acidobacteriaceae; Acidobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..239</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..239</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Acidobacterium ailaaui whole genome shotgun (WGS) project has the project accession JAIMBP000000000. This version of the project (01) has the accession number JAIMBP010000000, and consists of sequences JAIMBP010000001-JAIMBP010000239.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 22.2x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:03:31 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,283 ; CDSs (total) :: 2,249 ; Genes (coding) :: 2,239 ; CDSs (with protein) :: 2,239 ; Genes (RNA) :: 34 ; rRNAs :: 1 (16S) ; partial rRNAs :: 1 (16S) ; tRNAs :: 29 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 10 ; CDSs (without protein) :: 10 ; Pseudo Genes (ambiguous residues) :: 0 of 10 ; Pseudo Genes (frameshifted) :: 4 of 10 ; Pseudo Genes (incomplete) :: 5 of 10 ; Pseudo Genes (internal stop) :: 3 of 10 ; Pseudo Genes (multiple problems) :: 2 of 10 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..239</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>239</GBInterval_to>\\n            <GBInterval_accession>JAIMBP000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Acidobacterium ailaaui</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.78</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1382359</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBP010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBP010000239</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023765</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Acidobacterium ailaaui isolate HOT.MB2.78, whole genome shotgun sequencing project', 'Acidobacterium ailaaui (hot springs metagenome)', 'Acidobacterium ailaaui', 'Bacteria; Acidobacteria; Acidobacteriales; Acidobacteriaceae; Acidobacterium']\n",
      "Id 2240894455\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBU010000000</GBSeq_locus>\\n    <GBSeq_length>914</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Gorillibacterium sp. isolate HOT.MB2.94, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBU000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBU000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBU000000000.1|JAIMBU010000000</GBSeqid>\\n      <GBSeqid>gi|2240894455</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Gorillibacterium sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Gorillibacterium sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Paenibacillaceae; Gorillibacterium; unclassified Gorillibacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..914</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..914</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Gorillibacterium sp. whole genome shotgun (WGS) project has the project accession JAIMBU000000000. This version of the project (01) has the accession number JAIMBU010000000, and consists of sequences JAIMBU010000001-JAIMBU010000914.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 21.6x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:48:59 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,963 ; CDSs (total) :: 3,945 ; Genes (coding) :: 3,880 ; CDSs (with protein) :: 3,880 ; Genes (RNA) :: 18 ; rRNAs :: 2, 1 (16S, 23S) ; partial rRNAs :: 2, 1 (16S, 23S) ; tRNAs :: 13 ; ncRNAs :: 2 ; Pseudo Genes (total) :: 65 ; CDSs (without protein) :: 65 ; Pseudo Genes (ambiguous residues) :: 0 of 65 ; Pseudo Genes (frameshifted) :: 15 of 65 ; Pseudo Genes (incomplete) :: 34 of 65 ; Pseudo Genes (internal stop) :: 20 of 65 ; Pseudo Genes (multiple problems) :: 4 of 65 ; CRISPR Arrays :: 7 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..914</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>914</GBInterval_to>\\n            <GBInterval_accession>JAIMBU000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Gorillibacterium sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.94</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1978774</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBU010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBU010000914</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023770</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Gorillibacterium sp. isolate HOT.MB2.94, whole genome shotgun sequencing project', 'Gorillibacterium sp. (hot springs metagenome)', 'Gorillibacterium sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Paenibacillaceae; Gorillibacterium; unclassified Gorillibacterium']\n",
      "Id 2240894454\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JADRFJ010000000</GBSeq_locus>\\n    <GBSeq_length>1788</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Janthinobacterium lividum isolate MAG-1, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JADRFJ000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JADRFJ000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JADRFJ000000000.1|JADRFJ010000000</GBSeqid>\\n      <GBSeqid>gi|2240894454</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA674475</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Janthinobacterium lividum (snow metagenome)</GBSeq_source>\\n    <GBSeq_organism>Janthinobacterium lividum</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Proteobacteria; Betaproteobacteria; Burkholderiales; Oxalobacteraceae; Janthinobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..1788</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Rahlff,J.</GBAuthor>\\n          <GBAuthor>Bornemann,T.L.V.</GBAuthor>\\n          <GBAuthor>Lopatina,A.</GBAuthor>\\n          <GBAuthor>Severinov,K.</GBAuthor>\\n          <GBAuthor>Probst,A.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Host-Associated Phages Disperse across the Extraterrestrial Analogue Antarctica</GBReference_title>\\n        <GBReference_journal>Appl Environ Microbiol, e0031522 (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1128/aem.00315-22</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35499326</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..1788</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Rahlff,J.</GBAuthor>\\n          <GBAuthor>Bornemann,T.L.V.</GBAuthor>\\n          <GBAuthor>Lopatina,A.</GBAuthor>\\n          <GBAuthor>Severinov,K.</GBAuthor>\\n          <GBAuthor>Probst,A.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (15-NOV-2020) Department of Chemistry, University Duisburg-Essen, Universitaetsstrasse 5, Essen 45147, Germany</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Janthinobacterium lividum whole genome shotgun (WGS) project has the project accession JADRFJ000000000. This version of the project (01) has the accession number JADRFJ010000000, and consists of sequences JADRFJ010000001-JADRFJ010001788.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: SPAdes v. 3.13 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 7.5x ; Sequencing Technology :: Illumina MiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 11/25/2020 14:27:59 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 4.13 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 4,920 ; CDSs (total) :: 4,873 ; Genes (coding) :: 4,762 ; CDSs (with protein) :: 4,762 ; Genes (RNA) :: 47 ; tRNAs :: 43 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 111 ; CDSs (without protein) :: 111 ; Pseudo Genes (ambiguous residues) :: 0 of 111 ; Pseudo Genes (frameshifted) :: 22 of 111 ; Pseudo Genes (incomplete) :: 61 of 111 ; Pseudo Genes (internal stop) :: 37 of 111 ; Pseudo Genes (multiple problems) :: 9 of 111 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..1788</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>1788</GBInterval_to>\\n            <GBInterval_accession>JADRFJ000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Janthinobacterium lividum</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>MAG-1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>surface snow</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:29581</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Antarctica: Druzhnaja</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>69.44462 S 73.42109 E</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2009-12-17</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>snow metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JADRFJ010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JADRFJ010001788</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA674475</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN16789483</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Janthinobacterium lividum isolate MAG-1, whole genome shotgun sequencing project', 'Janthinobacterium lividum (snow metagenome)', 'Janthinobacterium lividum', 'Bacteria; Proteobacteria; Betaproteobacteria; Burkholderiales; Oxalobacteraceae; Janthinobacterium']\n",
      "Id 2240894453\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBS010000000</GBSeq_locus>\\n    <GBSeq_length>627</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Alicyclobacillus sp. isolate HOT.MB2.88, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBS000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBS000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBS000000000.1|JAIMBS010000000</GBSeqid>\\n      <GBSeqid>gi|2240894453</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Alicyclobacillus sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Alicyclobacillus sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..627</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..627</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Alicyclobacillus sp. whole genome shotgun (WGS) project has the project accession JAIMBS000000000. This version of the project (01) has the accession number JAIMBS010000000, and consists of sequences JAIMBS010000001-JAIMBS010000627.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 29.1x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:47:25 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 4,860 ; CDSs (total) :: 4,820 ; Genes (coding) :: 4,690 ; CDSs (with protein) :: 4,690 ; Genes (RNA) :: 40 ; rRNAs :: 1, 1, 2 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 2 (16S, 23S) ; tRNAs :: 33 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 130 ; CDSs (without protein) :: 130 ; Pseudo Genes (ambiguous residues) :: 0 of 130 ; Pseudo Genes (frameshifted) :: 43 of 130 ; Pseudo Genes (incomplete) :: 77 of 130 ; Pseudo Genes (internal stop) :: 27 of 130 ; Pseudo Genes (multiple problems) :: 16 of 130 ; CRISPR Arrays :: 2 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..627</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>627</GBInterval_to>\\n            <GBInterval_accession>JAIMBS000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Alicyclobacillus sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.88</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:61169</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBS010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBS010000627</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023768</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Alicyclobacillus sp. isolate HOT.MB2.88, whole genome shotgun sequencing project', 'Alicyclobacillus sp. (hot springs metagenome)', 'Alicyclobacillus sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus']\n",
      "Id 2240894452\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBR010000000</GBSeq_locus>\\n    <GBSeq_length>254</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Peptococcaceae bacterium isolate HOT.MB2.86, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBR000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBR000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBR000000000.1|JAIMBR010000000</GBSeqid>\\n      <GBSeqid>gi|2240894452</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Peptococcaceae bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Peptococcaceae bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Clostridia; Eubacteriales; Peptococcaceae</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..254</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..254</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Peptococcaceae bacterium whole genome shotgun (WGS) project has the project accession JAIMBR000000000. This version of the project (01) has the accession number JAIMBR010000000, and consists of sequences JAIMBR010000001-JAIMBR010000254.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 11x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:25:57 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,419 ; CDSs (total) :: 3,376 ; Genes (coding) :: 3,306 ; CDSs (with protein) :: 3,306 ; Genes (RNA) :: 43 ; rRNAs :: 2, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; tRNAs :: 36 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 70 ; CDSs (without protein) :: 70 ; Pseudo Genes (ambiguous residues) :: 0 of 70 ; Pseudo Genes (frameshifted) :: 27 of 70 ; Pseudo Genes (incomplete) :: 36 of 70 ; Pseudo Genes (internal stop) :: 12 of 70 ; Pseudo Genes (multiple problems) :: 5 of 70 ; CRISPR Arrays :: 2 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..254</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>254</GBInterval_to>\\n            <GBInterval_accession>JAIMBR000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Peptococcaceae bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.86</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2052179</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBR010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBR010000254</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023767</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Peptococcaceae bacterium isolate HOT.MB2.86, whole genome shotgun sequencing project', 'Peptococcaceae bacterium (hot springs metagenome)', 'Peptococcaceae bacterium', 'Bacteria; Firmicutes; Clostridia; Eubacteriales; Peptococcaceae']\n",
      "Id 2240894451\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBQ010000000</GBSeq_locus>\\n    <GBSeq_length>228</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Thermoleophilum sp. isolate HOT.MB2.84, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBQ000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBQ000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBQ000000000.1|JAIMBQ010000000</GBSeqid>\\n      <GBSeqid>gi|2240894451</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Thermoleophilum sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Thermoleophilum sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Actinobacteria; Thermoleophilia; Thermoleophilales; Thermoleophilaceae; Thermoleophilum; unclassified Thermoleophilum</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..228</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..228</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Thermoleophilum sp. whole genome shotgun (WGS) project has the project accession JAIMBQ000000000. This version of the project (01) has the accession number JAIMBQ010000000, and consists of sequences JAIMBQ010000001-JAIMBQ010000228.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 54.2x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 09:48:08 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,034 ; CDSs (total) :: 1,987 ; Genes (coding) :: 1,982 ; CDSs (with protein) :: 1,982 ; Genes (RNA) :: 47 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 41 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 5 ; CDSs (without protein) :: 5 ; Pseudo Genes (ambiguous residues) :: 0 of 5 ; Pseudo Genes (frameshifted) :: 0 of 5 ; Pseudo Genes (incomplete) :: 5 of 5 ; Pseudo Genes (internal stop) :: 0 of 5 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..228</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>228</GBInterval_to>\\n            <GBInterval_accession>JAIMBQ000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Thermoleophilum sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.84</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2480089</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBQ010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBQ010000228</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023766</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Thermoleophilum sp. isolate HOT.MB2.84, whole genome shotgun sequencing project', 'Thermoleophilum sp. (hot springs metagenome)', 'Thermoleophilum sp.', 'Bacteria; Actinobacteria; Thermoleophilia; Thermoleophilales; Thermoleophilaceae; Thermoleophilum; unclassified Thermoleophilum']\n",
      "Id 2240894450\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBE010000000</GBSeq_locus>\\n    <GBSeq_length>235</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Firmicutes bacterium isolate HOT.MB2.40, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBE000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBE000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBE000000000.1|JAIMBE010000000</GBSeqid>\\n      <GBSeqid>gi|2240894450</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Firmicutes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Firmicutes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..235</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..235</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Firmicutes bacterium whole genome shotgun (WGS) project has the project accession JAIMBE000000000. This version of the project (01) has the accession number JAIMBE010000000, and consists of sequences JAIMBE010000001-JAIMBE010000235.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 13.7x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 11:53:26 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,522 ; CDSs (total) :: 2,481 ; Genes (coding) :: 2,468 ; CDSs (with protein) :: 2,468 ; Genes (RNA) :: 41 ; tRNAs :: 37 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 13 ; CDSs (without protein) :: 13 ; Pseudo Genes (ambiguous residues) :: 0 of 13 ; Pseudo Genes (frameshifted) :: 3 of 13 ; Pseudo Genes (incomplete) :: 8 of 13 ; Pseudo Genes (internal stop) :: 2 of 13 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..235</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>235</GBInterval_to>\\n            <GBInterval_accession>JAIMBE000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Firmicutes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.40</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1879010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBE010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBE010000235</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023754</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Firmicutes bacterium isolate HOT.MB2.40, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes']\n",
      "Id 2240894449\n",
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JADRFN010000000</GBSeq_locus>\\n    <GBSeq_length>147</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Flavobacterium micromati isolate MAG-5, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JADRFN000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JADRFN000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JADRFN000000000.1|JADRFN010000000</GBSeqid>\\n      <GBSeqid>gi|2240894449</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA674475</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Flavobacterium micromati (snow metagenome)</GBSeq_source>\\n    <GBSeq_organism>Flavobacterium micromati</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Bacteroidetes; Flavobacteriia; Flavobacteriales; Flavobacteriaceae; Flavobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..147</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Rahlff,J.</GBAuthor>\\n          <GBAuthor>Bornemann,T.L.V.</GBAuthor>\\n          <GBAuthor>Lopatina,A.</GBAuthor>\\n          <GBAuthor>Severinov,K.</GBAuthor>\\n          <GBAuthor>Probst,A.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Host-Associated Phages Disperse across the Extraterrestrial Analogue Antarctica</GBReference_title>\\n        <GBReference_journal>Appl Environ Microbiol, e0031522 (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1128/aem.00315-22</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35499326</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..147</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Rahlff,J.</GBAuthor>\\n          <GBAuthor>Bornemann,T.L.V.</GBAuthor>\\n          <GBAuthor>Lopatina,A.</GBAuthor>\\n          <GBAuthor>Severinov,K.</GBAuthor>\\n          <GBAuthor>Probst,A.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (15-NOV-2020) Department of Chemistry, University Duisburg-Essen, Universitaetsstrasse 5, Essen 45147, Germany</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Flavobacterium micromati whole genome shotgun (WGS) project has the project accession JADRFN000000000. This version of the project (01) has the accession number JADRFN010000000, and consists of sequences JADRFN010000001-JADRFN010000147.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: SPAdes v. 3.13 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 19.2x ; Sequencing Technology :: Illumina MiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 11/25/2020 13:13:31 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 4.13 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,713 ; CDSs (total) :: 2,685 ; Genes (coding) :: 2,638 ; CDSs (with protein) :: 2,638 ; Genes (RNA) :: 28 ; tRNAs :: 25 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 47 ; CDSs (without protein) :: 47 ; Pseudo Genes (ambiguous residues) :: 6 of 47 ; Pseudo Genes (frameshifted) :: 17 of 47 ; Pseudo Genes (incomplete) :: 7 of 47 ; Pseudo Genes (internal stop) :: 25 of 47 ; Pseudo Genes (multiple problems) :: 8 of 47 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..147</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>147</GBInterval_to>\\n            <GBInterval_accession>JADRFN000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Flavobacterium micromati</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>MAG-5</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>surface snow</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:229205</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Antarctica: Progress</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>69.40059 S 76.37939 E</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2009-12-28</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>snow metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JADRFN010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JADRFN010000147</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA674475</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN16789487</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Flavobacterium micromati isolate MAG-5, whole genome shotgun sequencing project', 'Flavobacterium micromati (snow metagenome)', 'Flavobacterium micromati', 'Bacteria; Bacteroidetes; Flavobacteriia; Flavobacteriales; Flavobacteriaceae; Flavobacterium']\n",
      "Id 2240894448\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBV010000000</GBSeq_locus>\\n    <GBSeq_length>256</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Alicyclobacillus mali isolate HOT.MB2.97, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBV000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBV000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBV000000000.1|JAIMBV010000000</GBSeqid>\\n      <GBSeqid>gi|2240894448</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Alicyclobacillus mali (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Alicyclobacillus mali</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..256</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..256</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Alicyclobacillus mali whole genome shotgun (WGS) project has the project accession JAIMBV000000000. This version of the project (01) has the accession number JAIMBV010000000, and consists of sequences JAIMBV010000001-JAIMBV010000256.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 15.4x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:39:04 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,956 ; CDSs (total) :: 2,918 ; Genes (coding) :: 2,870 ; CDSs (with protein) :: 2,870 ; Genes (RNA) :: 38 ; rRNAs :: 2 (5S) ; partial rRNAs :: 2 (5S) ; tRNAs :: 33 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 48 ; CDSs (without protein) :: 48 ; Pseudo Genes (ambiguous residues) :: 0 of 48 ; Pseudo Genes (frameshifted) :: 15 of 48 ; Pseudo Genes (incomplete) :: 29 of 48 ; Pseudo Genes (internal stop) :: 7 of 48 ; Pseudo Genes (multiple problems) :: 2 of 48 ; CRISPR Arrays :: 2 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..256</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>256</GBInterval_to>\\n            <GBInterval_accession>JAIMBV000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Alicyclobacillus mali</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.97</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1123961</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBV010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBV010000256</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023771</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Alicyclobacillus mali isolate HOT.MB2.97, whole genome shotgun sequencing project', 'Alicyclobacillus mali (hot springs metagenome)', 'Alicyclobacillus mali', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus']\n",
      "Id 2240893217\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000000</GBSeq_locus>\\n    <GBSeq_length>2342000</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>BCT</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB00000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB00000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB00000000.1|NZ_AYHB01000000</GBSeqid>\\n      <GBSeqid>gi|2240893217</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..2342000</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..2342000</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB00000000.1.; The Fusobacterium nucleatum YWH7199 whole genome shotgun (WGS) project has the project accession NZ_AYHB00000000. This version of the project (01) has the accession number NZ_AYHB01000000, and consists of sequences AYHB01000001-AYHB01000276.; Annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (released 2013). Information about the Pipeline can be found here: http://www.ncbi.nlm.nih.gov/genome/annotation_prok/; Bacteria and source DNA available from Yiping W. Han, Department of Periodontics, School of Dental Medicine, Case Western Reserve University, Cleveland, OH 44106-4905, USA.; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..2342000</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>2342000</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB00000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>AYHB01000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>AYHB01000276</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS_SCAFLD</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>NZ_AYHB01000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>NZ_AYHB01000276</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199, whole genome shotgun sequencing project', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893216\n",
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000276</GBSeq_locus>\\n    <GBSeq_length>5598</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0283, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000276</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000276.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000276.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0283</GBSeqid>\\n      <GBSeqid>gi|2240893216</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..5598</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..5598</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000276.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..5598</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>5598</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000276.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0283</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>&lt;1..428</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>428</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000276.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11845</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12245</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>&lt;1..428</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>428</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000276.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11845</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12245</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005894563.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>3</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>DUF3798 domain-containing protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527467.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>ADLPSPTMGYPGAFGIEFSDDEKGNWPKILEEVEKAVIAAGGSGRMGTWAYSYNFAGVEGLTDLAIKSIESGDRDFTLDKLLASLNVATPDAKWNGSIMKDNNGVDVPNAFFIYQDTYIFGKGYMGVTSVEIPEKYTNLGK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>complement(487..1146)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1146</GBInterval_from>\\n            <GBInterval_to>487</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000276.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11850</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12250</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>complement(487..1146)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1146</GBInterval_from>\\n            <GBInterval_to>487</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000276.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11850</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12250</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008800337.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>helix-turn-helix transcriptional regulator</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_022069210.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MSFGKTLKRIRLKHKDSLRGLAKKIDLHFTFIDKVEKGTAPISKNFIENVIAVYPDEREILKKEYLKETLPNVFQKEDAIKIVSNSEILNLPVYGKASAGRGYMNMDSPDYYMPILRGNFSKRSFFVEITGNSMEPTLEDGEFALVDPDNTAYSKNKIYVVTYNDEGYIKRLEMNNKLKLITLKSDNPDYEDIDIPEEMQEYFSVNGRVVEVISKKKLL</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>1948..3039</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1948</GBInterval_from>\\n            <GBInterval_to>3039</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000276.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11855</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12260</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>1948..3039</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1948</GBInterval_from>\\n            <GBInterval_to>3039</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000276.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11855</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12260</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008795007.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>dipeptide epimerase</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_060798043.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKITKIKLGIISVPLRVPFKTALRSVNSVEDVIVEIHTDTGNVGYGEAPPTGVITGDTTGAIVGALKDHIIKTLIGRDVDDFENLMKDLNSCIVKNTSAKAAADIALWDLYGQLHKIPVYKLLGGSRNKIVTDITISVNPPEEMARDAINAIKRGYDTLKVKVGIDPTLDVARLSAIRDAIGKDYRIRIDANQAWSPKQAIRLLNQMQDRGLDIELVEQPVKAHDFDGLRYVTKHSDVPVLADESVFSPEDAFKILQMKAADLINIKLMKCGGIYNALKIINLAEVVGVECMIGCMLEAKVSVNAAVHLACAKQVITKIDLDGPVLCSEDPIIGGAVFNEKEITISDDFGLGIKGINGIKYID</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>3061..4242</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3061</GBInterval_from>\\n            <GBInterval_to>4242</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000276.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11860</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12265</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>3061..4242</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3061</GBInterval_from>\\n            <GBInterval_to>4242</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000276.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11860</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12265</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005971875.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>DUF819 family protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527468.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MVITNGFTYIAFLMCLAGCLLLLEKYSKWRIFNVVPALVFIYILNMAFCTMGLFNSEACSKAYSVLKNNLLYAMIFVMLLRCDFRKLAKLGGRMVAIFLACSFTLFIGFVVGYPIFKSFLGVDVWGAVAALYASWVGGSANMAAMQAALPVDAGAYSCALALDTVCYSVWIALLLLMVRYSSKWDNATKADTSKLQAVADAAAKEVEKEKKTASAADWVFLIGISLMVSAISQMIGGALQNGFASIGLEMFDRGTITTVFVTILGLVCALTPLGKLPAVEELSTVYLYAVVSLLASTASVVDLLTAPMWIVYGLFILVIHVVLMFVFSKIFHWDLCMVSTASLANIGGSASAPIVASAYNPSYAGIGVLMGVLGAAVGNFFGIGIGQILKMMS</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>4471..5256</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>4471</GBInterval_from>\\n            <GBInterval_to>5256</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000276.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11865</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12270</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>4471..5256</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>4471</GBInterval_from>\\n            <GBInterval_to>5256</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000276.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11865</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12270</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005910563.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>class A beta-lactamase-related serine hydrolase</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527469.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MEKYTEWKKEIEKIISQVKGSVCINFYDLVKNTGFSIDGDKKVLSASMIKLLILAELMKKVSENKFSLSDAITVTEAMKTGGDGVLKELNSDHHFTLKELATLMIIVSDNQATNILIDFLGMKNINLLGKELRLKETFLGRKMMDIEARKKGYDNYTCADDISLLLKLIYQKKLINKEASQLMLDILLRQQQEERLQRYLPSDIKIAHKCGDLDNLENDGGIIWIGDKAYILAVLTNGMPNFQCKQTIGKISKFVYDKMEE</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>5259..&gt;5598</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>5259</GBInterval_from>\\n            <GBInterval_to>5598</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000276.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11870</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12275</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>5259..&gt;5598</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>5259</GBInterval_from>\\n            <GBInterval_to>5598</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000276.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11870</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12275</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008795009.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>DUF3870 domain-containing protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527470.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MNNQTIYVTGEARITIDNAITKMFGTFYIAFEVILSTDEIIDMDCNATLRLTRDFIKRLFLNHNIIKDEEILKKEIATRYFGSSSKAILTAYHDALQHSEPLTTDTLRVLESQ</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000276.1:1..5598)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0283, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893215\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000275</GBSeq_locus>\\n    <GBSeq_length>545</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0282, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000275</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000275.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000275.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0282</GBSeqid>\\n      <GBSeqid>gi|2240893215</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..545</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..545</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000275.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..545</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>545</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000275.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0282</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>&lt;1..&gt;545</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>545</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000275.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11840</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12240</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>&lt;1..&gt;545</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>545</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000275.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11840</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12240</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005894563.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>2</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>DUF3798 domain-containing protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527466.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>DDPKMKAIVVAEGVPGTYPAFKAIREKRPDILLFVNNNHEDPVQVSTVADVVMNSDSIARGYLIVKTAHDLGATKFMHISFPRHLSYETISRRRAIMEQTAKDLGMEYIEMSAPDPLSDVGVPGAQQFILEQVPNWVKKYGKDIAFFATNDAQTEPLLKQIAAYGGYFIKCVRTLVALAVR</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000275.1:1..545)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0282, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893214\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000274</GBSeq_locus>\\n    <GBSeq_length>8773</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0281, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000274</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000274.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000274.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0281</GBSeqid>\\n      <GBSeqid>gi|2240893214</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..8773</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..8773</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000274.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..8773</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>8773</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0281</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>414..776</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>414</GBInterval_from>\\n            <GBInterval_to>776</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11785</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12175</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>414..776</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>414</GBInterval_from>\\n            <GBInterval_to>776</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11785</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12175</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005910548.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_016340075.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MLKNLLLYTNDFLALFVMQLTIITVISIQIYALIKVKLLQRKYYIFLLIKILEIIFFKYEIFYMYLILFIIGVILNISLAIRGKKIDIKEMILGTIIFSTVPLIITFGILYLILLNFIYE</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>1172..1810</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1172</GBInterval_from>\\n            <GBInterval_to>1810</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11790</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12185</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>1172..1810</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1172</GBInterval_from>\\n            <GBInterval_to>1810</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11790</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12185</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005910549.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_009007580.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MLKNRKELIELIEFGYDIKEIINSWDPMGLMEFCPEDEYEAEIKGLRNLVVNNRNTDKKLLGKEIRKLFRFYFSNRYNSKRDVEENIAGKIIEKSKKYKLSCTVSNYYDIENIIFKNEKEIDIYINLYTKINKIINSWDPLKIMDISFSNEYSYEINRIIEELLKNITIQNLSKEINKIFKNAYNGLYKIEKNEEIEITEKIFEEYNNISKL</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>2223..2504</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>2223</GBInterval_from>\\n            <GBInterval_to>2504</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11795</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12195</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>2223..2504</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>2223</GBInterval_from>\\n            <GBInterval_to>2504</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11795</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12195</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: ab initio prediction:GeneMarkS-2+</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: GeneMarkS-2+.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527464.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MIVIDCNRCEKYKYIIEELFIKEDKIIIFHINKKKRIEKYKIKFDEITDLEYKDGFFLNPYRPDTFFHKNIEKCRLLKIKLKSKKVVSFGFFL</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>2634..3209</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>2634</GBInterval_from>\\n            <GBInterval_to>3209</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>pth</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11800</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12200</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>2634..3209</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>2634</GBInterval_from>\\n            <GBInterval_to>3209</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>pth</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11800</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12200</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>EC_number</GBQualifier_name>\\n            <GBQualifier_value>3.1.1.29</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_009007578.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_function</GBQualifier_name>\\n            <GBQualifier_value>GO:0004045 - aminoacyl-tRNA hydrolase activity [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_process</GBQualifier_name>\\n            <GBQualifier_value>GO:0006412 - translation [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>aminoacyl-tRNA hydrolase</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_005910552.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKVVIGLGNPGKKYERTRHNMGFIAVDSLRKKFNLNDEREKFQALVSEKNIDGEKVIFFKPQTFMNLSGNAVIEIINFYKLDPKKDIIVIYDDMDLPFGDIRIREKGSSGGHNGIKSIISHIGEEFVRIKCGIGEKEKDAIEHVLGEFNQTEQKNLDEILENIDNCIIEMLTVQNLDRIMQKYNKKKEKLK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>3283..4599</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3283</GBInterval_from>\\n            <GBInterval_to>4599</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>rsxC</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11805</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12205</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>3283..4599</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3283</GBInterval_from>\\n            <GBInterval_to>4599</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>rsxC</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11805</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12205</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008693384.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_component</GBQualifier_name>\\n            <GBQualifier_value>GO:0016020 - membrane [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_function</GBQualifier_name>\\n            <GBQualifier_value>GO:0051539 - 4 iron, 4 sulfur cluster binding [Evidence IEA]; GO:0009055 - electron transfer activity [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>electron transport complex subunit RsxC</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_005905476.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MTFFGFRGGVHPPENKIQTEHLPIEKLESPDEIFVPLLQHIGVPLNPLVNVGDRVLKGQKIADAEGLAVPIHSPVSGTVTKIESRVFPLTGKVMTVFIENDKKEEWAELTKIENWEEADKKALLDIIREKGIVGIGGATFPTHVKLNPPPNTKLDSLILNGAECEPYLNSDNRLMLENPKSIVEGIKIIKKILNVPDVYVGIEDNKPEAIESMRKATEGTGINIVPLKTKYPQGGEKQLIKSVLNREVPSGQLPSAVGVVVQNTGTAAAIYEAVVNGKPLIEKIVTVSGKAIKNPKNLKIAIGTPFSYILDYCGVDREKMERLVMGGPMMGLAQMTEDATVIKGTSGLLALTTEEMRPYKTKSCISCSKCVSACPMGLEPLMFDRLAANKQYDEMAEYNLMDCIECGSCNYICPANRPLAEAIKTGKSKLRAKAKKKK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>4625..5572</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>4625</GBInterval_from>\\n            <GBInterval_to>5572</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11810</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12210</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>4625..5572</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>4625</GBInterval_from>\\n            <GBInterval_to>5572</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11810</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12210</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005919433.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>RnfABCDGE type electron transport complex subunit D</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_005905473.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MSTILKTGPAPHIRTAETVESVMYDVVIALIPAFAMAVYSFGVRALILTAVSVATCIATEYIWQKILKRDIEAFDGSAILTGILFAFVVPVGMGLQYVIVGNFVAITLGKIVYGGLGHNIFNPALIGRAFVQASWPVAITTFAYDGKAGATVLDAMKRGIPLTDSLIYEGGNQYINAFLGQMGGCLGETSALALLIGGVYLIYKKQIDWKVPAVMIGTVFVLTWAMGANPFMQILSGGLFLGAFFMATDMVTSPITGKGRVIFALGIGILVSLIRIKGGYPEGTAYAILIMNGVVPLIDRYIRAKKFGGVSKNGK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>5562..6095</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>5562</GBInterval_from>\\n            <GBInterval_to>6095</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11815</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12215</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>5562..6095</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>5562</GBInterval_from>\\n            <GBInterval_to>6095</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11815</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12215</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008821913.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_component</GBQualifier_name>\\n            <GBQualifier_value>GO:0005886 - plasma membrane [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_function</GBQualifier_name>\\n            <GBQualifier_value>GO:0009055 - electron transfer activity [Evidence IEA]; GO:0010181 - FMN binding [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_process</GBQualifier_name>\\n            <GBQualifier_value>GO:0022900 - electron transport chain [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>RnfABCDGE type electron transport complex subunit G</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_008693383.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MENRYIHFGIVLGLIAAISAGILGGVNGFTSKVIAENTKKIVNEARQEVLPEAVSFKEDEAKVADEIQYIPGFNDAGEVVGYVASVTEPGYGGDINFVVGIDKDAKVTGLNVVTSSETPGLGAKINGKEWQEHWIGKDATYEFNKSVDAFAGATISPKAVYTGVIKALNTYQNEVSK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>6095..6712</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>6095</GBInterval_from>\\n            <GBInterval_to>6712</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11820</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12220</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>6095..6712</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>6095</GBInterval_from>\\n            <GBInterval_to>6712</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11820</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12220</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005969446.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>electron transport complex subunit E</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_005910556.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKKFEVLTSGIFKENPIFVLMLGLCPTLGVTSSAINGFSMGLAVIAVLACSNGLISLFKKFIPDEVRIPAFIMIIASLVTVVDMVMNAYTPDLYKVLGIFIPLIVVNCIVLGRAESFASKNGVIDSILDGIGSGIGFTLSLTFLGSIREILGNGSIFGIPLVPANFSPALIFILAPGGFITIGIIMGCINMKKERDAKKKKVAKK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>6709..7293</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>6709</GBInterval_from>\\n            <GBInterval_to>7293</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>rsxA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11825</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12225</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>6709..7293</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>6709</GBInterval_from>\\n            <GBInterval_to>7293</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>rsxA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11825</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12225</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_009424153.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_component</GBQualifier_name>\\n            <GBQualifier_value>GO:0016020 - membrane [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_process</GBQualifier_name>\\n            <GBQualifier_value>GO:0022900 - electron transport chain [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>electron transport complex subunit RsxA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_022069213.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MSISGLFSIVITSIFINNIIFAKFLGCCPFMGVSKKVDSSLGMGMAVTFVITIASGVTWIVYRLILEPHGLGYLQTIAFILIIASLVQFVEMAIKKTSPSLYKALGVFLPLITTNCAVLGVAIINIQVGYNFIETLVNGFGVAVGFSLALLLLAGIRERLEFANTPKNFKGVPIAFITAGLLAMAFMGFSGMQI</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>7320..8282</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>7320</GBInterval_from>\\n            <GBInterval_to>8282</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11830</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12230</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>7320..8282</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>7320</GBInterval_from>\\n            <GBInterval_to>8282</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11830</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12230</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005910558.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_function</GBQualifier_name>\\n            <GBQualifier_value>GO:0009055 - electron transfer activity [Evidence IEA]; GO:0051536 - iron-sulfur cluster binding [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_process</GBQualifier_name>\\n            <GBQualifier_value>GO:0022900 - electron transport chain [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>RnfABCDGE type electron transport complex subunit B</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249524214.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MEAIIMPVVVLGITGVLMGLFLAFASKKFEVEVDPKVEAILAVLPGANCGACGFPGCAGYAAGVALEGAKMTLCAPGGPKVAAKIGDIMGVAVEMPVKKKPAAKKPVEKKVVQTGEPISASQEFIEKNKRMLMKFKDAFDAGDKEGYEKLENLAKMAKKDELLKYYEEIKSGKIVPDGSAPVAAGAASTNPISATKEFVEKNKRMLMKFKEAFDAGDKEGYEKLENLAKMAKKDELLKFYEEIKAGKIVPDPATMVDSPAVKEETPKVETKVGDSKKQEAAYCSTLGDGLCVPEQNEKTKEDLKKQAEPPKTAEETEKDK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>8469..&gt;8773</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>8469</GBInterval_from>\\n            <GBInterval_to>8773</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11835</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>8469..&gt;8773</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>8469</GBInterval_from>\\n            <GBInterval_to>8773</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11835</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005894563.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>DUF3798 domain-containing protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527465.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKMKRILFGLLTIFMLMLAVACGKKEAPTEDANAQQQESANEAVQDYHIGVVTTSVSQSEDNFRGAEAILKKYGAANERGKITIVTIPDNFMQEQETTISQ</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000274.1:1..8773)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0281, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893213\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000273</GBSeq_locus>\\n    <GBSeq_length>4901</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0280, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000273</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000273.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000273.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0280</GBSeqid>\\n      <GBSeqid>gi|2240893213</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..4901</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..4901</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000273.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..4901</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>4901</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0280</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>&lt;1..662</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>662</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11740</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12125</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>&lt;1..662</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>662</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11740</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12125</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008798111.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>incomplete; too short partial abutting assembly gap; missing N-terminus; Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>3</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>644..880</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>644</GBInterval_from>\\n            <GBInterval_to>880</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11745</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12130</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>644..880</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>644</GBInterval_from>\\n            <GBInterval_to>880</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11745</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12130</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_009423188.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>serine kinase</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527457.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MECKKIKKVYMLYHINERKNEKLIGFLSSKEKAKSIIKELVEKPGFKDCPNGFRIKTMIIGKDYYTKGFKSKCAPKDE</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>926..1195</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>926</GBInterval_from>\\n            <GBInterval_to>1195</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11750</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12135</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>926..1195</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>926</GBInterval_from>\\n            <GBInterval_to>1195</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11750</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12135</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008821871.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527458.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKIYVLSHEYCYGDYKYKYKEESRFIGIYLTRKEALRALEKFKKIRGFSSHLDGFYIDRTEINQLEWLDGYRTGCFSMELGMHEEKEEE</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>1197..1469</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1197</GBInterval_from>\\n            <GBInterval_to>1469</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11755</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12140</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>1197..1469</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1197</GBInterval_from>\\n            <GBInterval_to>1469</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11755</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12140</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_009423190.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527459.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKIYVLSHSYECDVEKTKDEGRFLGVYLTKKEAIKKAEMYKKIKGFSSHISNFYIGCYIVDKTLKWLDNYSIDNWYAVGFMGKFYRQNIE</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>1475..1750</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1475</GBInterval_from>\\n            <GBInterval_to>1750</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11760</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12145</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>1475..1750</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1475</GBInterval_from>\\n            <GBInterval_to>1750</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11760</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12145</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005910539.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527460.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKIDDDVQKDQVSVMNKLLNDALRAKGYAGPDIKMVLTDVTALNEPYYTNTFVVGGSLGIINGELIYDSETSELSAFRLGITASAISLIKN</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>1841..2689</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1841</GBInterval_from>\\n            <GBInterval_to>2689</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11765</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12150</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>1841..2689</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1841</GBInterval_from>\\n            <GBInterval_to>2689</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11765</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12150</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005910540.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527461.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MEIENTKESTVIVFKNNSVFIGVYIVPLFIIVPLIISKITVQSIIMSLIVFTYLNIGNYIAMRNIGSIETITIKNESLIIRRLKRNKKITYEKEIFFDKILKIYYQEIFLGFHKRNFNFDVKRTLKIKTYFCIYSFGYKMSYEDFKKANTTIEEKIKNYKNFIEKKETKKLYIEIYNLEIEKRYEKILNIILDEEKLYLLKKNDCYIVNGDSEAIKNLKIFKNMNFKEIDFYIFYVNYLSKKEYEDKKVLVGYNGVDGKKVTMSKFKEDINEIRDSGSTFRN</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>3026..3532</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3026</GBInterval_from>\\n            <GBInterval_to>3532</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11770</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12155</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>3026..3532</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3026</GBInterval_from>\\n            <GBInterval_to>3532</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11770</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12155</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005910541.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>DUF2247 family protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527462.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MITLNDFKNNNLKINWKVINIGCLGSEIFKNELSYDDIINFSLEKFDEKNKLILRIIASDRDEYQEIGYLVQELANMEKSEYKLEFEKWKLVYVKKNFPKLNKNIIQGLIELNNLWVKLDFPEDSPYILQGVKNNISPQEYYTEKNYIYLYNRHLKWIRDKSDYLDGK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>&lt;3532..&gt;4080</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3532</GBInterval_from>\\n            <GBInterval_to>4080</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11775</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12160</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>&lt;3532..&gt;4080</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3532</GBInterval_from>\\n            <GBInterval_to>4080</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11775</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12160</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_017144793.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>incomplete; partial in the middle of a contig; missing N-terminus and C-terminus; Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hemin transporter</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>4156..&gt;4901</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>4156</GBInterval_from>\\n            <GBInterval_to>4901</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11780</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12165</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>4156..&gt;4901</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>4156</GBInterval_from>\\n            <GBInterval_to>4901</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11780</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12165</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_011016284.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527463.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MRKVENRQHFNLYIYALISLIVTDIAIIIFLVGLIFSRPIIYVIIVILLIMVYFITRAIVNCLKYYISKEECYCENGTLIYKRILLKKFILKEVEIPLLNIQKVIDKGHIPSNNAEPDYLNPFHYVVLFFKYYERVLLEMKTGNKYEIFVYAFPYGTRGKSFEKIYNDNDFLKSFTELKEMIEEEQSKILFNQKVENLRERYNFPLDKRYNYILNKIIDEEKLFISEKDNNFIINGDSEAIKNLEIFK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000273.1:1..4901)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0280, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893212\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000272</GBSeq_locus>\\n    <GBSeq_length>798</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0279, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000272</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000272.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000272.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0279</GBSeqid>\\n      <GBSeqid>gi|2240893212</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..798</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..798</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000272.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..798</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>798</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000272.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0279</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>&lt;1..612</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>612</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000272.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11735</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>&lt;1..612</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>612</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000272.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11735</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_011016284.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527456.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>KGHIPSHNARRDVLNPLHYVVLFFNYYERILLEMKTGDKYEIFVYAFPYGTRTEELEKIYNDNDFLKSFDELKEMIEEEQKKILFSQKVKNLSEKYNSPLEERYNYVLNKILDEEKLFLFKKDNNFVINGDSEAIKDLEIFKNINFEEIDFYIFYVNYLSKKEYENKKVLVGYNGVDGKEVTMSKFKEDINEIRDSRSTFKNQ</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000272.1:1..798)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0279, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893211\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000271</GBSeq_locus>\\n    <GBSeq_length>1462</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0277, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000271</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000271.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000271.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0277</GBSeqid>\\n      <GBSeqid>gi|2240893211</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..1462</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..1462</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000271.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..1462</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>1462</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000271.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0277</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>76..1416</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>76</GBInterval_from>\\n            <GBInterval_to>1416</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000271.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11730</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12105</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>76..1416</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>76</GBInterval_from>\\n            <GBInterval_to>1416</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000271.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11730</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12105</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005974393.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>MATE family efflux transporter</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_005909886.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MNSIGNVGKKTLISLTIPIFFELLLVTVVGNIDTIMLGYYSDKAVGAIGGITQLLNIQNVIFSFINLATAILTAQFLGARDYKRVKQVISVSLVLNVLLGVVLGGIYLFFWRGLLQKINLPEELVGIGKYYFQMVGGLCVFQGIILSCGAILKSHGRPTETLIINVGVNILNILGNGMFIFGWLGMPVLGTTGVGISTVVSRGIGCVAAFYMMCKYCNFTFRKKYIKPFPFKIVKNILSIGFPTAGENLSWNVGQLLIVAMVNTMGTTIIASRTYLMLISNFVMTLSIALGQGTAIQVGHLVGAGEVKEVYSKCLKSVKIAFIFAFVATSIVCFLRKPIMNIFTTNPDILSASLKIFPLMIILEMGRVFNIVIINSLHAAGDIKFPMFMSIICVYAVAVLFSYIFGISLGWGLAGIWIANAMDEWIRGIAMYLRWKSRKWQNKSFV</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000271.1:1..1462)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0277, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893210\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000270</GBSeq_locus>\\n    <GBSeq_length>12725</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0276, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000270</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000270.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000270.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0276</GBSeqid>\\n      <GBSeqid>gi|2240893210</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..12725</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..12725</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000270.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..12725</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>12725</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0276</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>184..663</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>184</GBInterval_from>\\n            <GBInterval_to>663</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11660</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12035</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>184..663</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>184</GBInterval_from>\\n            <GBInterval_to>663</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11660</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12035</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_016339983.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527450.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MNKISRYLGRELANKDYFFISKENREKEQTKANPNFTGNHEFFAFGKTRLYQSVYGSVKRLENGKYDVDITVMFQYTDRFEDVKNINNSSVAKQGLNKEFEGGKSFSFKTEPKIITIKKQVNNLDEITGLLKNRLRNIDDNSEEGKDNIRADFYDSVIF</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>644..1132</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>644</GBInterval_from>\\n            <GBInterval_to>1132</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11665</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12040</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>644..1132</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>644</GBInterval_from>\\n            <GBInterval_to>1132</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11665</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12040</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008798426.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527451.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MIVLYSKTSNEKEENFKRKYFELFYLFIFFIYFSFAIISGYFQHYEYIFTDINNTKEYAVFSEGRYVYIRLENNMLTEGQFPDFRIKEVYEDKNNFIGCDFEYNYETDKKEERYYIVIDKNKASMKIFSEKEFKEKYNISDGKFINIYTFLKKKGTKIGRYR</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>1151..1618</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1151</GBInterval_from>\\n            <GBInterval_to>1618</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11670</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12045</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>1151..1618</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1151</GBInterval_from>\\n            <GBInterval_to>1618</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11670</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12045</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_016361408.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527452.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKYGENKIIKISVALTTVLNIFISIIEIRYIEFIYILIFIGFNIYVRGTKTHNEKENITYVEKKEIVKSNKVLFIGTNILIFFYIFKEPYFFLSKYKILILLISIIIGYSSLIFLQKIRIKLSLNFFKEIIKIFLISGIIYYFPIVCVQILLKIT</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>1840..2838</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1840</GBInterval_from>\\n            <GBInterval_to>2838</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>plsX</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11675</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12050</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>1840..2838</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1840</GBInterval_from>\\n            <GBInterval_to>2838</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>plsX</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11675</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12050</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>EC_number</GBQualifier_name>\\n            <GBQualifier_value>2.3.1.274</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_011016179.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_function</GBQualifier_name>\\n            <GBQualifier_value>GO:0043811 - phosphate:acyl-[acyl carrier protein] acyltransferase activity [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_process</GBQualifier_name>\\n            <GBQualifier_value>GO:0008610 - lipid biosynthetic process [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>phosphate acyltransferase PlsX</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_008694253.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKIALDAMSGDFAPISTVKGAIEALQEIEGLEIILVGKEGIIKEELKKYKYDTKRIEIKNADEVIEMTDDPVKAVREKKNSSMNVCIDLVKEKIAQASVSCGNTGALLASSQLKLKRIKGVLRPAIAVLFPNKKEQGTLFLDLGANSDSKPEFLNQFATMGSKYMEIFSGKKNPKVALLNIGEEETKGNELTRETYALLKENKDIDFYGNIESTKIMDGEVDVVVTDGYTGNVLLKTSEGVGKFIFHIVKEAIMESWISKLGALLIKGAMKKVKKKTEASEYGGAIFLGLSELSLKAHGNSNSRAIKNALKVASKFIELNFIEELRKTMEVE</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>2838..3824</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>2838</GBInterval_from>\\n            <GBInterval_to>3824</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11680</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12055</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>2838..3824</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>2838</GBInterval_from>\\n            <GBInterval_to>3824</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11680</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12055</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_009424177.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>ketoacyl-ACP synthase III</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527453.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MQSIGIKGIGYYVPENVFTNFDFEKIIDTSDEWIRTRTGIVERRFASKDQATSDLACEAALKVIKNAKINKEDVDMIILATTTPDYIAQGASCIVQHKLGLKNIPCFDVNAACTGFIYSLSVAYALVNSKLYKNILVIGAETLTRIIDIKNRNTAVLFGDGAAAAIVGEVEEGYGILGFSIGAEGEDDMILKVPAGGSKKPNDEETVKNRENFVVMKGQEVFKFAVRVLPKVTLEALKKAKLNVRDLSMIFPHQANLRIIESAAKRMKFPLEKFYMNLSRYGNTSSASVGIALGEAIEKGLIKKGDNIALTGFGGGLTYGSVIIKWAY</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>3857..4756</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3857</GBInterval_from>\\n            <GBInterval_to>4756</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>fabD</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11685</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12060</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>3857..4756</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3857</GBInterval_from>\\n            <GBInterval_to>4756</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>fabD</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11685</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12060</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>EC_number</GBQualifier_name>\\n            <GBQualifier_value>2.3.1.39</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_011016181.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_function</GBQualifier_name>\\n            <GBQualifier_value>GO:0004314 - [acyl-carrier-protein] S-malonyltransferase activity [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_process</GBQualifier_name>\\n            <GBQualifier_value>GO:0006633 - fatty acid biosynthetic process [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>ACP S-malonyltransferase</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_060676636.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MGKVAFVYPGQGTQYVGMGKELYEKNKRAKELFDKIFNSLDIDLKKVMFEGPEDLLKRTDYTQPAIVSLSLVLTELLKEKGIEPDYVAGHSVGEFAAFGGANYLSIEDAVKLVAARGRIMREVAEKVNGSMAAVLGMDAEKIKEVLKSVDGVVEAVNFNEPNQTVIAGSKEAVEKACVVLKEAGAKRAIPLAVSGPFHSSLMKEAGEKLKEEAQKYTFNIGKVKIVANTTAELLESDSEVKDEIYRQSFGPVKWVDTINKLKSLGVTKIYEIGAGKVLSGLIKKIDKEIEVENIELIEA</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>4825..5052</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>4825</GBInterval_from>\\n            <GBInterval_to>5052</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>acpP</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11690</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12065</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>4825..5052</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>4825</GBInterval_from>\\n            <GBInterval_to>5052</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>acpP</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11690</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12065</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005902726.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_process</GBQualifier_name>\\n            <GBQualifier_value>GO:0006633 - fatty acid biosynthetic process [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>acyl carrier protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_005906540.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MLDKVKEIIVEQLGVDADQIKPESNFVDDLGADSLDTVELIMSFEEEFGVEIPDTEAEKIKTVQDVINYIEAHKK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>5379..6620</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>5379</GBInterval_from>\\n            <GBInterval_to>6620</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>fabF</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11695</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12070</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>5379..6620</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>5379</GBInterval_from>\\n            <GBInterval_to>6620</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>fabF</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11695</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12070</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>EC_number</GBQualifier_name>\\n            <GBQualifier_value>2.3.1.179</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005915867.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_function</GBQualifier_name>\\n            <GBQualifier_value>GO:0004315 - 3-oxoacyl-[acyl-carrier-protein] synthase activity [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_process</GBQualifier_name>\\n            <GBQualifier_value>GO:0006633 - fatty acid biosynthetic process [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>beta-ketoacyl-ACP synthase II</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527454.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKRVVVTGLGLISALGIGLEESWKKLIAGETGIDLIKSYDTTDQPVRIAGEVKGFEPTDYGIEKKEVKKLARNTQFALVATKMALEDANFKIDETNADDVGVIVSAGVGGIEIMEEQYGNMLTKGYRRISPFTIPAMIENMAAGNIAIYYGAKGPNRSIVTACASGTHSIGDGFDLIRHGRAKAMIVGGTEASVTQFCINSFANMKALSTRNETPKTASRPFSKDRDGFVMGEGAGILILEELESALARGAKIYAEMVGFGDTSDANHITAPIETGEGATKAMRAALKDANIPLEDVTYINAHGTSTPTNDIVETRAIKALFGDKAKSLYVSSTKGATGHALGGAGGIEGVIIAKTISEGIIPPTINLYETEEECDLNYVPNKAIKADVKVAMSNSLGFGGHNSVIVMKKFEK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>6636..7340</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>6636</GBInterval_from>\\n            <GBInterval_to>7340</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>rnc</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11700</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12075</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>6636..7340</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>6636</GBInterval_from>\\n            <GBInterval_to>7340</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>rnc</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11700</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12075</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>EC_number</GBQualifier_name>\\n            <GBQualifier_value>3.1.26.3</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_009006826.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_function</GBQualifier_name>\\n            <GBQualifier_value>GO:0004525 - ribonuclease III activity [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_process</GBQualifier_name>\\n            <GBQualifier_value>GO:0006396 - RNA processing [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>ribonuclease III</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_005909892.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKNLLDLEHKLNYYFNDRNLLKNALLHKSLGNERKEYKNQNNERLELLGDAVLDLIVAEYLYKSYKNASEGTIAKLKAMIVSEPILAKISRQIGVGKFLMLSRGEVMSGGRNRESILADSFEAILGAVYIDSNLDDARNFTLSHIKQYIDHIEENEDILDFKSILQEYVQKEFRTVPTYKLIAERGPDHMKEFEIQVIVGNYREKAVAKNKKKAEQLSAKALCIKLGVNYHEAL</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>7327..8373</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>7327</GBInterval_from>\\n            <GBInterval_to>8373</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11705</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12080</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>7327..8373</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>7327</GBInterval_from>\\n            <GBInterval_to>8373</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11705</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12080</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_011016184.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_function</GBQualifier_name>\\n            <GBQualifier_value>GO:0051536 - iron-sulfur cluster binding [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>radical SAM protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_022070416.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKHYNIPVFISHFGCPNACVFCNQKKINGRETDVSLDDLKNIIDSYLKTLPKNSIKQVAFFGGTFTGISMNLQKEYLEVVKKYIDNNDVEGVRISTRPECIDDEILTQLKKYGVKTIELGIQSLDDKVLKATGRNYAYDVVKKSCDLIKNYGFELGIQLMIGLPKSDFKSDLQSAIKSLELNPDIARIYPTLVIKGTELELMYKKNLYQSLSIEEAVDRTVPIYSLLELKNINVIRVGLQPAEDLTADGVIISGPFHPAFRDLIENKIYFNFLSKIYKKEKKLDIEVNERNVSKIVGQKAINKKTFYPNFKILINNNLDLDELMINSKKYTRKEILEGEFNEKISDFI</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>8348..9724</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>8348</GBInterval_from>\\n            <GBInterval_to>9724</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11710</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12085</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>8348..9724</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>8348</GBInterval_from>\\n            <GBInterval_to>9724</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11710</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12085</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008702438.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>ribonuclease E/G</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527455.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKKYLILSKSVCETKLALLEDNKLDEMYIERNNQKEIIGNIYKGKVVDILNNGEIIFVDIGLEKNALLSFENKKKVPKFNIDDKLIVQTESEPRDEKGAKLTLDYSINGENLVLLPKSKNISISKKIKDIEEINRLKNIFLSIDNGLILRTNSEEKSEESLLEEYRKLEDIDNQINKDFEKINIGLLYDVNSILKKAVTLFNNTIEEFIIDDEHIFEEIKILLEKTGKKDLIKKLRKYFKDEDIFEYYNINSQIERALDRKVYLDSGAYIIIEKTEALISIDVNTGQNIGNKTSQELIFQTNLEATKEIARQIKLRNLAGIIIVDFIDMKKFSDRKRVLEEFKKYLSVDRAEINSVEYTNLGLIQFTRKRQGKELAFYYKEKCQYCEGTGYFLSKDRIILNLLEDLNSQIKSQDIKKIVIRAKKDIIKELNKYIGNNKIEYIEDNNFYKIGYKMELYK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>9755..10252</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>9755</GBInterval_from>\\n            <GBInterval_to>10252</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>coaD</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11715</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12090</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>9755..10252</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>9755</GBInterval_from>\\n            <GBInterval_to>10252</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>coaD</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11715</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12090</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>EC_number</GBQualifier_name>\\n            <GBQualifier_value>2.7.7.3</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005909889.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_function</GBQualifier_name>\\n            <GBQualifier_value>GO:0004595 - pantetheine-phosphate adenylyltransferase activity [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_process</GBQualifier_name>\\n            <GBQualifier_value>GO:0015937 - coenzyme A biosynthetic process [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>pantetheine-phosphate adenylyltransferase</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_005909889.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKTGVYAGSFDPITKGHQDIIERALKIVDKLIVVVMNNPKKNYWFNLDERKNLISKIFEGDKSIKVDEHAGLLVDFMAKNSCGILIKGLRDVKDFSEEMTYSFANKKLSNGEVDTIFIPTSEKYTYVSSTFVKELAFYNQSLAGYVDDKVIVDILNRAKEYRDRG</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>10257..11615</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>10257</GBInterval_from>\\n            <GBInterval_to>11615</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>radA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11720</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12095</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>10257..11615</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>10257</GBInterval_from>\\n            <GBInterval_to>11615</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>radA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11720</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12095</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005895523.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_function</GBQualifier_name>\\n            <GBQualifier_value>GO:0005524 - ATP binding [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_process</GBQualifier_name>\\n            <GBQualifier_value>GO:0006281 - DNA repair [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>DNA repair protein RadA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_005909888.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MAKGTVYYCSECGYKSAKWVGKCPQCGAWSSFEEVDELPKDVKKTKNSDIKVYEFKDVEYTSEDRYKTKYEEFDRLLGGGLLKGEVVLVTGNPGIGKSTLLLQVANSYKDYGNILYISGEESPAQIKNRGERLKISGESIYIMAEMDILNIYEYVVSKKPKVVIVDSIQTLYNSSMDSISGTPTQIRECTLKIIEIAKKYNISFFIVGHITKDGKVAGPKLLEHMVDAVFNFEGDEGLYYRILRSVKNRFGSTNEIAVFSMEENGMKEIKNSSEYFLSEREEKNIGSMVVPILEGTKVFLLEVQSLITDSGIGIPKRVVQGYDRNRIQILTAIAEKKLYVPLSMKDLFVNVPGGLSIEDPAADLAVLMSILSVHKGFAISQKIAAIGELGLRGEIRKVFFLERRLKELEKLGFTGVYVPESNRKELEKKKFKLKIIYLKNLDELLERMNKND</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>11608..12657</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>11608</GBInterval_from>\\n            <GBInterval_to>12657</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>disA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11725</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12100</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>11608..12657</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>11608</GBInterval_from>\\n            <GBInterval_to>12657</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>disA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11725</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12100</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>EC_number</GBQualifier_name>\\n            <GBQualifier_value>2.7.7.85</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005966456.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>DNA integrity scanning diadenylate cyclase DisA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_005909887.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MTKQDLMDIIVKVAPGSPLREGVDYILDAGIGALIVIGYDDEVEKVRDGGFFIDCDYTPERIFELSKMDGAIILNDDCSKILYANVHIQPNISFITTESGTRHRTAERTAKHLKREVVAISERKKNVTLYKGELKYRLKNFDELNIEVGQVLKTLESYRHVLNRSLDNLTILELDDLVTVLDVANTLQRFEMVRRISEEITRYLLELGTRGRLVNMQVSELIWDLDDEEESFLKDYIDDGMTTDSVRRYLHSLSDSELLDIENVVVALGYSKSSSVFDNKIAAKGYRVLEKISKLTKKDIEKIVNTYKDISEIQEVTDEDLSAIKISKFKIKALRAGINRLKFTIEMQR</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000270.1:1..12725)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0276, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893209\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000269</GBSeq_locus>\\n    <GBSeq_length>3387</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0275, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000269</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000269.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000269.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0275</GBSeqid>\\n      <GBSeqid>gi|2240893209</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..3387</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..3387</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000269.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..3387</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>3387</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0275</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>&lt;1..230</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>230</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11620</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11990</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>&lt;1..230</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>230</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11620</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11990</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008701199.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>incomplete; too short partial abutting assembly gap; missing N-terminus; Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>3</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>621..773</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>621</GBInterval_from>\\n            <GBInterval_to>773</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11625</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>621..773</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>621</GBInterval_from>\\n            <GBInterval_to>773</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11625</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: ab initio prediction:GeneMarkS-2+</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: GeneMarkS-2+.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527444.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MTGIKQWQIKPETILLKEIADSQQLQGTQYIILEELNKYLFINLGNIKHY</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>826..1119</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>826</GBInterval_from>\\n            <GBInterval_to>1119</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11630</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12000</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>826..1119</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>826</GBInterval_from>\\n            <GBInterval_to>1119</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11630</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12000</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008794343.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527449.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MLEKVIQQCQKTLDKIEEELSKPEPKLTPYDIEMGNFDEVPRLILKEAKRQIKIMMQVLDKNKYMPDYTYPLIDSYLIDTELCDLLFETKSIYKKYT</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>1340..1663</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1340</GBInterval_from>\\n            <GBInterval_to>1663</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11635</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12005</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>1340..1663</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1340</GBInterval_from>\\n            <GBInterval_to>1663</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11635</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12005</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_009008008.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527445.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MEELSRQFKAGEINEKQLKEAARDIVKGYGKDIGIDYEVVYLDESTMPKDPKGSTGSAYIFKLGDDNNSNISLTTDGIDLTNADVGEKVGDFFPKQEIDTAGGVKKV</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>1702..2121</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1702</GBInterval_from>\\n            <GBInterval_to>2121</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11640</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12010</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>1702..2121</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1702</GBInterval_from>\\n            <GBInterval_to>2121</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11640</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: ab initio prediction:GeneMarkS-2+</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: GeneMarkS-2+.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527446.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MLDLFRKTGEFTLRALKAGREDLSLNHIKFLKGFNIILYHEDTKNLEKELEELKDGWGKGSFSDKEATIKYHYNKHKEEVGADSLLQYLRKAKAFSKNLRGAKKSNVMGATKGVIKYEKNGKYIEIAPNGDIISFGTIK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>2167..2568</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>2167</GBInterval_from>\\n            <GBInterval_to>2568</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11645</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12015</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>2167..2568</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>2167</GBInterval_from>\\n            <GBInterval_to>2568</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11645</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12015</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005910007.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_060798864.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MLVYNFEISNGEKVFINNGVISYMFDSFKWIKTFNRLRMKRNKGLFYHGSTYIEKENIDKMKEVVFAWKELFNEASEEFILLGFFNKKLDDYERWKCNKRQVIESLKKLIILCDKARKNNKIIRCRKITIKAQ</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>2583..2975</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>2583</GBInterval_from>\\n            <GBInterval_to>2975</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11650</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12020</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>2583..2975</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>2583</GBInterval_from>\\n            <GBInterval_to>2975</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11650</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12020</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005910006.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>coproporphyrinogen III oxidase</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527447.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MLIHDFGIVGEKKDVHLYDDLILYMIDTFEWIKTFSELESNIEKNGLNHSGITYFKGESVTKLKNIILHWINIFNLGEKTIELRGLFSINEKKHSYNKISKKHLIESLKKLVLLCEKAEKENKIIEHWGI</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>3120..&gt;3387</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3120</GBInterval_from>\\n            <GBInterval_to>3387</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11655</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12025</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>3120..&gt;3387</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3120</GBInterval_from>\\n            <GBInterval_to>3387</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11655</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12025</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008700668.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>HD domain-containing protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527448.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKYNFNELKEIVKSKMSLKRFTHTLGVVEMSEKLAKIYNADIEKCKVAALLHDICKEMDMKYIKNICKNNFMNKLSEEDLENNEILHGF</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000269.1:1..3387)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0275, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893208\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000268</GBSeq_locus>\\n    <GBSeq_length>1484</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0274, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000268</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000268.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000268.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0274</GBSeqid>\\n      <GBSeqid>gi|2240893208</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..1484</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..1484</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000268.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..1484</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>1484</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000268.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0274</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>&lt;1..&gt;530</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>530</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000268.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11605</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>&lt;1..&gt;530</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>530</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000268.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11605</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005909906.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>frameshifted; incomplete; partial in the middle of a contig; missing N-terminus and C-terminus; Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>2</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hemolysin</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>564..1040</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>564</GBInterval_from>\\n            <GBInterval_to>1040</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000268.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11610</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11980</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>564..1040</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>564</GBInterval_from>\\n            <GBInterval_to>1040</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000268.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11610</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11980</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_016339539.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527442.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MMNKKFNIKKNIGNLIFIVFLFPAYFFSTLISNYVQHYEYIFTDKNRIKKYKVFSDRRTLYAYIDLEPDMYTEGEFSDFKIKKVYENSDYFIGYNFDVNRDSEEIENKYYVVVNKNKATMEFFSEKEFKEKLRNINDEKFIYVYPFLKKRGTKIGWYR</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>1064..&gt;1484</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1064</GBInterval_from>\\n            <GBInterval_to>1484</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000268.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11615</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11985</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>1064..&gt;1484</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1064</GBInterval_from>\\n            <GBInterval_to>1484</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000268.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11615</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11985</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_016361408.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527443.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKYKIIETVKISMIMTTILNFFICIATELRYIEFIYLLLFLGINLYMIGVKGYFKIETYNYSLYTIRKKIVDSQRATTLSINILIFFYIFKEPYFFLSKYKILILLISIVIGYSSLIFLQKIKIKLSLNFFKEIIKIFLI</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000268.1:1..1484)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0274, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893207\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000267</GBSeq_locus>\\n    <GBSeq_length>7464</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0273, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000267</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000267.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000267.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0273</GBSeqid>\\n      <GBSeqid>gi|2240893207</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..7464</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..7464</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000267.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..7464</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>7464</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0273</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>&lt;1..1801</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>1801</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11570</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>&lt;1..1801</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>1801</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11570</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: ab initio prediction:GeneMarkS-2+</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>incomplete; too short partial abutting assembly gap; missing N-terminus; Derived by automated computational analysis using gene prediction method: GeneMarkS-2+.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>2</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>1812..2288</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1812</GBInterval_from>\\n            <GBInterval_to>2288</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11575</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11935</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>1812..2288</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1812</GBInterval_from>\\n            <GBInterval_to>2288</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11575</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11935</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005973529.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_022070496.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MDVIVRISKEKKEKDFRMVVIPSGRDKIGIIQTKDYGIIAYPDKKDFEKIGEIINWAFNESDDKVIENSSDTKIEKQFYNCSSYRKVTNEYNEIWFEFFKGIYSISLLKKDGNFFVTFKDENKEAVKHIFKEKPTALELGTKVMEMFEYKERYDELIE</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>3650..4303</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3650</GBInterval_from>\\n            <GBInterval_to>4303</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11580</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11940</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>3650..4303</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3650</GBInterval_from>\\n            <GBInterval_to>4303</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11580</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11940</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: ab initio prediction:GeneMarkS-2+</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: GeneMarkS-2+.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527437.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MVKEIENVGDTTKTIGQKNKQGQIGQSNPMKNNNLNKSQADISDVSNKQSTPAYANKIKNGYKPVYLTKPEQGGTLVIAAGNNPIKGVYNIDMKENIKIGVFKGDATNLVNVPTGSQAKVIIENPNGFDPLNPEILRVVKEGGEIEITGIKSNKEFYNIYSGKVEVPKGFEIIEVGEIPENFQKQGFRTDGKLIGTKNGEGFPKKTDKIIRIRKIKK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>4316..5137</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>4316</GBInterval_from>\\n            <GBInterval_to>5137</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11585</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11945</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>4316..5137</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>4316</GBInterval_from>\\n            <GBInterval_to>5137</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11585</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11945</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008794358.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>stage V sporulation protein K</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527438.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKIKLKTQSLDDWELNLFESYSTMPIIINRTMIFGAYNVEFEMTANIWNQLPEEYKRKIYKYNWKKMIKSMLITVTDITAYSFCFGCNVKLKESLIMEEIYKDFDENKKIEFLTPGCDFPDSSMSVYFQHLGEVYAEVDLDELVAISDEENSYQYSIMPKLMEASNYRKRRENNTGKLEQIYNEQLIIKNLVNKNINELSKEEIQKVLDNFLLINNLKYLIKIIKKSKEFEIVISNKLKNEIEHWLRDIQIKIKTEEEEKIHQEIKEILKEQL</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>5134..5979</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>5134</GBInterval_from>\\n            <GBInterval_to>5979</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11590</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11950</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>5134..5979</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>5134</GBInterval_from>\\n            <GBInterval_to>5979</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11590</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11950</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008794769.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527439.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKIKIKLESLENTLNLSNSFCTIPIIMDKSMMFGAYGVDLNMPEYIWYQLPEECKKKIYNYKEHTIKAMIITITNISAYSITISHHEKFKEAITMEEFYEDFSENKKIERFLCECDFPYSNMSVYFQNLGEIYAEFELEDWVYYEKEAKEEWKIKERERRNIREIIKEALLERIEEKQSNFEDSLKYWVKVVELTEEKIKEIFLKFPTVLHRGGFRLLLELKYMMSIIENSEKFKLIIPENVKNEIGYWLTDMQGEIRTEEEKNLFKEIREKLNLKKVYGD</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>6463..6660</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>6463</GBInterval_from>\\n            <GBInterval_to>6660</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11595</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11960</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>6463..6660</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>6463</GBInterval_from>\\n            <GBInterval_to>6660</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11595</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11960</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: ab initio prediction:GeneMarkS-2+</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: GeneMarkS-2+.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527440.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MAYRFIQKYSFLFGVRWLLRRFNIHLNAYYNYVKNKKKKNIKEKENIKSIIKDMYHSNNGILGHR</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>6971..&gt;7464</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>6971</GBInterval_from>\\n            <GBInterval_to>7464</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11600</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>6971..&gt;7464</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>6971</GBInterval_from>\\n            <GBInterval_to>7464</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11600</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005903864.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hemagglutinin repeat-containing protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527441.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MGATIKADKDLKVTAQGTKLSLDAVKADNTVGKDNVKLAEANNNFYANMGVNLGFNKSSSKSNSHSESGVVTAIKGKDENSSITYNNVKNIEYVGTQAKDTKFIYNNVDNITKKAVELNNYSLSDSKSSGVSAGVNIGYGRKVLTDNASVSVSASKSNMNSNGTT</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000267.1:1..7464)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0273, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893206\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000266</GBSeq_locus>\\n    <GBSeq_length>494</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0272, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000266</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000266.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000266.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0272</GBSeqid>\\n      <GBSeqid>gi|2240893206</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..494</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..494</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000266.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..494</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>494</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000266.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0272</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>&lt;1..&gt;494</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>494</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000266.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11565</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>&lt;1..&gt;494</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>494</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000266.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11565</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005895493.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>incomplete; too short partial abutting assembly gap; missing N-terminus and C-terminus; Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>3</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>ShlB/FhaC/HecB family hemolysin secretion/activation protein</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000266.1:1..494)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0272, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893205\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000265</GBSeq_locus>\\n    <GBSeq_length>811</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0271, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000265</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000265.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000265.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0271</GBSeqid>\\n      <GBSeqid>gi|2240893205</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..811</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..811</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000265.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..811</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>811</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000265.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0271</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>122..198</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>122</GBInterval_from>\\n            <GBInterval_to>198</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000265.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11535</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11895</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>tRNA</GBFeature_key>\\n        <GBFeature_location>122..198</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>122</GBInterval_from>\\n            <GBInterval_to>198</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000265.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11535</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11895</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>tRNA-Pro</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: profile:tRNAscan-SE:2.0.7</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: tRNAscan-SE.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>anticodon</GBQualifier_name>\\n            <GBQualifier_value>(pos:156..158,aa:Pro,seq:tgg)</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transcription</GBQualifier_name>\\n            <GBQualifier_value>CGGAATATAGCGCAGTCCGGTAGCGCACCTGCCTTGGGAGCAGGGGGCCGCAAGTTCAAATCTTGCTATTCCGACCA</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>206..281</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>206</GBInterval_from>\\n            <GBInterval_to>281</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000265.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11540</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11900</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>tRNA</GBFeature_key>\\n        <GBFeature_location>206..281</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>206</GBInterval_from>\\n            <GBInterval_to>281</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000265.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11540</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11900</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>tRNA-Gly</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: profile:tRNAscan-SE:2.0.7</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: tRNAscan-SE.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>anticodon</GBQualifier_name>\\n            <GBQualifier_value>(pos:239..241,aa:Gly,seq:tcc)</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transcription</GBQualifier_name>\\n            <GBQualifier_value>GCGGGAATAGCTCAGTTGGTAGAGCGTCAGCCTTCCAAGCTGAATGTCGCGAGTTCGATTCTCGTTTCCCGCTCCA</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>294..369</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>294</GBInterval_from>\\n            <GBInterval_to>369</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000265.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11545</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11905</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>tRNA</GBFeature_key>\\n        <GBFeature_location>294..369</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>294</GBInterval_from>\\n            <GBInterval_to>369</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000265.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11545</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11905</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>tRNA-His</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: profile:tRNAscan-SE:2.0.7</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: tRNAscan-SE.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>anticodon</GBQualifier_name>\\n            <GBQualifier_value>(pos:327..329,aa:His,seq:gtg)</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transcription</GBQualifier_name>\\n            <GBQualifier_value>GTGACCGTAGTTCAGTTGGTAGAGCGCCAGTTTGTGGCACTGGTTGTCGCGAGTTCGATCCTCGTCGGTCACCCCA</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>373..448</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>373</GBInterval_from>\\n            <GBInterval_to>448</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000265.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11550</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11910</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>tRNA</GBFeature_key>\\n        <GBFeature_location>373..448</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>373</GBInterval_from>\\n            <GBInterval_to>448</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000265.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11550</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11910</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>tRNA-Lys</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: profile:tRNAscan-SE:2.0.7</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: tRNAscan-SE.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>anticodon</GBQualifier_name>\\n            <GBQualifier_value>(pos:406..408,aa:Lys,seq:ttt)</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transcription</GBQualifier_name>\\n            <GBQualifier_value>GCGTCATTAGCTCAGTTGGTAGAGCACACGACTTTTAATCGTGTTGTCACAAGTTCAAATCTTGTATGACGCACCA</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>461..544</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>461</GBInterval_from>\\n            <GBInterval_to>544</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000265.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11555</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11915</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>tRNA</GBFeature_key>\\n        <GBFeature_location>461..544</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>461</GBInterval_from>\\n            <GBInterval_to>544</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000265.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11555</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11915</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>tRNA-Leu</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: profile:tRNAscan-SE:2.0.7</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: tRNAscan-SE.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>anticodon</GBQualifier_name>\\n            <GBQualifier_value>(pos:495..497,aa:Leu,seq:tag)</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transcription</GBQualifier_name>\\n            <GBQualifier_value>GCGGGGATGGCGGAATTGGCAGACGCGCTAGACTTAGGATCTAGTGTCCCAGACGTGAGAGTTCAAGTCTCTCTCTCCGCACCA</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>586..660</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>586</GBInterval_from>\\n            <GBInterval_to>660</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000265.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11560</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11920</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>tRNA</GBFeature_key>\\n        <GBFeature_location>586..660</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>586</GBInterval_from>\\n            <GBInterval_to>660</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000265.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11560</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11920</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>tRNA-Gln</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: profile:tRNAscan-SE:2.0.7</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: tRNAscan-SE.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>anticodon</GBQualifier_name>\\n            <GBQualifier_value>(pos:618..620,aa:Gln,seq:ttg)</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transcription</GBQualifier_name>\\n            <GBQualifier_value>TGGGGTGTCGCCAAGCGGTAAGGCAACGGACTTTGACTCCGTTATGCGTTGGTTCGAATCCAGCCACCCCAGCCA</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000265.1:1..811)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0271, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893204\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000264</GBSeq_locus>\\n    <GBSeq_length>351</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0270, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000264</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000264.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000264.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0270</GBSeqid>\\n      <GBSeqid>gi|2240893204</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..351</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..351</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000264.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..351</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>351</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000264.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0270</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>117..&gt;351</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>117</GBInterval_from>\\n            <GBInterval_to>351</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000264.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11530</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>117..&gt;351</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>117</GBInterval_from>\\n            <GBInterval_to>351</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000264.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11530</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_016361159.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>helix-turn-helix domain-containing protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527436.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MEKAYKFRFYPTKTQIKILNSIFGCVRYVYNHFLDLKQKLYSTEKKSISYSECSKELTVLKKEKEWLKDVDKFSLQNS</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000264.1:1..351)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0270, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893203\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000263</GBSeq_locus>\\n    <GBSeq_length>816</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0268, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000263</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000263.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000263.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0268</GBSeqid>\\n      <GBSeqid>gi|2240893203</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..816</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..816</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000263.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..816</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>816</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000263.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0268</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>&lt;1..230</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>230</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000263.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11520</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11880</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>&lt;1..230</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>230</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000263.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11520</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11880</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008701199.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>incomplete; too short partial abutting assembly gap; missing N-terminus; Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>3</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>431..&gt;816</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>431</GBInterval_from>\\n            <GBInterval_to>816</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000263.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11525</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11885</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>431..&gt;816</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>431</GBInterval_from>\\n            <GBInterval_to>816</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000263.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11525</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11885</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008695093.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527435.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MTLITIFSLIILFLLQFIYVVPESHYSILDQTGKVELKDYPELKEISFMYSTDLYIEFTEPINLELEKINLLLNDKVIGTIEINRNINDLKDFAEPYIDEKSEEKIIRKTYPLQKEFLRILGRNAEIY</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000263.1:1..816)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0268, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893202\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000262</GBSeq_locus>\\n    <GBSeq_length>1866</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0267, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000262</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000262.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000262.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0267</GBSeqid>\\n      <GBSeqid>gi|2240893202</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..1866</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..1866</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000262.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..1866</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>1866</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000262.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0267</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>261..806</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>261</GBInterval_from>\\n            <GBInterval_to>806</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000262.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>infC</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11495</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11855</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>261..806</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>261</GBInterval_from>\\n            <GBInterval_to>806</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000262.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>infC</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11495</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11855</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005895946.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_function</GBQualifier_name>\\n            <GBQualifier_value>GO:0003743 - translation initiation factor activity [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_process</GBQualifier_name>\\n            <GBQualifier_value>GO:0006413 - translational initiation [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>translation initiation factor IF-3</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_022069803.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MPVLFFQWRCSVISDKTRINEKIRGKEFRIISFDGEQLGIMTAEQALNLASSQGYDLVEIAPSATPPVCKIMDYSKYKYEQTRKLKEAKKNQKQVVIKEIKVTARIDSHDLETKLNQVTKFLEKENKVKVTLVLFGREKMHANLGVTTLDEIAEKFAEIAEVEKKYADKQKHLILSPKKIK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>883..1089</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>883</GBInterval_from>\\n            <GBInterval_to>1089</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000262.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>rpmI</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11500</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11860</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>883..1089</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>883</GBInterval_from>\\n            <GBInterval_to>1089</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000262.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>rpmI</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11500</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11860</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005893833.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_component</GBQualifier_name>\\n            <GBQualifier_value>GO:0022625 - cytosolic large ribosomal subunit [Evidence IEA]; GO:0000315 - organellar large ribosomal subunit [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_function</GBQualifier_name>\\n            <GBQualifier_value>GO:0003735 - structural constituent of ribosome [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_process</GBQualifier_name>\\n            <GBQualifier_value>GO:0006412 - translation [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>50S ribosomal protein L35</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_005906263.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MPKMKTHRGAKKRIKVTGTGKFIIKHSGKSHILTKKDRKRKNHLKKDAVVNETYKKHMQGLLPYGEGR</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>1114..1464</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1114</GBInterval_from>\\n            <GBInterval_to>1464</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000262.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>rplT</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11505</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11865</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>1114..1464</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1114</GBInterval_from>\\n            <GBInterval_to>1464</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000262.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>rplT</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11505</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11865</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_016339621.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_component</GBQualifier_name>\\n            <GBQualifier_value>GO:0005840 - ribosome [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_function</GBQualifier_name>\\n            <GBQualifier_value>GO:0019843 - rRNA binding [Evidence IEA]; GO:0003735 - structural constituent of ribosome [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_process</GBQualifier_name>\\n            <GBQualifier_value>GO:0006412 - translation [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>50S ribosomal protein L20</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527434.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MRVKTGIIRRKRHKRVLKAAKGFRGASGDAFKQAKQATRKAMAYATRDRKVNKRRMRQLWITRINSAARMNGVSYSVLMSGLKKAGILLDRKVLADIALNNATEFTKLVEAAKSAL</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>1512..1595</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1512</GBInterval_from>\\n            <GBInterval_to>1595</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000262.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11510</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11870</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>tRNA</GBFeature_key>\\n        <GBFeature_location>1512..1595</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1512</GBInterval_from>\\n            <GBInterval_to>1595</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000262.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11510</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11870</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>tRNA-Ser</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: profile:tRNAscan-SE:2.0.7</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: tRNAscan-SE.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>anticodon</GBQualifier_name>\\n            <GBQualifier_value>(pos:1545..1547,aa:Ser,seq:tga)</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transcription</GBQualifier_name>\\n            <GBQualifier_value>GGAAGGCTATCCTAATTGGTAAGGAACCGGTCTTGAAAACCGGCGTCGTAAGACTTTAGAGTTCGAGTCTCTAGCCTTCCGCCA</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>1618..1707</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1618</GBInterval_from>\\n            <GBInterval_to>1707</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000262.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11515</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11875</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>tRNA</GBFeature_key>\\n        <GBFeature_location>1618..1707</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1618</GBInterval_from>\\n            <GBInterval_to>1707</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000262.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11515</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11875</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>tRNA-Ser</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: profile:tRNAscan-SE:2.0.7</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: tRNAscan-SE.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>anticodon</GBQualifier_name>\\n            <GBQualifier_value>(pos:1652..1654,aa:Ser,seq:gct)</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transcription</GBQualifier_name>\\n            <GBQualifier_value>GGTGAGATGGCAGAGTGGCCTAATGCACTGACCTGCTAAGTCAGAGTACCAATTATGGTACCGAGGGTTCAAATCCCTCTCTCACCGCCA</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000262.1:1..1866)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0267, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893201\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000261</GBSeq_locus>\\n    <GBSeq_length>868</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0266, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000261</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000261.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000261.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0266</GBSeqid>\\n      <GBSeqid>gi|2240893201</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..868</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..868</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000261.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..868</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>868</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000261.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0266</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>&lt;1..&gt;868</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>868</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000261.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11490</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>&lt;1..&gt;868</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>868</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000261.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11490</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005912642.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>incomplete; too short partial abutting assembly gap; missing N-terminus and C-terminus; Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>2</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hemolysin</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000261.1:1..868)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0266, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893200\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000260</GBSeq_locus>\\n    <GBSeq_length>1556</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0265, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000260</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000260.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000260.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0265</GBSeqid>\\n      <GBSeqid>gi|2240893200</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..1556</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..1556</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000260.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..1556</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>1556</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000260.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0265</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>&lt;1..&gt;1556</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>1556</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000260.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11485</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>&lt;1..&gt;1556</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>1556</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000260.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11485</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008795587.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>incomplete; too short partial abutting assembly gap; missing N-terminus and C-terminus; Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hemolysin</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000260.1:1..1556)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0265, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893199\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000259</GBSeq_locus>\\n    <GBSeq_length>455</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0264, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000259</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000259.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000259.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0264</GBSeqid>\\n      <GBSeqid>gi|2240893199</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..455</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..455</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000259.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..455</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>455</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000259.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0264</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>complement(16..363)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>363</GBInterval_from>\\n            <GBInterval_to>16</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000259.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11480</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11840</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>complement(16..363)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>363</GBInterval_from>\\n            <GBInterval_to>16</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000259.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11480</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11840</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: protein motif:HMM:NF013776.2</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>AAA family ATPase</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527433.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKDWDTLFDYLLTKILNEKFILVIDEFQYLCMVNKNFSSIFQRIYDEKLKNKNIMIILCGSLISMMYSETLAYESALYGRRTAQIKLQPIKFKYYNEFFKNKSIQDLIKNMVAHL</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000259.1:1..455)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0264, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893198\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000258</GBSeq_locus>\\n    <GBSeq_length>3806</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0263, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000258</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000258.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000258.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0263</GBSeqid>\\n      <GBSeqid>gi|2240893198</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..3806</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..3806</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000258.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..3806</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>3806</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000258.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0263</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>complement(179..793)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>793</GBInterval_from>\\n            <GBInterval_to>179</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000258.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11450</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11810</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>complement(179..793)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>793</GBInterval_from>\\n            <GBInterval_to>179</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000258.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11450</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11810</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: ab initio prediction:GeneMarkS-2+</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: GeneMarkS-2+.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527429.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MRRILIFIIGTIFLSGCMISKTSYRIKSGNIEHKYYSLSYSARASNIIYLTKENAKIKNKVFLVLEFPEYGIKSKGDVIGIETSIEGDNEDKKKYQYPELLEDIYIMKNNKQKIKISRENLVYSYKKDPNTKFPGVHIYTNQKFTGPVTLELGKVKINGEIYNIPPLHLQRYKESANWGLVDDFLMQGNVRNKPLYESGKWLDD</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>complement(799..1329)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1329</GBInterval_from>\\n            <GBInterval_to>799</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000258.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11455</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11815</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>complement(799..1329)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1329</GBInterval_from>\\n            <GBInterval_to>799</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000258.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11455</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11815</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: ab initio prediction:GeneMarkS-2+</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: GeneMarkS-2+.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527430.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKVFFNPSHGIIADVIESALDVTGFYTGGLIRSSNSKKLAELAIKNPELYNGTLSHSQGSAILGNSMKVVNLKGKKEVLQGKDISMNGSPLSMWAMKNFAEEKGFKFSYKVNVVDPITWLGANKWNTDVWWHTTYSKYQEWNNEKNKYIMQKTKDLEKPIFIKDVKDFTKKYYKGK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>complement(1478..1648)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1648</GBInterval_from>\\n            <GBInterval_to>1478</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000258.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11460</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11820</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>complement(1478..1648)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1648</GBInterval_from>\\n            <GBInterval_to>1478</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000258.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11460</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11820</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_009423194.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>thioredoxinprotein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249524326.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKKILLIILGIFLFNACMTNDYYYISPTGEKSKIVKPNPPIKIEKNEVINKVSIYQ</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>complement(1716..2240)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>2240</GBInterval_from>\\n            <GBInterval_to>1716</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000258.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11465</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11825</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>complement(1716..2240)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>2240</GBInterval_from>\\n            <GBInterval_to>1716</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000258.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11465</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11825</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008794299.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527431.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MAKKIIVIVFSILIFNACSMLDKQYSFSTIEYIKGISIVESPKNIYTENSSLSSIWISDSRNWDGKKRKLKLLDNKIKIIKDGREYVIPYSDIENKNDIYIYIKVVNIVNEDFLLYIGRVQLDNGEIVNIPPLHFKKYIYIQKNGILAGIDPVGRFKSYSGTLEEYKKNGWKDE</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>complement(2225..2638)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>2638</GBInterval_from>\\n            <GBInterval_to>2225</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000258.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11470</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11830</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>complement(2225..2638)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>2638</GBInterval_from>\\n            <GBInterval_to>2225</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000258.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11470</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11830</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: ab initio prediction:GeneMarkS-2+</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: GeneMarkS-2+.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527432.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MIAHSQGTIIYASKLRDDLKTEEGRLKISKMKENYFLGIAVVPWILDDIIDKMNALGTKAEKRRNNYDFVSIITLNLKPNTNGKGHGIIGYDPYYFRWRDDIGKYFYDEKNKEIGKIPKIKENEVINKAGVYQWQKK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>complement(2752..3255)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3255</GBInterval_from>\\n            <GBInterval_to>2752</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000258.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11475</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11835</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>complement(2752..3255)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3255</GBInterval_from>\\n            <GBInterval_to>2752</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000258.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11475</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11835</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: ab initio prediction:GeneMarkS-2+</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: GeneMarkS-2+.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249525425.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MGEYGKNLGSDLDKMSGGKYDLENKLGQGFYNTYREIEDFIDYKLYNKFTGIIPTKNVQGGLAGELQSPSSLLNKNLIPVVITYSGEGDKDNNFKIKSVVEDLTQEKYKEIKDKNSGEIPLINGVGNDVSDFTDGRFAGTLTPDEHKRIKKGETIKKIGIFNPTNGL</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000258.1:1..3806)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0263, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893197\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000257</GBSeq_locus>\\n    <GBSeq_length>609</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0262, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000257</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000257.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000257.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0262</GBSeqid>\\n      <GBSeqid>gi|2240893197</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..609</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..609</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000257.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..609</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>609</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000257.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0262</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>complement(&lt;1..&gt;609)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>609</GBInterval_from>\\n            <GBInterval_to>1</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000257.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11445</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>complement(&lt;1..&gt;609)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>609</GBInterval_from>\\n            <GBInterval_to>1</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000257.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11445</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_020788966.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>3</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hemagglutinin repeat-containing protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527428.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>ILNQTASQIVSNKDINIEATDKVKAKSVDIYAKNDINISGDNGVEISTANNSYDNTTKQSSSRIGTSVGINSAIVNTVENVRDIKKLTDFSGNSYDILNNASKVVGAIKDGAKATIAVADTNYNGATDAGYNNLKIMNDVFKANISYNKSKSKSSVHNETVEKSSLVSGRNMNIKSKNGSITISGTDVKVGNDLDLNAKKDI</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000257.1:1..609)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0262, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893196\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000256</GBSeq_locus>\\n    <GBSeq_length>1035</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0261, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000256</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000256.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000256.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0261</GBSeqid>\\n      <GBSeqid>gi|2240893196</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..1035</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..1035</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000256.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..1035</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>1035</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000256.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0261</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>complement(&lt;1..&gt;1035)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1035</GBInterval_from>\\n            <GBInterval_to>1</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000256.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11440</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>complement(&lt;1..&gt;1035)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1035</GBInterval_from>\\n            <GBInterval_to>1</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000256.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11440</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008795587.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>3</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hemagglutinin repeat-containing protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527427.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>ADLSISKAGTKGRGNGTNYINSTIDVGGKLKTNSENLTLSGANVEADKLDINAKNLVIESKQDKSEKKDSSYGGSFSIDLVNPSSFSANINGSKGSGEKEWVNKQTTLIARNGGKVDTDSLTNIGAVIGSENEKEKLKVSANKVVVKDLEDKNKYENIGGGITIGTDVPNTSIKHDKIDKEQMDRATAINTDFEISGKSTSGEDLGFNTDIDKSQEKTKDEEKHLDAELHTDLLGKDKQEELKKAGGIISDLTTALGNKSKTEGDFLERYKQLSMMRAIGDQVAKNPEYLSILDKKAIKNGKIDDDVQKDQVSVMNKLLNDALRAKGYKGPDIKMVLTDVTDPN</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000256.1:1..1035)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0261, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893195\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000255</GBSeq_locus>\\n    <GBSeq_length>739</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0260, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000255</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000255.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000255.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0260</GBSeqid>\\n      <GBSeqid>gi|2240893195</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..739</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..739</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000255.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..739</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>739</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000255.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0260</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>complement(&lt;1..457)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>457</GBInterval_from>\\n            <GBInterval_to>1</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000255.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11430</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>complement(&lt;1..457)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>457</GBInterval_from>\\n            <GBInterval_to>1</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000255.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11430</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: protein motif:HMM:NF015803.2</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>ShlB/FhaC/HecB family hemolysin secretion/activation protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527426.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MTLQDIDLDTHNIFGLNESLDFSYQNKLQRQYKERDTKNFSFGVSVPFKYWTFSYNYDSSEYLRTINIYNRKYRATGKTKNQTFGIRKMLHRNENHKIDIGAKITLKDSKNYIDDVRLVSSSRKLSVLTVDTTYTGRILSGLLSTNVGVSFG</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>complement(423..&gt;739)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>739</GBInterval_from>\\n            <GBInterval_to>423</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000255.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11435</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>complement(423..&gt;739)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>739</GBInterval_from>\\n            <GBInterval_to>423</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000255.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11435</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: protein motif:HMM:NF036420.2</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>incomplete; too short partial abutting assembly gap; missing N-terminus; Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>3</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>ShlB/FhaC/HecB family hemolysin secretion/activation protein</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000255.1:1..739)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0260, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893194\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000254</GBSeq_locus>\\n    <GBSeq_length>531</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0259, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000254</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000254.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000254.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0259</GBSeqid>\\n      <GBSeqid>gi|2240893194</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..531</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..531</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000254.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..531</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>531</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000254.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0259</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>&lt;1..&gt;531</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>531</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000254.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11425</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>&lt;1..&gt;531</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>531</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000254.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11425</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005910012.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>incomplete; too short partial abutting assembly gap; missing N-terminus and C-terminus; Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>2</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hemolysin</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000254.1:1..531)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0259, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "this is the id list [['2240894608'], ['2240894606'], ['2240894605'], ['2240894604'], ['2240894603'], ['2240894602'], ['2240894601'], ['2240894593'], ['2240894592'], ['2240894591'], ['2240894590'], ['2240894589'], ['2240894588'], ['2240894583'], ['2240894582'], ['2240894581'], ['2240894580'], ['2240894574'], ['2240894573'], ['2240894572'], ['2240894571'], ['2240894570'], ['2240894569'], ['2240894568'], ['2240894567'], ['2240894566'], ['2240894565'], ['2240894564'], ['2240894563'], ['2240894562'], ['2240894555'], ['2240894553'], ['2240894552'], ['2240894551'], ['2240894550'], ['2240894549'], ['2240894548'], ['2240894547'], ['2240894546'], ['2240894545'], ['2240894544'], ['2240894509'], ['2240894508'], ['2240894507'], ['2240894506'], ['2240894505'], ['2240894498'], ['2240894485'], ['2240894483'], ['2240894482'], ['2240894481'], ['2240894480'], ['2240894479'], ['2240894478'], ['2240894477'], ['2240894475'], ['2240894467'], ['2240894466'], ['2240894465'], ['2240894464'], ['2240894463'], ['2240894462'], ['2240894461'], ['2240894460'], ['2240894459'], ['2240894458'], ['2240894457'], ['2240894456'], ['2240894455'], ['2240894454'], ['2240894453'], ['2240894452'], ['2240894451'], ['2240894450'], ['2240894449'], ['2240894448'], ['2240893217'], ['2240893216'], ['2240893215'], ['2240893214'], ['2240893213'], ['2240893212'], ['2240893211'], ['2240893210'], ['2240893209'], ['2240893208'], ['2240893207'], ['2240893206'], ['2240893205'], ['2240893204'], ['2240893203'], ['2240893202'], ['2240893201'], ['2240893200'], ['2240893199'], ['2240893198'], ['2240893197'], ['2240893196'], ['2240893195'], ['2240893194']]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Id 2240894608\n",
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZC010000000</GBSeq_locus>\\n    <GBSeq_length>322</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Candidatus Bipolaricaulota bacterium isolate HOT.Cluster.56, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZC000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZC000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZC000000000.1|JAILZC010000000</GBSeqid>\\n      <GBSeqid>gi|2240894608</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Candidatus Bipolaricaulota bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Candidatus Bipolaricaulota bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Candidatus Bipolaricaulota</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..322</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..322</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Candidatus Bipolaricaulota bacterium whole genome shotgun (WGS) project has the project accession JAILZC000000000. This version of the project (01) has the accession number JAILZC010000000, and consists of sequences JAILZC010000001-JAILZC010000322.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 41.4x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:28:30 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 1,727 ; CDSs (total) :: 1,689 ; Genes (coding) :: 1,663 ; CDSs (with protein) :: 1,663 ; Genes (RNA) :: 38 ; rRNAs :: 2, 1 (16S, 23S) ; partial rRNAs :: 2, 1 (16S, 23S) ; tRNAs :: 32 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 26 ; CDSs (without protein) :: 26 ; Pseudo Genes (ambiguous residues) :: 0 of 26 ; Pseudo Genes (frameshifted) :: 6 of 26 ; Pseudo Genes (incomplete) :: 17 of 26 ; Pseudo Genes (internal stop) :: 4 of 26 ; Pseudo Genes (multiple problems) :: 1 of 26 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..322</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>322</GBInterval_to>\\n            <GBInterval_accession>JAILZC000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Candidatus Bipolaricaulota bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.Cluster.56</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2306012</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZC010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZC010000322</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023700</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Candidatus Bipolaricaulota bacterium isolate HOT.Cluster.56, whole genome shotgun sequencing project', 'Candidatus Bipolaricaulota bacterium (hot springs metagenome)', 'Candidatus Bipolaricaulota bacterium', 'Bacteria; Candidatus Bipolaricaulota']\n",
      "Id 2240894606\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZD010000000</GBSeq_locus>\\n    <GBSeq_length>679</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Dehalococcoidia bacterium isolate HOT.Cluster.58, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZD000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZD000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZD000000000.1|JAILZD010000000</GBSeqid>\\n      <GBSeqid>gi|2240894606</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Dehalococcoidia bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Dehalococcoidia bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Chloroflexi; Dehalococcoidia</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..679</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..679</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Dehalococcoidia bacterium whole genome shotgun (WGS) project has the project accession JAILZD000000000. This version of the project (01) has the accession number JAILZD010000000, and consists of sequences JAILZD010000001-JAILZD010000679.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 23.7x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 13:39:08 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,476 ; CDSs (total) :: 3,420 ; Genes (coding) :: 3,342 ; CDSs (with protein) :: 3,342 ; Genes (RNA) :: 56 ; rRNAs :: 1, 1 (5S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1 (23S) ; tRNAs :: 50 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 78 ; CDSs (without protein) :: 78 ; Pseudo Genes (ambiguous residues) :: 0 of 78 ; Pseudo Genes (frameshifted) :: 4 of 78 ; Pseudo Genes (incomplete) :: 66 of 78 ; Pseudo Genes (internal stop) :: 9 of 78 ; Pseudo Genes (multiple problems) :: 1 of 78 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..679</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>679</GBInterval_to>\\n            <GBInterval_accession>JAILZD000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Dehalococcoidia bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.Cluster.58</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2026734</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZD010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZD010000679</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023701</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Dehalococcoidia bacterium isolate HOT.Cluster.58, whole genome shotgun sequencing project', 'Dehalococcoidia bacterium (hot springs metagenome)', 'Dehalococcoidia bacterium', 'Bacteria; Chloroflexi; Dehalococcoidia']\n",
      "Id 2240894605\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZB010000000</GBSeq_locus>\\n    <GBSeq_length>461</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Chloroflexi bacterium isolate HOT.Cluster.3, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZB000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZB000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZB000000000.1|JAILZB010000000</GBSeqid>\\n      <GBSeqid>gi|2240894605</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Chloroflexi bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Chloroflexi bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Chloroflexi</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..461</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..461</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Chloroflexi bacterium whole genome shotgun (WGS) project has the project accession JAILZB000000000. This version of the project (01) has the accession number JAILZB010000000, and consists of sequences JAILZB010000001-JAILZB010000461.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 13.1x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 13:46:36 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 4,087 ; CDSs (total) :: 4,034 ; Genes (coding) :: 3,968 ; CDSs (with protein) :: 3,968 ; Genes (RNA) :: 53 ; rRNAs :: 1, 1, 2 (5S, 16S, 23S) ; complete rRNAs :: 1, 1 (5S, 16S) ; partial rRNAs :: 2 (23S) ; tRNAs :: 46 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 66 ; CDSs (without protein) :: 66 ; Pseudo Genes (ambiguous residues) :: 0 of 66 ; Pseudo Genes (frameshifted) :: 13 of 66 ; Pseudo Genes (incomplete) :: 44 of 66 ; Pseudo Genes (internal stop) :: 14 of 66 ; Pseudo Genes (multiple problems) :: 5 of 66 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..461</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>461</GBInterval_to>\\n            <GBInterval_accession>JAILZB000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Chloroflexi bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.Cluster.3</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2026724</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZB010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZB010000461</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023699</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Chloroflexi bacterium isolate HOT.Cluster.3, whole genome shotgun sequencing project', 'Chloroflexi bacterium (hot springs metagenome)', 'Chloroflexi bacterium', 'Bacteria; Chloroflexi']\n",
      "Id 2240894604\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZG010000000</GBSeq_locus>\\n    <GBSeq_length>172</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Firmicutes bacterium isolate HOT.Cluster.78, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZG000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZG000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZG000000000.1|JAILZG010000000</GBSeqid>\\n      <GBSeqid>gi|2240894604</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Firmicutes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Firmicutes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..172</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..172</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Firmicutes bacterium whole genome shotgun (WGS) project has the project accession JAILZG000000000. This version of the project (01) has the accession number JAILZG010000000, and consists of sequences JAILZG010000001-JAILZG010000172.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 17.5x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:42:59 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 1,737 ; CDSs (total) :: 1,715 ; Genes (coding) :: 1,686 ; CDSs (with protein) :: 1,686 ; Genes (RNA) :: 22 ; rRNAs :: 1, 1 (5S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1 (23S) ; tRNAs :: 19 ; ncRNAs :: 1 ; Pseudo Genes (total) :: 29 ; CDSs (without protein) :: 29 ; Pseudo Genes (ambiguous residues) :: 0 of 29 ; Pseudo Genes (frameshifted) :: 6 of 29 ; Pseudo Genes (incomplete) :: 20 of 29 ; Pseudo Genes (internal stop) :: 3 of 29 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..172</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>172</GBInterval_to>\\n            <GBInterval_accession>JAILZG000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Firmicutes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.Cluster.78</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1879010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZG010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZG010000172</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023704</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Firmicutes bacterium isolate HOT.Cluster.78, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes']\n",
      "Id 2240894603\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZE010000000</GBSeq_locus>\\n    <GBSeq_length>555</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Candidatus Rokubacteria bacterium isolate HOT.Cluster.5, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZE000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZE000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZE000000000.1|JAILZE010000000</GBSeqid>\\n      <GBSeqid>gi|2240894603</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Candidatus Rokubacteria bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Candidatus Rokubacteria bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Candidatus Rokubacteria; unclassified Candidatus Rokubacteria</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..555</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..555</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Candidatus Rokubacteria bacterium whole genome shotgun (WGS) project has the project accession JAILZE000000000. This version of the project (01) has the accession number JAILZE010000000, and consists of sequences JAILZE010000001-JAILZE010000555.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 14.4x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 11:26:23 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,007 ; CDSs (total) :: 2,000 ; Genes (coding) :: 1,962 ; CDSs (with protein) :: 1,962 ; Genes (RNA) :: 7 ; tRNAs :: 7 ; ncRNAs :: 0 ; Pseudo Genes (total) :: 38 ; CDSs (without protein) :: 38 ; Pseudo Genes (ambiguous residues) :: 0 of 38 ; Pseudo Genes (frameshifted) :: 3 of 38 ; Pseudo Genes (incomplete) :: 24 of 38 ; Pseudo Genes (internal stop) :: 12 of 38 ; Pseudo Genes (multiple problems) :: 1 of 38 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..555</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>555</GBInterval_to>\\n            <GBInterval_accession>JAILZE000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Candidatus Rokubacteria bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.Cluster.5</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2053607</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZE010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZE010000555</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023702</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Candidatus Rokubacteria bacterium isolate HOT.Cluster.5, whole genome shotgun sequencing project', 'Candidatus Rokubacteria bacterium (hot springs metagenome)', 'Candidatus Rokubacteria bacterium', 'Bacteria; Candidatus Rokubacteria; unclassified Candidatus Rokubacteria']\n",
      "Id 2240894602\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZF010000000</GBSeq_locus>\\n    <GBSeq_length>214</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Alicyclobacillus sp. isolate HOT.Cluster.61, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZF000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZF000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZF000000000.1|JAILZF010000000</GBSeqid>\\n      <GBSeqid>gi|2240894602</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Alicyclobacillus sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Alicyclobacillus sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..214</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..214</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Alicyclobacillus sp. whole genome shotgun (WGS) project has the project accession JAILZF000000000. This version of the project (01) has the accession number JAILZF010000000, and consists of sequences JAILZF010000001-JAILZF010000214.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 16.9x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 09:47:40 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 1,773 ; CDSs (total) :: 1,759 ; Genes (coding) :: 1,747 ; CDSs (with protein) :: 1,747 ; Genes (RNA) :: 14 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 9 ; ncRNAs :: 2 ; Pseudo Genes (total) :: 12 ; CDSs (without protein) :: 12 ; Pseudo Genes (ambiguous residues) :: 0 of 12 ; Pseudo Genes (frameshifted) :: 1 of 12 ; Pseudo Genes (incomplete) :: 8 of 12 ; Pseudo Genes (internal stop) :: 3 of 12 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..214</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>214</GBInterval_to>\\n            <GBInterval_accession>JAILZF000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Alicyclobacillus sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.Cluster.61</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:61169</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZF010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZF010000214</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023703</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Alicyclobacillus sp. isolate HOT.Cluster.61, whole genome shotgun sequencing project', 'Alicyclobacillus sp. (hot springs metagenome)', 'Alicyclobacillus sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus']\n",
      "Id 2240894601\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZH010000000</GBSeq_locus>\\n    <GBSeq_length>341</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Peptococcaceae bacterium isolate HOT.Cluster.86, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZH000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZH000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZH000000000.1|JAILZH010000000</GBSeqid>\\n      <GBSeqid>gi|2240894601</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Peptococcaceae bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Peptococcaceae bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Clostridia; Eubacteriales; Peptococcaceae</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..341</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..341</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Peptococcaceae bacterium whole genome shotgun (WGS) project has the project accession JAILZH000000000. This version of the project (01) has the accession number JAILZH010000000, and consists of sequences JAILZH010000001-JAILZH010000341.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 38.1x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 09:49:04 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,231 ; CDSs (total) :: 2,204 ; Genes (coding) :: 2,158 ; CDSs (with protein) :: 2,158 ; Genes (RNA) :: 27 ; rRNAs :: 1, 1 (5S, 16S) ; partial rRNAs :: 1, 1 (5S, 16S) ; tRNAs :: 23 ; ncRNAs :: 2 ; Pseudo Genes (total) :: 46 ; CDSs (without protein) :: 46 ; Pseudo Genes (ambiguous residues) :: 0 of 46 ; Pseudo Genes (frameshifted) :: 12 of 46 ; Pseudo Genes (incomplete) :: 29 of 46 ; Pseudo Genes (internal stop) :: 7 of 46 ; Pseudo Genes (multiple problems) :: 2 of 46 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..341</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>341</GBInterval_to>\\n            <GBInterval_accession>JAILZH000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Peptococcaceae bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.Cluster.86</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2052179</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZH010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZH010000341</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023705</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Peptococcaceae bacterium isolate HOT.Cluster.86, whole genome shotgun sequencing project', 'Peptococcaceae bacterium (hot springs metagenome)', 'Peptococcaceae bacterium', 'Bacteria; Firmicutes; Clostridia; Eubacteriales; Peptococcaceae']\n",
      "Id 2240894593\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZJ010000000</GBSeq_locus>\\n    <GBSeq_length>217</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Alicyclobacillus herbarius isolate HOT.CON.104, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZJ000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZJ000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZJ000000000.1|JAILZJ010000000</GBSeqid>\\n      <GBSeqid>gi|2240894593</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Alicyclobacillus herbarius (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Alicyclobacillus herbarius</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..217</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..217</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Alicyclobacillus herbarius whole genome shotgun (WGS) project has the project accession JAILZJ000000000. This version of the project (01) has the accession number JAILZJ010000000, and consists of sequences JAILZJ010000001-JAILZJ010000217.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 42.2x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:21:23 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,235 ; CDSs (total) :: 3,197 ; Genes (coding) :: 3,155 ; CDSs (with protein) :: 3,155 ; Genes (RNA) :: 38 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 30 ; ncRNAs :: 5 ; Pseudo Genes (total) :: 42 ; CDSs (without protein) :: 42 ; Pseudo Genes (ambiguous residues) :: 0 of 42 ; Pseudo Genes (frameshifted) :: 16 of 42 ; Pseudo Genes (incomplete) :: 26 of 42 ; Pseudo Genes (internal stop) :: 5 of 42 ; Pseudo Genes (multiple problems) :: 5 of 42 ; CRISPR Arrays :: 3 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..217</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>217</GBInterval_to>\\n            <GBInterval_accession>JAILZJ000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Alicyclobacillus herbarius</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.104</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:122960</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZJ010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZJ010000217</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023707</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Alicyclobacillus herbarius isolate HOT.CON.104, whole genome shotgun sequencing project', 'Alicyclobacillus herbarius (hot springs metagenome)', 'Alicyclobacillus herbarius', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus']\n",
      "Id 2240894592\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZR010000000</GBSeq_locus>\\n    <GBSeq_length>729</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Firmicutes bacterium isolate HOT.CON.146, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZR000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZR000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZR000000000.1|JAILZR010000000</GBSeqid>\\n      <GBSeqid>gi|2240894592</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Firmicutes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Firmicutes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..729</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..729</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Firmicutes bacterium whole genome shotgun (WGS) project has the project accession JAILZR000000000. This version of the project (01) has the accession number JAILZR010000000, and consists of sequences JAILZR010000001-JAILZR010000729.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 14.5x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 11:55:08 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,616 ; CDSs (total) :: 2,581 ; Genes (coding) :: 2,519 ; CDSs (with protein) :: 2,519 ; Genes (RNA) :: 35 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 30 ; ncRNAs :: 2 ; Pseudo Genes (total) :: 62 ; CDSs (without protein) :: 62 ; Pseudo Genes (ambiguous residues) :: 0 of 62 ; Pseudo Genes (frameshifted) :: 16 of 62 ; Pseudo Genes (incomplete) :: 40 of 62 ; Pseudo Genes (internal stop) :: 8 of 62 ; Pseudo Genes (multiple problems) :: 2 of 62 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..729</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>729</GBInterval_to>\\n            <GBInterval_accession>JAILZR000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Firmicutes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.146</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1879010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZR010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZR010000729</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023715</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Firmicutes bacterium isolate HOT.CON.146, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes']\n",
      "Id 2240894591\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZK010000000</GBSeq_locus>\\n    <GBSeq_length>104</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Thermomonas hydrothermalis isolate HOT.CON.106, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZK000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZK000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZK000000000.1|JAILZK010000000</GBSeqid>\\n      <GBSeqid>gi|2240894591</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Thermomonas hydrothermalis (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Thermomonas hydrothermalis</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Proteobacteria; Gammaproteobacteria; Xanthomonadales; Xanthomonadaceae; Thermomonas</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..104</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..104</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Thermomonas hydrothermalis whole genome shotgun (WGS) project has the project accession JAILZK000000000. This version of the project (01) has the accession number JAILZK010000000, and consists of sequences JAILZK010000001-JAILZK010000104.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 14.3x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 09:56:04 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,366 ; CDSs (total) :: 2,319 ; Genes (coding) :: 2,301 ; CDSs (with protein) :: 2,301 ; Genes (RNA) :: 47 ; tRNAs :: 43 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 18 ; CDSs (without protein) :: 18 ; Pseudo Genes (ambiguous residues) :: 0 of 18 ; Pseudo Genes (frameshifted) :: 5 of 18 ; Pseudo Genes (incomplete) :: 13 of 18 ; Pseudo Genes (internal stop) :: 1 of 18 ; Pseudo Genes (multiple problems) :: 1 of 18 ; CRISPR Arrays :: 2 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..104</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>104</GBInterval_to>\\n            <GBInterval_accession>JAILZK000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Thermomonas hydrothermalis</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.106</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:213588</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZK010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZK010000104</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023708</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Thermomonas hydrothermalis isolate HOT.CON.106, whole genome shotgun sequencing project', 'Thermomonas hydrothermalis (hot springs metagenome)', 'Thermomonas hydrothermalis', 'Bacteria; Proteobacteria; Gammaproteobacteria; Xanthomonadales; Xanthomonadaceae; Thermomonas']\n",
      "Id 2240894590\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZO010000000</GBSeq_locus>\\n    <GBSeq_length>556</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Alicyclobacillus shizuokensis isolate HOT.CON.119, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZO000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZO000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZO000000000.1|JAILZO010000000</GBSeqid>\\n      <GBSeqid>gi|2240894590</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Alicyclobacillus shizuokensis (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Alicyclobacillus shizuokensis</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..556</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..556</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Alicyclobacillus shizuokensis whole genome shotgun (WGS) project has the project accession JAILZO000000000. This version of the project (01) has the accession number JAILZO010000000, and consists of sequences JAILZO010000001-JAILZO010000556.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 13.2x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:26:26 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,086 ; CDSs (total) :: 3,061 ; Genes (coding) :: 3,014 ; CDSs (with protein) :: 3,014 ; Genes (RNA) :: 25 ; rRNAs :: 1, 1 (5S, 16S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1 (16S) ; tRNAs :: 20 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 47 ; CDSs (without protein) :: 47 ; Pseudo Genes (ambiguous residues) :: 0 of 47 ; Pseudo Genes (frameshifted) :: 20 of 47 ; Pseudo Genes (incomplete) :: 26 of 47 ; Pseudo Genes (internal stop) :: 7 of 47 ; Pseudo Genes (multiple problems) :: 5 of 47 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..556</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>556</GBInterval_to>\\n            <GBInterval_accession>JAILZO000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Alicyclobacillus shizuokensis</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.119</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:392014</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZO010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZO010000556</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023712</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Alicyclobacillus shizuokensis isolate HOT.CON.119, whole genome shotgun sequencing project', 'Alicyclobacillus shizuokensis (hot springs metagenome)', 'Alicyclobacillus shizuokensis', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus']\n",
      "Id 2240894589\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZM010000000</GBSeq_locus>\\n    <GBSeq_length>878</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Anoxybacillus ayderensis isolate HOT.CON.111, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZM000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZM000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZM000000000.1|JAILZM010000000</GBSeqid>\\n      <GBSeqid>gi|2240894589</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Anoxybacillus ayderensis (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Anoxybacillus ayderensis</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Bacillaceae; Anoxybacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..878</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..878</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Anoxybacillus ayderensis whole genome shotgun (WGS) project has the project accession JAILZM000000000. This version of the project (01) has the accession number JAILZM010000000, and consists of sequences JAILZM010000001-JAILZM010000878.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 14x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 09:58:52 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,790 ; CDSs (total) :: 2,776 ; Genes (coding) :: 2,642 ; CDSs (with protein) :: 2,642 ; Genes (RNA) :: 14 ; rRNAs :: 1 (16S) ; partial rRNAs :: 1 (16S) ; tRNAs :: 10 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 134 ; CDSs (without protein) :: 134 ; Pseudo Genes (ambiguous residues) :: 0 of 134 ; Pseudo Genes (frameshifted) :: 13 of 134 ; Pseudo Genes (incomplete) :: 81 of 134 ; Pseudo Genes (internal stop) :: 45 of 134 ; Pseudo Genes (multiple problems) :: 5 of 134 ; CRISPR Arrays :: 2 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..878</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>878</GBInterval_to>\\n            <GBInterval_accession>JAILZM000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Anoxybacillus ayderensis</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.111</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:265546</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZM010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZM010000878</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023710</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Anoxybacillus ayderensis isolate HOT.CON.111, whole genome shotgun sequencing project', 'Anoxybacillus ayderensis (hot springs metagenome)', 'Anoxybacillus ayderensis', 'Bacteria; Firmicutes; Bacilli; Bacillales; Bacillaceae; Anoxybacillus']\n",
      "Id 2240894588\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZN010000000</GBSeq_locus>\\n    <GBSeq_length>1094</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Armatimonadetes bacterium isolate HOT.CON.113, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZN000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZN000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZN000000000.1|JAILZN010000000</GBSeqid>\\n      <GBSeqid>gi|2240894588</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Armatimonadetes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Armatimonadetes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Armatimonadetes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..1094</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..1094</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Armatimonadetes bacterium whole genome shotgun (WGS) project has the project accession JAILZN000000000. This version of the project (01) has the accession number JAILZN010000000, and consists of sequences JAILZN010000001-JAILZN010001094.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 14.4x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 11:48:59 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,086 ; CDSs (total) :: 3,056 ; Genes (coding) :: 3,023 ; CDSs (with protein) :: 3,023 ; Genes (RNA) :: 30 ; rRNAs :: 1, 2 (5S, 23S) ; partial rRNAs :: 1, 2 (5S, 23S) ; tRNAs :: 24 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 33 ; CDSs (without protein) :: 33 ; Pseudo Genes (ambiguous residues) :: 0 of 33 ; Pseudo Genes (frameshifted) :: 2 of 33 ; Pseudo Genes (incomplete) :: 17 of 33 ; Pseudo Genes (internal stop) :: 14 of 33 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..1094</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>1094</GBInterval_to>\\n            <GBInterval_accession>JAILZN000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Armatimonadetes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.113</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2033014</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZN010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZN010001094</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023711</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Armatimonadetes bacterium isolate HOT.CON.113, whole genome shotgun sequencing project', 'Armatimonadetes bacterium (hot springs metagenome)', 'Armatimonadetes bacterium', 'Bacteria; Armatimonadetes']\n",
      "Id 2240894583\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZI010000000</GBSeq_locus>\\n    <GBSeq_length>661</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Fimbriimonadales bacterium isolate HOT.CON.102, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZI000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZI000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZI000000000.1|JAILZI010000000</GBSeqid>\\n      <GBSeqid>gi|2240894583</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fimbriimonadales bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Fimbriimonadales bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Armatimonadetes; Fimbriimonadia; Fimbriimonadales; unclassified Fimbriimonadales</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..661</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..661</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Fimbriimonadales bacterium whole genome shotgun (WGS) project has the project accession JAILZI000000000. This version of the project (01) has the accession number JAILZI010000000, and consists of sequences JAILZI010000001-JAILZI010000661.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 34.8x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 09:37:11 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,016 ; CDSs (total) :: 1,985 ; Genes (coding) :: 1,968 ; CDSs (with protein) :: 1,968 ; Genes (RNA) :: 31 ; rRNAs :: 1, 1 (16S, 23S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 27 ; ncRNAs :: 2 ; Pseudo Genes (total) :: 17 ; CDSs (without protein) :: 17 ; Pseudo Genes (ambiguous residues) :: 0 of 17 ; Pseudo Genes (frameshifted) :: 1 of 17 ; Pseudo Genes (incomplete) :: 16 of 17 ; Pseudo Genes (internal stop) :: 0 of 17 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..661</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>661</GBInterval_to>\\n            <GBInterval_accession>JAILZI000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fimbriimonadales bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.102</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2682145</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZI010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZI010000661</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023706</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Fimbriimonadales bacterium isolate HOT.CON.102, whole genome shotgun sequencing project', 'Fimbriimonadales bacterium (hot springs metagenome)', 'Fimbriimonadales bacterium', 'Bacteria; Armatimonadetes; Fimbriimonadia; Fimbriimonadales; unclassified Fimbriimonadales']\n",
      "Id 2240894582\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZL010000000</GBSeq_locus>\\n    <GBSeq_length>73</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Syntrophobacterales bacterium isolate HOT.CON.10, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZL000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZL000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZL000000000.1|JAILZL010000000</GBSeqid>\\n      <GBSeqid>gi|2240894582</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Syntrophobacterales bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Syntrophobacterales bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Proteobacteria; Deltaproteobacteria; Syntrophobacterales</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..73</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..73</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Syntrophobacterales bacterium whole genome shotgun (WGS) project has the project accession JAILZL000000000. This version of the project (01) has the accession number JAILZL010000000, and consists of sequences JAILZL010000001-JAILZL010000073.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 15x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:12:39 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,581 ; CDSs (total) :: 2,531 ; Genes (coding) :: 2,523 ; CDSs (with protein) :: 2,523 ; Genes (RNA) :: 50 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 44 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 8 ; CDSs (without protein) :: 8 ; Pseudo Genes (ambiguous residues) :: 0 of 8 ; Pseudo Genes (frameshifted) :: 2 of 8 ; Pseudo Genes (incomplete) :: 5 of 8 ; Pseudo Genes (internal stop) :: 2 of 8 ; Pseudo Genes (multiple problems) :: 1 of 8 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..73</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>73</GBInterval_to>\\n            <GBInterval_accession>JAILZL000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Syntrophobacterales bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.10</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2282155</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZL010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZL010000073</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023709</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Syntrophobacterales bacterium isolate HOT.CON.10, whole genome shotgun sequencing project', 'Syntrophobacterales bacterium (hot springs metagenome)', 'Syntrophobacterales bacterium', 'Bacteria; Proteobacteria; Deltaproteobacteria; Syntrophobacterales']\n",
      "Id 2240894581\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZP010000000</GBSeq_locus>\\n    <GBSeq_length>80</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Peptococcaceae bacterium isolate HOT.CON.130, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZP000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZP000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZP000000000.1|JAILZP010000000</GBSeqid>\\n      <GBSeqid>gi|2240894581</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Peptococcaceae bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Peptococcaceae bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Clostridia; Eubacteriales; Peptococcaceae</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..80</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..80</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Peptococcaceae bacterium whole genome shotgun (WGS) project has the project accession JAILZP000000000. This version of the project (01) has the accession number JAILZP010000000, and consists of sequences JAILZP010000001-JAILZP010000080.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 15x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:25:56 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,161 ; CDSs (total) :: 3,119 ; Genes (coding) :: 3,098 ; CDSs (with protein) :: 3,098 ; Genes (RNA) :: 42 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 34 ; ncRNAs :: 5 ; Pseudo Genes (total) :: 21 ; CDSs (without protein) :: 21 ; Pseudo Genes (ambiguous residues) :: 0 of 21 ; Pseudo Genes (frameshifted) :: 7 of 21 ; Pseudo Genes (incomplete) :: 15 of 21 ; Pseudo Genes (internal stop) :: 1 of 21 ; Pseudo Genes (multiple problems) :: 2 of 21 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..80</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>80</GBInterval_to>\\n            <GBInterval_accession>JAILZP000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Peptococcaceae bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.130</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2052179</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZP010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZP010000080</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023713</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Peptococcaceae bacterium isolate HOT.CON.130, whole genome shotgun sequencing project', 'Peptococcaceae bacterium (hot springs metagenome)', 'Peptococcaceae bacterium', 'Bacteria; Firmicutes; Clostridia; Eubacteriales; Peptococcaceae']\n",
      "Id 2240894580\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZQ010000000</GBSeq_locus>\\n    <GBSeq_length>263</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Geminicoccaceae bacterium isolate HOT.CON.139, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZQ000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZQ000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZQ000000000.1|JAILZQ010000000</GBSeqid>\\n      <GBSeqid>gi|2240894580</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Geminicoccaceae bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Geminicoccaceae bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Proteobacteria; Alphaproteobacteria; Rhodospirillales; Geminicoccaceae</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..263</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..263</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Geminicoccaceae bacterium whole genome shotgun (WGS) project has the project accession JAILZQ000000000. This version of the project (01) has the accession number JAILZQ010000000, and consists of sequences JAILZQ010000001-JAILZQ010000263.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 14x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:34:32 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,518 ; CDSs (total) :: 3,478 ; Genes (coding) :: 3,452 ; CDSs (with protein) :: 3,452 ; Genes (RNA) :: 40 ; rRNAs :: 1 (16S) ; partial rRNAs :: 1 (16S) ; tRNAs :: 37 ; ncRNAs :: 2 ; Pseudo Genes (total) :: 26 ; CDSs (without protein) :: 26 ; Pseudo Genes (ambiguous residues) :: 0 of 26 ; Pseudo Genes (frameshifted) :: 5 of 26 ; Pseudo Genes (incomplete) :: 17 of 26 ; Pseudo Genes (internal stop) :: 4 of 26 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..263</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>263</GBInterval_to>\\n            <GBInterval_accession>JAILZQ000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Geminicoccaceae bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.139</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2448052</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZQ010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZQ010000263</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023714</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Geminicoccaceae bacterium isolate HOT.CON.139, whole genome shotgun sequencing project', 'Geminicoccaceae bacterium (hot springs metagenome)', 'Geminicoccaceae bacterium', 'Bacteria; Proteobacteria; Alphaproteobacteria; Rhodospirillales; Geminicoccaceae']\n",
      "Id 2240894574\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZT010000000</GBSeq_locus>\\n    <GBSeq_length>49</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Paenibacillus sp. isolate HOT.CON.154, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZT000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZT000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZT000000000.1|JAILZT010000000</GBSeqid>\\n      <GBSeqid>gi|2240894574</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Paenibacillus sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Paenibacillus sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Paenibacillaceae; Paenibacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..49</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..49</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Paenibacillus sp. whole genome shotgun (WGS) project has the project accession JAILZT000000000. This version of the project (01) has the accession number JAILZT010000000, and consists of sequences JAILZT010000001-JAILZT010000049.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 17x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 11:34:20 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 5,322 ; CDSs (total) :: 5,268 ; Genes (coding) :: 5,214 ; CDSs (with protein) :: 5,214 ; Genes (RNA) :: 54 ; rRNAs :: 2, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 2, 1 (5S, 16S) ; partial rRNAs :: 1, 3 (16S, 23S) ; tRNAs :: 43 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 54 ; CDSs (without protein) :: 54 ; Pseudo Genes (ambiguous residues) :: 0 of 54 ; Pseudo Genes (frameshifted) :: 23 of 54 ; Pseudo Genes (incomplete) :: 30 of 54 ; Pseudo Genes (internal stop) :: 11 of 54 ; Pseudo Genes (multiple problems) :: 8 of 54 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..49</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>49</GBInterval_to>\\n            <GBInterval_accession>JAILZT000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Paenibacillus sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.154</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:58172</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZT010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZT010000049</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023717</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Paenibacillus sp. isolate HOT.CON.154, whole genome shotgun sequencing project', 'Paenibacillus sp. (hot springs metagenome)', 'Paenibacillus sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Paenibacillaceae; Paenibacillus']\n",
      "Id 2240894573\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZW010000000</GBSeq_locus>\\n    <GBSeq_length>349</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Alicyclobacillus macrosporangiidus isolate HOT.CON.25, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZW000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZW000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZW000000000.1|JAILZW010000000</GBSeqid>\\n      <GBSeqid>gi|2240894573</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Alicyclobacillus macrosporangiidus (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Alicyclobacillus macrosporangiidus</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..349</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..349</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Alicyclobacillus macrosporangiidus whole genome shotgun (WGS) project has the project accession JAILZW000000000. This version of the project (01) has the accession number JAILZW010000000, and consists of sequences JAILZW010000001-JAILZW010000349.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 24.9x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:44:22 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 4,249 ; CDSs (total) :: 4,205 ; Genes (coding) :: 4,125 ; CDSs (with protein) :: 4,125 ; Genes (RNA) :: 44 ; rRNAs :: 1, 2, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 1 (16S, 23S) ; tRNAs :: 38 ; ncRNAs :: 2 ; Pseudo Genes (total) :: 80 ; CDSs (without protein) :: 80 ; Pseudo Genes (ambiguous residues) :: 0 of 80 ; Pseudo Genes (frameshifted) :: 31 of 80 ; Pseudo Genes (incomplete) :: 45 of 80 ; Pseudo Genes (internal stop) :: 14 of 80 ; Pseudo Genes (multiple problems) :: 8 of 80 ; CRISPR Arrays :: 2 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..349</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>349</GBInterval_to>\\n            <GBInterval_accession>JAILZW000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Alicyclobacillus macrosporangiidus</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.25</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:392015</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZW010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZW010000349</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023720</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Alicyclobacillus macrosporangiidus isolate HOT.CON.25, whole genome shotgun sequencing project', 'Alicyclobacillus macrosporangiidus (hot springs metagenome)', 'Alicyclobacillus macrosporangiidus', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus']\n",
      "Id 2240894572\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZV010000000</GBSeq_locus>\\n    <GBSeq_length>349</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Anoxybacillus sp. isolate HOT.CON.20, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZV000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZV000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZV000000000.1|JAILZV010000000</GBSeqid>\\n      <GBSeqid>gi|2240894572</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Anoxybacillus sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Anoxybacillus sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Bacillaceae; Anoxybacillus; unclassified Anoxybacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..349</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..349</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Anoxybacillus sp. whole genome shotgun (WGS) project has the project accession JAILZV000000000. This version of the project (01) has the accession number JAILZV010000000, and consists of sequences JAILZV010000001-JAILZV010000349.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 58x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:31:31 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,558 ; CDSs (total) :: 3,499 ; Genes (coding) :: 3,416 ; CDSs (with protein) :: 3,416 ; Genes (RNA) :: 59 ; rRNAs :: 2, 3, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 3, 1 (5S, 16S, 23S) ; tRNAs :: 45 ; ncRNAs :: 8 ; Pseudo Genes (total) :: 83 ; CDSs (without protein) :: 83 ; Pseudo Genes (ambiguous residues) :: 0 of 83 ; Pseudo Genes (frameshifted) :: 24 of 83 ; Pseudo Genes (incomplete) :: 57 of 83 ; Pseudo Genes (internal stop) :: 11 of 83 ; Pseudo Genes (multiple problems) :: 8 of 83 ; CRISPR Arrays :: 3 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..349</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>349</GBInterval_to>\\n            <GBInterval_accession>JAILZV000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Anoxybacillus sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.20</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1872573</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZV010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZV010000349</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023719</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Anoxybacillus sp. isolate HOT.CON.20, whole genome shotgun sequencing project', 'Anoxybacillus sp. (hot springs metagenome)', 'Anoxybacillus sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Bacillaceae; Anoxybacillus; unclassified Anoxybacillus']\n",
      "Id 2240894571\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZS010000000</GBSeq_locus>\\n    <GBSeq_length>104</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Firmicutes bacterium isolate HOT.CON.150, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZS000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZS000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZS000000000.1|JAILZS010000000</GBSeqid>\\n      <GBSeqid>gi|2240894571</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Firmicutes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Firmicutes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..104</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..104</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Firmicutes bacterium whole genome shotgun (WGS) project has the project accession JAILZS000000000. This version of the project (01) has the accession number JAILZS010000000, and consists of sequences JAILZS010000001-JAILZS010000104.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 15.2x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 14:27:22 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,606 ; CDSs (total) :: 2,556 ; Genes (coding) :: 2,531 ; CDSs (with protein) :: 2,531 ; Genes (RNA) :: 50 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 43 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 25 ; CDSs (without protein) :: 25 ; Pseudo Genes (ambiguous residues) :: 0 of 25 ; Pseudo Genes (frameshifted) :: 6 of 25 ; Pseudo Genes (incomplete) :: 17 of 25 ; Pseudo Genes (internal stop) :: 3 of 25 ; Pseudo Genes (multiple problems) :: 1 of 25 ; CRISPR Arrays :: 2 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..104</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>104</GBInterval_to>\\n            <GBInterval_accession>JAILZS000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Firmicutes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.150</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1879010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZS010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZS010000104</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023716</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Firmicutes bacterium isolate HOT.CON.150, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes']\n",
      "Id 2240894570\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZU010000000</GBSeq_locus>\\n    <GBSeq_length>806</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Alicyclobacillus sp. isolate HOT.CON.16, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZU000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZU000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZU000000000.1|JAILZU010000000</GBSeqid>\\n      <GBSeqid>gi|2240894570</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Alicyclobacillus sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Alicyclobacillus sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..806</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..806</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Alicyclobacillus sp. whole genome shotgun (WGS) project has the project accession JAILZU000000000. This version of the project (01) has the accession number JAILZU010000000, and consists of sequences JAILZU010000001-JAILZU010000806.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 29.8x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 09:53:29 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,283 ; CDSs (total) :: 2,269 ; Genes (coding) :: 2,247 ; CDSs (with protein) :: 2,247 ; Genes (RNA) :: 14 ; rRNAs :: 1, 1 (5S, 16S) ; partial rRNAs :: 1, 1 (5S, 16S) ; tRNAs :: 10 ; ncRNAs :: 2 ; Pseudo Genes (total) :: 22 ; CDSs (without protein) :: 22 ; Pseudo Genes (ambiguous residues) :: 0 of 22 ; Pseudo Genes (frameshifted) :: 10 of 22 ; Pseudo Genes (incomplete) :: 8 of 22 ; Pseudo Genes (internal stop) :: 6 of 22 ; Pseudo Genes (multiple problems) :: 2 of 22 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..806</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>806</GBInterval_to>\\n            <GBInterval_accession>JAILZU000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Alicyclobacillus sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.16</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:61169</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZU010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZU010000806</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023718</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Alicyclobacillus sp. isolate HOT.CON.16, whole genome shotgun sequencing project', 'Alicyclobacillus sp. (hot springs metagenome)', 'Alicyclobacillus sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus']\n",
      "Id 2240894569\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZX010000000</GBSeq_locus>\\n    <GBSeq_length>340</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Firmicutes bacterium isolate HOT.CON.49, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZX000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZX000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZX000000000.1|JAILZX010000000</GBSeqid>\\n      <GBSeqid>gi|2240894569</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Firmicutes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Firmicutes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..340</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..340</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Firmicutes bacterium whole genome shotgun (WGS) project has the project accession JAILZX000000000. This version of the project (01) has the accession number JAILZX010000000, and consists of sequences JAILZX010000001-JAILZX010000340.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 14.8x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 12:11:24 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,080 ; CDSs (total) :: 2,041 ; Genes (coding) :: 2,008 ; CDSs (with protein) :: 2,008 ; Genes (RNA) :: 39 ; rRNAs :: 1, 1 (5S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1 (23S) ; tRNAs :: 34 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 33 ; CDSs (without protein) :: 33 ; Pseudo Genes (ambiguous residues) :: 0 of 33 ; Pseudo Genes (frameshifted) :: 6 of 33 ; Pseudo Genes (incomplete) :: 25 of 33 ; Pseudo Genes (internal stop) :: 3 of 33 ; Pseudo Genes (multiple problems) :: 1 of 33 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..340</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>340</GBInterval_to>\\n            <GBInterval_accession>JAILZX000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Firmicutes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.49</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1879010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZX010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZX010000340</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023721</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Firmicutes bacterium isolate HOT.CON.49, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes']\n",
      "Id 2240894568\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAA010000000</GBSeq_locus>\\n    <GBSeq_length>195</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Candidatus Bathyarchaeota archaeon isolate HOT.CON.75, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAA000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAA000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAA000000000.1|JAIMAA010000000</GBSeqid>\\n      <GBSeqid>gi|2240894568</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Candidatus Bathyarchaeota archaeon (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Candidatus Bathyarchaeota archaeon</GBSeq_organism>\\n    <GBSeq_taxonomy>Archaea; Candidatus Bathyarchaeota</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..195</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..195</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Candidatus Bathyarchaeota archaeon whole genome shotgun (WGS) project has the project accession JAIMAA000000000. This version of the project (01) has the accession number JAIMAA010000000, and consists of sequences JAIMAA010000001-JAIMAA010000195.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 15x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:32:13 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,309 ; CDSs (total) :: 2,251 ; Genes (coding) :: 2,235 ; CDSs (with protein) :: 2,235 ; Genes (RNA) :: 58 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1, 2, 1 (5S, 16S, 23S) ; partial rRNAs :: 2 (23S) ; tRNAs :: 50 ; ncRNAs :: 2 ; Pseudo Genes (total) :: 16 ; CDSs (without protein) :: 16 ; Pseudo Genes (ambiguous residues) :: 0 of 16 ; Pseudo Genes (frameshifted) :: 2 of 16 ; Pseudo Genes (incomplete) :: 10 of 16 ; Pseudo Genes (internal stop) :: 4 of 16 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..195</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>195</GBInterval_to>\\n            <GBInterval_accession>JAIMAA000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Candidatus Bathyarchaeota archaeon</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.75</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2026714</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAA010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAA010000195</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023724</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Candidatus Bathyarchaeota archaeon isolate HOT.CON.75, whole genome shotgun sequencing project', 'Candidatus Bathyarchaeota archaeon (hot springs metagenome)', 'Candidatus Bathyarchaeota archaeon', 'Archaea; Candidatus Bathyarchaeota']\n",
      "Id 2240894567\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAB010000000</GBSeq_locus>\\n    <GBSeq_length>173</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Firmicutes bacterium isolate HOT.CON.82, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAB000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAB000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAB000000000.1|JAIMAB010000000</GBSeqid>\\n      <GBSeqid>gi|2240894567</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Firmicutes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Firmicutes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..173</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..173</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Firmicutes bacterium whole genome shotgun (WGS) project has the project accession JAIMAB000000000. This version of the project (01) has the accession number JAIMAB010000000, and consists of sequences JAIMAB010000001-JAIMAB010000173.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 24.4x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 13:25:34 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 4,118 ; CDSs (total) :: 4,063 ; Genes (coding) :: 4,016 ; CDSs (with protein) :: 4,016 ; Genes (RNA) :: 55 ; rRNAs :: 1, 1, 2 (5S, 16S, 23S) ; complete rRNAs :: 1, 1 (5S, 16S) ; partial rRNAs :: 2 (23S) ; tRNAs :: 47 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 47 ; CDSs (without protein) :: 47 ; Pseudo Genes (ambiguous residues) :: 0 of 47 ; Pseudo Genes (frameshifted) :: 17 of 47 ; Pseudo Genes (incomplete) :: 30 of 47 ; Pseudo Genes (internal stop) :: 4 of 47 ; Pseudo Genes (multiple problems) :: 3 of 47 ; CRISPR Arrays :: 9 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..173</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>173</GBInterval_to>\\n            <GBInterval_accession>JAIMAB000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Firmicutes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.82</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1879010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAB010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAB010000173</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023725</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Firmicutes bacterium isolate HOT.CON.82, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes']\n",
      "Id 2240894566\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZY010000000</GBSeq_locus>\\n    <GBSeq_length>172</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: bacterium isolate HOT.CON.50, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZY000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZY000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZY000000000.1|JAILZY010000000</GBSeqid>\\n      <GBSeqid>gi|2240894566</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..172</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..172</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The bacterium whole genome shotgun (WGS) project has the project accession JAILZY000000000. This version of the project (01) has the accession number JAILZY010000000, and consists of sequences JAILZY010000001-JAILZY010000172.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 23.9x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 12:15:36 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,972 ; CDSs (total) :: 2,926 ; Genes (coding) :: 2,900 ; CDSs (with protein) :: 2,900 ; Genes (RNA) :: 46 ; tRNAs :: 42 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 26 ; CDSs (without protein) :: 26 ; Pseudo Genes (ambiguous residues) :: 0 of 26 ; Pseudo Genes (frameshifted) :: 7 of 26 ; Pseudo Genes (incomplete) :: 12 of 26 ; Pseudo Genes (internal stop) :: 9 of 26 ; Pseudo Genes (multiple problems) :: 2 of 26 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..172</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>172</GBInterval_to>\\n            <GBInterval_accession>JAILZY000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.50</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1869227</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZY010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZY010000172</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023722</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: bacterium isolate HOT.CON.50, whole genome shotgun sequencing project', 'bacterium (hot springs metagenome)', 'bacterium', 'Bacteria']\n",
      "Id 2240894565\n",
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAILZZ010000000</GBSeq_locus>\\n    <GBSeq_length>124</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Kyrpidia sp. isolate HOT.CON.60, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAILZZ000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAILZZ000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAILZZ000000000.1|JAILZZ010000000</GBSeqid>\\n      <GBSeqid>gi|2240894565</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Kyrpidia sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Kyrpidia sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Kyrpidia; unclassified Kyrpidia</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..124</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..124</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Kyrpidia sp. whole genome shotgun (WGS) project has the project accession JAILZZ000000000. This version of the project (01) has the accession number JAILZZ010000000, and consists of sequences JAILZZ010000001-JAILZZ010000124.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 16.4x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:21:48 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,370 ; CDSs (total) :: 3,316 ; Genes (coding) :: 3,245 ; CDSs (with protein) :: 3,245 ; Genes (RNA) :: 54 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 47 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 71 ; CDSs (without protein) :: 71 ; Pseudo Genes (ambiguous residues) :: 0 of 71 ; Pseudo Genes (frameshifted) :: 21 of 71 ; Pseudo Genes (incomplete) :: 50 of 71 ; Pseudo Genes (internal stop) :: 11 of 71 ; Pseudo Genes (multiple problems) :: 11 of 71 ; CRISPR Arrays :: 5 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..124</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>124</GBInterval_to>\\n            <GBInterval_accession>JAILZZ000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Kyrpidia sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.60</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2073077</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAILZZ010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAILZZ010000124</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023723</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Kyrpidia sp. isolate HOT.CON.60, whole genome shotgun sequencing project', 'Kyrpidia sp. (hot springs metagenome)', 'Kyrpidia sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Kyrpidia; unclassified Kyrpidia']\n",
      "Id 2240894564\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAD010000000</GBSeq_locus>\\n    <GBSeq_length>638</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Bacillus sp. (in: Bacteria) isolate HOT.CON.97, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAD000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAD000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAD000000000.1|JAIMAD010000000</GBSeqid>\\n      <GBSeqid>gi|2240894564</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Bacillus sp. (in: Bacteria) (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Bacillus sp. (in: Bacteria)</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Bacillaceae; Bacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..638</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..638</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Bacillus sp. (in: Bacteria) whole genome shotgun (WGS) project has the project accession JAIMAD000000000. This version of the project (01) has the accession number JAIMAD010000000, and consists of sequences JAIMAD010000001-JAIMAD010000638.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 257.7x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:52:06 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 4,598 ; CDSs (total) :: 4,564 ; Genes (coding) :: 4,460 ; CDSs (with protein) :: 4,460 ; Genes (RNA) :: 34 ; rRNAs :: 2, 2, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 2, 1 (5S, 16S, 23S) ; tRNAs :: 25 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 104 ; CDSs (without protein) :: 104 ; Pseudo Genes (ambiguous residues) :: 0 of 104 ; Pseudo Genes (frameshifted) :: 32 of 104 ; Pseudo Genes (incomplete) :: 47 of 104 ; Pseudo Genes (internal stop) :: 41 of 104 ; Pseudo Genes (multiple problems) :: 13 of 104 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..638</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>638</GBInterval_to>\\n            <GBInterval_accession>JAIMAD000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Bacillus sp. (in: Bacteria)</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.97</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1409</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAD010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAD010000638</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023727</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Bacillus sp. (in: Bacteria) isolate HOT.CON.97, whole genome shotgun sequencing project', 'Bacillus sp. (in: Bacteria) (hot springs metagenome)', 'Bacillus sp. (in: Bacteria)', 'Bacteria; Firmicutes; Bacilli; Bacillales; Bacillaceae; Bacillus']\n",
      "Id 2240894563\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAJ010000000</GBSeq_locus>\\n    <GBSeq_length>1051</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Firmicutes bacterium isolate HOT.MAX.058, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAJ000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAJ000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAJ000000000.1|JAIMAJ010000000</GBSeqid>\\n      <GBSeqid>gi|2240894563</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Firmicutes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Firmicutes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..1051</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..1051</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Firmicutes bacterium whole genome shotgun (WGS) project has the project accession JAIMAJ000000000. This version of the project (01) has the accession number JAIMAJ010000000, and consists of sequences JAIMAJ010000001-JAIMAJ010001051.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 16.3x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 12:45:59 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 4,363 ; CDSs (total) :: 4,319 ; Genes (coding) :: 4,185 ; CDSs (with protein) :: 4,185 ; Genes (RNA) :: 44 ; rRNAs :: 3, 2 (16S, 23S) ; partial rRNAs :: 3, 2 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 0 ; Pseudo Genes (total) :: 134 ; CDSs (without protein) :: 134 ; Pseudo Genes (ambiguous residues) :: 0 of 134 ; Pseudo Genes (frameshifted) :: 30 of 134 ; Pseudo Genes (incomplete) :: 92 of 134 ; Pseudo Genes (internal stop) :: 23 of 134 ; Pseudo Genes (multiple problems) :: 10 of 134 ; CRISPR Arrays :: 5 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..1051</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>1051</GBInterval_to>\\n            <GBInterval_accession>JAIMAJ000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Firmicutes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MAX.058</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1879010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAJ010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAJ010001051</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023733</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Firmicutes bacterium isolate HOT.MAX.058, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes']\n",
      "Id 2240894562\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAK010000000</GBSeq_locus>\\n    <GBSeq_length>410</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Firmicutes bacterium isolate HOT.MAX.062, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAK000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAK000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAK000000000.1|JAIMAK010000000</GBSeqid>\\n      <GBSeqid>gi|2240894562</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Firmicutes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Firmicutes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..410</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..410</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Firmicutes bacterium whole genome shotgun (WGS) project has the project accession JAIMAK000000000. This version of the project (01) has the accession number JAIMAK010000000, and consists of sequences JAIMAK010000001-JAIMAK010000410.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 14.2x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 15:09:07 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,324 ; CDSs (total) :: 3,267 ; Genes (coding) :: 3,230 ; CDSs (with protein) :: 3,230 ; Genes (RNA) :: 57 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 51 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 37 ; CDSs (without protein) :: 37 ; Pseudo Genes (ambiguous residues) :: 0 of 37 ; Pseudo Genes (frameshifted) :: 3 of 37 ; Pseudo Genes (incomplete) :: 30 of 37 ; Pseudo Genes (internal stop) :: 6 of 37 ; Pseudo Genes (multiple problems) :: 2 of 37 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..410</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>410</GBInterval_to>\\n            <GBInterval_accession>JAIMAK000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Firmicutes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MAX.062</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1879010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAK010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAK010000410</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023734</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Firmicutes bacterium isolate HOT.MAX.062, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes']\n",
      "Id 2240894555\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAH010000000</GBSeq_locus>\\n    <GBSeq_length>376</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Alicyclobacillus sp. isolate HOT.MAX.041, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAH000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAH000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAH000000000.1|JAIMAH010000000</GBSeqid>\\n      <GBSeqid>gi|2240894555</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Alicyclobacillus sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Alicyclobacillus sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..376</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..376</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Alicyclobacillus sp. whole genome shotgun (WGS) project has the project accession JAIMAH000000000. This version of the project (01) has the accession number JAIMAH010000000, and consists of sequences JAIMAH010000001-JAIMAH010000376.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 16.2x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 09:53:16 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,158 ; CDSs (total) :: 2,132 ; Genes (coding) :: 2,110 ; CDSs (with protein) :: 2,110 ; Genes (RNA) :: 26 ; tRNAs :: 23 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 22 ; CDSs (without protein) :: 22 ; Pseudo Genes (ambiguous residues) :: 0 of 22 ; Pseudo Genes (frameshifted) :: 6 of 22 ; Pseudo Genes (incomplete) :: 13 of 22 ; Pseudo Genes (internal stop) :: 5 of 22 ; Pseudo Genes (multiple problems) :: 1 of 22 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..376</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>376</GBInterval_to>\\n            <GBInterval_accession>JAIMAH000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Alicyclobacillus sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MAX.041</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:61169</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAH010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAH010000376</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023731</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Alicyclobacillus sp. isolate HOT.MAX.041, whole genome shotgun sequencing project', 'Alicyclobacillus sp. (hot springs metagenome)', 'Alicyclobacillus sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus']\n",
      "Id 2240894553\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAI010000000</GBSeq_locus>\\n    <GBSeq_length>75</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Acidothermus sp. isolate HOT.MAX.052, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAI000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAI000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAI000000000.1|JAIMAI010000000</GBSeqid>\\n      <GBSeqid>gi|2240894553</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Acidothermus sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Acidothermus sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Actinobacteria; Acidothermales; Acidothermaceae; Acidothermus; unclassified Acidothermus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..75</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..75</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Acidothermus sp. whole genome shotgun (WGS) project has the project accession JAIMAI000000000. This version of the project (01) has the accession number JAIMAI010000000, and consists of sequences JAIMAI010000001-JAIMAI010000075.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 14.7x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 09:48:49 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,075 ; CDSs (total) :: 2,029 ; Genes (coding) :: 1,995 ; CDSs (with protein) :: 1,995 ; Genes (RNA) :: 46 ; tRNAs :: 43 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 34 ; CDSs (without protein) :: 34 ; Pseudo Genes (ambiguous residues) :: 0 of 34 ; Pseudo Genes (frameshifted) :: 2 of 34 ; Pseudo Genes (incomplete) :: 31 of 34 ; Pseudo Genes (internal stop) :: 1 of 34 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..75</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>75</GBInterval_to>\\n            <GBInterval_accession>JAIMAI000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Acidothermus sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MAX.052</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1872121</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAI010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAI010000075</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023732</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Acidothermus sp. isolate HOT.MAX.052, whole genome shotgun sequencing project', 'Acidothermus sp. (hot springs metagenome)', 'Acidothermus sp.', 'Bacteria; Actinobacteria; Acidothermales; Acidothermaceae; Acidothermus; unclassified Acidothermus']\n",
      "Id 2240894552\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAL010000000</GBSeq_locus>\\n    <GBSeq_length>44</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Burkholderiales bacterium isolate HOT.MAX.068, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAL000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAL000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAL000000000.1|JAIMAL010000000</GBSeqid>\\n      <GBSeqid>gi|2240894552</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Burkholderiales bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Burkholderiales bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Proteobacteria; Betaproteobacteria; Burkholderiales</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..44</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..44</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Burkholderiales bacterium whole genome shotgun (WGS) project has the project accession JAIMAL000000000. This version of the project (01) has the accession number JAIMAL010000000, and consists of sequences JAIMAL010000001-JAIMAL010000044.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 14.4x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:13:49 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,525 ; CDSs (total) :: 2,474 ; Genes (coding) :: 2,463 ; CDSs (with protein) :: 2,463 ; Genes (RNA) :: 51 ; rRNAs :: 1, 1, 2 (5S, 16S, 23S) ; complete rRNAs :: 1, 1 (5S, 16S) ; partial rRNAs :: 2 (23S) ; tRNAs :: 43 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 11 ; CDSs (without protein) :: 11 ; Pseudo Genes (ambiguous residues) :: 0 of 11 ; Pseudo Genes (frameshifted) :: 3 of 11 ; Pseudo Genes (incomplete) :: 6 of 11 ; Pseudo Genes (internal stop) :: 5 of 11 ; Pseudo Genes (multiple problems) :: 3 of 11 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..44</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>44</GBInterval_to>\\n            <GBInterval_accession>JAIMAL000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Burkholderiales bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MAX.068</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1891238</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAL010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAL010000044</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023735</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Burkholderiales bacterium isolate HOT.MAX.068, whole genome shotgun sequencing project', 'Burkholderiales bacterium (hot springs metagenome)', 'Burkholderiales bacterium', 'Bacteria; Proteobacteria; Betaproteobacteria; Burkholderiales']\n",
      "Id 2240894551\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAR010000000</GBSeq_locus>\\n    <GBSeq_length>394</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Bryobacteraceae bacterium isolate HOT.MAX.089, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAR000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAR000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAR000000000.1|JAIMAR010000000</GBSeqid>\\n      <GBSeqid>gi|2240894551</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Bryobacteraceae bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Bryobacteraceae bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Acidobacteria; Bryobacterales; Bryobacteraceae</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..394</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..394</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Bryobacteraceae bacterium whole genome shotgun (WGS) project has the project accession JAIMAR000000000. This version of the project (01) has the accession number JAIMAR010000000, and consists of sequences JAIMAR010000001-JAIMAR010000394.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 16.7x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:19:39 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,768 ; CDSs (total) :: 3,710 ; Genes (coding) :: 3,688 ; CDSs (with protein) :: 3,688 ; Genes (RNA) :: 58 ; rRNAs :: 2, 4, 4 (5S, 16S, 23S) ; complete rRNAs :: 2, 1 (5S, 23S) ; partial rRNAs :: 4, 3 (16S, 23S) ; tRNAs :: 45 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 22 ; CDSs (without protein) :: 22 ; Pseudo Genes (ambiguous residues) :: 0 of 22 ; Pseudo Genes (frameshifted) :: 2 of 22 ; Pseudo Genes (incomplete) :: 16 of 22 ; Pseudo Genes (internal stop) :: 4 of 22 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..394</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>394</GBInterval_to>\\n            <GBInterval_accession>JAIMAR000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Bryobacteraceae bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MAX.089</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2212468</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAR010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAR010000394</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023741</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Bryobacteraceae bacterium isolate HOT.MAX.089, whole genome shotgun sequencing project', 'Bryobacteraceae bacterium (hot springs metagenome)', 'Bryobacteraceae bacterium', 'Bacteria; Acidobacteria; Bryobacterales; Bryobacteraceae']\n",
      "Id 2240894550\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAE010000000</GBSeq_locus>\\n    <GBSeq_length>405</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Firmicutes bacterium isolate HOT.CON.99, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAE000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAE000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAE000000000.1|JAIMAE010000000</GBSeqid>\\n      <GBSeqid>gi|2240894550</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Firmicutes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Firmicutes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..405</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..405</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Firmicutes bacterium whole genome shotgun (WGS) project has the project accession JAIMAE000000000. This version of the project (01) has the accession number JAIMAE010000000, and consists of sequences JAIMAE010000001-JAIMAE010000405.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 31x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 13:35:38 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,492 ; CDSs (total) :: 3,447 ; Genes (coding) :: 3,417 ; CDSs (with protein) :: 3,417 ; Genes (RNA) :: 45 ; rRNAs :: 1, 1 (16S, 23S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 30 ; CDSs (without protein) :: 30 ; Pseudo Genes (ambiguous residues) :: 0 of 30 ; Pseudo Genes (frameshifted) :: 9 of 30 ; Pseudo Genes (incomplete) :: 18 of 30 ; Pseudo Genes (internal stop) :: 9 of 30 ; Pseudo Genes (multiple problems) :: 5 of 30 ; CRISPR Arrays :: 2 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..405</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>405</GBInterval_to>\\n            <GBInterval_accession>JAIMAE000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Firmicutes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.99</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1879010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAE010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAE010000405</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023728</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Firmicutes bacterium isolate HOT.CON.99, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes']\n",
      "Id 2240894549\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAC010000000</GBSeq_locus>\\n    <GBSeq_length>1082</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Meiothermus silvanus isolate HOT.CON.95, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAC000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAC000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAC000000000.1|JAIMAC010000000</GBSeqid>\\n      <GBSeqid>gi|2240894549</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Meiothermus silvanus (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Meiothermus silvanus</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Deinococcus-Thermus; Deinococci; Thermales; Thermaceae; Meiothermus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..1082</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..1082</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Meiothermus silvanus whole genome shotgun (WGS) project has the project accession JAIMAC000000000. This version of the project (01) has the accession number JAIMAC010000000, and consists of sequences JAIMAC010000001-JAIMAC010001082.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 17x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 09:57:43 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,913 ; CDSs (total) :: 2,880 ; Genes (coding) :: 2,758 ; CDSs (with protein) :: 2,758 ; Genes (RNA) :: 33 ; rRNAs :: 2 (16S) ; partial rRNAs :: 2 (16S) ; tRNAs :: 30 ; ncRNAs :: 1 ; Pseudo Genes (total) :: 122 ; CDSs (without protein) :: 122 ; Pseudo Genes (ambiguous residues) :: 0 of 122 ; Pseudo Genes (frameshifted) :: 14 of 122 ; Pseudo Genes (incomplete) :: 100 of 122 ; Pseudo Genes (internal stop) :: 17 of 122 ; Pseudo Genes (multiple problems) :: 8 of 122 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..1082</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>1082</GBInterval_to>\\n            <GBInterval_accession>JAIMAC000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Meiothermus silvanus</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.CON.95</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:52022</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAC010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAC010001082</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023726</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Meiothermus silvanus isolate HOT.CON.95, whole genome shotgun sequencing project', 'Meiothermus silvanus (hot springs metagenome)', 'Meiothermus silvanus', 'Bacteria; Deinococcus-Thermus; Deinococci; Thermales; Thermaceae; Meiothermus']\n",
      "Id 2240894548\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAG010000000</GBSeq_locus>\\n    <GBSeq_length>51</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Acidobacteriia bacterium isolate HOT.MAX.039, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAG000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAG000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAG000000000.1|JAIMAG010000000</GBSeqid>\\n      <GBSeqid>gi|2240894548</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Acidobacteriia bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Acidobacteriia bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Acidobacteria</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..51</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..51</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Acidobacteriia bacterium whole genome shotgun (WGS) project has the project accession JAIMAG000000000. This version of the project (01) has the accession number JAIMAG010000000, and consists of sequences JAIMAG010000001-JAIMAG010000051.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 14.9x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 12:11:52 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,569 ; CDSs (total) :: 2,518 ; Genes (coding) :: 2,504 ; CDSs (with protein) :: 2,504 ; Genes (RNA) :: 51 ; rRNAs :: 1, 2, 2 (5S, 16S, 23S) ; complete rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 44 ; ncRNAs :: 2 ; Pseudo Genes (total) :: 14 ; CDSs (without protein) :: 14 ; Pseudo Genes (ambiguous residues) :: 0 of 14 ; Pseudo Genes (frameshifted) :: 5 of 14 ; Pseudo Genes (incomplete) :: 7 of 14 ; Pseudo Genes (internal stop) :: 2 of 14 ; CRISPR Arrays :: 2 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..51</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>51</GBInterval_to>\\n            <GBInterval_accession>JAIMAG000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Acidobacteriia bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MAX.039</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2081523</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAG010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAG010000051</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023730</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Acidobacteriia bacterium isolate HOT.MAX.039, whole genome shotgun sequencing project', 'Acidobacteriia bacterium (hot springs metagenome)', 'Acidobacteriia bacterium', 'Bacteria; Acidobacteria']\n",
      "Id 2240894547\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAM010000000</GBSeq_locus>\\n    <GBSeq_length>140</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Armatimonadetes bacterium isolate HOT.MAX.070, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAM000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAM000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAM000000000.1|JAIMAM010000000</GBSeqid>\\n      <GBSeqid>gi|2240894547</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Armatimonadetes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Armatimonadetes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Armatimonadetes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..140</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..140</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Armatimonadetes bacterium whole genome shotgun (WGS) project has the project accession JAIMAM000000000. This version of the project (01) has the accession number JAIMAM010000000, and consists of sequences JAIMAM010000001-JAIMAM010000140.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 15.9x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 12:21:12 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,854 ; CDSs (total) :: 2,804 ; Genes (coding) :: 2,786 ; CDSs (with protein) :: 2,786 ; Genes (RNA) :: 50 ; rRNAs :: 1 (16S) ; partial rRNAs :: 1 (16S) ; tRNAs :: 46 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 18 ; CDSs (without protein) :: 18 ; Pseudo Genes (ambiguous residues) :: 0 of 18 ; Pseudo Genes (frameshifted) :: 3 of 18 ; Pseudo Genes (incomplete) :: 15 of 18 ; Pseudo Genes (internal stop) :: 0 of 18 ; CRISPR Arrays :: 5 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..140</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>140</GBInterval_to>\\n            <GBInterval_accession>JAIMAM000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Armatimonadetes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MAX.070</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2033014</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAM010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAM010000140</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023736</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Armatimonadetes bacterium isolate HOT.MAX.070, whole genome shotgun sequencing project', 'Armatimonadetes bacterium (hot springs metagenome)', 'Armatimonadetes bacterium', 'Bacteria; Armatimonadetes']\n",
      "Id 2240894546\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAF010000000</GBSeq_locus>\\n    <GBSeq_length>185</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Acidothermus cellulolyticus isolate HOT.MAX.011, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAF000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAF000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAF000000000.1|JAIMAF010000000</GBSeqid>\\n      <GBSeqid>gi|2240894546</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Acidothermus cellulolyticus (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Acidothermus cellulolyticus</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Actinobacteria; Acidothermales; Acidothermaceae; Acidothermus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..185</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..185</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Acidothermus cellulolyticus whole genome shotgun (WGS) project has the project accession JAIMAF000000000. This version of the project (01) has the accession number JAIMAF010000000, and consists of sequences JAIMAF010000001-JAIMAF010000185.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 14.8x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 09:58:50 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,171 ; CDSs (total) :: 2,123 ; Genes (coding) :: 2,112 ; CDSs (with protein) :: 2,112 ; Genes (RNA) :: 48 ; rRNAs :: 1, 1 (5S, 16S) ; partial rRNAs :: 1, 1 (5S, 16S) ; tRNAs :: 43 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 11 ; CDSs (without protein) :: 11 ; Pseudo Genes (ambiguous residues) :: 0 of 11 ; Pseudo Genes (frameshifted) :: 3 of 11 ; Pseudo Genes (incomplete) :: 8 of 11 ; Pseudo Genes (internal stop) :: 1 of 11 ; Pseudo Genes (multiple problems) :: 1 of 11 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..185</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>185</GBInterval_to>\\n            <GBInterval_accession>JAIMAF000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Acidothermus cellulolyticus</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MAX.011</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:28049</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAF010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAF010000185</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023729</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Acidothermus cellulolyticus isolate HOT.MAX.011, whole genome shotgun sequencing project', 'Acidothermus cellulolyticus (hot springs metagenome)', 'Acidothermus cellulolyticus', 'Bacteria; Actinobacteria; Acidothermales; Acidothermaceae; Acidothermus']\n",
      "Id 2240894545\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAO010000000</GBSeq_locus>\\n    <GBSeq_length>206</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Roseiflexus sp. isolate HOT.MAX.081, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAO000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAO000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAO000000000.1|JAIMAO010000000</GBSeqid>\\n      <GBSeqid>gi|2240894545</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Roseiflexus sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Roseiflexus sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Chloroflexi; Chloroflexia; Chloroflexales; Roseiflexineae; Roseiflexaceae; Roseiflexus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..206</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..206</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Roseiflexus sp. whole genome shotgun (WGS) project has the project accession JAIMAO000000000. This version of the project (01) has the accession number JAIMAO010000000, and consists of sequences JAIMAO010000001-JAIMAO010000206.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 27.7x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:50:43 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 4,771 ; CDSs (total) :: 4,715 ; Genes (coding) :: 4,616 ; CDSs (with protein) :: 4,616 ; Genes (RNA) :: 56 ; rRNAs :: 2, 1, 2 (5S, 16S, 23S) ; complete rRNAs :: 2, 1, 1 (5S, 16S, 23S) ; partial rRNAs :: 1 (23S) ; tRNAs :: 48 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 99 ; CDSs (without protein) :: 99 ; Pseudo Genes (ambiguous residues) :: 0 of 99 ; Pseudo Genes (frameshifted) :: 36 of 99 ; Pseudo Genes (incomplete) :: 61 of 99 ; Pseudo Genes (internal stop) :: 14 of 99 ; Pseudo Genes (multiple problems) :: 10 of 99 ; CRISPR Arrays :: 6 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..206</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>206</GBInterval_to>\\n            <GBInterval_accession>JAIMAO000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Roseiflexus sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MAX.081</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2562120</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAO010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAO010000206</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023738</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Roseiflexus sp. isolate HOT.MAX.081, whole genome shotgun sequencing project', 'Roseiflexus sp. (hot springs metagenome)', 'Roseiflexus sp.', 'Bacteria; Chloroflexi; Chloroflexia; Chloroflexales; Roseiflexineae; Roseiflexaceae; Roseiflexus']\n",
      "Id 2240894544\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAN010000000</GBSeq_locus>\\n    <GBSeq_length>122</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Armatimonadetes bacterium isolate HOT.MAX.079, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAN000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAN000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAN000000000.1|JAIMAN010000000</GBSeqid>\\n      <GBSeqid>gi|2240894544</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Armatimonadetes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Armatimonadetes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Armatimonadetes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..122</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..122</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Armatimonadetes bacterium whole genome shotgun (WGS) project has the project accession JAIMAN000000000. This version of the project (01) has the accession number JAIMAN010000000, and consists of sequences JAIMAN010000001-JAIMAN010000122.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 18.2x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 13:16:57 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,610 ; CDSs (total) :: 2,558 ; Genes (coding) :: 2,544 ; CDSs (with protein) :: 2,544 ; Genes (RNA) :: 52 ; rRNAs :: 1, 1 (5S, 16S) ; partial rRNAs :: 1, 1 (5S, 16S) ; tRNAs :: 47 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 14 ; CDSs (without protein) :: 14 ; Pseudo Genes (ambiguous residues) :: 0 of 14 ; Pseudo Genes (frameshifted) :: 2 of 14 ; Pseudo Genes (incomplete) :: 12 of 14 ; Pseudo Genes (internal stop) :: 0 of 14 ; CRISPR Arrays :: 2 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..122</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>122</GBInterval_to>\\n            <GBInterval_accession>JAIMAN000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Armatimonadetes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MAX.079</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2033014</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAN010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAN010000122</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023737</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Armatimonadetes bacterium isolate HOT.MAX.079, whole genome shotgun sequencing project', 'Armatimonadetes bacterium (hot springs metagenome)', 'Armatimonadetes bacterium', 'Bacteria; Armatimonadetes']\n",
      "Id 2240894509\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAP010000000</GBSeq_locus>\\n    <GBSeq_length>64</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Meiothermus ruber isolate HOT.MAX.085, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAP000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAP000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAP000000000.1|JAIMAP010000000</GBSeqid>\\n      <GBSeqid>gi|2240894509</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Meiothermus ruber (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Meiothermus ruber</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Deinococcus-Thermus; Deinococci; Thermales; Thermaceae; Meiothermus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..64</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..64</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Meiothermus ruber whole genome shotgun (WGS) project has the project accession JAIMAP000000000. This version of the project (01) has the accession number JAIMAP010000000, and consists of sequences JAIMAP010000001-JAIMAP010000064.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 21.5x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:27:06 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,247 ; CDSs (total) :: 3,189 ; Genes (coding) :: 3,160 ; CDSs (with protein) :: 3,160 ; Genes (RNA) :: 58 ; rRNAs :: 1, 4, 2 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 4, 2 (16S, 23S) ; tRNAs :: 48 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 29 ; CDSs (without protein) :: 29 ; Pseudo Genes (ambiguous residues) :: 0 of 29 ; Pseudo Genes (frameshifted) :: 7 of 29 ; Pseudo Genes (incomplete) :: 19 of 29 ; Pseudo Genes (internal stop) :: 4 of 29 ; Pseudo Genes (multiple problems) :: 1 of 29 ; CRISPR Arrays :: 6 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..64</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>64</GBInterval_to>\\n            <GBInterval_accession>JAIMAP000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Meiothermus ruber</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MAX.085</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:277</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAP010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAP010000064</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023739</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Meiothermus ruber isolate HOT.MAX.085, whole genome shotgun sequencing project', 'Meiothermus ruber (hot springs metagenome)', 'Meiothermus ruber', 'Bacteria; Deinococcus-Thermus; Deinococci; Thermales; Thermaceae; Meiothermus']\n",
      "Id 2240894508\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAS010000000</GBSeq_locus>\\n    <GBSeq_length>96</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Thermaceae bacterium isolate HOT.MAX.091, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAS000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAS000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAS000000000.1|JAIMAS010000000</GBSeqid>\\n      <GBSeqid>gi|2240894508</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Thermaceae bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Thermaceae bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Deinococcus-Thermus; Deinococci; Thermales; Thermaceae; unclassified Thermaceae</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..96</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..96</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Thermaceae bacterium whole genome shotgun (WGS) project has the project accession JAIMAS000000000. This version of the project (01) has the accession number JAIMAS010000000, and consists of sequences JAIMAS010000001-JAIMAS010000096.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 14.6x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:14:17 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,328 ; CDSs (total) :: 3,270 ; Genes (coding) :: 3,245 ; CDSs (with protein) :: 3,245 ; Genes (RNA) :: 58 ; rRNAs :: 1, 4, 2 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 4, 2 (16S, 23S) ; tRNAs :: 48 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 25 ; CDSs (without protein) :: 25 ; Pseudo Genes (ambiguous residues) :: 0 of 25 ; Pseudo Genes (frameshifted) :: 12 of 25 ; Pseudo Genes (incomplete) :: 13 of 25 ; Pseudo Genes (internal stop) :: 4 of 25 ; Pseudo Genes (multiple problems) :: 4 of 25 ; CRISPR Arrays :: 5 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..96</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>96</GBInterval_to>\\n            <GBInterval_accession>JAIMAS000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Thermaceae bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MAX.091</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2614257</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAS010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAS010000096</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023742</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Thermaceae bacterium isolate HOT.MAX.091, whole genome shotgun sequencing project', 'Thermaceae bacterium (hot springs metagenome)', 'Thermaceae bacterium', 'Bacteria; Deinococcus-Thermus; Deinococci; Thermales; Thermaceae; unclassified Thermaceae']\n",
      "Id 2240894507\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAQ010000000</GBSeq_locus>\\n    <GBSeq_length>35</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Thermoflavifilum sp. isolate HOT.MAX.088, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAQ000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAQ000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAQ000000000.1|JAIMAQ010000000</GBSeqid>\\n      <GBSeqid>gi|2240894507</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Thermoflavifilum sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Thermoflavifilum sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Bacteroidetes; Chitinophagia; Chitinophagales; Chitinophagaceae; Thermoflavifilum</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..35</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..35</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Thermoflavifilum sp. whole genome shotgun (WGS) project has the project accession JAIMAQ000000000. This version of the project (01) has the accession number JAIMAQ010000000, and consists of sequences JAIMAQ010000001-JAIMAQ010000035.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 19.6x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 09:56:32 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,269 ; CDSs (total) :: 2,228 ; Genes (coding) :: 2,223 ; CDSs (with protein) :: 2,223 ; Genes (RNA) :: 41 ; rRNAs :: 1 (16S) ; partial rRNAs :: 1 (16S) ; tRNAs :: 37 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 5 ; CDSs (without protein) :: 5 ; Pseudo Genes (ambiguous residues) :: 0 of 5 ; Pseudo Genes (frameshifted) :: 2 of 5 ; Pseudo Genes (incomplete) :: 2 of 5 ; Pseudo Genes (internal stop) :: 1 of 5 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..35</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>35</GBInterval_to>\\n            <GBInterval_accession>JAIMAQ000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Thermoflavifilum sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MAX.088</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1968839</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAQ010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAQ010000035</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023740</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Thermoflavifilum sp. isolate HOT.MAX.088, whole genome shotgun sequencing project', 'Thermoflavifilum sp. (hot springs metagenome)', 'Thermoflavifilum sp.', 'Bacteria; Bacteroidetes; Chitinophagia; Chitinophagales; Chitinophagaceae; Thermoflavifilum']\n",
      "Id 2240894506\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAU010000000</GBSeq_locus>\\n    <GBSeq_length>35</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Armatimonadetes bacterium isolate HOT.MB2.102, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAU000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAU000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAU000000000.1|JAIMAU010000000</GBSeqid>\\n      <GBSeqid>gi|2240894506</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Armatimonadetes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Armatimonadetes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Armatimonadetes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..35</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..35</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Armatimonadetes bacterium whole genome shotgun (WGS) project has the project accession JAIMAU000000000. This version of the project (01) has the accession number JAIMAU010000000, and consists of sequences JAIMAU010000001-JAIMAU010000035.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 54.7x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 11:45:01 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,611 ; CDSs (total) :: 2,559 ; Genes (coding) :: 2,551 ; CDSs (with protein) :: 2,551 ; Genes (RNA) :: 52 ; rRNAs :: 1, 3, 2 (5S, 16S, 23S) ; complete rRNAs :: 1, 1, 2 (5S, 16S, 23S) ; partial rRNAs :: 2 (16S) ; tRNAs :: 42 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 8 ; CDSs (without protein) :: 8 ; Pseudo Genes (ambiguous residues) :: 0 of 8 ; Pseudo Genes (frameshifted) :: 2 of 8 ; Pseudo Genes (incomplete) :: 3 of 8 ; Pseudo Genes (internal stop) :: 3 of 8 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..35</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>35</GBInterval_to>\\n            <GBInterval_accession>JAIMAU000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Armatimonadetes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.102</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2033014</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAU010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAU010000035</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023744</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Armatimonadetes bacterium isolate HOT.MB2.102, whole genome shotgun sequencing project', 'Armatimonadetes bacterium (hot springs metagenome)', 'Armatimonadetes bacterium', 'Bacteria; Armatimonadetes']\n",
      "Id 2240894505\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAT010000000</GBSeq_locus>\\n    <GBSeq_length>156</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Firmicutes bacterium isolate HOT.MB2.100, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAT000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAT000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAT000000000.1|JAIMAT010000000</GBSeqid>\\n      <GBSeqid>gi|2240894505</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Firmicutes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Firmicutes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..156</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..156</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Firmicutes bacterium whole genome shotgun (WGS) project has the project accession JAIMAT000000000. This version of the project (01) has the accession number JAIMAT010000000, and consists of sequences JAIMAT010000001-JAIMAT010000156.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 16.1x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 13:22:10 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,335 ; CDSs (total) :: 2,290 ; Genes (coding) :: 2,267 ; CDSs (with protein) :: 2,267 ; Genes (RNA) :: 45 ; rRNAs :: 1 (16S) ; partial rRNAs :: 1 (16S) ; tRNAs :: 41 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 23 ; CDSs (without protein) :: 23 ; Pseudo Genes (ambiguous residues) :: 0 of 23 ; Pseudo Genes (frameshifted) :: 6 of 23 ; Pseudo Genes (incomplete) :: 17 of 23 ; Pseudo Genes (internal stop) :: 2 of 23 ; Pseudo Genes (multiple problems) :: 2 of 23 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..156</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>156</GBInterval_to>\\n            <GBInterval_accession>JAIMAT000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Firmicutes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.100</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1879010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAT010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAT010000156</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023743</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Firmicutes bacterium isolate HOT.MB2.100, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes']\n",
      "Id 2240894498\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAV010000000</GBSeq_locus>\\n    <GBSeq_length>165</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Alicyclobacillus sp. isolate HOT.MB2.104, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAV000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAV000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAV000000000.1|JAIMAV010000000</GBSeqid>\\n      <GBSeqid>gi|2240894498</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Alicyclobacillus sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Alicyclobacillus sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..165</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..165</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Alicyclobacillus sp. whole genome shotgun (WGS) project has the project accession JAIMAV000000000. This version of the project (01) has the accession number JAIMAV010000000, and consists of sequences JAIMAV010000001-JAIMAV010000165.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 20.2x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:26:11 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,947 ; CDSs (total) :: 2,915 ; Genes (coding) :: 2,895 ; CDSs (with protein) :: 2,895 ; Genes (RNA) :: 32 ; rRNAs :: 1, 1 (5S, 16S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1 (16S) ; tRNAs :: 25 ; ncRNAs :: 5 ; Pseudo Genes (total) :: 20 ; CDSs (without protein) :: 20 ; Pseudo Genes (ambiguous residues) :: 0 of 20 ; Pseudo Genes (frameshifted) :: 4 of 20 ; Pseudo Genes (incomplete) :: 16 of 20 ; Pseudo Genes (internal stop) :: 1 of 20 ; Pseudo Genes (multiple problems) :: 1 of 20 ; CRISPR Arrays :: 2 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..165</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>165</GBInterval_to>\\n            <GBInterval_accession>JAIMAV000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Alicyclobacillus sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.104</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:61169</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAV010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAV010000165</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023745</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Alicyclobacillus sp. isolate HOT.MB2.104, whole genome shotgun sequencing project', 'Alicyclobacillus sp. (hot springs metagenome)', 'Alicyclobacillus sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus']\n",
      "Id 2240894485\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAY010000000</GBSeq_locus>\\n    <GBSeq_length>76</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Alicyclobacillus sp. isolate HOT.MB2.19, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAY000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAY000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAY000000000.1|JAIMAY010000000</GBSeqid>\\n      <GBSeqid>gi|2240894485</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Alicyclobacillus sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Alicyclobacillus sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..76</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..76</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Alicyclobacillus sp. whole genome shotgun (WGS) project has the project accession JAIMAY000000000. This version of the project (01) has the accession number JAIMAY010000000, and consists of sequences JAIMAY010000001-JAIMAY010000076.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 18.5x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:21:29 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,589 ; CDSs (total) :: 2,551 ; Genes (coding) :: 2,532 ; CDSs (with protein) :: 2,532 ; Genes (RNA) :: 38 ; rRNAs :: 1, 1, 2 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 2 (16S, 23S) ; tRNAs :: 30 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 19 ; CDSs (without protein) :: 19 ; Pseudo Genes (ambiguous residues) :: 0 of 19 ; Pseudo Genes (frameshifted) :: 11 of 19 ; Pseudo Genes (incomplete) :: 6 of 19 ; Pseudo Genes (internal stop) :: 2 of 19 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..76</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>76</GBInterval_to>\\n            <GBInterval_accession>JAIMAY000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Alicyclobacillus sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.19</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:61169</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAY010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAY010000076</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023748</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Alicyclobacillus sp. isolate HOT.MB2.19, whole genome shotgun sequencing project', 'Alicyclobacillus sp. (hot springs metagenome)', 'Alicyclobacillus sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus']\n",
      "Id 2240894483\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBB010000000</GBSeq_locus>\\n    <GBSeq_length>302</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Anaerolineae bacterium isolate HOT.MB2.30, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBB000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBB000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBB000000000.1|JAIMBB010000000</GBSeqid>\\n      <GBSeqid>gi|2240894483</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Anaerolineae bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Anaerolineae bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Chloroflexi; Anaerolineae; unclassified Anaerolineae</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..302</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..302</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Anaerolineae bacterium whole genome shotgun (WGS) project has the project accession JAIMBB000000000. This version of the project (01) has the accession number JAIMBB010000000, and consists of sequences JAIMBB010000001-JAIMBB010000302.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 21.2x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 13:17:40 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,809 ; CDSs (total) :: 3,765 ; Genes (coding) :: 3,721 ; CDSs (with protein) :: 3,721 ; Genes (RNA) :: 44 ; rRNAs :: 1, 1 (5S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1 (23S) ; tRNAs :: 38 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 44 ; CDSs (without protein) :: 44 ; Pseudo Genes (ambiguous residues) :: 0 of 44 ; Pseudo Genes (frameshifted) :: 9 of 44 ; Pseudo Genes (incomplete) :: 30 of 44 ; Pseudo Genes (internal stop) :: 6 of 44 ; Pseudo Genes (multiple problems) :: 1 of 44 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..302</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>302</GBInterval_to>\\n            <GBInterval_accession>JAIMBB000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Anaerolineae bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.30</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2052143</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBB010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBB010000302</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023751</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Anaerolineae bacterium isolate HOT.MB2.30, whole genome shotgun sequencing project', 'Anaerolineae bacterium (hot springs metagenome)', 'Anaerolineae bacterium', 'Bacteria; Chloroflexi; Anaerolineae; unclassified Anaerolineae']\n",
      "Id 2240894482\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBA010000000</GBSeq_locus>\\n    <GBSeq_length>839</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Bryobacteraceae bacterium isolate HOT.MB2.26, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBA000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBA000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBA000000000.1|JAIMBA010000000</GBSeqid>\\n      <GBSeqid>gi|2240894482</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Bryobacteraceae bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Bryobacteraceae bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Acidobacteria; Bryobacterales; Bryobacteraceae</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..839</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..839</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Bryobacteraceae bacterium whole genome shotgun (WGS) project has the project accession JAIMBA000000000. This version of the project (01) has the accession number JAIMBA010000000, and consists of sequences JAIMBA010000001-JAIMBA010000839.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 20.6x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:17:19 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,746 ; CDSs (total) :: 3,703 ; Genes (coding) :: 3,663 ; CDSs (with protein) :: 3,663 ; Genes (RNA) :: 43 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 38 ; ncRNAs :: 2 ; Pseudo Genes (total) :: 40 ; CDSs (without protein) :: 40 ; Pseudo Genes (ambiguous residues) :: 0 of 40 ; Pseudo Genes (frameshifted) :: 4 of 40 ; Pseudo Genes (incomplete) :: 22 of 40 ; Pseudo Genes (internal stop) :: 15 of 40 ; Pseudo Genes (multiple problems) :: 1 of 40 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..839</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>839</GBInterval_to>\\n            <GBInterval_accession>JAIMBA000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Bryobacteraceae bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.26</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2212468</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBA010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBA010000839</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023750</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Bryobacteraceae bacterium isolate HOT.MB2.26, whole genome shotgun sequencing project', 'Bryobacteraceae bacterium (hot springs metagenome)', 'Bryobacteraceae bacterium', 'Bacteria; Acidobacteria; Bryobacterales; Bryobacteraceae']\n",
      "Id 2240894481\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBC010000000</GBSeq_locus>\\n    <GBSeq_length>422</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Pirellulales bacterium isolate HOT.MB2.34, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBC000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBC000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBC000000000.1|JAIMBC010000000</GBSeqid>\\n      <GBSeqid>gi|2240894481</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Pirellulales bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Pirellulales bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Planctomycetes; Planctomycetia; Pirellulales; unclassified Pirellulales</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..422</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..422</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Pirellulales bacterium whole genome shotgun (WGS) project has the project accession JAIMBC000000000. This version of the project (01) has the accession number JAIMBC010000000, and consists of sequences JAIMBC010000001-JAIMBC010000422.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 41.3x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:50:28 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 4,428 ; CDSs (total) :: 4,377 ; Genes (coding) :: 4,355 ; CDSs (with protein) :: 4,355 ; Genes (RNA) :: 51 ; rRNAs :: 1, 1 (16S, 23S) ; complete rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 46 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 22 ; CDSs (without protein) :: 22 ; Pseudo Genes (ambiguous residues) :: 0 of 22 ; Pseudo Genes (frameshifted) :: 4 of 22 ; Pseudo Genes (incomplete) :: 15 of 22 ; Pseudo Genes (internal stop) :: 4 of 22 ; Pseudo Genes (multiple problems) :: 1 of 22 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..422</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>422</GBInterval_to>\\n            <GBInterval_accession>JAIMBC000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Pirellulales bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.34</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2715253</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBC010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBC010000422</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023752</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Pirellulales bacterium isolate HOT.MB2.34, whole genome shotgun sequencing project', 'Pirellulales bacterium (hot springs metagenome)', 'Pirellulales bacterium', 'Bacteria; Planctomycetes; Planctomycetia; Pirellulales; unclassified Pirellulales']\n",
      "Id 2240894480\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAX010000000</GBSeq_locus>\\n    <GBSeq_length>456</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Firmicutes bacterium isolate HOT.MB2.13, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAX000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAX000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAX000000000.1|JAIMAX010000000</GBSeqid>\\n      <GBSeqid>gi|2240894480</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Firmicutes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Firmicutes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..456</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..456</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Firmicutes bacterium whole genome shotgun (WGS) project has the project accession JAIMAX000000000. This version of the project (01) has the accession number JAIMAX010000000, and consists of sequences JAIMAX010000001-JAIMAX010000456.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 31.6x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 12:11:10 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,077 ; CDSs (total) :: 2,045 ; Genes (coding) :: 2,003 ; CDSs (with protein) :: 2,003 ; Genes (RNA) :: 32 ; tRNAs :: 29 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 42 ; CDSs (without protein) :: 42 ; Pseudo Genes (ambiguous residues) :: 0 of 42 ; Pseudo Genes (frameshifted) :: 8 of 42 ; Pseudo Genes (incomplete) :: 26 of 42 ; Pseudo Genes (internal stop) :: 8 of 42 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..456</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>456</GBInterval_to>\\n            <GBInterval_accession>JAIMAX000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Firmicutes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.13</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1879010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAX010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAX010000456</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023747</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Firmicutes bacterium isolate HOT.MB2.13, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes']\n",
      "Id 2240894479\n",
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBT010000000</GBSeq_locus>\\n    <GBSeq_length>42</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Alicyclobacillus sp. isolate HOT.MB2.91, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBT000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBT000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBT000000000.1|JAIMBT010000000</GBSeqid>\\n      <GBSeqid>gi|2240894479</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Alicyclobacillus sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Alicyclobacillus sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..42</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..42</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Alicyclobacillus sp. whole genome shotgun (WGS) project has the project accession JAIMBT000000000. This version of the project (01) has the accession number JAIMBT010000000, and consists of sequences JAIMBT010000001-JAIMBT010000042.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 17x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:42:45 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,270 ; CDSs (total) :: 3,216 ; Genes (coding) :: 3,183 ; CDSs (with protein) :: 3,183 ; Genes (RNA) :: 54 ; rRNAs :: 1, 2, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 1 (16S, 23S) ; tRNAs :: 46 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 33 ; CDSs (without protein) :: 33 ; Pseudo Genes (ambiguous residues) :: 0 of 33 ; Pseudo Genes (frameshifted) :: 8 of 33 ; Pseudo Genes (incomplete) :: 25 of 33 ; Pseudo Genes (internal stop) :: 2 of 33 ; Pseudo Genes (multiple problems) :: 2 of 33 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..42</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>42</GBInterval_to>\\n            <GBInterval_accession>JAIMBT000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Alicyclobacillus sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.91</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:61169</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBT010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBT010000042</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023769</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Alicyclobacillus sp. isolate HOT.MB2.91, whole genome shotgun sequencing project', 'Alicyclobacillus sp. (hot springs metagenome)', 'Alicyclobacillus sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus']\n",
      "Id 2240894478\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBN010000000</GBSeq_locus>\\n    <GBSeq_length>196</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: candidate division WOR-3 bacterium isolate HOT.MB2.69, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBN000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBN000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBN000000000.1|JAIMBN010000000</GBSeqid>\\n      <GBSeqid>gi|2240894478</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>candidate division WOR-3 bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>candidate division WOR-3 bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; candidate division WOR-3</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..196</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..196</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The candidate division WOR-3 bacterium whole genome shotgun (WGS) project has the project accession JAIMBN000000000. This version of the project (01) has the accession number JAIMBN010000000, and consists of sequences JAIMBN010000001-JAIMBN010000196.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 12x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:40:21 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 1,620 ; CDSs (total) :: 1,573 ; Genes (coding) :: 1,570 ; CDSs (with protein) :: 1,570 ; Genes (RNA) :: 47 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; tRNAs :: 41 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 3 ; CDSs (without protein) :: 3 ; Pseudo Genes (ambiguous residues) :: 0 of 3 ; Pseudo Genes (frameshifted) :: 2 of 3 ; Pseudo Genes (incomplete) :: 1 of 3 ; Pseudo Genes (internal stop) :: 0 of 3 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..196</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>196</GBInterval_to>\\n            <GBInterval_accession>JAIMBN000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>candidate division WOR-3 bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.69</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2052148</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBN010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBN010000196</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023763</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: candidate division WOR-3 bacterium isolate HOT.MB2.69, whole genome shotgun sequencing project', 'candidate division WOR-3 bacterium (hot springs metagenome)', 'candidate division WOR-3 bacterium', 'Bacteria; candidate division WOR-3']\n",
      "Id 2240894477\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JADRFK010000000</GBSeq_locus>\\n    <GBSeq_length>1761</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Ralstonia sp. isolate MAG-2, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JADRFK000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JADRFK000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JADRFK000000000.1|JADRFK010000000</GBSeqid>\\n      <GBSeqid>gi|2240894477</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA674475</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Ralstonia sp. (snow metagenome)</GBSeq_source>\\n    <GBSeq_organism>Ralstonia sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Proteobacteria; Betaproteobacteria; Burkholderiales; Burkholderiaceae; Ralstonia</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..1761</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Rahlff,J.</GBAuthor>\\n          <GBAuthor>Bornemann,T.L.V.</GBAuthor>\\n          <GBAuthor>Lopatina,A.</GBAuthor>\\n          <GBAuthor>Severinov,K.</GBAuthor>\\n          <GBAuthor>Probst,A.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Host-Associated Phages Disperse across the Extraterrestrial Analogue Antarctica</GBReference_title>\\n        <GBReference_journal>Appl Environ Microbiol, e0031522 (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1128/aem.00315-22</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35499326</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..1761</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Rahlff,J.</GBAuthor>\\n          <GBAuthor>Bornemann,T.L.V.</GBAuthor>\\n          <GBAuthor>Lopatina,A.</GBAuthor>\\n          <GBAuthor>Severinov,K.</GBAuthor>\\n          <GBAuthor>Probst,A.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (15-NOV-2020) Department of Chemistry, University Duisburg-Essen, Universitaetsstrasse 5, Essen 45147, Germany</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Ralstonia sp. whole genome shotgun (WGS) project has the project accession JADRFK000000000. This version of the project (01) has the accession number JADRFK010000000, and consists of sequences JADRFK010000001-JADRFK010001761.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: SPAdes v. 3.13 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 8.6x ; Sequencing Technology :: Illumina MiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 11/25/2020 14:26:07 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 4.13 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 4,616 ; CDSs (total) :: 4,586 ; Genes (coding) :: 4,493 ; CDSs (with protein) :: 4,493 ; Genes (RNA) :: 30 ; tRNAs :: 27 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 93 ; CDSs (without protein) :: 93 ; Pseudo Genes (ambiguous residues) :: 1 of 93 ; Pseudo Genes (frameshifted) :: 22 of 93 ; Pseudo Genes (incomplete) :: 50 of 93 ; Pseudo Genes (internal stop) :: 26 of 93 ; Pseudo Genes (multiple problems) :: 6 of 93 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..1761</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>1761</GBInterval_to>\\n            <GBInterval_accession>JADRFK000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Ralstonia sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>MAG-2</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>surface snow</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:54061</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Antarctica: Druzhnaja</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>69.44462 S 73.42109 E</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2009-12-17</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>snow metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JADRFK010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JADRFK010001761</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA674475</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN16789484</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Ralstonia sp. isolate MAG-2, whole genome shotgun sequencing project', 'Ralstonia sp. (snow metagenome)', 'Ralstonia sp.', 'Bacteria; Proteobacteria; Betaproteobacteria; Burkholderiales; Burkholderiaceae; Ralstonia']\n",
      "Id 2240894475\n",
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBD010000000</GBSeq_locus>\\n    <GBSeq_length>40</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Firmicutes bacterium isolate HOT.MB2.4, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBD000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBD000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBD000000000.1|JAIMBD010000000</GBSeqid>\\n      <GBSeqid>gi|2240894475</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Firmicutes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Firmicutes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..40</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..40</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Firmicutes bacterium whole genome shotgun (WGS) project has the project accession JAIMBD000000000. This version of the project (01) has the accession number JAIMBD010000000, and consists of sequences JAIMBD010000001-JAIMBD010000040.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 11.8x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 12:37:04 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,473 ; CDSs (total) :: 2,425 ; Genes (coding) :: 2,396 ; CDSs (with protein) :: 2,396 ; Genes (RNA) :: 48 ; rRNAs :: 1, 1 (16S, 23S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 42 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 29 ; CDSs (without protein) :: 29 ; Pseudo Genes (ambiguous residues) :: 0 of 29 ; Pseudo Genes (frameshifted) :: 17 of 29 ; Pseudo Genes (incomplete) :: 19 of 29 ; Pseudo Genes (internal stop) :: 5 of 29 ; Pseudo Genes (multiple problems) :: 9 of 29 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..40</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>40</GBInterval_to>\\n            <GBInterval_accession>JAIMBD000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Firmicutes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.4</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1879010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBD010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBD010000040</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023753</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Firmicutes bacterium isolate HOT.MB2.4, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes']\n",
      "Id 2240894467\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAW010000000</GBSeq_locus>\\n    <GBSeq_length>90</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Candidatus Pacearchaeota archaeon isolate HOT.MB2.108, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAW000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAW000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAW000000000.1|JAIMAW010000000</GBSeqid>\\n      <GBSeqid>gi|2240894467</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Candidatus Pacearchaeota archaeon (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Candidatus Pacearchaeota archaeon</GBSeq_organism>\\n    <GBSeq_taxonomy>Archaea; Candidatus Pacearchaeota</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..90</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..90</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Candidatus Pacearchaeota archaeon whole genome shotgun (WGS) project has the project accession JAIMAW000000000. This version of the project (01) has the accession number JAIMAW010000000, and consists of sequences JAIMAW010000001-JAIMAW010000090.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 13.6x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 09:35:31 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 662 ; CDSs (total) :: 629 ; Genes (coding) :: 622 ; CDSs (with protein) :: 622 ; Genes (RNA) :: 33 ; rRNAs :: 1 (5S) ; complete rRNAs :: 1 (5S) ; tRNAs :: 31 ; ncRNAs :: 1 ; Pseudo Genes (total) :: 7 ; CDSs (without protein) :: 7 ; Pseudo Genes (ambiguous residues) :: 0 of 7 ; Pseudo Genes (frameshifted) :: 0 of 7 ; Pseudo Genes (incomplete) :: 6 of 7 ; Pseudo Genes (internal stop) :: 1 of 7 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..90</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>90</GBInterval_to>\\n            <GBInterval_accession>JAIMAW000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Candidatus Pacearchaeota archaeon</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.108</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2026773</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAW010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAW010000090</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023746</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Candidatus Pacearchaeota archaeon isolate HOT.MB2.108, whole genome shotgun sequencing project', 'Candidatus Pacearchaeota archaeon (hot springs metagenome)', 'Candidatus Pacearchaeota archaeon', 'Archaea; Candidatus Pacearchaeota']\n",
      "Id 2240894466\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JADRFL010000000</GBSeq_locus>\\n    <GBSeq_length>165</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Ralstonia pickettii isolate MAG-3, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JADRFL000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JADRFL000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JADRFL000000000.1|JADRFL010000000</GBSeqid>\\n      <GBSeqid>gi|2240894466</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA674475</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Ralstonia pickettii (snow metagenome)</GBSeq_source>\\n    <GBSeq_organism>Ralstonia pickettii</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Proteobacteria; Betaproteobacteria; Burkholderiales; Burkholderiaceae; Ralstonia</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..165</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Rahlff,J.</GBAuthor>\\n          <GBAuthor>Bornemann,T.L.V.</GBAuthor>\\n          <GBAuthor>Lopatina,A.</GBAuthor>\\n          <GBAuthor>Severinov,K.</GBAuthor>\\n          <GBAuthor>Probst,A.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Host-Associated Phages Disperse across the Extraterrestrial Analogue Antarctica</GBReference_title>\\n        <GBReference_journal>Appl Environ Microbiol, e0031522 (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1128/aem.00315-22</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35499326</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..165</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Rahlff,J.</GBAuthor>\\n          <GBAuthor>Bornemann,T.L.V.</GBAuthor>\\n          <GBAuthor>Lopatina,A.</GBAuthor>\\n          <GBAuthor>Severinov,K.</GBAuthor>\\n          <GBAuthor>Probst,A.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (15-NOV-2020) Department of Chemistry, University Duisburg-Essen, Universitaetsstrasse 5, Essen 45147, Germany</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Ralstonia pickettii whole genome shotgun (WGS) project has the project accession JADRFL000000000. This version of the project (01) has the accession number JADRFL010000000, and consists of sequences JADRFL010000001-JADRFL010000165.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: SPAdes v. 3.13 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 8.7x ; Sequencing Technology :: Illumina MiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 11/25/2020 12:24:30 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 4.13 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 1,230 ; CDSs (total) :: 1,225 ; Genes (coding) :: 1,189 ; CDSs (with protein) :: 1,189 ; Genes (RNA) :: 5 ; tRNAs :: 5 ; ncRNAs :: 0 ; Pseudo Genes (total) :: 36 ; CDSs (without protein) :: 36 ; Pseudo Genes (ambiguous residues) :: 0 of 36 ; Pseudo Genes (frameshifted) :: 7 of 36 ; Pseudo Genes (incomplete) :: 19 of 36 ; Pseudo Genes (internal stop) :: 12 of 36 ; Pseudo Genes (multiple problems) :: 2 of 36 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..165</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>165</GBInterval_to>\\n            <GBInterval_accession>JADRFL000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Ralstonia pickettii</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>MAG-3</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>surface snow</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:329</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Antarctica: Mirnii</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>66.33245 S 93.00296 E</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2009-12-18</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>snow metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JADRFL010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JADRFL010000165</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA674475</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN16789485</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Ralstonia pickettii isolate MAG-3, whole genome shotgun sequencing project', 'Ralstonia pickettii (snow metagenome)', 'Ralstonia pickettii', 'Bacteria; Proteobacteria; Betaproteobacteria; Burkholderiales; Burkholderiaceae; Ralstonia']\n",
      "Id 2240894465\n",
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMAZ010000000</GBSeq_locus>\\n    <GBSeq_length>432</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Ignavibacterium sp. isolate HOT.MB2.1, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMAZ000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMAZ000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMAZ000000000.1|JAIMAZ010000000</GBSeqid>\\n      <GBSeqid>gi|2240894465</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Ignavibacterium sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Ignavibacterium sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Ignavibacteriae; Ignavibacteria; Ignavibacteriales; Ignavibacteriaceae; Ignavibacterium; unclassified Ignavibacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..432</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..432</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Ignavibacterium sp. whole genome shotgun (WGS) project has the project accession JAIMAZ000000000. This version of the project (01) has the accession number JAIMAZ010000000, and consists of sequences JAIMAZ010000001-JAIMAZ010000432.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 38.6x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:12:11 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,940 ; CDSs (total) :: 2,900 ; Genes (coding) :: 2,873 ; CDSs (with protein) :: 2,873 ; Genes (RNA) :: 40 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 34 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 27 ; CDSs (without protein) :: 27 ; Pseudo Genes (ambiguous residues) :: 0 of 27 ; Pseudo Genes (frameshifted) :: 1 of 27 ; Pseudo Genes (incomplete) :: 20 of 27 ; Pseudo Genes (internal stop) :: 6 of 27 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..432</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>432</GBInterval_to>\\n            <GBInterval_accession>JAIMAZ000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Ignavibacterium sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2651167</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMAZ010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMAZ010000432</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023749</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Ignavibacterium sp. isolate HOT.MB2.1, whole genome shotgun sequencing project', 'Ignavibacterium sp. (hot springs metagenome)', 'Ignavibacterium sp.', 'Bacteria; Ignavibacteriae; Ignavibacteria; Ignavibacteriales; Ignavibacteriaceae; Ignavibacterium; unclassified Ignavibacterium']\n",
      "Id 2240894464\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBF010000000</GBSeq_locus>\\n    <GBSeq_length>201</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Acidobacteriia bacterium isolate HOT.MB2.42, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBF000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBF000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBF000000000.1|JAIMBF010000000</GBSeqid>\\n      <GBSeqid>gi|2240894464</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Acidobacteriia bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Acidobacteriia bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Acidobacteria</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..201</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..201</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Acidobacteriia bacterium whole genome shotgun (WGS) project has the project accession JAIMBF000000000. This version of the project (01) has the accession number JAIMBF010000000, and consists of sequences JAIMBF010000001-JAIMBF010000201.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 13.4x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 12:11:22 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,024 ; CDSs (total) :: 2,969 ; Genes (coding) :: 2,951 ; CDSs (with protein) :: 2,951 ; Genes (RNA) :: 55 ; rRNAs :: 2, 2, 1 (5S, 16S, 23S) ; complete rRNAs :: 2, 1 (5S, 23S) ; partial rRNAs :: 2 (16S) ; tRNAs :: 47 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 18 ; CDSs (without protein) :: 18 ; Pseudo Genes (ambiguous residues) :: 0 of 18 ; Pseudo Genes (frameshifted) :: 5 of 18 ; Pseudo Genes (incomplete) :: 16 of 18 ; Pseudo Genes (internal stop) :: 0 of 18 ; Pseudo Genes (multiple problems) :: 3 of 18 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..201</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>201</GBInterval_to>\\n            <GBInterval_accession>JAIMBF000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Acidobacteriia bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.42</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2081523</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBF010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBF010000201</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023755</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Acidobacteriia bacterium isolate HOT.MB2.42, whole genome shotgun sequencing project', 'Acidobacteriia bacterium (hot springs metagenome)', 'Acidobacteriia bacterium', 'Bacteria; Acidobacteria']\n",
      "Id 2240894463\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBL010000000</GBSeq_locus>\\n    <GBSeq_length>383</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Leptolyngbyaceae cyanobacterium HOT.MB2.61, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBL000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBL000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBL000000000.1|JAIMBL010000000</GBSeqid>\\n      <GBSeqid>gi|2240894463</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Leptolyngbyaceae cyanobacterium HOT.MB2.61 (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Leptolyngbyaceae cyanobacterium HOT.MB2.61</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Cyanobacteria; Pseudanabaenales; Leptolyngbyaceae; unclassified Leptolyngbyaceae</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..383</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..383</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Leptolyngbyaceae cyanobacterium HOT.MB2.61 whole genome shotgun (WGS) project has the project accession JAIMBL000000000. This version of the project (01) has the accession number JAIMBL010000000, and consists of sequences JAIMBL010000001-JAIMBL010000383.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 16.3x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 11:07:14 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 4,543 ; CDSs (total) :: 4,496 ; Genes (coding) :: 4,370 ; CDSs (with protein) :: 4,370 ; Genes (RNA) :: 47 ; rRNAs :: 1, 1 (16S, 23S) ; complete rRNAs :: 1 (16S) ; partial rRNAs :: 1 (23S) ; tRNAs :: 41 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 126 ; CDSs (without protein) :: 126 ; Pseudo Genes (ambiguous residues) :: 0 of 126 ; Pseudo Genes (frameshifted) :: 52 of 126 ; Pseudo Genes (incomplete) :: 64 of 126 ; Pseudo Genes (internal stop) :: 34 of 126 ; Pseudo Genes (multiple problems) :: 21 of 126 ; CRISPR Arrays :: 3 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..383</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>383</GBInterval_to>\\n            <GBInterval_accession>JAIMBL000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Leptolyngbyaceae cyanobacterium HOT.MB2.61</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.61</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2855467</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBL010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBL010000383</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023761</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Leptolyngbyaceae cyanobacterium HOT.MB2.61, whole genome shotgun sequencing project', 'Leptolyngbyaceae cyanobacterium HOT.MB2.61 (hot springs metagenome)', 'Leptolyngbyaceae cyanobacterium HOT.MB2.61', 'Bacteria; Cyanobacteria; Pseudanabaenales; Leptolyngbyaceae; unclassified Leptolyngbyaceae']\n",
      "Id 2240894462\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBJ010000000</GBSeq_locus>\\n    <GBSeq_length>73</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Firmicutes bacterium isolate HOT.MB2.53, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBJ000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBJ000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBJ000000000.1|JAIMBJ010000000</GBSeqid>\\n      <GBSeqid>gi|2240894462</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Firmicutes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Firmicutes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..73</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..73</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Firmicutes bacterium whole genome shotgun (WGS) project has the project accession JAIMBJ000000000. This version of the project (01) has the accession number JAIMBJ010000000, and consists of sequences JAIMBJ010000001-JAIMBJ010000073.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 17x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 12:48:30 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,449 ; CDSs (total) :: 3,398 ; Genes (coding) :: 3,358 ; CDSs (with protein) :: 3,358 ; Genes (RNA) :: 51 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 45 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 40 ; CDSs (without protein) :: 40 ; Pseudo Genes (ambiguous residues) :: 0 of 40 ; Pseudo Genes (frameshifted) :: 8 of 40 ; Pseudo Genes (incomplete) :: 25 of 40 ; Pseudo Genes (internal stop) :: 7 of 40 ; CRISPR Arrays :: 2 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..73</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>73</GBInterval_to>\\n            <GBInterval_accession>JAIMBJ000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Firmicutes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.53</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1879010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBJ010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBJ010000073</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023759</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Firmicutes bacterium isolate HOT.MB2.53, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes']\n",
      "Id 2240894461\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBG010000000</GBSeq_locus>\\n    <GBSeq_length>243</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Acetobacteraceae bacterium isolate HOT.MB2.46, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBG000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBG000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBG000000000.1|JAIMBG010000000</GBSeqid>\\n      <GBSeqid>gi|2240894461</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Acetobacteraceae bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Acetobacteraceae bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Proteobacteria; Alphaproteobacteria; Rhodospirillales; Acetobacteraceae; unclassified Acetobacteraceae</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..243</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..243</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Acetobacteraceae bacterium whole genome shotgun (WGS) project has the project accession JAIMBG000000000. This version of the project (01) has the accession number JAIMBG010000000, and consists of sequences JAIMBG010000001-JAIMBG010000243.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 39.7x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:36:40 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,382 ; CDSs (total) :: 2,343 ; Genes (coding) :: 2,324 ; CDSs (with protein) :: 2,324 ; Genes (RNA) :: 39 ; rRNAs :: 1 (16S) ; partial rRNAs :: 1 (16S) ; tRNAs :: 34 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 19 ; CDSs (without protein) :: 19 ; Pseudo Genes (ambiguous residues) :: 0 of 19 ; Pseudo Genes (frameshifted) :: 1 of 19 ; Pseudo Genes (incomplete) :: 17 of 19 ; Pseudo Genes (internal stop) :: 1 of 19 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..243</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>243</GBInterval_to>\\n            <GBInterval_accession>JAIMBG000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Acetobacteraceae bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.46</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1909293</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBG010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBG010000243</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023756</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Acetobacteraceae bacterium isolate HOT.MB2.46, whole genome shotgun sequencing project', 'Acetobacteraceae bacterium (hot springs metagenome)', 'Acetobacteraceae bacterium', 'Bacteria; Proteobacteria; Alphaproteobacteria; Rhodospirillales; Acetobacteraceae; unclassified Acetobacteraceae']\n",
      "Id 2240894460\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBM010000000</GBSeq_locus>\\n    <GBSeq_length>757</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Anaerolineae bacterium isolate HOT.MB2.62, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBM000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBM000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBM000000000.1|JAIMBM010000000</GBSeqid>\\n      <GBSeqid>gi|2240894460</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Anaerolineae bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Anaerolineae bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Chloroflexi; Anaerolineae; unclassified Anaerolineae</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..757</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..757</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Anaerolineae bacterium whole genome shotgun (WGS) project has the project accession JAIMBM000000000. This version of the project (01) has the accession number JAIMBM010000000, and consists of sequences JAIMBM010000001-JAIMBM010000757.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 13x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 14:09:41 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,461 ; CDSs (total) :: 3,415 ; Genes (coding) :: 3,369 ; CDSs (with protein) :: 3,369 ; Genes (RNA) :: 46 ; tRNAs :: 42 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 46 ; CDSs (without protein) :: 46 ; Pseudo Genes (ambiguous residues) :: 0 of 46 ; Pseudo Genes (frameshifted) :: 7 of 46 ; Pseudo Genes (incomplete) :: 38 of 46 ; Pseudo Genes (internal stop) :: 3 of 46 ; Pseudo Genes (multiple problems) :: 2 of 46 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..757</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>757</GBInterval_to>\\n            <GBInterval_accession>JAIMBM000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Anaerolineae bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.62</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2052143</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBM010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBM010000757</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023762</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Anaerolineae bacterium isolate HOT.MB2.62, whole genome shotgun sequencing project', 'Anaerolineae bacterium (hot springs metagenome)', 'Anaerolineae bacterium', 'Bacteria; Chloroflexi; Anaerolineae; unclassified Anaerolineae']\n",
      "Id 2240894459\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBK010000000</GBSeq_locus>\\n    <GBSeq_length>136</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Armatimonadetes bacterium isolate HOT.MB2.58, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBK000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBK000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBK000000000.1|JAIMBK010000000</GBSeqid>\\n      <GBSeqid>gi|2240894459</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Armatimonadetes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Armatimonadetes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Armatimonadetes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..136</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..136</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Armatimonadetes bacterium whole genome shotgun (WGS) project has the project accession JAIMBK000000000. This version of the project (01) has the accession number JAIMBK010000000, and consists of sequences JAIMBK010000001-JAIMBK010000136.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 18.1x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 11:59:15 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,262 ; CDSs (total) :: 3,213 ; Genes (coding) :: 3,135 ; CDSs (with protein) :: 3,135 ; Genes (RNA) :: 49 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 42 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 78 ; CDSs (without protein) :: 78 ; Pseudo Genes (ambiguous residues) :: 0 of 78 ; Pseudo Genes (frameshifted) :: 28 of 78 ; Pseudo Genes (incomplete) :: 52 of 78 ; Pseudo Genes (internal stop) :: 5 of 78 ; Pseudo Genes (multiple problems) :: 6 of 78 ; CRISPR Arrays :: 3 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..136</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>136</GBInterval_to>\\n            <GBInterval_accession>JAIMBK000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Armatimonadetes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.58</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2033014</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBK010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBK010000136</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023760</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Armatimonadetes bacterium isolate HOT.MB2.58, whole genome shotgun sequencing project', 'Armatimonadetes bacterium (hot springs metagenome)', 'Armatimonadetes bacterium', 'Bacteria; Armatimonadetes']\n",
      "Id 2240894458\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBH010000000</GBSeq_locus>\\n    <GBSeq_length>58</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Firmicutes bacterium isolate HOT.MB2.5, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBH000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBH000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBH000000000.1|JAIMBH010000000</GBSeqid>\\n      <GBSeqid>gi|2240894458</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Firmicutes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Firmicutes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..58</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..58</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Firmicutes bacterium whole genome shotgun (WGS) project has the project accession JAIMBH000000000. This version of the project (01) has the accession number JAIMBH010000000, and consists of sequences JAIMBH010000001-JAIMBH010000058.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 24.3x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 11:14:31 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,311 ; CDSs (total) :: 2,257 ; Genes (coding) :: 2,238 ; CDSs (with protein) :: 2,238 ; Genes (RNA) :: 54 ; rRNAs :: 1, 1, 2 (5S, 16S, 23S) ; complete rRNAs :: 1, 1 (5S, 16S) ; partial rRNAs :: 2 (23S) ; tRNAs :: 46 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 19 ; CDSs (without protein) :: 19 ; Pseudo Genes (ambiguous residues) :: 0 of 19 ; Pseudo Genes (frameshifted) :: 5 of 19 ; Pseudo Genes (incomplete) :: 11 of 19 ; Pseudo Genes (internal stop) :: 4 of 19 ; Pseudo Genes (multiple problems) :: 1 of 19 ; CRISPR Arrays :: 2 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..58</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>58</GBInterval_to>\\n            <GBInterval_accession>JAIMBH000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Firmicutes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.5</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1879010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBH010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBH010000058</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023757</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Firmicutes bacterium isolate HOT.MB2.5, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes']\n",
      "Id 2240894457\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBO010000000</GBSeq_locus>\\n    <GBSeq_length>218</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Rubrobacteraceae bacterium isolate HOT.MB2.74, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBO000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBO000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBO000000000.1|JAIMBO010000000</GBSeqid>\\n      <GBSeqid>gi|2240894457</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Rubrobacteraceae bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Rubrobacteraceae bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Actinobacteria; Rubrobacteria; Rubrobacterales; Rubrobacteraceae; unclassified Rubrobacteraceae</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..218</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..218</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Rubrobacteraceae bacterium whole genome shotgun (WGS) project has the project accession JAIMBO000000000. This version of the project (01) has the accession number JAIMBO010000000, and consists of sequences JAIMBO010000001-JAIMBO010000218.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 13x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:07:44 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,950 ; CDSs (total) :: 2,904 ; Genes (coding) :: 2,897 ; CDSs (with protein) :: 2,897 ; Genes (RNA) :: 46 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 40 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 7 ; CDSs (without protein) :: 7 ; Pseudo Genes (ambiguous residues) :: 0 of 7 ; Pseudo Genes (frameshifted) :: 3 of 7 ; Pseudo Genes (incomplete) :: 4 of 7 ; Pseudo Genes (internal stop) :: 2 of 7 ; Pseudo Genes (multiple problems) :: 2 of 7 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..218</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>218</GBInterval_to>\\n            <GBInterval_accession>JAIMBO000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Rubrobacteraceae bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.74</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2740537</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBO010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBO010000218</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023764</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Rubrobacteraceae bacterium isolate HOT.MB2.74, whole genome shotgun sequencing project', 'Rubrobacteraceae bacterium (hot springs metagenome)', 'Rubrobacteraceae bacterium', 'Bacteria; Actinobacteria; Rubrobacteria; Rubrobacterales; Rubrobacteraceae; unclassified Rubrobacteraceae']\n",
      "Id 2240894456\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBP010000000</GBSeq_locus>\\n    <GBSeq_length>239</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Acidobacterium ailaaui isolate HOT.MB2.78, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBP000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBP000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBP000000000.1|JAIMBP010000000</GBSeqid>\\n      <GBSeqid>gi|2240894456</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Acidobacterium ailaaui (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Acidobacterium ailaaui</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Acidobacteria; Acidobacteriales; Acidobacteriaceae; Acidobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..239</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..239</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Acidobacterium ailaaui whole genome shotgun (WGS) project has the project accession JAIMBP000000000. This version of the project (01) has the accession number JAIMBP010000000, and consists of sequences JAIMBP010000001-JAIMBP010000239.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 22.2x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:03:31 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,283 ; CDSs (total) :: 2,249 ; Genes (coding) :: 2,239 ; CDSs (with protein) :: 2,239 ; Genes (RNA) :: 34 ; rRNAs :: 1 (16S) ; partial rRNAs :: 1 (16S) ; tRNAs :: 29 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 10 ; CDSs (without protein) :: 10 ; Pseudo Genes (ambiguous residues) :: 0 of 10 ; Pseudo Genes (frameshifted) :: 4 of 10 ; Pseudo Genes (incomplete) :: 5 of 10 ; Pseudo Genes (internal stop) :: 3 of 10 ; Pseudo Genes (multiple problems) :: 2 of 10 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..239</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>239</GBInterval_to>\\n            <GBInterval_accession>JAIMBP000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Acidobacterium ailaaui</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.78</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1382359</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBP010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBP010000239</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023765</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Acidobacterium ailaaui isolate HOT.MB2.78, whole genome shotgun sequencing project', 'Acidobacterium ailaaui (hot springs metagenome)', 'Acidobacterium ailaaui', 'Bacteria; Acidobacteria; Acidobacteriales; Acidobacteriaceae; Acidobacterium']\n",
      "Id 2240894455\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBU010000000</GBSeq_locus>\\n    <GBSeq_length>914</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Gorillibacterium sp. isolate HOT.MB2.94, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBU000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBU000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBU000000000.1|JAIMBU010000000</GBSeqid>\\n      <GBSeqid>gi|2240894455</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Gorillibacterium sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Gorillibacterium sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Paenibacillaceae; Gorillibacterium; unclassified Gorillibacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..914</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..914</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Gorillibacterium sp. whole genome shotgun (WGS) project has the project accession JAIMBU000000000. This version of the project (01) has the accession number JAIMBU010000000, and consists of sequences JAIMBU010000001-JAIMBU010000914.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 21.6x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:48:59 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,963 ; CDSs (total) :: 3,945 ; Genes (coding) :: 3,880 ; CDSs (with protein) :: 3,880 ; Genes (RNA) :: 18 ; rRNAs :: 2, 1 (16S, 23S) ; partial rRNAs :: 2, 1 (16S, 23S) ; tRNAs :: 13 ; ncRNAs :: 2 ; Pseudo Genes (total) :: 65 ; CDSs (without protein) :: 65 ; Pseudo Genes (ambiguous residues) :: 0 of 65 ; Pseudo Genes (frameshifted) :: 15 of 65 ; Pseudo Genes (incomplete) :: 34 of 65 ; Pseudo Genes (internal stop) :: 20 of 65 ; Pseudo Genes (multiple problems) :: 4 of 65 ; CRISPR Arrays :: 7 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..914</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>914</GBInterval_to>\\n            <GBInterval_accession>JAIMBU000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Gorillibacterium sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.94</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1978774</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBU010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBU010000914</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023770</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Gorillibacterium sp. isolate HOT.MB2.94, whole genome shotgun sequencing project', 'Gorillibacterium sp. (hot springs metagenome)', 'Gorillibacterium sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Paenibacillaceae; Gorillibacterium; unclassified Gorillibacterium']\n",
      "Id 2240894454\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JADRFJ010000000</GBSeq_locus>\\n    <GBSeq_length>1788</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Janthinobacterium lividum isolate MAG-1, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JADRFJ000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JADRFJ000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JADRFJ000000000.1|JADRFJ010000000</GBSeqid>\\n      <GBSeqid>gi|2240894454</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA674475</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Janthinobacterium lividum (snow metagenome)</GBSeq_source>\\n    <GBSeq_organism>Janthinobacterium lividum</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Proteobacteria; Betaproteobacteria; Burkholderiales; Oxalobacteraceae; Janthinobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..1788</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Rahlff,J.</GBAuthor>\\n          <GBAuthor>Bornemann,T.L.V.</GBAuthor>\\n          <GBAuthor>Lopatina,A.</GBAuthor>\\n          <GBAuthor>Severinov,K.</GBAuthor>\\n          <GBAuthor>Probst,A.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Host-Associated Phages Disperse across the Extraterrestrial Analogue Antarctica</GBReference_title>\\n        <GBReference_journal>Appl Environ Microbiol, e0031522 (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1128/aem.00315-22</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35499326</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..1788</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Rahlff,J.</GBAuthor>\\n          <GBAuthor>Bornemann,T.L.V.</GBAuthor>\\n          <GBAuthor>Lopatina,A.</GBAuthor>\\n          <GBAuthor>Severinov,K.</GBAuthor>\\n          <GBAuthor>Probst,A.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (15-NOV-2020) Department of Chemistry, University Duisburg-Essen, Universitaetsstrasse 5, Essen 45147, Germany</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Janthinobacterium lividum whole genome shotgun (WGS) project has the project accession JADRFJ000000000. This version of the project (01) has the accession number JADRFJ010000000, and consists of sequences JADRFJ010000001-JADRFJ010001788.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: SPAdes v. 3.13 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 7.5x ; Sequencing Technology :: Illumina MiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 11/25/2020 14:27:59 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 4.13 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 4,920 ; CDSs (total) :: 4,873 ; Genes (coding) :: 4,762 ; CDSs (with protein) :: 4,762 ; Genes (RNA) :: 47 ; tRNAs :: 43 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 111 ; CDSs (without protein) :: 111 ; Pseudo Genes (ambiguous residues) :: 0 of 111 ; Pseudo Genes (frameshifted) :: 22 of 111 ; Pseudo Genes (incomplete) :: 61 of 111 ; Pseudo Genes (internal stop) :: 37 of 111 ; Pseudo Genes (multiple problems) :: 9 of 111 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..1788</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>1788</GBInterval_to>\\n            <GBInterval_accession>JADRFJ000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Janthinobacterium lividum</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>MAG-1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>surface snow</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:29581</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Antarctica: Druzhnaja</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>69.44462 S 73.42109 E</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2009-12-17</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>snow metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JADRFJ010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JADRFJ010001788</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA674475</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN16789483</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Janthinobacterium lividum isolate MAG-1, whole genome shotgun sequencing project', 'Janthinobacterium lividum (snow metagenome)', 'Janthinobacterium lividum', 'Bacteria; Proteobacteria; Betaproteobacteria; Burkholderiales; Oxalobacteraceae; Janthinobacterium']\n",
      "Id 2240894453\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBS010000000</GBSeq_locus>\\n    <GBSeq_length>627</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Alicyclobacillus sp. isolate HOT.MB2.88, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBS000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBS000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBS000000000.1|JAIMBS010000000</GBSeqid>\\n      <GBSeqid>gi|2240894453</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Alicyclobacillus sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Alicyclobacillus sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..627</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..627</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Alicyclobacillus sp. whole genome shotgun (WGS) project has the project accession JAIMBS000000000. This version of the project (01) has the accession number JAIMBS010000000, and consists of sequences JAIMBS010000001-JAIMBS010000627.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 29.1x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:47:25 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 4,860 ; CDSs (total) :: 4,820 ; Genes (coding) :: 4,690 ; CDSs (with protein) :: 4,690 ; Genes (RNA) :: 40 ; rRNAs :: 1, 1, 2 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 2 (16S, 23S) ; tRNAs :: 33 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 130 ; CDSs (without protein) :: 130 ; Pseudo Genes (ambiguous residues) :: 0 of 130 ; Pseudo Genes (frameshifted) :: 43 of 130 ; Pseudo Genes (incomplete) :: 77 of 130 ; Pseudo Genes (internal stop) :: 27 of 130 ; Pseudo Genes (multiple problems) :: 16 of 130 ; CRISPR Arrays :: 2 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..627</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>627</GBInterval_to>\\n            <GBInterval_accession>JAIMBS000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Alicyclobacillus sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.88</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:61169</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBS010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBS010000627</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023768</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Alicyclobacillus sp. isolate HOT.MB2.88, whole genome shotgun sequencing project', 'Alicyclobacillus sp. (hot springs metagenome)', 'Alicyclobacillus sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus']\n",
      "Id 2240894452\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBR010000000</GBSeq_locus>\\n    <GBSeq_length>254</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Peptococcaceae bacterium isolate HOT.MB2.86, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBR000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBR000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBR000000000.1|JAIMBR010000000</GBSeqid>\\n      <GBSeqid>gi|2240894452</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Peptococcaceae bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Peptococcaceae bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Clostridia; Eubacteriales; Peptococcaceae</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..254</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..254</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Peptococcaceae bacterium whole genome shotgun (WGS) project has the project accession JAIMBR000000000. This version of the project (01) has the accession number JAIMBR010000000, and consists of sequences JAIMBR010000001-JAIMBR010000254.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 11x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:25:57 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 3,419 ; CDSs (total) :: 3,376 ; Genes (coding) :: 3,306 ; CDSs (with protein) :: 3,306 ; Genes (RNA) :: 43 ; rRNAs :: 2, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; tRNAs :: 36 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 70 ; CDSs (without protein) :: 70 ; Pseudo Genes (ambiguous residues) :: 0 of 70 ; Pseudo Genes (frameshifted) :: 27 of 70 ; Pseudo Genes (incomplete) :: 36 of 70 ; Pseudo Genes (internal stop) :: 12 of 70 ; Pseudo Genes (multiple problems) :: 5 of 70 ; CRISPR Arrays :: 2 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..254</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>254</GBInterval_to>\\n            <GBInterval_accession>JAIMBR000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Peptococcaceae bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.86</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2052179</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBR010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBR010000254</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023767</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Peptococcaceae bacterium isolate HOT.MB2.86, whole genome shotgun sequencing project', 'Peptococcaceae bacterium (hot springs metagenome)', 'Peptococcaceae bacterium', 'Bacteria; Firmicutes; Clostridia; Eubacteriales; Peptococcaceae']\n",
      "Id 2240894451\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBQ010000000</GBSeq_locus>\\n    <GBSeq_length>228</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Thermoleophilum sp. isolate HOT.MB2.84, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBQ000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBQ000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBQ000000000.1|JAIMBQ010000000</GBSeqid>\\n      <GBSeqid>gi|2240894451</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Thermoleophilum sp. (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Thermoleophilum sp.</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Actinobacteria; Thermoleophilia; Thermoleophilales; Thermoleophilaceae; Thermoleophilum; unclassified Thermoleophilum</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..228</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..228</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Thermoleophilum sp. whole genome shotgun (WGS) project has the project accession JAIMBQ000000000. This version of the project (01) has the accession number JAIMBQ010000000, and consists of sequences JAIMBQ010000001-JAIMBQ010000228.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 54.2x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 09:48:08 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,034 ; CDSs (total) :: 1,987 ; Genes (coding) :: 1,982 ; CDSs (with protein) :: 1,982 ; Genes (RNA) :: 47 ; rRNAs :: 1, 1, 1 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 1, 1 (16S, 23S) ; tRNAs :: 41 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 5 ; CDSs (without protein) :: 5 ; Pseudo Genes (ambiguous residues) :: 0 of 5 ; Pseudo Genes (frameshifted) :: 0 of 5 ; Pseudo Genes (incomplete) :: 5 of 5 ; Pseudo Genes (internal stop) :: 0 of 5 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..228</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>228</GBInterval_to>\\n            <GBInterval_accession>JAIMBQ000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Thermoleophilum sp.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.84</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:2480089</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBQ010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBQ010000228</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023766</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Thermoleophilum sp. isolate HOT.MB2.84, whole genome shotgun sequencing project', 'Thermoleophilum sp. (hot springs metagenome)', 'Thermoleophilum sp.', 'Bacteria; Actinobacteria; Thermoleophilia; Thermoleophilales; Thermoleophilaceae; Thermoleophilum; unclassified Thermoleophilum']\n",
      "Id 2240894450\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBE010000000</GBSeq_locus>\\n    <GBSeq_length>235</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Firmicutes bacterium isolate HOT.MB2.40, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBE000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBE000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBE000000000.1|JAIMBE010000000</GBSeqid>\\n      <GBSeqid>gi|2240894450</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Firmicutes bacterium (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Firmicutes bacterium</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..235</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..235</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Firmicutes bacterium whole genome shotgun (WGS) project has the project accession JAIMBE000000000. This version of the project (01) has the accession number JAIMBE010000000, and consists of sequences JAIMBE010000001-JAIMBE010000235.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 13.7x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 11:53:26 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,522 ; CDSs (total) :: 2,481 ; Genes (coding) :: 2,468 ; CDSs (with protein) :: 2,468 ; Genes (RNA) :: 41 ; tRNAs :: 37 ; ncRNAs :: 4 ; Pseudo Genes (total) :: 13 ; CDSs (without protein) :: 13 ; Pseudo Genes (ambiguous residues) :: 0 of 13 ; Pseudo Genes (frameshifted) :: 3 of 13 ; Pseudo Genes (incomplete) :: 8 of 13 ; Pseudo Genes (internal stop) :: 2 of 13 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..235</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>235</GBInterval_to>\\n            <GBInterval_accession>JAIMBE000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Firmicutes bacterium</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.40</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1879010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBE010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBE010000235</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023754</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Firmicutes bacterium isolate HOT.MB2.40, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes']\n",
      "Id 2240894449\n",
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JADRFN010000000</GBSeq_locus>\\n    <GBSeq_length>147</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Flavobacterium micromati isolate MAG-5, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JADRFN000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JADRFN000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JADRFN000000000.1|JADRFN010000000</GBSeqid>\\n      <GBSeqid>gi|2240894449</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA674475</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Flavobacterium micromati (snow metagenome)</GBSeq_source>\\n    <GBSeq_organism>Flavobacterium micromati</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Bacteroidetes; Flavobacteriia; Flavobacteriales; Flavobacteriaceae; Flavobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..147</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Rahlff,J.</GBAuthor>\\n          <GBAuthor>Bornemann,T.L.V.</GBAuthor>\\n          <GBAuthor>Lopatina,A.</GBAuthor>\\n          <GBAuthor>Severinov,K.</GBAuthor>\\n          <GBAuthor>Probst,A.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Host-Associated Phages Disperse across the Extraterrestrial Analogue Antarctica</GBReference_title>\\n        <GBReference_journal>Appl Environ Microbiol, e0031522 (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1128/aem.00315-22</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35499326</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..147</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Rahlff,J.</GBAuthor>\\n          <GBAuthor>Bornemann,T.L.V.</GBAuthor>\\n          <GBAuthor>Lopatina,A.</GBAuthor>\\n          <GBAuthor>Severinov,K.</GBAuthor>\\n          <GBAuthor>Probst,A.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (15-NOV-2020) Department of Chemistry, University Duisburg-Essen, Universitaetsstrasse 5, Essen 45147, Germany</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Flavobacterium micromati whole genome shotgun (WGS) project has the project accession JADRFN000000000. This version of the project (01) has the accession number JADRFN010000000, and consists of sequences JADRFN010000001-JADRFN010000147.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: SPAdes v. 3.13 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 19.2x ; Sequencing Technology :: Illumina MiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 11/25/2020 13:13:31 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 4.13 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,713 ; CDSs (total) :: 2,685 ; Genes (coding) :: 2,638 ; CDSs (with protein) :: 2,638 ; Genes (RNA) :: 28 ; tRNAs :: 25 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 47 ; CDSs (without protein) :: 47 ; Pseudo Genes (ambiguous residues) :: 6 of 47 ; Pseudo Genes (frameshifted) :: 17 of 47 ; Pseudo Genes (incomplete) :: 7 of 47 ; Pseudo Genes (internal stop) :: 25 of 47 ; Pseudo Genes (multiple problems) :: 8 of 47 ; CRISPR Arrays :: 1 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..147</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>147</GBInterval_to>\\n            <GBInterval_accession>JADRFN000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Flavobacterium micromati</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>MAG-5</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>surface snow</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:229205</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Antarctica: Progress</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>69.40059 S 76.37939 E</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2009-12-28</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>snow metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JADRFN010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JADRFN010000147</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA674475</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN16789487</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Flavobacterium micromati isolate MAG-5, whole genome shotgun sequencing project', 'Flavobacterium micromati (snow metagenome)', 'Flavobacterium micromati', 'Bacteria; Bacteroidetes; Flavobacteriia; Flavobacteriales; Flavobacteriaceae; Flavobacterium']\n",
      "Id 2240894448\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>JAIMBV010000000</GBSeq_locus>\\n    <GBSeq_length>256</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>ENV</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>MAG: Alicyclobacillus mali isolate HOT.MB2.97, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>JAIMBV000000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>JAIMBV000000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>gb|JAIMBV000000000.1|JAIMBV010000000</GBSeqid>\\n      <GBSeqid>gi|2240894448</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA743362</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>ENV</GBKeyword>\\n      <GBKeyword>Metagenome Assembled Genome</GBKeyword>\\n      <GBKeyword>MAG</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Alicyclobacillus mali (hot springs metagenome)</GBSeq_source>\\n    <GBSeq_organism>Alicyclobacillus mali</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..256</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n          <GBAuthor>Navas,L.E.</GBAuthor>\\n          <GBAuthor>Fetherolf,M.M.</GBAuthor>\\n          <GBAuthor>Liu,L.Y.</GBAuthor>\\n          <GBAuthor>Dalhuisen,T.</GBAuthor>\\n          <GBAuthor>Renneckar,S.</GBAuthor>\\n          <GBAuthor>Eltis,L.D.</GBAuthor>\\n          <GBAuthor>Mohn,W.W.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Discovery of lignin-transforming bacteria and enzymes in thermophilic environments using stable isotope probing</GBReference_title>\\n        <GBReference_journal>ISME J (2022) In press</GBReference_journal>\\n        <GBReference_xref>\\n          <GBXref>\\n            <GBXref_dbname>doi</GBXref_dbname>\\n            <GBXref_id>10.1038/s41396-022-01241-8</GBXref_id>\\n          </GBXref>\\n        </GBReference_xref>\\n        <GBReference_pubmed>35501417</GBReference_pubmed>\\n        <GBReference_remark>Publication Status: Available-Online prior to print</GBReference_remark>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..256</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Levy-Booth,D.J.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (20-AUG-2021) Microbiology and Immunology, University of British Columbia, 2350 Health Sciences Mall, Vancouver, BC V6T 1Z3, Canada</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>The Alicyclobacillus mali whole genome shotgun (WGS) project has the project accession JAIMBV000000000. This version of the project (01) has the accession number JAIMBV010000000, and consists of sequences JAIMBV010000001-JAIMBV010000256.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Date :: 19-DEC-2019 ; Assembly Method :: SPAdes v. 3.11.1 ; Genome Representation :: Full ; Expected Final Version :: Yes ; Genome Coverage :: 15.4x ; Sequencing Technology :: Illumina NextSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI ; Annotation Date :: 08/24/2021 10:39:04 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 5.2 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,956 ; CDSs (total) :: 2,918 ; Genes (coding) :: 2,870 ; CDSs (with protein) :: 2,870 ; Genes (RNA) :: 38 ; rRNAs :: 2 (5S) ; partial rRNAs :: 2 (5S) ; tRNAs :: 33 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 48 ; CDSs (without protein) :: 48 ; Pseudo Genes (ambiguous residues) :: 0 of 48 ; Pseudo Genes (frameshifted) :: 15 of 48 ; Pseudo Genes (incomplete) :: 29 of 48 ; Pseudo Genes (internal stop) :: 7 of 48 ; Pseudo Genes (multiple problems) :: 2 of 48 ; CRISPR Arrays :: 2 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..256</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>256</GBInterval_to>\\n            <GBInterval_accession>JAIMBV000000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Alicyclobacillus mali</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolate</GBQualifier_name>\\n            <GBQualifier_value>HOT.MB2.97</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>hot spring sediment</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1123961</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>environmental_sample</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>Canada:Terrace, Lakelse Hot Spring</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>lat_lon</GBQualifier_name>\\n            <GBQualifier_value>54.358300 N 128.541133 W</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>collection_date</GBQualifier_name>\\n            <GBQualifier_value>2018-08-21</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>metagenome_source</GBQualifier_name>\\n            <GBQualifier_value>hot springs metagenome</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>metagenomic</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>JAIMBV010000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>JAIMBV010000256</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA743362</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN20023771</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['MAG: Alicyclobacillus mali isolate HOT.MB2.97, whole genome shotgun sequencing project', 'Alicyclobacillus mali (hot springs metagenome)', 'Alicyclobacillus mali', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus']\n",
      "Id 2240893217\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000000</GBSeq_locus>\\n    <GBSeq_length>2342000</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>BCT</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199, whole genome shotgun sequencing project</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB00000000</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB00000000.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB00000000.1|NZ_AYHB01000000</GBSeqid>\\n      <GBSeqid>gi|2240893217</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..2342000</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..2342000</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB00000000.1.; The Fusobacterium nucleatum YWH7199 whole genome shotgun (WGS) project has the project accession NZ_AYHB00000000. This version of the project (01) has the accession number NZ_AYHB01000000, and consists of sequences AYHB01000001-AYHB01000276.; Annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (released 2013). Information about the Pipeline can be found here: http://www.ncbi.nlm.nih.gov/genome/annotation_prok/; Bacteria and source DNA available from Yiping W. Han, Department of Periodontics, School of Dental Medicine, Case Western Reserve University, Cleveland, OH 44106-4905, USA.; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..2342000</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>2342000</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB00000000.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_alt-seq>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>AYHB01000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>AYHB01000276</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n      <GBAltSeqData>\\n        <GBAltSeqData_name>WGS_SCAFLD</GBAltSeqData_name>\\n        <GBAltSeqData_items>\\n          <GBAltSeqItem>\\n          <GBAltSeqItem_first-accn>NZ_AYHB01000001</GBAltSeqItem_first-accn>\\n          <GBAltSeqItem_last-accn>NZ_AYHB01000276</GBAltSeqItem_last-accn>\\n          </GBAltSeqItem>\\n        </GBAltSeqData_items>\\n      </GBAltSeqData>\\n    </GBSeq_alt-seq>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199, whole genome shotgun sequencing project', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893216\n",
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000276</GBSeq_locus>\\n    <GBSeq_length>5598</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0283, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000276</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000276.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000276.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0283</GBSeqid>\\n      <GBSeqid>gi|2240893216</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..5598</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..5598</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000276.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..5598</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>5598</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000276.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0283</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>&lt;1..428</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>428</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000276.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11845</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12245</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>&lt;1..428</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>428</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000276.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11845</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12245</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005894563.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>3</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>DUF3798 domain-containing protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527467.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>ADLPSPTMGYPGAFGIEFSDDEKGNWPKILEEVEKAVIAAGGSGRMGTWAYSYNFAGVEGLTDLAIKSIESGDRDFTLDKLLASLNVATPDAKWNGSIMKDNNGVDVPNAFFIYQDTYIFGKGYMGVTSVEIPEKYTNLGK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>complement(487..1146)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1146</GBInterval_from>\\n            <GBInterval_to>487</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000276.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11850</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12250</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>complement(487..1146)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1146</GBInterval_from>\\n            <GBInterval_to>487</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000276.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11850</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12250</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008800337.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>helix-turn-helix transcriptional regulator</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_022069210.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MSFGKTLKRIRLKHKDSLRGLAKKIDLHFTFIDKVEKGTAPISKNFIENVIAVYPDEREILKKEYLKETLPNVFQKEDAIKIVSNSEILNLPVYGKASAGRGYMNMDSPDYYMPILRGNFSKRSFFVEITGNSMEPTLEDGEFALVDPDNTAYSKNKIYVVTYNDEGYIKRLEMNNKLKLITLKSDNPDYEDIDIPEEMQEYFSVNGRVVEVISKKKLL</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>1948..3039</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1948</GBInterval_from>\\n            <GBInterval_to>3039</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000276.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11855</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12260</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>1948..3039</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1948</GBInterval_from>\\n            <GBInterval_to>3039</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000276.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11855</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12260</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008795007.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>dipeptide epimerase</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_060798043.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKITKIKLGIISVPLRVPFKTALRSVNSVEDVIVEIHTDTGNVGYGEAPPTGVITGDTTGAIVGALKDHIIKTLIGRDVDDFENLMKDLNSCIVKNTSAKAAADIALWDLYGQLHKIPVYKLLGGSRNKIVTDITISVNPPEEMARDAINAIKRGYDTLKVKVGIDPTLDVARLSAIRDAIGKDYRIRIDANQAWSPKQAIRLLNQMQDRGLDIELVEQPVKAHDFDGLRYVTKHSDVPVLADESVFSPEDAFKILQMKAADLINIKLMKCGGIYNALKIINLAEVVGVECMIGCMLEAKVSVNAAVHLACAKQVITKIDLDGPVLCSEDPIIGGAVFNEKEITISDDFGLGIKGINGIKYID</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>3061..4242</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3061</GBInterval_from>\\n            <GBInterval_to>4242</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000276.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11860</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12265</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>3061..4242</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3061</GBInterval_from>\\n            <GBInterval_to>4242</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000276.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11860</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12265</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005971875.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>DUF819 family protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527468.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MVITNGFTYIAFLMCLAGCLLLLEKYSKWRIFNVVPALVFIYILNMAFCTMGLFNSEACSKAYSVLKNNLLYAMIFVMLLRCDFRKLAKLGGRMVAIFLACSFTLFIGFVVGYPIFKSFLGVDVWGAVAALYASWVGGSANMAAMQAALPVDAGAYSCALALDTVCYSVWIALLLLMVRYSSKWDNATKADTSKLQAVADAAAKEVEKEKKTASAADWVFLIGISLMVSAISQMIGGALQNGFASIGLEMFDRGTITTVFVTILGLVCALTPLGKLPAVEELSTVYLYAVVSLLASTASVVDLLTAPMWIVYGLFILVIHVVLMFVFSKIFHWDLCMVSTASLANIGGSASAPIVASAYNPSYAGIGVLMGVLGAAVGNFFGIGIGQILKMMS</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>4471..5256</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>4471</GBInterval_from>\\n            <GBInterval_to>5256</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000276.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11865</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12270</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>4471..5256</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>4471</GBInterval_from>\\n            <GBInterval_to>5256</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000276.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11865</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12270</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005910563.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>class A beta-lactamase-related serine hydrolase</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527469.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MEKYTEWKKEIEKIISQVKGSVCINFYDLVKNTGFSIDGDKKVLSASMIKLLILAELMKKVSENKFSLSDAITVTEAMKTGGDGVLKELNSDHHFTLKELATLMIIVSDNQATNILIDFLGMKNINLLGKELRLKETFLGRKMMDIEARKKGYDNYTCADDISLLLKLIYQKKLINKEASQLMLDILLRQQQEERLQRYLPSDIKIAHKCGDLDNLENDGGIIWIGDKAYILAVLTNGMPNFQCKQTIGKISKFVYDKMEE</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>5259..&gt;5598</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>5259</GBInterval_from>\\n            <GBInterval_to>5598</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000276.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11870</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12275</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>5259..&gt;5598</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>5259</GBInterval_from>\\n            <GBInterval_to>5598</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000276.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11870</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12275</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008795009.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>DUF3870 domain-containing protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527470.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MNNQTIYVTGEARITIDNAITKMFGTFYIAFEVILSTDEIIDMDCNATLRLTRDFIKRLFLNHNIIKDEEILKKEIATRYFGSSSKAILTAYHDALQHSEPLTTDTLRVLESQ</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000276.1:1..5598)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0283, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893215\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000275</GBSeq_locus>\\n    <GBSeq_length>545</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0282, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000275</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000275.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000275.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0282</GBSeqid>\\n      <GBSeqid>gi|2240893215</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..545</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..545</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000275.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..545</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>545</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000275.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0282</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>&lt;1..&gt;545</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>545</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000275.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11840</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12240</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>&lt;1..&gt;545</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>545</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000275.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11840</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12240</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005894563.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>2</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>DUF3798 domain-containing protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527466.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>DDPKMKAIVVAEGVPGTYPAFKAIREKRPDILLFVNNNHEDPVQVSTVADVVMNSDSIARGYLIVKTAHDLGATKFMHISFPRHLSYETISRRRAIMEQTAKDLGMEYIEMSAPDPLSDVGVPGAQQFILEQVPNWVKKYGKDIAFFATNDAQTEPLLKQIAAYGGYFIKCVRTLVALAVR</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000275.1:1..545)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0282, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893214\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000274</GBSeq_locus>\\n    <GBSeq_length>8773</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0281, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000274</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000274.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000274.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0281</GBSeqid>\\n      <GBSeqid>gi|2240893214</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..8773</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..8773</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000274.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..8773</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>8773</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0281</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>414..776</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>414</GBInterval_from>\\n            <GBInterval_to>776</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11785</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12175</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>414..776</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>414</GBInterval_from>\\n            <GBInterval_to>776</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11785</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12175</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005910548.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_016340075.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MLKNLLLYTNDFLALFVMQLTIITVISIQIYALIKVKLLQRKYYIFLLIKILEIIFFKYEIFYMYLILFIIGVILNISLAIRGKKIDIKEMILGTIIFSTVPLIITFGILYLILLNFIYE</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>1172..1810</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1172</GBInterval_from>\\n            <GBInterval_to>1810</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11790</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12185</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>1172..1810</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1172</GBInterval_from>\\n            <GBInterval_to>1810</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11790</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12185</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005910549.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_009007580.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MLKNRKELIELIEFGYDIKEIINSWDPMGLMEFCPEDEYEAEIKGLRNLVVNNRNTDKKLLGKEIRKLFRFYFSNRYNSKRDVEENIAGKIIEKSKKYKLSCTVSNYYDIENIIFKNEKEIDIYINLYTKINKIINSWDPLKIMDISFSNEYSYEINRIIEELLKNITIQNLSKEINKIFKNAYNGLYKIEKNEEIEITEKIFEEYNNISKL</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>2223..2504</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>2223</GBInterval_from>\\n            <GBInterval_to>2504</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11795</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12195</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>2223..2504</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>2223</GBInterval_from>\\n            <GBInterval_to>2504</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11795</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12195</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: ab initio prediction:GeneMarkS-2+</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: GeneMarkS-2+.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527464.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MIVIDCNRCEKYKYIIEELFIKEDKIIIFHINKKKRIEKYKIKFDEITDLEYKDGFFLNPYRPDTFFHKNIEKCRLLKIKLKSKKVVSFGFFL</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>2634..3209</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>2634</GBInterval_from>\\n            <GBInterval_to>3209</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>pth</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11800</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12200</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>2634..3209</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>2634</GBInterval_from>\\n            <GBInterval_to>3209</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>pth</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11800</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12200</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>EC_number</GBQualifier_name>\\n            <GBQualifier_value>3.1.1.29</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_009007578.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_function</GBQualifier_name>\\n            <GBQualifier_value>GO:0004045 - aminoacyl-tRNA hydrolase activity [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_process</GBQualifier_name>\\n            <GBQualifier_value>GO:0006412 - translation [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>aminoacyl-tRNA hydrolase</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_005910552.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKVVIGLGNPGKKYERTRHNMGFIAVDSLRKKFNLNDEREKFQALVSEKNIDGEKVIFFKPQTFMNLSGNAVIEIINFYKLDPKKDIIVIYDDMDLPFGDIRIREKGSSGGHNGIKSIISHIGEEFVRIKCGIGEKEKDAIEHVLGEFNQTEQKNLDEILENIDNCIIEMLTVQNLDRIMQKYNKKKEKLK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>3283..4599</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3283</GBInterval_from>\\n            <GBInterval_to>4599</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>rsxC</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11805</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12205</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>3283..4599</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3283</GBInterval_from>\\n            <GBInterval_to>4599</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>rsxC</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11805</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12205</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008693384.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_component</GBQualifier_name>\\n            <GBQualifier_value>GO:0016020 - membrane [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_function</GBQualifier_name>\\n            <GBQualifier_value>GO:0051539 - 4 iron, 4 sulfur cluster binding [Evidence IEA]; GO:0009055 - electron transfer activity [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>electron transport complex subunit RsxC</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_005905476.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MTFFGFRGGVHPPENKIQTEHLPIEKLESPDEIFVPLLQHIGVPLNPLVNVGDRVLKGQKIADAEGLAVPIHSPVSGTVTKIESRVFPLTGKVMTVFIENDKKEEWAELTKIENWEEADKKALLDIIREKGIVGIGGATFPTHVKLNPPPNTKLDSLILNGAECEPYLNSDNRLMLENPKSIVEGIKIIKKILNVPDVYVGIEDNKPEAIESMRKATEGTGINIVPLKTKYPQGGEKQLIKSVLNREVPSGQLPSAVGVVVQNTGTAAAIYEAVVNGKPLIEKIVTVSGKAIKNPKNLKIAIGTPFSYILDYCGVDREKMERLVMGGPMMGLAQMTEDATVIKGTSGLLALTTEEMRPYKTKSCISCSKCVSACPMGLEPLMFDRLAANKQYDEMAEYNLMDCIECGSCNYICPANRPLAEAIKTGKSKLRAKAKKKK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>4625..5572</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>4625</GBInterval_from>\\n            <GBInterval_to>5572</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11810</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12210</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>4625..5572</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>4625</GBInterval_from>\\n            <GBInterval_to>5572</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11810</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12210</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005919433.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>RnfABCDGE type electron transport complex subunit D</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_005905473.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MSTILKTGPAPHIRTAETVESVMYDVVIALIPAFAMAVYSFGVRALILTAVSVATCIATEYIWQKILKRDIEAFDGSAILTGILFAFVVPVGMGLQYVIVGNFVAITLGKIVYGGLGHNIFNPALIGRAFVQASWPVAITTFAYDGKAGATVLDAMKRGIPLTDSLIYEGGNQYINAFLGQMGGCLGETSALALLIGGVYLIYKKQIDWKVPAVMIGTVFVLTWAMGANPFMQILSGGLFLGAFFMATDMVTSPITGKGRVIFALGIGILVSLIRIKGGYPEGTAYAILIMNGVVPLIDRYIRAKKFGGVSKNGK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>5562..6095</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>5562</GBInterval_from>\\n            <GBInterval_to>6095</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11815</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12215</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>5562..6095</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>5562</GBInterval_from>\\n            <GBInterval_to>6095</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11815</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12215</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008821913.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_component</GBQualifier_name>\\n            <GBQualifier_value>GO:0005886 - plasma membrane [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_function</GBQualifier_name>\\n            <GBQualifier_value>GO:0009055 - electron transfer activity [Evidence IEA]; GO:0010181 - FMN binding [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_process</GBQualifier_name>\\n            <GBQualifier_value>GO:0022900 - electron transport chain [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>RnfABCDGE type electron transport complex subunit G</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_008693383.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MENRYIHFGIVLGLIAAISAGILGGVNGFTSKVIAENTKKIVNEARQEVLPEAVSFKEDEAKVADEIQYIPGFNDAGEVVGYVASVTEPGYGGDINFVVGIDKDAKVTGLNVVTSSETPGLGAKINGKEWQEHWIGKDATYEFNKSVDAFAGATISPKAVYTGVIKALNTYQNEVSK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>6095..6712</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>6095</GBInterval_from>\\n            <GBInterval_to>6712</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11820</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12220</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>6095..6712</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>6095</GBInterval_from>\\n            <GBInterval_to>6712</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11820</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12220</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005969446.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>electron transport complex subunit E</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_005910556.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKKFEVLTSGIFKENPIFVLMLGLCPTLGVTSSAINGFSMGLAVIAVLACSNGLISLFKKFIPDEVRIPAFIMIIASLVTVVDMVMNAYTPDLYKVLGIFIPLIVVNCIVLGRAESFASKNGVIDSILDGIGSGIGFTLSLTFLGSIREILGNGSIFGIPLVPANFSPALIFILAPGGFITIGIIMGCINMKKERDAKKKKVAKK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>6709..7293</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>6709</GBInterval_from>\\n            <GBInterval_to>7293</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>rsxA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11825</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12225</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>6709..7293</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>6709</GBInterval_from>\\n            <GBInterval_to>7293</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>rsxA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11825</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12225</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_009424153.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_component</GBQualifier_name>\\n            <GBQualifier_value>GO:0016020 - membrane [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_process</GBQualifier_name>\\n            <GBQualifier_value>GO:0022900 - electron transport chain [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>electron transport complex subunit RsxA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_022069213.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MSISGLFSIVITSIFINNIIFAKFLGCCPFMGVSKKVDSSLGMGMAVTFVITIASGVTWIVYRLILEPHGLGYLQTIAFILIIASLVQFVEMAIKKTSPSLYKALGVFLPLITTNCAVLGVAIINIQVGYNFIETLVNGFGVAVGFSLALLLLAGIRERLEFANTPKNFKGVPIAFITAGLLAMAFMGFSGMQI</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>7320..8282</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>7320</GBInterval_from>\\n            <GBInterval_to>8282</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11830</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12230</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>7320..8282</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>7320</GBInterval_from>\\n            <GBInterval_to>8282</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11830</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12230</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005910558.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_function</GBQualifier_name>\\n            <GBQualifier_value>GO:0009055 - electron transfer activity [Evidence IEA]; GO:0051536 - iron-sulfur cluster binding [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_process</GBQualifier_name>\\n            <GBQualifier_value>GO:0022900 - electron transport chain [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>RnfABCDGE type electron transport complex subunit B</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249524214.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MEAIIMPVVVLGITGVLMGLFLAFASKKFEVEVDPKVEAILAVLPGANCGACGFPGCAGYAAGVALEGAKMTLCAPGGPKVAAKIGDIMGVAVEMPVKKKPAAKKPVEKKVVQTGEPISASQEFIEKNKRMLMKFKDAFDAGDKEGYEKLENLAKMAKKDELLKYYEEIKSGKIVPDGSAPVAAGAASTNPISATKEFVEKNKRMLMKFKEAFDAGDKEGYEKLENLAKMAKKDELLKFYEEIKAGKIVPDPATMVDSPAVKEETPKVETKVGDSKKQEAAYCSTLGDGLCVPEQNEKTKEDLKKQAEPPKTAEETEKDK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>8469..&gt;8773</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>8469</GBInterval_from>\\n            <GBInterval_to>8773</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11835</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>8469..&gt;8773</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>8469</GBInterval_from>\\n            <GBInterval_to>8773</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000274.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11835</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005894563.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>DUF3798 domain-containing protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527465.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKMKRILFGLLTIFMLMLAVACGKKEAPTEDANAQQQESANEAVQDYHIGVVTTSVSQSEDNFRGAEAILKKYGAANERGKITIVTIPDNFMQEQETTISQ</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000274.1:1..8773)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0281, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893213\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000273</GBSeq_locus>\\n    <GBSeq_length>4901</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0280, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000273</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000273.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000273.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0280</GBSeqid>\\n      <GBSeqid>gi|2240893213</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..4901</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..4901</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000273.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..4901</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>4901</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0280</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>&lt;1..662</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>662</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11740</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12125</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>&lt;1..662</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>662</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11740</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12125</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008798111.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>incomplete; too short partial abutting assembly gap; missing N-terminus; Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>3</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>644..880</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>644</GBInterval_from>\\n            <GBInterval_to>880</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11745</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12130</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>644..880</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>644</GBInterval_from>\\n            <GBInterval_to>880</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11745</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12130</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_009423188.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>serine kinase</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527457.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MECKKIKKVYMLYHINERKNEKLIGFLSSKEKAKSIIKELVEKPGFKDCPNGFRIKTMIIGKDYYTKGFKSKCAPKDE</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>926..1195</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>926</GBInterval_from>\\n            <GBInterval_to>1195</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11750</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12135</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>926..1195</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>926</GBInterval_from>\\n            <GBInterval_to>1195</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11750</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12135</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008821871.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527458.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKIYVLSHEYCYGDYKYKYKEESRFIGIYLTRKEALRALEKFKKIRGFSSHLDGFYIDRTEINQLEWLDGYRTGCFSMELGMHEEKEEE</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>1197..1469</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1197</GBInterval_from>\\n            <GBInterval_to>1469</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11755</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12140</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>1197..1469</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1197</GBInterval_from>\\n            <GBInterval_to>1469</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11755</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12140</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_009423190.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527459.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKIYVLSHSYECDVEKTKDEGRFLGVYLTKKEAIKKAEMYKKIKGFSSHISNFYIGCYIVDKTLKWLDNYSIDNWYAVGFMGKFYRQNIE</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>1475..1750</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1475</GBInterval_from>\\n            <GBInterval_to>1750</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11760</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12145</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>1475..1750</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1475</GBInterval_from>\\n            <GBInterval_to>1750</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11760</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12145</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005910539.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527460.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKIDDDVQKDQVSVMNKLLNDALRAKGYAGPDIKMVLTDVTALNEPYYTNTFVVGGSLGIINGELIYDSETSELSAFRLGITASAISLIKN</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>1841..2689</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1841</GBInterval_from>\\n            <GBInterval_to>2689</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11765</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12150</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>1841..2689</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1841</GBInterval_from>\\n            <GBInterval_to>2689</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11765</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12150</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005910540.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527461.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MEIENTKESTVIVFKNNSVFIGVYIVPLFIIVPLIISKITVQSIIMSLIVFTYLNIGNYIAMRNIGSIETITIKNESLIIRRLKRNKKITYEKEIFFDKILKIYYQEIFLGFHKRNFNFDVKRTLKIKTYFCIYSFGYKMSYEDFKKANTTIEEKIKNYKNFIEKKETKKLYIEIYNLEIEKRYEKILNIILDEEKLYLLKKNDCYIVNGDSEAIKNLKIFKNMNFKEIDFYIFYVNYLSKKEYEDKKVLVGYNGVDGKKVTMSKFKEDINEIRDSGSTFRN</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>3026..3532</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3026</GBInterval_from>\\n            <GBInterval_to>3532</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11770</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12155</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>3026..3532</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3026</GBInterval_from>\\n            <GBInterval_to>3532</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11770</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12155</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005910541.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>DUF2247 family protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527462.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MITLNDFKNNNLKINWKVINIGCLGSEIFKNELSYDDIINFSLEKFDEKNKLILRIIASDRDEYQEIGYLVQELANMEKSEYKLEFEKWKLVYVKKNFPKLNKNIIQGLIELNNLWVKLDFPEDSPYILQGVKNNISPQEYYTEKNYIYLYNRHLKWIRDKSDYLDGK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>&lt;3532..&gt;4080</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3532</GBInterval_from>\\n            <GBInterval_to>4080</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11775</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12160</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>&lt;3532..&gt;4080</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3532</GBInterval_from>\\n            <GBInterval_to>4080</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11775</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12160</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_017144793.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>incomplete; partial in the middle of a contig; missing N-terminus and C-terminus; Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hemin transporter</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>4156..&gt;4901</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>4156</GBInterval_from>\\n            <GBInterval_to>4901</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11780</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12165</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>4156..&gt;4901</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>4156</GBInterval_from>\\n            <GBInterval_to>4901</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000273.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11780</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12165</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_011016284.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527463.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MRKVENRQHFNLYIYALISLIVTDIAIIIFLVGLIFSRPIIYVIIVILLIMVYFITRAIVNCLKYYISKEECYCENGTLIYKRILLKKFILKEVEIPLLNIQKVIDKGHIPSNNAEPDYLNPFHYVVLFFKYYERVLLEMKTGNKYEIFVYAFPYGTRGKSFEKIYNDNDFLKSFTELKEMIEEEQSKILFNQKVENLRERYNFPLDKRYNYILNKIIDEEKLFISEKDNNFIINGDSEAIKNLEIFK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000273.1:1..4901)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0280, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893212\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000272</GBSeq_locus>\\n    <GBSeq_length>798</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0279, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000272</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000272.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000272.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0279</GBSeqid>\\n      <GBSeqid>gi|2240893212</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..798</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..798</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000272.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..798</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>798</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000272.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0279</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>&lt;1..612</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>612</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000272.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11735</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>&lt;1..612</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>612</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000272.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11735</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_011016284.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527456.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>KGHIPSHNARRDVLNPLHYVVLFFNYYERILLEMKTGDKYEIFVYAFPYGTRTEELEKIYNDNDFLKSFDELKEMIEEEQKKILFSQKVKNLSEKYNSPLEERYNYVLNKILDEEKLFLFKKDNNFVINGDSEAIKDLEIFKNINFEEIDFYIFYVNYLSKKEYENKKVLVGYNGVDGKEVTMSKFKEDINEIRDSRSTFKNQ</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000272.1:1..798)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0279, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893211\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000271</GBSeq_locus>\\n    <GBSeq_length>1462</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0277, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000271</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000271.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000271.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0277</GBSeqid>\\n      <GBSeqid>gi|2240893211</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..1462</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..1462</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000271.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..1462</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>1462</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000271.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0277</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>76..1416</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>76</GBInterval_from>\\n            <GBInterval_to>1416</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000271.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11730</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12105</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>76..1416</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>76</GBInterval_from>\\n            <GBInterval_to>1416</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000271.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11730</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12105</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005974393.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>MATE family efflux transporter</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_005909886.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MNSIGNVGKKTLISLTIPIFFELLLVTVVGNIDTIMLGYYSDKAVGAIGGITQLLNIQNVIFSFINLATAILTAQFLGARDYKRVKQVISVSLVLNVLLGVVLGGIYLFFWRGLLQKINLPEELVGIGKYYFQMVGGLCVFQGIILSCGAILKSHGRPTETLIINVGVNILNILGNGMFIFGWLGMPVLGTTGVGISTVVSRGIGCVAAFYMMCKYCNFTFRKKYIKPFPFKIVKNILSIGFPTAGENLSWNVGQLLIVAMVNTMGTTIIASRTYLMLISNFVMTLSIALGQGTAIQVGHLVGAGEVKEVYSKCLKSVKIAFIFAFVATSIVCFLRKPIMNIFTTNPDILSASLKIFPLMIILEMGRVFNIVIINSLHAAGDIKFPMFMSIICVYAVAVLFSYIFGISLGWGLAGIWIANAMDEWIRGIAMYLRWKSRKWQNKSFV</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000271.1:1..1462)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0277, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893210\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000270</GBSeq_locus>\\n    <GBSeq_length>12725</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0276, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000270</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000270.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000270.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0276</GBSeqid>\\n      <GBSeqid>gi|2240893210</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..12725</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..12725</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000270.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..12725</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>12725</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0276</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>184..663</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>184</GBInterval_from>\\n            <GBInterval_to>663</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11660</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12035</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>184..663</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>184</GBInterval_from>\\n            <GBInterval_to>663</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11660</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12035</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_016339983.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527450.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MNKISRYLGRELANKDYFFISKENREKEQTKANPNFTGNHEFFAFGKTRLYQSVYGSVKRLENGKYDVDITVMFQYTDRFEDVKNINNSSVAKQGLNKEFEGGKSFSFKTEPKIITIKKQVNNLDEITGLLKNRLRNIDDNSEEGKDNIRADFYDSVIF</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>644..1132</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>644</GBInterval_from>\\n            <GBInterval_to>1132</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11665</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12040</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>644..1132</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>644</GBInterval_from>\\n            <GBInterval_to>1132</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11665</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12040</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008798426.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527451.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MIVLYSKTSNEKEENFKRKYFELFYLFIFFIYFSFAIISGYFQHYEYIFTDINNTKEYAVFSEGRYVYIRLENNMLTEGQFPDFRIKEVYEDKNNFIGCDFEYNYETDKKEERYYIVIDKNKASMKIFSEKEFKEKYNISDGKFINIYTFLKKKGTKIGRYR</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>1151..1618</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1151</GBInterval_from>\\n            <GBInterval_to>1618</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11670</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12045</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>1151..1618</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1151</GBInterval_from>\\n            <GBInterval_to>1618</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11670</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12045</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_016361408.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527452.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKYGENKIIKISVALTTVLNIFISIIEIRYIEFIYILIFIGFNIYVRGTKTHNEKENITYVEKKEIVKSNKVLFIGTNILIFFYIFKEPYFFLSKYKILILLISIIIGYSSLIFLQKIRIKLSLNFFKEIIKIFLISGIIYYFPIVCVQILLKIT</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>1840..2838</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1840</GBInterval_from>\\n            <GBInterval_to>2838</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>plsX</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11675</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12050</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>1840..2838</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1840</GBInterval_from>\\n            <GBInterval_to>2838</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>plsX</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11675</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12050</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>EC_number</GBQualifier_name>\\n            <GBQualifier_value>2.3.1.274</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_011016179.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_function</GBQualifier_name>\\n            <GBQualifier_value>GO:0043811 - phosphate:acyl-[acyl carrier protein] acyltransferase activity [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_process</GBQualifier_name>\\n            <GBQualifier_value>GO:0008610 - lipid biosynthetic process [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>phosphate acyltransferase PlsX</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_008694253.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKIALDAMSGDFAPISTVKGAIEALQEIEGLEIILVGKEGIIKEELKKYKYDTKRIEIKNADEVIEMTDDPVKAVREKKNSSMNVCIDLVKEKIAQASVSCGNTGALLASSQLKLKRIKGVLRPAIAVLFPNKKEQGTLFLDLGANSDSKPEFLNQFATMGSKYMEIFSGKKNPKVALLNIGEEETKGNELTRETYALLKENKDIDFYGNIESTKIMDGEVDVVVTDGYTGNVLLKTSEGVGKFIFHIVKEAIMESWISKLGALLIKGAMKKVKKKTEASEYGGAIFLGLSELSLKAHGNSNSRAIKNALKVASKFIELNFIEELRKTMEVE</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>2838..3824</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>2838</GBInterval_from>\\n            <GBInterval_to>3824</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11680</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12055</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>2838..3824</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>2838</GBInterval_from>\\n            <GBInterval_to>3824</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11680</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12055</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_009424177.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>ketoacyl-ACP synthase III</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527453.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MQSIGIKGIGYYVPENVFTNFDFEKIIDTSDEWIRTRTGIVERRFASKDQATSDLACEAALKVIKNAKINKEDVDMIILATTTPDYIAQGASCIVQHKLGLKNIPCFDVNAACTGFIYSLSVAYALVNSKLYKNILVIGAETLTRIIDIKNRNTAVLFGDGAAAAIVGEVEEGYGILGFSIGAEGEDDMILKVPAGGSKKPNDEETVKNRENFVVMKGQEVFKFAVRVLPKVTLEALKKAKLNVRDLSMIFPHQANLRIIESAAKRMKFPLEKFYMNLSRYGNTSSASVGIALGEAIEKGLIKKGDNIALTGFGGGLTYGSVIIKWAY</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>3857..4756</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3857</GBInterval_from>\\n            <GBInterval_to>4756</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>fabD</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11685</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12060</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>3857..4756</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3857</GBInterval_from>\\n            <GBInterval_to>4756</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>fabD</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11685</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12060</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>EC_number</GBQualifier_name>\\n            <GBQualifier_value>2.3.1.39</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_011016181.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_function</GBQualifier_name>\\n            <GBQualifier_value>GO:0004314 - [acyl-carrier-protein] S-malonyltransferase activity [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_process</GBQualifier_name>\\n            <GBQualifier_value>GO:0006633 - fatty acid biosynthetic process [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>ACP S-malonyltransferase</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_060676636.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MGKVAFVYPGQGTQYVGMGKELYEKNKRAKELFDKIFNSLDIDLKKVMFEGPEDLLKRTDYTQPAIVSLSLVLTELLKEKGIEPDYVAGHSVGEFAAFGGANYLSIEDAVKLVAARGRIMREVAEKVNGSMAAVLGMDAEKIKEVLKSVDGVVEAVNFNEPNQTVIAGSKEAVEKACVVLKEAGAKRAIPLAVSGPFHSSLMKEAGEKLKEEAQKYTFNIGKVKIVANTTAELLESDSEVKDEIYRQSFGPVKWVDTINKLKSLGVTKIYEIGAGKVLSGLIKKIDKEIEVENIELIEA</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>4825..5052</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>4825</GBInterval_from>\\n            <GBInterval_to>5052</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>acpP</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11690</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12065</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>4825..5052</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>4825</GBInterval_from>\\n            <GBInterval_to>5052</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>acpP</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11690</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12065</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005902726.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_process</GBQualifier_name>\\n            <GBQualifier_value>GO:0006633 - fatty acid biosynthetic process [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>acyl carrier protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_005906540.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MLDKVKEIIVEQLGVDADQIKPESNFVDDLGADSLDTVELIMSFEEEFGVEIPDTEAEKIKTVQDVINYIEAHKK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>5379..6620</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>5379</GBInterval_from>\\n            <GBInterval_to>6620</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>fabF</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11695</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12070</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>5379..6620</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>5379</GBInterval_from>\\n            <GBInterval_to>6620</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>fabF</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11695</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12070</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>EC_number</GBQualifier_name>\\n            <GBQualifier_value>2.3.1.179</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005915867.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_function</GBQualifier_name>\\n            <GBQualifier_value>GO:0004315 - 3-oxoacyl-[acyl-carrier-protein] synthase activity [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_process</GBQualifier_name>\\n            <GBQualifier_value>GO:0006633 - fatty acid biosynthetic process [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>beta-ketoacyl-ACP synthase II</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527454.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKRVVVTGLGLISALGIGLEESWKKLIAGETGIDLIKSYDTTDQPVRIAGEVKGFEPTDYGIEKKEVKKLARNTQFALVATKMALEDANFKIDETNADDVGVIVSAGVGGIEIMEEQYGNMLTKGYRRISPFTIPAMIENMAAGNIAIYYGAKGPNRSIVTACASGTHSIGDGFDLIRHGRAKAMIVGGTEASVTQFCINSFANMKALSTRNETPKTASRPFSKDRDGFVMGEGAGILILEELESALARGAKIYAEMVGFGDTSDANHITAPIETGEGATKAMRAALKDANIPLEDVTYINAHGTSTPTNDIVETRAIKALFGDKAKSLYVSSTKGATGHALGGAGGIEGVIIAKTISEGIIPPTINLYETEEECDLNYVPNKAIKADVKVAMSNSLGFGGHNSVIVMKKFEK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>6636..7340</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>6636</GBInterval_from>\\n            <GBInterval_to>7340</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>rnc</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11700</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12075</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>6636..7340</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>6636</GBInterval_from>\\n            <GBInterval_to>7340</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>rnc</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11700</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12075</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>EC_number</GBQualifier_name>\\n            <GBQualifier_value>3.1.26.3</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_009006826.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_function</GBQualifier_name>\\n            <GBQualifier_value>GO:0004525 - ribonuclease III activity [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_process</GBQualifier_name>\\n            <GBQualifier_value>GO:0006396 - RNA processing [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>ribonuclease III</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_005909892.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKNLLDLEHKLNYYFNDRNLLKNALLHKSLGNERKEYKNQNNERLELLGDAVLDLIVAEYLYKSYKNASEGTIAKLKAMIVSEPILAKISRQIGVGKFLMLSRGEVMSGGRNRESILADSFEAILGAVYIDSNLDDARNFTLSHIKQYIDHIEENEDILDFKSILQEYVQKEFRTVPTYKLIAERGPDHMKEFEIQVIVGNYREKAVAKNKKKAEQLSAKALCIKLGVNYHEAL</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>7327..8373</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>7327</GBInterval_from>\\n            <GBInterval_to>8373</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11705</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12080</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>7327..8373</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>7327</GBInterval_from>\\n            <GBInterval_to>8373</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11705</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12080</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_011016184.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_function</GBQualifier_name>\\n            <GBQualifier_value>GO:0051536 - iron-sulfur cluster binding [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>radical SAM protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_022070416.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKHYNIPVFISHFGCPNACVFCNQKKINGRETDVSLDDLKNIIDSYLKTLPKNSIKQVAFFGGTFTGISMNLQKEYLEVVKKYIDNNDVEGVRISTRPECIDDEILTQLKKYGVKTIELGIQSLDDKVLKATGRNYAYDVVKKSCDLIKNYGFELGIQLMIGLPKSDFKSDLQSAIKSLELNPDIARIYPTLVIKGTELELMYKKNLYQSLSIEEAVDRTVPIYSLLELKNINVIRVGLQPAEDLTADGVIISGPFHPAFRDLIENKIYFNFLSKIYKKEKKLDIEVNERNVSKIVGQKAINKKTFYPNFKILINNNLDLDELMINSKKYTRKEILEGEFNEKISDFI</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>8348..9724</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>8348</GBInterval_from>\\n            <GBInterval_to>9724</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11710</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12085</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>8348..9724</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>8348</GBInterval_from>\\n            <GBInterval_to>9724</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11710</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12085</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008702438.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>ribonuclease E/G</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527455.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKKYLILSKSVCETKLALLEDNKLDEMYIERNNQKEIIGNIYKGKVVDILNNGEIIFVDIGLEKNALLSFENKKKVPKFNIDDKLIVQTESEPRDEKGAKLTLDYSINGENLVLLPKSKNISISKKIKDIEEINRLKNIFLSIDNGLILRTNSEEKSEESLLEEYRKLEDIDNQINKDFEKINIGLLYDVNSILKKAVTLFNNTIEEFIIDDEHIFEEIKILLEKTGKKDLIKKLRKYFKDEDIFEYYNINSQIERALDRKVYLDSGAYIIIEKTEALISIDVNTGQNIGNKTSQELIFQTNLEATKEIARQIKLRNLAGIIIVDFIDMKKFSDRKRVLEEFKKYLSVDRAEINSVEYTNLGLIQFTRKRQGKELAFYYKEKCQYCEGTGYFLSKDRIILNLLEDLNSQIKSQDIKKIVIRAKKDIIKELNKYIGNNKIEYIEDNNFYKIGYKMELYK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>9755..10252</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>9755</GBInterval_from>\\n            <GBInterval_to>10252</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>coaD</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11715</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12090</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>9755..10252</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>9755</GBInterval_from>\\n            <GBInterval_to>10252</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>coaD</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11715</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12090</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>EC_number</GBQualifier_name>\\n            <GBQualifier_value>2.7.7.3</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005909889.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_function</GBQualifier_name>\\n            <GBQualifier_value>GO:0004595 - pantetheine-phosphate adenylyltransferase activity [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_process</GBQualifier_name>\\n            <GBQualifier_value>GO:0015937 - coenzyme A biosynthetic process [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>pantetheine-phosphate adenylyltransferase</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_005909889.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKTGVYAGSFDPITKGHQDIIERALKIVDKLIVVVMNNPKKNYWFNLDERKNLISKIFEGDKSIKVDEHAGLLVDFMAKNSCGILIKGLRDVKDFSEEMTYSFANKKLSNGEVDTIFIPTSEKYTYVSSTFVKELAFYNQSLAGYVDDKVIVDILNRAKEYRDRG</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>10257..11615</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>10257</GBInterval_from>\\n            <GBInterval_to>11615</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>radA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11720</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12095</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>10257..11615</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>10257</GBInterval_from>\\n            <GBInterval_to>11615</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>radA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11720</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12095</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005895523.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_function</GBQualifier_name>\\n            <GBQualifier_value>GO:0005524 - ATP binding [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_process</GBQualifier_name>\\n            <GBQualifier_value>GO:0006281 - DNA repair [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>DNA repair protein RadA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_005909888.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MAKGTVYYCSECGYKSAKWVGKCPQCGAWSSFEEVDELPKDVKKTKNSDIKVYEFKDVEYTSEDRYKTKYEEFDRLLGGGLLKGEVVLVTGNPGIGKSTLLLQVANSYKDYGNILYISGEESPAQIKNRGERLKISGESIYIMAEMDILNIYEYVVSKKPKVVIVDSIQTLYNSSMDSISGTPTQIRECTLKIIEIAKKYNISFFIVGHITKDGKVAGPKLLEHMVDAVFNFEGDEGLYYRILRSVKNRFGSTNEIAVFSMEENGMKEIKNSSEYFLSEREEKNIGSMVVPILEGTKVFLLEVQSLITDSGIGIPKRVVQGYDRNRIQILTAIAEKKLYVPLSMKDLFVNVPGGLSIEDPAADLAVLMSILSVHKGFAISQKIAAIGELGLRGEIRKVFFLERRLKELEKLGFTGVYVPESNRKELEKKKFKLKIIYLKNLDELLERMNKND</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>11608..12657</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>11608</GBInterval_from>\\n            <GBInterval_to>12657</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>disA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11725</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12100</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>11608..12657</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>11608</GBInterval_from>\\n            <GBInterval_to>12657</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000270.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>disA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11725</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12100</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>EC_number</GBQualifier_name>\\n            <GBQualifier_value>2.7.7.85</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005966456.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>DNA integrity scanning diadenylate cyclase DisA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_005909887.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MTKQDLMDIIVKVAPGSPLREGVDYILDAGIGALIVIGYDDEVEKVRDGGFFIDCDYTPERIFELSKMDGAIILNDDCSKILYANVHIQPNISFITTESGTRHRTAERTAKHLKREVVAISERKKNVTLYKGELKYRLKNFDELNIEVGQVLKTLESYRHVLNRSLDNLTILELDDLVTVLDVANTLQRFEMVRRISEEITRYLLELGTRGRLVNMQVSELIWDLDDEEESFLKDYIDDGMTTDSVRRYLHSLSDSELLDIENVVVALGYSKSSSVFDNKIAAKGYRVLEKISKLTKKDIEKIVNTYKDISEIQEVTDEDLSAIKISKFKIKALRAGINRLKFTIEMQR</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000270.1:1..12725)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0276, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893209\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000269</GBSeq_locus>\\n    <GBSeq_length>3387</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0275, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000269</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000269.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000269.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0275</GBSeqid>\\n      <GBSeqid>gi|2240893209</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..3387</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..3387</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000269.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..3387</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>3387</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0275</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>&lt;1..230</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>230</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11620</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11990</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>&lt;1..230</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>230</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11620</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11990</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008701199.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>incomplete; too short partial abutting assembly gap; missing N-terminus; Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>3</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>621..773</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>621</GBInterval_from>\\n            <GBInterval_to>773</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11625</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>621..773</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>621</GBInterval_from>\\n            <GBInterval_to>773</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11625</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: ab initio prediction:GeneMarkS-2+</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: GeneMarkS-2+.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527444.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MTGIKQWQIKPETILLKEIADSQQLQGTQYIILEELNKYLFINLGNIKHY</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>826..1119</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>826</GBInterval_from>\\n            <GBInterval_to>1119</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11630</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12000</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>826..1119</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>826</GBInterval_from>\\n            <GBInterval_to>1119</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11630</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12000</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008794343.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527449.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MLEKVIQQCQKTLDKIEEELSKPEPKLTPYDIEMGNFDEVPRLILKEAKRQIKIMMQVLDKNKYMPDYTYPLIDSYLIDTELCDLLFETKSIYKKYT</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>1340..1663</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1340</GBInterval_from>\\n            <GBInterval_to>1663</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11635</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12005</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>1340..1663</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1340</GBInterval_from>\\n            <GBInterval_to>1663</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11635</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12005</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_009008008.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527445.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MEELSRQFKAGEINEKQLKEAARDIVKGYGKDIGIDYEVVYLDESTMPKDPKGSTGSAYIFKLGDDNNSNISLTTDGIDLTNADVGEKVGDFFPKQEIDTAGGVKKV</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>1702..2121</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1702</GBInterval_from>\\n            <GBInterval_to>2121</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11640</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12010</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>1702..2121</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1702</GBInterval_from>\\n            <GBInterval_to>2121</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11640</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12010</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: ab initio prediction:GeneMarkS-2+</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: GeneMarkS-2+.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527446.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MLDLFRKTGEFTLRALKAGREDLSLNHIKFLKGFNIILYHEDTKNLEKELEELKDGWGKGSFSDKEATIKYHYNKHKEEVGADSLLQYLRKAKAFSKNLRGAKKSNVMGATKGVIKYEKNGKYIEIAPNGDIISFGTIK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>2167..2568</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>2167</GBInterval_from>\\n            <GBInterval_to>2568</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11645</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12015</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>2167..2568</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>2167</GBInterval_from>\\n            <GBInterval_to>2568</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11645</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12015</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005910007.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_060798864.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MLVYNFEISNGEKVFINNGVISYMFDSFKWIKTFNRLRMKRNKGLFYHGSTYIEKENIDKMKEVVFAWKELFNEASEEFILLGFFNKKLDDYERWKCNKRQVIESLKKLIILCDKARKNNKIIRCRKITIKAQ</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>2583..2975</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>2583</GBInterval_from>\\n            <GBInterval_to>2975</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11650</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12020</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>2583..2975</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>2583</GBInterval_from>\\n            <GBInterval_to>2975</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11650</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12020</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005910006.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>coproporphyrinogen III oxidase</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527447.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MLIHDFGIVGEKKDVHLYDDLILYMIDTFEWIKTFSELESNIEKNGLNHSGITYFKGESVTKLKNIILHWINIFNLGEKTIELRGLFSINEKKHSYNKISKKHLIESLKKLVLLCEKAEKENKIIEHWGI</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>3120..&gt;3387</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3120</GBInterval_from>\\n            <GBInterval_to>3387</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11655</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12025</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>3120..&gt;3387</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3120</GBInterval_from>\\n            <GBInterval_to>3387</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000269.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11655</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_12025</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008700668.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>HD domain-containing protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527448.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKYNFNELKEIVKSKMSLKRFTHTLGVVEMSEKLAKIYNADIEKCKVAALLHDICKEMDMKYIKNICKNNFMNKLSEEDLENNEILHGF</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000269.1:1..3387)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0275, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893208\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000268</GBSeq_locus>\\n    <GBSeq_length>1484</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0274, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000268</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000268.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000268.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0274</GBSeqid>\\n      <GBSeqid>gi|2240893208</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..1484</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..1484</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000268.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..1484</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>1484</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000268.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0274</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>&lt;1..&gt;530</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>530</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000268.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11605</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>&lt;1..&gt;530</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>530</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000268.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11605</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005909906.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>frameshifted; incomplete; partial in the middle of a contig; missing N-terminus and C-terminus; Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>2</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hemolysin</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>564..1040</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>564</GBInterval_from>\\n            <GBInterval_to>1040</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000268.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11610</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11980</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>564..1040</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>564</GBInterval_from>\\n            <GBInterval_to>1040</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000268.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11610</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11980</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_016339539.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527442.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MMNKKFNIKKNIGNLIFIVFLFPAYFFSTLISNYVQHYEYIFTDKNRIKKYKVFSDRRTLYAYIDLEPDMYTEGEFSDFKIKKVYENSDYFIGYNFDVNRDSEEIENKYYVVVNKNKATMEFFSEKEFKEKLRNINDEKFIYVYPFLKKRGTKIGWYR</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>1064..&gt;1484</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1064</GBInterval_from>\\n            <GBInterval_to>1484</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000268.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11615</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11985</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>1064..&gt;1484</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1064</GBInterval_from>\\n            <GBInterval_to>1484</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000268.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11615</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11985</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_016361408.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527443.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKYKIIETVKISMIMTTILNFFICIATELRYIEFIYLLLFLGINLYMIGVKGYFKIETYNYSLYTIRKKIVDSQRATTLSINILIFFYIFKEPYFFLSKYKILILLISIVIGYSSLIFLQKIKIKLSLNFFKEIIKIFLI</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000268.1:1..1484)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0274, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893207\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000267</GBSeq_locus>\\n    <GBSeq_length>7464</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0273, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000267</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000267.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000267.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0273</GBSeqid>\\n      <GBSeqid>gi|2240893207</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..7464</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..7464</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000267.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..7464</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>7464</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0273</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>&lt;1..1801</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>1801</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11570</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>&lt;1..1801</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>1801</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11570</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: ab initio prediction:GeneMarkS-2+</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>incomplete; too short partial abutting assembly gap; missing N-terminus; Derived by automated computational analysis using gene prediction method: GeneMarkS-2+.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>2</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>1812..2288</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1812</GBInterval_from>\\n            <GBInterval_to>2288</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11575</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11935</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>1812..2288</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1812</GBInterval_from>\\n            <GBInterval_to>2288</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11575</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11935</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005973529.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_022070496.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MDVIVRISKEKKEKDFRMVVIPSGRDKIGIIQTKDYGIIAYPDKKDFEKIGEIINWAFNESDDKVIENSSDTKIEKQFYNCSSYRKVTNEYNEIWFEFFKGIYSISLLKKDGNFFVTFKDENKEAVKHIFKEKPTALELGTKVMEMFEYKERYDELIE</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>3650..4303</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3650</GBInterval_from>\\n            <GBInterval_to>4303</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11580</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11940</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>3650..4303</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3650</GBInterval_from>\\n            <GBInterval_to>4303</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11580</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11940</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: ab initio prediction:GeneMarkS-2+</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: GeneMarkS-2+.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527437.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MVKEIENVGDTTKTIGQKNKQGQIGQSNPMKNNNLNKSQADISDVSNKQSTPAYANKIKNGYKPVYLTKPEQGGTLVIAAGNNPIKGVYNIDMKENIKIGVFKGDATNLVNVPTGSQAKVIIENPNGFDPLNPEILRVVKEGGEIEITGIKSNKEFYNIYSGKVEVPKGFEIIEVGEIPENFQKQGFRTDGKLIGTKNGEGFPKKTDKIIRIRKIKK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>4316..5137</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>4316</GBInterval_from>\\n            <GBInterval_to>5137</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11585</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11945</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>4316..5137</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>4316</GBInterval_from>\\n            <GBInterval_to>5137</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11585</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11945</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008794358.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>stage V sporulation protein K</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527438.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKIKLKTQSLDDWELNLFESYSTMPIIINRTMIFGAYNVEFEMTANIWNQLPEEYKRKIYKYNWKKMIKSMLITVTDITAYSFCFGCNVKLKESLIMEEIYKDFDENKKIEFLTPGCDFPDSSMSVYFQHLGEVYAEVDLDELVAISDEENSYQYSIMPKLMEASNYRKRRENNTGKLEQIYNEQLIIKNLVNKNINELSKEEIQKVLDNFLLINNLKYLIKIIKKSKEFEIVISNKLKNEIEHWLRDIQIKIKTEEEEKIHQEIKEILKEQL</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>5134..5979</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>5134</GBInterval_from>\\n            <GBInterval_to>5979</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11590</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11950</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>5134..5979</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>5134</GBInterval_from>\\n            <GBInterval_to>5979</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11590</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11950</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008794769.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527439.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKIKIKLESLENTLNLSNSFCTIPIIMDKSMMFGAYGVDLNMPEYIWYQLPEECKKKIYNYKEHTIKAMIITITNISAYSITISHHEKFKEAITMEEFYEDFSENKKIERFLCECDFPYSNMSVYFQNLGEIYAEFELEDWVYYEKEAKEEWKIKERERRNIREIIKEALLERIEEKQSNFEDSLKYWVKVVELTEEKIKEIFLKFPTVLHRGGFRLLLELKYMMSIIENSEKFKLIIPENVKNEIGYWLTDMQGEIRTEEEKNLFKEIREKLNLKKVYGD</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>6463..6660</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>6463</GBInterval_from>\\n            <GBInterval_to>6660</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11595</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11960</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>6463..6660</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>6463</GBInterval_from>\\n            <GBInterval_to>6660</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11595</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11960</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: ab initio prediction:GeneMarkS-2+</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: GeneMarkS-2+.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527440.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MAYRFIQKYSFLFGVRWLLRRFNIHLNAYYNYVKNKKKKNIKEKENIKSIIKDMYHSNNGILGHR</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>6971..&gt;7464</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>6971</GBInterval_from>\\n            <GBInterval_to>7464</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11600</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>6971..&gt;7464</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>6971</GBInterval_from>\\n            <GBInterval_to>7464</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000267.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11600</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005903864.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hemagglutinin repeat-containing protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527441.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MGATIKADKDLKVTAQGTKLSLDAVKADNTVGKDNVKLAEANNNFYANMGVNLGFNKSSSKSNSHSESGVVTAIKGKDENSSITYNNVKNIEYVGTQAKDTKFIYNNVDNITKKAVELNNYSLSDSKSSGVSAGVNIGYGRKVLTDNASVSVSASKSNMNSNGTT</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000267.1:1..7464)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0273, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893206\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000266</GBSeq_locus>\\n    <GBSeq_length>494</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0272, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000266</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000266.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000266.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0272</GBSeqid>\\n      <GBSeqid>gi|2240893206</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..494</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..494</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000266.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..494</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>494</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000266.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0272</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>&lt;1..&gt;494</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>494</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000266.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11565</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>&lt;1..&gt;494</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>494</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000266.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11565</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005895493.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>incomplete; too short partial abutting assembly gap; missing N-terminus and C-terminus; Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>3</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>ShlB/FhaC/HecB family hemolysin secretion/activation protein</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000266.1:1..494)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0272, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893205\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000265</GBSeq_locus>\\n    <GBSeq_length>811</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0271, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000265</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000265.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000265.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0271</GBSeqid>\\n      <GBSeqid>gi|2240893205</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..811</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..811</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000265.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..811</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>811</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000265.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0271</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>122..198</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>122</GBInterval_from>\\n            <GBInterval_to>198</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000265.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11535</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11895</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>tRNA</GBFeature_key>\\n        <GBFeature_location>122..198</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>122</GBInterval_from>\\n            <GBInterval_to>198</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000265.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11535</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11895</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>tRNA-Pro</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: profile:tRNAscan-SE:2.0.7</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: tRNAscan-SE.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>anticodon</GBQualifier_name>\\n            <GBQualifier_value>(pos:156..158,aa:Pro,seq:tgg)</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transcription</GBQualifier_name>\\n            <GBQualifier_value>CGGAATATAGCGCAGTCCGGTAGCGCACCTGCCTTGGGAGCAGGGGGCCGCAAGTTCAAATCTTGCTATTCCGACCA</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>206..281</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>206</GBInterval_from>\\n            <GBInterval_to>281</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000265.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11540</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11900</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>tRNA</GBFeature_key>\\n        <GBFeature_location>206..281</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>206</GBInterval_from>\\n            <GBInterval_to>281</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000265.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11540</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11900</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>tRNA-Gly</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: profile:tRNAscan-SE:2.0.7</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: tRNAscan-SE.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>anticodon</GBQualifier_name>\\n            <GBQualifier_value>(pos:239..241,aa:Gly,seq:tcc)</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transcription</GBQualifier_name>\\n            <GBQualifier_value>GCGGGAATAGCTCAGTTGGTAGAGCGTCAGCCTTCCAAGCTGAATGTCGCGAGTTCGATTCTCGTTTCCCGCTCCA</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>294..369</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>294</GBInterval_from>\\n            <GBInterval_to>369</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000265.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11545</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11905</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>tRNA</GBFeature_key>\\n        <GBFeature_location>294..369</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>294</GBInterval_from>\\n            <GBInterval_to>369</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000265.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11545</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11905</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>tRNA-His</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: profile:tRNAscan-SE:2.0.7</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: tRNAscan-SE.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>anticodon</GBQualifier_name>\\n            <GBQualifier_value>(pos:327..329,aa:His,seq:gtg)</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transcription</GBQualifier_name>\\n            <GBQualifier_value>GTGACCGTAGTTCAGTTGGTAGAGCGCCAGTTTGTGGCACTGGTTGTCGCGAGTTCGATCCTCGTCGGTCACCCCA</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>373..448</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>373</GBInterval_from>\\n            <GBInterval_to>448</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000265.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11550</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11910</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>tRNA</GBFeature_key>\\n        <GBFeature_location>373..448</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>373</GBInterval_from>\\n            <GBInterval_to>448</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000265.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11550</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11910</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>tRNA-Lys</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: profile:tRNAscan-SE:2.0.7</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: tRNAscan-SE.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>anticodon</GBQualifier_name>\\n            <GBQualifier_value>(pos:406..408,aa:Lys,seq:ttt)</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transcription</GBQualifier_name>\\n            <GBQualifier_value>GCGTCATTAGCTCAGTTGGTAGAGCACACGACTTTTAATCGTGTTGTCACAAGTTCAAATCTTGTATGACGCACCA</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>461..544</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>461</GBInterval_from>\\n            <GBInterval_to>544</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000265.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11555</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11915</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>tRNA</GBFeature_key>\\n        <GBFeature_location>461..544</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>461</GBInterval_from>\\n            <GBInterval_to>544</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000265.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11555</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11915</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>tRNA-Leu</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: profile:tRNAscan-SE:2.0.7</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: tRNAscan-SE.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>anticodon</GBQualifier_name>\\n            <GBQualifier_value>(pos:495..497,aa:Leu,seq:tag)</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transcription</GBQualifier_name>\\n            <GBQualifier_value>GCGGGGATGGCGGAATTGGCAGACGCGCTAGACTTAGGATCTAGTGTCCCAGACGTGAGAGTTCAAGTCTCTCTCTCCGCACCA</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>586..660</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>586</GBInterval_from>\\n            <GBInterval_to>660</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000265.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11560</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11920</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>tRNA</GBFeature_key>\\n        <GBFeature_location>586..660</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>586</GBInterval_from>\\n            <GBInterval_to>660</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000265.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11560</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11920</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>tRNA-Gln</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: profile:tRNAscan-SE:2.0.7</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: tRNAscan-SE.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>anticodon</GBQualifier_name>\\n            <GBQualifier_value>(pos:618..620,aa:Gln,seq:ttg)</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transcription</GBQualifier_name>\\n            <GBQualifier_value>TGGGGTGTCGCCAAGCGGTAAGGCAACGGACTTTGACTCCGTTATGCGTTGGTTCGAATCCAGCCACCCCAGCCA</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000265.1:1..811)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0271, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893204\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000264</GBSeq_locus>\\n    <GBSeq_length>351</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0270, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000264</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000264.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000264.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0270</GBSeqid>\\n      <GBSeqid>gi|2240893204</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..351</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..351</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000264.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..351</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>351</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000264.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0270</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>117..&gt;351</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>117</GBInterval_from>\\n            <GBInterval_to>351</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000264.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11530</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>117..&gt;351</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>117</GBInterval_from>\\n            <GBInterval_to>351</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000264.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11530</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_016361159.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>helix-turn-helix domain-containing protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527436.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MEKAYKFRFYPTKTQIKILNSIFGCVRYVYNHFLDLKQKLYSTEKKSISYSECSKELTVLKKEKEWLKDVDKFSLQNS</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000264.1:1..351)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0270, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893203\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000263</GBSeq_locus>\\n    <GBSeq_length>816</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0268, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000263</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000263.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000263.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0268</GBSeqid>\\n      <GBSeqid>gi|2240893203</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..816</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..816</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000263.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..816</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>816</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000263.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0268</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>&lt;1..230</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>230</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000263.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11520</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11880</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>&lt;1..230</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>230</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000263.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11520</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11880</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008701199.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>incomplete; too short partial abutting assembly gap; missing N-terminus; Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>3</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>431..&gt;816</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>431</GBInterval_from>\\n            <GBInterval_to>816</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000263.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11525</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11885</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>431..&gt;816</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>431</GBInterval_from>\\n            <GBInterval_to>816</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000263.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11525</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11885</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008695093.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527435.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MTLITIFSLIILFLLQFIYVVPESHYSILDQTGKVELKDYPELKEISFMYSTDLYIEFTEPINLELEKINLLLNDKVIGTIEINRNINDLKDFAEPYIDEKSEEKIIRKTYPLQKEFLRILGRNAEIY</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000263.1:1..816)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0268, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893202\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000262</GBSeq_locus>\\n    <GBSeq_length>1866</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0267, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000262</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000262.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000262.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0267</GBSeqid>\\n      <GBSeqid>gi|2240893202</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..1866</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..1866</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000262.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..1866</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>1866</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000262.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0267</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>261..806</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>261</GBInterval_from>\\n            <GBInterval_to>806</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000262.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>infC</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11495</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11855</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>261..806</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>261</GBInterval_from>\\n            <GBInterval_to>806</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000262.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>infC</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11495</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11855</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005895946.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_function</GBQualifier_name>\\n            <GBQualifier_value>GO:0003743 - translation initiation factor activity [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_process</GBQualifier_name>\\n            <GBQualifier_value>GO:0006413 - translational initiation [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>translation initiation factor IF-3</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_022069803.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MPVLFFQWRCSVISDKTRINEKIRGKEFRIISFDGEQLGIMTAEQALNLASSQGYDLVEIAPSATPPVCKIMDYSKYKYEQTRKLKEAKKNQKQVVIKEIKVTARIDSHDLETKLNQVTKFLEKENKVKVTLVLFGREKMHANLGVTTLDEIAEKFAEIAEVEKKYADKQKHLILSPKKIK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>883..1089</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>883</GBInterval_from>\\n            <GBInterval_to>1089</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000262.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>rpmI</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11500</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11860</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>883..1089</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>883</GBInterval_from>\\n            <GBInterval_to>1089</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000262.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>rpmI</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11500</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11860</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005893833.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_component</GBQualifier_name>\\n            <GBQualifier_value>GO:0022625 - cytosolic large ribosomal subunit [Evidence IEA]; GO:0000315 - organellar large ribosomal subunit [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_function</GBQualifier_name>\\n            <GBQualifier_value>GO:0003735 - structural constituent of ribosome [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_process</GBQualifier_name>\\n            <GBQualifier_value>GO:0006412 - translation [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>50S ribosomal protein L35</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_005906263.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MPKMKTHRGAKKRIKVTGTGKFIIKHSGKSHILTKKDRKRKNHLKKDAVVNETYKKHMQGLLPYGEGR</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>1114..1464</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1114</GBInterval_from>\\n            <GBInterval_to>1464</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000262.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>rplT</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11505</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11865</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>1114..1464</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1114</GBInterval_from>\\n            <GBInterval_to>1464</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000262.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>gene</GBQualifier_name>\\n            <GBQualifier_value>rplT</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11505</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11865</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_016339621.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_component</GBQualifier_name>\\n            <GBQualifier_value>GO:0005840 - ribosome [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_function</GBQualifier_name>\\n            <GBQualifier_value>GO:0019843 - rRNA binding [Evidence IEA]; GO:0003735 - structural constituent of ribosome [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>GO_process</GBQualifier_name>\\n            <GBQualifier_value>GO:0006412 - translation [Evidence IEA]</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>50S ribosomal protein L20</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527434.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MRVKTGIIRRKRHKRVLKAAKGFRGASGDAFKQAKQATRKAMAYATRDRKVNKRRMRQLWITRINSAARMNGVSYSVLMSGLKKAGILLDRKVLADIALNNATEFTKLVEAAKSAL</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>1512..1595</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1512</GBInterval_from>\\n            <GBInterval_to>1595</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000262.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11510</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11870</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>tRNA</GBFeature_key>\\n        <GBFeature_location>1512..1595</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1512</GBInterval_from>\\n            <GBInterval_to>1595</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000262.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11510</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11870</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>tRNA-Ser</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: profile:tRNAscan-SE:2.0.7</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: tRNAscan-SE.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>anticodon</GBQualifier_name>\\n            <GBQualifier_value>(pos:1545..1547,aa:Ser,seq:tga)</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transcription</GBQualifier_name>\\n            <GBQualifier_value>GGAAGGCTATCCTAATTGGTAAGGAACCGGTCTTGAAAACCGGCGTCGTAAGACTTTAGAGTTCGAGTCTCTAGCCTTCCGCCA</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>1618..1707</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1618</GBInterval_from>\\n            <GBInterval_to>1707</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000262.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11515</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11875</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>tRNA</GBFeature_key>\\n        <GBFeature_location>1618..1707</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1618</GBInterval_from>\\n            <GBInterval_to>1707</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000262.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11515</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11875</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>tRNA-Ser</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: profile:tRNAscan-SE:2.0.7</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: tRNAscan-SE.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>anticodon</GBQualifier_name>\\n            <GBQualifier_value>(pos:1652..1654,aa:Ser,seq:gct)</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transcription</GBQualifier_name>\\n            <GBQualifier_value>GGTGAGATGGCAGAGTGGCCTAATGCACTGACCTGCTAAGTCAGAGTACCAATTATGGTACCGAGGGTTCAAATCCCTCTCTCACCGCCA</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000262.1:1..1866)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0267, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893201\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000261</GBSeq_locus>\\n    <GBSeq_length>868</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0266, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000261</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000261.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000261.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0266</GBSeqid>\\n      <GBSeqid>gi|2240893201</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..868</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..868</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000261.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..868</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>868</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000261.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0266</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>&lt;1..&gt;868</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>868</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000261.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11490</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>&lt;1..&gt;868</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>868</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000261.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11490</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005912642.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>incomplete; too short partial abutting assembly gap; missing N-terminus and C-terminus; Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>2</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hemolysin</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000261.1:1..868)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0266, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893200\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000260</GBSeq_locus>\\n    <GBSeq_length>1556</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0265, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000260</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000260.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000260.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0265</GBSeqid>\\n      <GBSeqid>gi|2240893200</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..1556</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..1556</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000260.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..1556</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>1556</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000260.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0265</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>&lt;1..&gt;1556</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>1556</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000260.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11485</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>&lt;1..&gt;1556</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>1556</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000260.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11485</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008795587.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>incomplete; too short partial abutting assembly gap; missing N-terminus and C-terminus; Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hemolysin</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000260.1:1..1556)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0265, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893199\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000259</GBSeq_locus>\\n    <GBSeq_length>455</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0264, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000259</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000259.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000259.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0264</GBSeqid>\\n      <GBSeqid>gi|2240893199</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..455</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..455</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000259.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..455</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>455</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000259.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0264</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>complement(16..363)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>363</GBInterval_from>\\n            <GBInterval_to>16</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000259.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11480</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11840</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>complement(16..363)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>363</GBInterval_from>\\n            <GBInterval_to>16</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000259.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11480</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11840</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: protein motif:HMM:NF013776.2</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>AAA family ATPase</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527433.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKDWDTLFDYLLTKILNEKFILVIDEFQYLCMVNKNFSSIFQRIYDEKLKNKNIMIILCGSLISMMYSETLAYESALYGRRTAQIKLQPIKFKYYNEFFKNKSIQDLIKNMVAHL</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000259.1:1..455)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0264, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893198\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000258</GBSeq_locus>\\n    <GBSeq_length>3806</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0263, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000258</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000258.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000258.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0263</GBSeqid>\\n      <GBSeqid>gi|2240893198</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..3806</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..3806</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000258.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..3806</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>3806</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000258.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0263</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>complement(179..793)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>793</GBInterval_from>\\n            <GBInterval_to>179</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000258.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11450</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11810</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>complement(179..793)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>793</GBInterval_from>\\n            <GBInterval_to>179</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000258.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11450</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11810</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: ab initio prediction:GeneMarkS-2+</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: GeneMarkS-2+.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527429.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MRRILIFIIGTIFLSGCMISKTSYRIKSGNIEHKYYSLSYSARASNIIYLTKENAKIKNKVFLVLEFPEYGIKSKGDVIGIETSIEGDNEDKKKYQYPELLEDIYIMKNNKQKIKISRENLVYSYKKDPNTKFPGVHIYTNQKFTGPVTLELGKVKINGEIYNIPPLHLQRYKESANWGLVDDFLMQGNVRNKPLYESGKWLDD</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>complement(799..1329)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1329</GBInterval_from>\\n            <GBInterval_to>799</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000258.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11455</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11815</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>complement(799..1329)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1329</GBInterval_from>\\n            <GBInterval_to>799</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000258.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11455</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11815</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: ab initio prediction:GeneMarkS-2+</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: GeneMarkS-2+.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527430.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKVFFNPSHGIIADVIESALDVTGFYTGGLIRSSNSKKLAELAIKNPELYNGTLSHSQGSAILGNSMKVVNLKGKKEVLQGKDISMNGSPLSMWAMKNFAEEKGFKFSYKVNVVDPITWLGANKWNTDVWWHTTYSKYQEWNNEKNKYIMQKTKDLEKPIFIKDVKDFTKKYYKGK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>complement(1478..1648)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1648</GBInterval_from>\\n            <GBInterval_to>1478</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000258.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11460</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11820</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>complement(1478..1648)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1648</GBInterval_from>\\n            <GBInterval_to>1478</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000258.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11460</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11820</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_009423194.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>thioredoxinprotein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249524326.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MKKILLIILGIFLFNACMTNDYYYISPTGEKSKIVKPNPPIKIEKNEVINKVSIYQ</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>complement(1716..2240)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>2240</GBInterval_from>\\n            <GBInterval_to>1716</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000258.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11465</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11825</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>complement(1716..2240)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>2240</GBInterval_from>\\n            <GBInterval_to>1716</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000258.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11465</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11825</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008794299.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527431.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MAKKIIVIVFSILIFNACSMLDKQYSFSTIEYIKGISIVESPKNIYTENSSLSSIWISDSRNWDGKKRKLKLLDNKIKIIKDGREYVIPYSDIENKNDIYIYIKVVNIVNEDFLLYIGRVQLDNGEIVNIPPLHFKKYIYIQKNGILAGIDPVGRFKSYSGTLEEYKKNGWKDE</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>complement(2225..2638)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>2638</GBInterval_from>\\n            <GBInterval_to>2225</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000258.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11470</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11830</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>complement(2225..2638)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>2638</GBInterval_from>\\n            <GBInterval_to>2225</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000258.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11470</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11830</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: ab initio prediction:GeneMarkS-2+</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: GeneMarkS-2+.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527432.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MIAHSQGTIIYASKLRDDLKTEEGRLKISKMKENYFLGIAVVPWILDDIIDKMNALGTKAEKRRNNYDFVSIITLNLKPNTNGKGHGIIGYDPYYFRWRDDIGKYFYDEKNKEIGKIPKIKENEVINKAGVYQWQKK</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>complement(2752..3255)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3255</GBInterval_from>\\n            <GBInterval_to>2752</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000258.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11475</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11835</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>complement(2752..3255)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>3255</GBInterval_from>\\n            <GBInterval_to>2752</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000258.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11475</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>old_locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_11835</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: ab initio prediction:GeneMarkS-2+</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: GeneMarkS-2+.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hypothetical protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249525425.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MGEYGKNLGSDLDKMSGGKYDLENKLGQGFYNTYREIEDFIDYKLYNKFTGIIPTKNVQGGLAGELQSPSSLLNKNLIPVVITYSGEGDKDNNFKIKSVVEDLTQEKYKEIKDKNSGEIPLINGVGNDVSDFTDGRFAGTLTPDEHKRIKKGETIKKIGIFNPTNGL</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000258.1:1..3806)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0263, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893197\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000257</GBSeq_locus>\\n    <GBSeq_length>609</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0262, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000257</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000257.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000257.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0262</GBSeqid>\\n      <GBSeqid>gi|2240893197</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..609</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..609</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000257.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..609</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>609</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000257.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0262</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>complement(&lt;1..&gt;609)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>609</GBInterval_from>\\n            <GBInterval_to>1</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000257.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11445</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>complement(&lt;1..&gt;609)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>609</GBInterval_from>\\n            <GBInterval_to>1</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000257.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11445</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_020788966.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>3</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hemagglutinin repeat-containing protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527428.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>ILNQTASQIVSNKDINIEATDKVKAKSVDIYAKNDINISGDNGVEISTANNSYDNTTKQSSSRIGTSVGINSAIVNTVENVRDIKKLTDFSGNSYDILNNASKVVGAIKDGAKATIAVADTNYNGATDAGYNNLKIMNDVFKANISYNKSKSKSSVHNETVEKSSLVSGRNMNIKSKNGSITISGTDVKVGNDLDLNAKKDI</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000257.1:1..609)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0262, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893196\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000256</GBSeq_locus>\\n    <GBSeq_length>1035</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0261, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000256</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000256.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000256.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0261</GBSeqid>\\n      <GBSeqid>gi|2240893196</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..1035</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..1035</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000256.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..1035</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>1035</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000256.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0261</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>complement(&lt;1..&gt;1035)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1035</GBInterval_from>\\n            <GBInterval_to>1</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000256.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11440</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>complement(&lt;1..&gt;1035)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1035</GBInterval_from>\\n            <GBInterval_to>1</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000256.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11440</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_008795587.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>3</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hemagglutinin repeat-containing protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527427.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>ADLSISKAGTKGRGNGTNYINSTIDVGGKLKTNSENLTLSGANVEADKLDINAKNLVIESKQDKSEKKDSSYGGSFSIDLVNPSSFSANINGSKGSGEKEWVNKQTTLIARNGGKVDTDSLTNIGAVIGSENEKEKLKVSANKVVVKDLEDKNKYENIGGGITIGTDVPNTSIKHDKIDKEQMDRATAINTDFEISGKSTSGEDLGFNTDIDKSQEKTKDEEKHLDAELHTDLLGKDKQEELKKAGGIISDLTTALGNKSKTEGDFLERYKQLSMMRAIGDQVAKNPEYLSILDKKAIKNGKIDDDVQKDQVSVMNKLLNDALRAKGYKGPDIKMVLTDVTDPN</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000256.1:1..1035)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0261, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893195\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000255</GBSeq_locus>\\n    <GBSeq_length>739</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0260, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000255</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000255.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000255.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0260</GBSeqid>\\n      <GBSeqid>gi|2240893195</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..739</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..739</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000255.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..739</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>739</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000255.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0260</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>complement(&lt;1..457)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>457</GBInterval_from>\\n            <GBInterval_to>1</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000255.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11430</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>complement(&lt;1..457)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>457</GBInterval_from>\\n            <GBInterval_to>1</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000255.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11430</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: protein motif:HMM:NF015803.2</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>ShlB/FhaC/HecB family hemolysin secretion/activation protein</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>protein_id</GBQualifier_name>\\n            <GBQualifier_value>WP_249527426.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>translation</GBQualifier_name>\\n            <GBQualifier_value>MTLQDIDLDTHNIFGLNESLDFSYQNKLQRQYKERDTKNFSFGVSVPFKYWTFSYNYDSSEYLRTINIYNRKYRATGKTKNQTFGIRKMLHRNENHKIDIGAKITLKDSKNYIDDVRLVSSSRKLSVLTVDTTYTGRILSGLLSTNVGVSFG</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>complement(423..&gt;739)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>739</GBInterval_from>\\n            <GBInterval_to>423</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000255.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11435</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>complement(423..&gt;739)</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>739</GBInterval_from>\\n            <GBInterval_to>423</GBInterval_to>\\n            <GBInterval_iscomp value=\"true\"/>\\n            <GBInterval_accession>NZ_AYHB01000255.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11435</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: protein motif:HMM:NF036420.2</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>incomplete; too short partial abutting assembly gap; missing N-terminus; Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>3</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>ShlB/FhaC/HecB family hemolysin secretion/activation protein</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000255.1:1..739)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0260, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "Id 2240893194\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'<?xml version=\"1.0\" encoding=\"UTF-8\"  ?>\\n<!DOCTYPE GBSet PUBLIC \"-//NCBI//NCBI GBSeq/EN\" \"https://www.ncbi.nlm.nih.gov/dtd/NCBI_GBSeq.dtd\">\\n<GBSet>\\n  <GBSeq>\\n\\n    <GBSeq_locus>NZ_AYHB01000254</GBSeq_locus>\\n    <GBSeq_length>531</GBSeq_length>\\n    <GBSeq_strandedness>double</GBSeq_strandedness>\\n    <GBSeq_moltype>DNA</GBSeq_moltype>\\n    <GBSeq_topology>linear</GBSeq_topology>\\n    <GBSeq_division>CON</GBSeq_division>\\n    <GBSeq_update-date>19-MAY-2022</GBSeq_update-date>\\n    <GBSeq_create-date>19-MAY-2022</GBSeq_create-date>\\n    <GBSeq_definition>Fusobacterium nucleatum YWH7199 contig0259, whole genome shotgun sequence</GBSeq_definition>\\n    <GBSeq_primary-accession>NZ_AYHB01000254</GBSeq_primary-accession>\\n    <GBSeq_accession-version>NZ_AYHB01000254.1</GBSeq_accession-version>\\n    <GBSeq_other-seqids>\\n      <GBSeqid>ref|NZ_AYHB01000254.1|</GBSeqid>\\n      <GBSeqid>gnl|WGS:NZ_AYHB01|contig0259</GBSeqid>\\n      <GBSeqid>gi|2240893194</GBSeqid>\\n    </GBSeq_other-seqids>\\n    <GBSeq_project>PRJNA224116</GBSeq_project>\\n    <GBSeq_keywords>\\n      <GBKeyword>WGS</GBKeyword>\\n      <GBKeyword>RefSeq</GBKeyword>\\n    </GBSeq_keywords>\\n    <GBSeq_source>Fusobacterium nucleatum YWH7199</GBSeq_source>\\n    <GBSeq_organism>Fusobacterium nucleatum YWH7199</GBSeq_organism>\\n    <GBSeq_taxonomy>Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium</GBSeq_taxonomy>\\n    <GBSeq_references>\\n      <GBReference>\\n        <GBReference_reference>1</GBReference_reference>\\n        <GBReference_position>1..531</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Comparative genomics of Fusobacterium nucleatum clinical isolates</GBReference_title>\\n        <GBReference_journal>Unpublished</GBReference_journal>\\n      </GBReference>\\n      <GBReference>\\n        <GBReference_reference>2</GBReference_reference>\\n        <GBReference_position>1..531</GBReference_position>\\n        <GBReference_authors>\\n          <GBAuthor>Kittichotirat,W.</GBAuthor>\\n          <GBAuthor>Han,Y.W.</GBAuthor>\\n          <GBAuthor>Bumgarner,R.E.</GBAuthor>\\n        </GBReference_authors>\\n        <GBReference_title>Direct Submission</GBReference_title>\\n        <GBReference_journal>Submitted (06-FEB-2014) Department of Microbiology, University of Washington, 960 Republican St., Seattle, WA 98109, USA</GBReference_journal>\\n      </GBReference>\\n    </GBSeq_references>\\n    <GBSeq_comment>REFSEQ INFORMATION: The reference sequence is identical to AYHB01000254.1.; The annotation was added by the NCBI Prokaryotic Genome Annotation Pipeline (PGAP). Information about PGAP can be found here: https://www.ncbi.nlm.nih.gov/genome/annotation_prok/; ; ##Genome-Assembly-Data-START## ; Assembly Method :: Velvet v. 1.2.07 ; Genome Coverage :: 89.85x ; Sequencing Technology :: Illumina HiSeq ; ##Genome-Assembly-Data-END##; ; ##Genome-Annotation-Data-START## ; Annotation Provider :: NCBI RefSeq ; Annotation Date :: 05/18/2022 09:00:33 ; Annotation Pipeline :: NCBI Prokaryotic Genome Annotation Pipeline (PGAP) ; Annotation Method :: Best-placed reference protein set; GeneMarkS-2+ ; Annotation Software revision :: 6.1 ; Features Annotated :: Gene; CDS; rRNA; tRNA; ncRNA; repeat_region ; Genes (total) :: 2,374 ; CDSs (total) :: 2,326 ; Genes (coding) :: 2,194 ; CDSs (with protein) :: 2,194 ; Genes (RNA) :: 48 ; rRNAs :: 1, 2, 3 (5S, 16S, 23S) ; complete rRNAs :: 1 (5S) ; partial rRNAs :: 2, 3 (16S, 23S) ; tRNAs :: 39 ; ncRNAs :: 3 ; Pseudo Genes (total) :: 132 ; CDSs (without protein) :: 132 ; Pseudo Genes (ambiguous residues) :: 0 of 132 ; Pseudo Genes (frameshifted) :: 35 of 132 ; Pseudo Genes (incomplete) :: 103 of 132 ; Pseudo Genes (internal stop) :: 14 of 132 ; Pseudo Genes (multiple problems) :: 18 of 132 ; CRISPR Arrays :: 4 ; ##Genome-Annotation-Data-END##</GBSeq_comment>\\n    <GBSeq_feature-table>\\n      <GBFeature>\\n        <GBFeature_key>source</GBFeature_key>\\n        <GBFeature_location>1..531</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>531</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000254.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>organism</GBQualifier_name>\\n            <GBQualifier_value>Fusobacterium nucleatum YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>mol_type</GBQualifier_name>\\n            <GBQualifier_value>genomic DNA</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>submitter_seqid</GBQualifier_name>\\n            <GBQualifier_value>contig0259</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>strain</GBQualifier_name>\\n            <GBQualifier_value>YWH7199</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>isolation_source</GBQualifier_name>\\n            <GBQualifier_value>term stillborn infant</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>host</GBQualifier_name>\\n            <GBQualifier_value>Homo sapiens</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>db_xref</GBQualifier_name>\\n            <GBQualifier_value>taxon:1402206</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>country</GBQualifier_name>\\n            <GBQualifier_value>USA: OH</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>gene</GBFeature_key>\\n        <GBFeature_location>&lt;1..&gt;531</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>531</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000254.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11425</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n      <GBFeature>\\n        <GBFeature_key>CDS</GBFeature_key>\\n        <GBFeature_location>&lt;1..&gt;531</GBFeature_location>\\n        <GBFeature_intervals>\\n          <GBInterval>\\n            <GBInterval_from>1</GBInterval_from>\\n            <GBInterval_to>531</GBInterval_to>\\n            <GBInterval_accession>NZ_AYHB01000254.1</GBInterval_accession>\\n          </GBInterval>\\n        </GBFeature_intervals>\\n        <GBFeature_partial5 value=\"true\"/>\\n        <GBFeature_partial3 value=\"true\"/>\\n        <GBFeature_quals>\\n          <GBQualifier>\\n            <GBQualifier_name>locus_tag</GBQualifier_name>\\n            <GBQualifier_value>YWH7199_RS11425</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>inference</GBQualifier_name>\\n            <GBQualifier_value>COORDINATES: similar to AA sequence:RefSeq:WP_005910012.1</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>note</GBQualifier_name>\\n            <GBQualifier_value>incomplete; too short partial abutting assembly gap; missing N-terminus and C-terminus; Derived by automated computational analysis using gene prediction method: Protein Homology.</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>pseudo</GBQualifier_name>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>codon_start</GBQualifier_name>\\n            <GBQualifier_value>2</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>transl_table</GBQualifier_name>\\n            <GBQualifier_value>11</GBQualifier_value>\\n          </GBQualifier>\\n          <GBQualifier>\\n            <GBQualifier_name>product</GBQualifier_name>\\n            <GBQualifier_value>hemolysin</GBQualifier_value>\\n          </GBQualifier>\\n        </GBFeature_quals>\\n      </GBFeature>\\n    </GBSeq_feature-table>\\n    <GBSeq_contig>join(AYHB01000254.1:1..531)</GBSeq_contig>\\n    <GBSeq_xrefs>\\n      <GBXref>\\n        <GBXref_dbname>BioProject</GBXref_dbname>\\n        <GBXref_id>PRJNA224116</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>BioSample</GBXref_dbname>\\n        <GBXref_id>SAMN07819093</GBXref_id>\\n      </GBXref>\\n      <GBXref>\\n        <GBXref_dbname>Assembly</GBXref_dbname>\\n        <GBXref_id>GCF_023383645.1</GBXref_id>\\n      </GBXref>\\n    </GBSeq_xrefs>\\n  </GBSeq>\\n\\n</GBSet>\\n'\n",
      "['Fusobacterium nucleatum YWH7199 contig0259, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium']\n",
      "--------------------------------\n",
      "this is the id list [['2240894608'], ['2240894606'], ['2240894605'], ['2240894604'], ['2240894603'], ['2240894602'], ['2240894601'], ['2240894593'], ['2240894592'], ['2240894591'], ['2240894590'], ['2240894589'], ['2240894588'], ['2240894583'], ['2240894582'], ['2240894581'], ['2240894580'], ['2240894574'], ['2240894573'], ['2240894572'], ['2240894571'], ['2240894570'], ['2240894569'], ['2240894568'], ['2240894567'], ['2240894566'], ['2240894565'], ['2240894564'], ['2240894563'], ['2240894562'], ['2240894555'], ['2240894553'], ['2240894552'], ['2240894551'], ['2240894550'], ['2240894549'], ['2240894548'], ['2240894547'], ['2240894546'], ['2240894545'], ['2240894544'], ['2240894509'], ['2240894508'], ['2240894507'], ['2240894506'], ['2240894505'], ['2240894498'], ['2240894485'], ['2240894483'], ['2240894482'], ['2240894481'], ['2240894480'], ['2240894479'], ['2240894478'], ['2240894477'], ['2240894475'], ['2240894467'], ['2240894466'], ['2240894465'], ['2240894464'], ['2240894463'], ['2240894462'], ['2240894461'], ['2240894460'], ['2240894459'], ['2240894458'], ['2240894457'], ['2240894456'], ['2240894455'], ['2240894454'], ['2240894453'], ['2240894452'], ['2240894451'], ['2240894450'], ['2240894449'], ['2240894448'], ['2240893217'], ['2240893216'], ['2240893215'], ['2240893214'], ['2240893213'], ['2240893212'], ['2240893211'], ['2240893210'], ['2240893209'], ['2240893208'], ['2240893207'], ['2240893206'], ['2240893205'], ['2240893204'], ['2240893203'], ['2240893202'], ['2240893201'], ['2240893200'], ['2240893199'], ['2240893198'], ['2240893197'], ['2240893196'], ['2240893195'], ['2240893194']]\n",
      "Data Inserted in the table: \n",
      "(2240894608, 'MAG: Candidatus Bipolaricaulota bacterium isolate HOT.Cluster.56, whole genome shotgun sequencing project', 'Candidatus Bipolaricaulota bacterium (hot springs metagenome)', 'Candidatus Bipolaricaulota bacterium', 'Bacteria; Candidatus Bipolaricaulota')\n",
      "(2240894606, 'MAG: Dehalococcoidia bacterium isolate HOT.Cluster.58, whole genome shotgun sequencing project', 'Dehalococcoidia bacterium (hot springs metagenome)', 'Dehalococcoidia bacterium', 'Bacteria; Chloroflexi; Dehalococcoidia')\n",
      "(2240894605, 'MAG: Chloroflexi bacterium isolate HOT.Cluster.3, whole genome shotgun sequencing project', 'Chloroflexi bacterium (hot springs metagenome)', 'Chloroflexi bacterium', 'Bacteria; Chloroflexi')\n",
      "(2240894604, 'MAG: Firmicutes bacterium isolate HOT.Cluster.78, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes')\n",
      "(2240894603, 'MAG: Candidatus Rokubacteria bacterium isolate HOT.Cluster.5, whole genome shotgun sequencing project', 'Candidatus Rokubacteria bacterium (hot springs metagenome)', 'Candidatus Rokubacteria bacterium', 'Bacteria; Candidatus Rokubacteria; unclassified Candidatus Rokubacteria')\n",
      "(2240894602, 'MAG: Alicyclobacillus sp. isolate HOT.Cluster.61, whole genome shotgun sequencing project', 'Alicyclobacillus sp. (hot springs metagenome)', 'Alicyclobacillus sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus')\n",
      "(2240894601, 'MAG: Peptococcaceae bacterium isolate HOT.Cluster.86, whole genome shotgun sequencing project', 'Peptococcaceae bacterium (hot springs metagenome)', 'Peptococcaceae bacterium', 'Bacteria; Firmicutes; Clostridia; Eubacteriales; Peptococcaceae')\n",
      "(2240894593, 'MAG: Alicyclobacillus herbarius isolate HOT.CON.104, whole genome shotgun sequencing project', 'Alicyclobacillus herbarius (hot springs metagenome)', 'Alicyclobacillus herbarius', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus')\n",
      "(2240894592, 'MAG: Firmicutes bacterium isolate HOT.CON.146, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes')\n",
      "(2240894591, 'MAG: Thermomonas hydrothermalis isolate HOT.CON.106, whole genome shotgun sequencing project', 'Thermomonas hydrothermalis (hot springs metagenome)', 'Thermomonas hydrothermalis', 'Bacteria; Proteobacteria; Gammaproteobacteria; Xanthomonadales; Xanthomonadaceae; Thermomonas')\n",
      "(2240894590, 'MAG: Alicyclobacillus shizuokensis isolate HOT.CON.119, whole genome shotgun sequencing project', 'Alicyclobacillus shizuokensis (hot springs metagenome)', 'Alicyclobacillus shizuokensis', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus')\n",
      "(2240894589, 'MAG: Anoxybacillus ayderensis isolate HOT.CON.111, whole genome shotgun sequencing project', 'Anoxybacillus ayderensis (hot springs metagenome)', 'Anoxybacillus ayderensis', 'Bacteria; Firmicutes; Bacilli; Bacillales; Bacillaceae; Anoxybacillus')\n",
      "(2240894588, 'MAG: Armatimonadetes bacterium isolate HOT.CON.113, whole genome shotgun sequencing project', 'Armatimonadetes bacterium (hot springs metagenome)', 'Armatimonadetes bacterium', 'Bacteria; Armatimonadetes')\n",
      "(2240894583, 'MAG: Fimbriimonadales bacterium isolate HOT.CON.102, whole genome shotgun sequencing project', 'Fimbriimonadales bacterium (hot springs metagenome)', 'Fimbriimonadales bacterium', 'Bacteria; Armatimonadetes; Fimbriimonadia; Fimbriimonadales; unclassified Fimbriimonadales')\n",
      "(2240894582, 'MAG: Syntrophobacterales bacterium isolate HOT.CON.10, whole genome shotgun sequencing project', 'Syntrophobacterales bacterium (hot springs metagenome)', 'Syntrophobacterales bacterium', 'Bacteria; Proteobacteria; Deltaproteobacteria; Syntrophobacterales')\n",
      "(2240894581, 'MAG: Peptococcaceae bacterium isolate HOT.CON.130, whole genome shotgun sequencing project', 'Peptococcaceae bacterium (hot springs metagenome)', 'Peptococcaceae bacterium', 'Bacteria; Firmicutes; Clostridia; Eubacteriales; Peptococcaceae')\n",
      "(2240894580, 'MAG: Geminicoccaceae bacterium isolate HOT.CON.139, whole genome shotgun sequencing project', 'Geminicoccaceae bacterium (hot springs metagenome)', 'Geminicoccaceae bacterium', 'Bacteria; Proteobacteria; Alphaproteobacteria; Rhodospirillales; Geminicoccaceae')\n",
      "(2240894574, 'MAG: Paenibacillus sp. isolate HOT.CON.154, whole genome shotgun sequencing project', 'Paenibacillus sp. (hot springs metagenome)', 'Paenibacillus sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Paenibacillaceae; Paenibacillus')\n",
      "(2240894573, 'MAG: Alicyclobacillus macrosporangiidus isolate HOT.CON.25, whole genome shotgun sequencing project', 'Alicyclobacillus macrosporangiidus (hot springs metagenome)', 'Alicyclobacillus macrosporangiidus', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus')\n",
      "(2240894572, 'MAG: Anoxybacillus sp. isolate HOT.CON.20, whole genome shotgun sequencing project', 'Anoxybacillus sp. (hot springs metagenome)', 'Anoxybacillus sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Bacillaceae; Anoxybacillus; unclassified Anoxybacillus')\n",
      "(2240894571, 'MAG: Firmicutes bacterium isolate HOT.CON.150, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes')\n",
      "(2240894570, 'MAG: Alicyclobacillus sp. isolate HOT.CON.16, whole genome shotgun sequencing project', 'Alicyclobacillus sp. (hot springs metagenome)', 'Alicyclobacillus sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus')\n",
      "(2240894569, 'MAG: Firmicutes bacterium isolate HOT.CON.49, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes')\n",
      "(2240894568, 'MAG: Candidatus Bathyarchaeota archaeon isolate HOT.CON.75, whole genome shotgun sequencing project', 'Candidatus Bathyarchaeota archaeon (hot springs metagenome)', 'Candidatus Bathyarchaeota archaeon', 'Archaea; Candidatus Bathyarchaeota')\n",
      "(2240894567, 'MAG: Firmicutes bacterium isolate HOT.CON.82, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes')\n",
      "(2240894566, 'MAG: bacterium isolate HOT.CON.50, whole genome shotgun sequencing project', 'bacterium (hot springs metagenome)', 'bacterium', 'Bacteria')\n",
      "(2240894565, 'MAG: Kyrpidia sp. isolate HOT.CON.60, whole genome shotgun sequencing project', 'Kyrpidia sp. (hot springs metagenome)', 'Kyrpidia sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Kyrpidia; unclassified Kyrpidia')\n",
      "(2240894564, 'MAG: Bacillus sp. (in: Bacteria) isolate HOT.CON.97, whole genome shotgun sequencing project', 'Bacillus sp. (in: Bacteria) (hot springs metagenome)', 'Bacillus sp. (in: Bacteria)', 'Bacteria; Firmicutes; Bacilli; Bacillales; Bacillaceae; Bacillus')\n",
      "(2240894563, 'MAG: Firmicutes bacterium isolate HOT.MAX.058, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes')\n",
      "(2240894562, 'MAG: Firmicutes bacterium isolate HOT.MAX.062, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes')\n",
      "(2240894555, 'MAG: Alicyclobacillus sp. isolate HOT.MAX.041, whole genome shotgun sequencing project', 'Alicyclobacillus sp. (hot springs metagenome)', 'Alicyclobacillus sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus')\n",
      "(2240894553, 'MAG: Acidothermus sp. isolate HOT.MAX.052, whole genome shotgun sequencing project', 'Acidothermus sp. (hot springs metagenome)', 'Acidothermus sp.', 'Bacteria; Actinobacteria; Acidothermales; Acidothermaceae; Acidothermus; unclassified Acidothermus')\n",
      "(2240894552, 'MAG: Burkholderiales bacterium isolate HOT.MAX.068, whole genome shotgun sequencing project', 'Burkholderiales bacterium (hot springs metagenome)', 'Burkholderiales bacterium', 'Bacteria; Proteobacteria; Betaproteobacteria; Burkholderiales')\n",
      "(2240894551, 'MAG: Bryobacteraceae bacterium isolate HOT.MAX.089, whole genome shotgun sequencing project', 'Bryobacteraceae bacterium (hot springs metagenome)', 'Bryobacteraceae bacterium', 'Bacteria; Acidobacteria; Bryobacterales; Bryobacteraceae')\n",
      "(2240894550, 'MAG: Firmicutes bacterium isolate HOT.CON.99, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes')\n",
      "(2240894549, 'MAG: Meiothermus silvanus isolate HOT.CON.95, whole genome shotgun sequencing project', 'Meiothermus silvanus (hot springs metagenome)', 'Meiothermus silvanus', 'Bacteria; Deinococcus-Thermus; Deinococci; Thermales; Thermaceae; Meiothermus')\n",
      "(2240894548, 'MAG: Acidobacteriia bacterium isolate HOT.MAX.039, whole genome shotgun sequencing project', 'Acidobacteriia bacterium (hot springs metagenome)', 'Acidobacteriia bacterium', 'Bacteria; Acidobacteria')\n",
      "(2240894547, 'MAG: Armatimonadetes bacterium isolate HOT.MAX.070, whole genome shotgun sequencing project', 'Armatimonadetes bacterium (hot springs metagenome)', 'Armatimonadetes bacterium', 'Bacteria; Armatimonadetes')\n",
      "(2240894546, 'MAG: Acidothermus cellulolyticus isolate HOT.MAX.011, whole genome shotgun sequencing project', 'Acidothermus cellulolyticus (hot springs metagenome)', 'Acidothermus cellulolyticus', 'Bacteria; Actinobacteria; Acidothermales; Acidothermaceae; Acidothermus')\n",
      "(2240894545, 'MAG: Roseiflexus sp. isolate HOT.MAX.081, whole genome shotgun sequencing project', 'Roseiflexus sp. (hot springs metagenome)', 'Roseiflexus sp.', 'Bacteria; Chloroflexi; Chloroflexia; Chloroflexales; Roseiflexineae; Roseiflexaceae; Roseiflexus')\n",
      "(2240894544, 'MAG: Armatimonadetes bacterium isolate HOT.MAX.079, whole genome shotgun sequencing project', 'Armatimonadetes bacterium (hot springs metagenome)', 'Armatimonadetes bacterium', 'Bacteria; Armatimonadetes')\n",
      "(2240894509, 'MAG: Meiothermus ruber isolate HOT.MAX.085, whole genome shotgun sequencing project', 'Meiothermus ruber (hot springs metagenome)', 'Meiothermus ruber', 'Bacteria; Deinococcus-Thermus; Deinococci; Thermales; Thermaceae; Meiothermus')\n",
      "(2240894508, 'MAG: Thermaceae bacterium isolate HOT.MAX.091, whole genome shotgun sequencing project', 'Thermaceae bacterium (hot springs metagenome)', 'Thermaceae bacterium', 'Bacteria; Deinococcus-Thermus; Deinococci; Thermales; Thermaceae; unclassified Thermaceae')\n",
      "(2240894507, 'MAG: Thermoflavifilum sp. isolate HOT.MAX.088, whole genome shotgun sequencing project', 'Thermoflavifilum sp. (hot springs metagenome)', 'Thermoflavifilum sp.', 'Bacteria; Bacteroidetes; Chitinophagia; Chitinophagales; Chitinophagaceae; Thermoflavifilum')\n",
      "(2240894506, 'MAG: Armatimonadetes bacterium isolate HOT.MB2.102, whole genome shotgun sequencing project', 'Armatimonadetes bacterium (hot springs metagenome)', 'Armatimonadetes bacterium', 'Bacteria; Armatimonadetes')\n",
      "(2240894505, 'MAG: Firmicutes bacterium isolate HOT.MB2.100, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes')\n",
      "(2240894498, 'MAG: Alicyclobacillus sp. isolate HOT.MB2.104, whole genome shotgun sequencing project', 'Alicyclobacillus sp. (hot springs metagenome)', 'Alicyclobacillus sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus')\n",
      "(2240894485, 'MAG: Alicyclobacillus sp. isolate HOT.MB2.19, whole genome shotgun sequencing project', 'Alicyclobacillus sp. (hot springs metagenome)', 'Alicyclobacillus sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus')\n",
      "(2240894483, 'MAG: Anaerolineae bacterium isolate HOT.MB2.30, whole genome shotgun sequencing project', 'Anaerolineae bacterium (hot springs metagenome)', 'Anaerolineae bacterium', 'Bacteria; Chloroflexi; Anaerolineae; unclassified Anaerolineae')\n",
      "(2240894482, 'MAG: Bryobacteraceae bacterium isolate HOT.MB2.26, whole genome shotgun sequencing project', 'Bryobacteraceae bacterium (hot springs metagenome)', 'Bryobacteraceae bacterium', 'Bacteria; Acidobacteria; Bryobacterales; Bryobacteraceae')\n",
      "(2240894481, 'MAG: Pirellulales bacterium isolate HOT.MB2.34, whole genome shotgun sequencing project', 'Pirellulales bacterium (hot springs metagenome)', 'Pirellulales bacterium', 'Bacteria; Planctomycetes; Planctomycetia; Pirellulales; unclassified Pirellulales')\n",
      "(2240894480, 'MAG: Firmicutes bacterium isolate HOT.MB2.13, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes')\n",
      "(2240894479, 'MAG: Alicyclobacillus sp. isolate HOT.MB2.91, whole genome shotgun sequencing project', 'Alicyclobacillus sp. (hot springs metagenome)', 'Alicyclobacillus sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus')\n",
      "(2240894478, 'MAG: candidate division WOR-3 bacterium isolate HOT.MB2.69, whole genome shotgun sequencing project', 'candidate division WOR-3 bacterium (hot springs metagenome)', 'candidate division WOR-3 bacterium', 'Bacteria; candidate division WOR-3')\n",
      "(2240894477, 'MAG: Ralstonia sp. isolate MAG-2, whole genome shotgun sequencing project', 'Ralstonia sp. (snow metagenome)', 'Ralstonia sp.', 'Bacteria; Proteobacteria; Betaproteobacteria; Burkholderiales; Burkholderiaceae; Ralstonia')\n",
      "(2240894475, 'MAG: Firmicutes bacterium isolate HOT.MB2.4, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes')\n",
      "(2240894467, 'MAG: Candidatus Pacearchaeota archaeon isolate HOT.MB2.108, whole genome shotgun sequencing project', 'Candidatus Pacearchaeota archaeon (hot springs metagenome)', 'Candidatus Pacearchaeota archaeon', 'Archaea; Candidatus Pacearchaeota')\n",
      "(2240894466, 'MAG: Ralstonia pickettii isolate MAG-3, whole genome shotgun sequencing project', 'Ralstonia pickettii (snow metagenome)', 'Ralstonia pickettii', 'Bacteria; Proteobacteria; Betaproteobacteria; Burkholderiales; Burkholderiaceae; Ralstonia')\n",
      "(2240894465, 'MAG: Ignavibacterium sp. isolate HOT.MB2.1, whole genome shotgun sequencing project', 'Ignavibacterium sp. (hot springs metagenome)', 'Ignavibacterium sp.', 'Bacteria; Ignavibacteriae; Ignavibacteria; Ignavibacteriales; Ignavibacteriaceae; Ignavibacterium; unclassified Ignavibacterium')\n",
      "(2240894464, 'MAG: Acidobacteriia bacterium isolate HOT.MB2.42, whole genome shotgun sequencing project', 'Acidobacteriia bacterium (hot springs metagenome)', 'Acidobacteriia bacterium', 'Bacteria; Acidobacteria')\n",
      "(2240894463, 'MAG: Leptolyngbyaceae cyanobacterium HOT.MB2.61, whole genome shotgun sequencing project', 'Leptolyngbyaceae cyanobacterium HOT.MB2.61 (hot springs metagenome)', 'Leptolyngbyaceae cyanobacterium HOT.MB2.61', 'Bacteria; Cyanobacteria; Pseudanabaenales; Leptolyngbyaceae; unclassified Leptolyngbyaceae')\n",
      "(2240894462, 'MAG: Firmicutes bacterium isolate HOT.MB2.53, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes')\n",
      "(2240894461, 'MAG: Acetobacteraceae bacterium isolate HOT.MB2.46, whole genome shotgun sequencing project', 'Acetobacteraceae bacterium (hot springs metagenome)', 'Acetobacteraceae bacterium', 'Bacteria; Proteobacteria; Alphaproteobacteria; Rhodospirillales; Acetobacteraceae; unclassified Acetobacteraceae')\n",
      "(2240894460, 'MAG: Anaerolineae bacterium isolate HOT.MB2.62, whole genome shotgun sequencing project', 'Anaerolineae bacterium (hot springs metagenome)', 'Anaerolineae bacterium', 'Bacteria; Chloroflexi; Anaerolineae; unclassified Anaerolineae')\n",
      "(2240894459, 'MAG: Armatimonadetes bacterium isolate HOT.MB2.58, whole genome shotgun sequencing project', 'Armatimonadetes bacterium (hot springs metagenome)', 'Armatimonadetes bacterium', 'Bacteria; Armatimonadetes')\n",
      "(2240894458, 'MAG: Firmicutes bacterium isolate HOT.MB2.5, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes')\n",
      "(2240894457, 'MAG: Rubrobacteraceae bacterium isolate HOT.MB2.74, whole genome shotgun sequencing project', 'Rubrobacteraceae bacterium (hot springs metagenome)', 'Rubrobacteraceae bacterium', 'Bacteria; Actinobacteria; Rubrobacteria; Rubrobacterales; Rubrobacteraceae; unclassified Rubrobacteraceae')\n",
      "(2240894456, 'MAG: Acidobacterium ailaaui isolate HOT.MB2.78, whole genome shotgun sequencing project', 'Acidobacterium ailaaui (hot springs metagenome)', 'Acidobacterium ailaaui', 'Bacteria; Acidobacteria; Acidobacteriales; Acidobacteriaceae; Acidobacterium')\n",
      "(2240894455, 'MAG: Gorillibacterium sp. isolate HOT.MB2.94, whole genome shotgun sequencing project', 'Gorillibacterium sp. (hot springs metagenome)', 'Gorillibacterium sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Paenibacillaceae; Gorillibacterium; unclassified Gorillibacterium')\n",
      "(2240894454, 'MAG: Janthinobacterium lividum isolate MAG-1, whole genome shotgun sequencing project', 'Janthinobacterium lividum (snow metagenome)', 'Janthinobacterium lividum', 'Bacteria; Proteobacteria; Betaproteobacteria; Burkholderiales; Oxalobacteraceae; Janthinobacterium')\n",
      "(2240894453, 'MAG: Alicyclobacillus sp. isolate HOT.MB2.88, whole genome shotgun sequencing project', 'Alicyclobacillus sp. (hot springs metagenome)', 'Alicyclobacillus sp.', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus; unclassified Alicyclobacillus')\n",
      "(2240894452, 'MAG: Peptococcaceae bacterium isolate HOT.MB2.86, whole genome shotgun sequencing project', 'Peptococcaceae bacterium (hot springs metagenome)', 'Peptococcaceae bacterium', 'Bacteria; Firmicutes; Clostridia; Eubacteriales; Peptococcaceae')\n",
      "(2240894451, 'MAG: Thermoleophilum sp. isolate HOT.MB2.84, whole genome shotgun sequencing project', 'Thermoleophilum sp. (hot springs metagenome)', 'Thermoleophilum sp.', 'Bacteria; Actinobacteria; Thermoleophilia; Thermoleophilales; Thermoleophilaceae; Thermoleophilum; unclassified Thermoleophilum')\n",
      "(2240894450, 'MAG: Firmicutes bacterium isolate HOT.MB2.40, whole genome shotgun sequencing project', 'Firmicutes bacterium (hot springs metagenome)', 'Firmicutes bacterium', 'Bacteria; Firmicutes')\n",
      "(2240894449, 'MAG: Flavobacterium micromati isolate MAG-5, whole genome shotgun sequencing project', 'Flavobacterium micromati (snow metagenome)', 'Flavobacterium micromati', 'Bacteria; Bacteroidetes; Flavobacteriia; Flavobacteriales; Flavobacteriaceae; Flavobacterium')\n",
      "(2240894448, 'MAG: Alicyclobacillus mali isolate HOT.MB2.97, whole genome shotgun sequencing project', 'Alicyclobacillus mali (hot springs metagenome)', 'Alicyclobacillus mali', 'Bacteria; Firmicutes; Bacilli; Bacillales; Alicyclobacillaceae; Alicyclobacillus')\n",
      "(2240893217, 'Fusobacterium nucleatum YWH7199, whole genome shotgun sequencing project', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium')\n",
      "(2240893216, 'Fusobacterium nucleatum YWH7199 contig0283, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium')\n",
      "(2240893215, 'Fusobacterium nucleatum YWH7199 contig0282, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium')\n",
      "(2240893214, 'Fusobacterium nucleatum YWH7199 contig0281, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium')\n",
      "(2240893213, 'Fusobacterium nucleatum YWH7199 contig0280, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium')\n",
      "(2240893212, 'Fusobacterium nucleatum YWH7199 contig0279, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium')\n",
      "(2240893211, 'Fusobacterium nucleatum YWH7199 contig0277, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium')\n",
      "(2240893210, 'Fusobacterium nucleatum YWH7199 contig0276, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium')\n",
      "(2240893209, 'Fusobacterium nucleatum YWH7199 contig0275, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium')\n",
      "(2240893208, 'Fusobacterium nucleatum YWH7199 contig0274, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium')\n",
      "(2240893207, 'Fusobacterium nucleatum YWH7199 contig0273, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium')\n",
      "(2240893206, 'Fusobacterium nucleatum YWH7199 contig0272, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium')\n",
      "(2240893205, 'Fusobacterium nucleatum YWH7199 contig0271, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium')\n",
      "(2240893204, 'Fusobacterium nucleatum YWH7199 contig0270, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium')\n",
      "(2240893203, 'Fusobacterium nucleatum YWH7199 contig0268, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium')\n",
      "(2240893202, 'Fusobacterium nucleatum YWH7199 contig0267, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium')\n",
      "(2240893201, 'Fusobacterium nucleatum YWH7199 contig0266, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium')\n",
      "(2240893200, 'Fusobacterium nucleatum YWH7199 contig0265, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium')\n",
      "(2240893199, 'Fusobacterium nucleatum YWH7199 contig0264, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium')\n",
      "(2240893198, 'Fusobacterium nucleatum YWH7199 contig0263, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium')\n",
      "(2240893197, 'Fusobacterium nucleatum YWH7199 contig0262, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium')\n",
      "(2240893196, 'Fusobacterium nucleatum YWH7199 contig0261, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium')\n",
      "(2240893195, 'Fusobacterium nucleatum YWH7199 contig0260, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium')\n",
      "(2240893194, 'Fusobacterium nucleatum YWH7199 contig0259, whole genome shotgun sequence', 'Fusobacterium nucleatum YWH7199', 'Fusobacterium nucleatum YWH7199', 'Bacteria; Fusobacteria; Fusobacteriales; Fusobacteriaceae; Fusobacterium')\n",
      "This is the final list of Id's [['2240894608'], ['2240894606'], ['2240894605'], ['2240894604'], ['2240894603'], ['2240894602'], ['2240894601'], ['2240894593'], ['2240894592'], ['2240894591'], ['2240894590'], ['2240894589'], ['2240894588'], ['2240894583'], ['2240894582'], ['2240894581'], ['2240894580'], ['2240894574'], ['2240894573'], ['2240894572'], ['2240894571'], ['2240894570'], ['2240894569'], ['2240894568'], ['2240894567'], ['2240894566'], ['2240894565'], ['2240894564'], ['2240894563'], ['2240894562'], ['2240894555'], ['2240894553'], ['2240894552'], ['2240894551'], ['2240894550'], ['2240894549'], ['2240894548'], ['2240894547'], ['2240894546'], ['2240894545'], ['2240894544'], ['2240894509'], ['2240894508'], ['2240894507'], ['2240894506'], ['2240894505'], ['2240894498'], ['2240894485'], ['2240894483'], ['2240894482'], ['2240894481'], ['2240894480'], ['2240894479'], ['2240894478'], ['2240894477'], ['2240894475'], ['2240894467'], ['2240894466'], ['2240894465'], ['2240894464'], ['2240894463'], ['2240894462'], ['2240894461'], ['2240894460'], ['2240894459'], ['2240894458'], ['2240894457'], ['2240894456'], ['2240894455'], ['2240894454'], ['2240894453'], ['2240894452'], ['2240894451'], ['2240894450'], ['2240894449'], ['2240894448'], ['2240893217'], ['2240893216'], ['2240893215'], ['2240893214'], ['2240893213'], ['2240893212'], ['2240893211'], ['2240893210'], ['2240893209'], ['2240893208'], ['2240893207'], ['2240893206'], ['2240893205'], ['2240893204'], ['2240893203'], ['2240893202'], ['2240893201'], ['2240893200'], ['2240893199'], ['2240893198'], ['2240893197'], ['2240893196'], ['2240893195'], ['2240893194']]\n"
     ]
    }
   ],
   "source": [
    "# -*- coding: utf-8 -*-\n",
    "import pandas as pd\n",
    "import re\n",
    "import json\n",
    "import requests\n",
    "from contextlib import closing\n",
    "import sqlite3\n",
    "\n",
    "# Imprting prefect\n",
    "import prefect\n",
    "from prefect import task, Flow, Parameter\n",
    "\n",
    "\n",
    "from elasticsearch import helpers\n",
    "\n",
    "\n",
    "def fetch_data_from_id(id, query):\n",
    "    print('Id', id)\n",
    "    response = requests.get(\n",
    "        'https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=nuccore&retmode=xml&tool=rex-ncbi&id={}'.format(id))\n",
    "    if response.status_code == 400:\n",
    "        raise RuntimeError(\"Invalid PMID/Search Query ({})\".format(query))\n",
    "    return get_data_from_xml(response.content)\n",
    "\n",
    "#@task\n",
    "def fetch_data_from_query(query):\n",
    "    query_response = requests.get(\n",
    "            'https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=nuccore&retmode=json&tool=rex-ncbi&term=%s&retmax=100' % str(query)).json()\n",
    "    nuccore_list = query_response['esearchresult']['idlist']     \n",
    "    \n",
    "    ID_LIST=[]\n",
    "    all_data = []\n",
    "    json_dataset = []\n",
    "    ids_json = {}\n",
    "    for i in nuccore_list:\n",
    "        output = fetch_data_from_id(i, query)\n",
    "        #print(\"-------------------------------------------------------\")\n",
    "        #print(\"this is \", nuccore_list)\n",
    "        print(output)\n",
    "        \n",
    "        json_data = {}\n",
    "        json_data['id'] = i\n",
    "        json_data['title'] = output[0]\n",
    "        json_data['source'] = output[1] \n",
    "        json_data['organism'] = output[2]\n",
    "        json_data['taxonomy'] = output[3]\n",
    "        ids_json[query] = nuccore_list\n",
    "        json_dataset.append(json_data)\n",
    "        all_data.append([i, output[0], output[1], output[2] , output[3]])\n",
    "        ID_LIST.append([i])\n",
    "        \n",
    "    print(\"this is the id list\",ID_LIST)\n",
    "    return all_data\n",
    "\n",
    "\n",
    "\n",
    "#@task\n",
    "def fetch_id_from_query(query):\n",
    "    query_response = requests.get(\n",
    "            'https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=nuccore&retmode=json&tool=rex-ncbi&term=%s&retmax=100' % str(query)).json()\n",
    "    nuccore_list = query_response['esearchresult']['idlist']     \n",
    "    \n",
    "    ID_LIST=[]\n",
    "    for i in nuccore_list:\n",
    "        output = fetch_data_from_id(i, query)\n",
    "        #print(\"-------------------------------------------------------\")\n",
    "        #print(\"this is \", nuccore_list)\n",
    "        print(output)\n",
    "        \n",
    "        json_data = {}\n",
    "        json_data['id'] = i\n",
    "        ID_LIST.append([i])\n",
    "    print(\"--------------------------------\")\n",
    "    print(\"this is the id list\",ID_LIST)\n",
    "    return ID_LIST\n",
    "\n",
    "\n",
    "\n",
    "#fetching only the id's from the nuccore db -- checpoint1\n",
    "\n",
    "def fetch_data_from_id(id, query):\n",
    "    print('Id', id)\n",
    "    response = requests.get(\n",
    "        'https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=nuccore&retmode=xml&tool=rex-ncbi&id={}'.format(id))\n",
    "    if response.status_code == 400:\n",
    "        raise RuntimeError(\"Invalid PMID/Search Query ({})\".format(query))\n",
    "    return get_data_from_xml(response.content)\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "import xmltodict\n",
    "def get_data_from_xml(response):\n",
    "    print(response)\n",
    "    # url = \"https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=nuccore&retmode=xml&tool=rex-ncbi&id=2230715355\"\n",
    "    # response = requests.get(url)\n",
    "    data = xmltodict.parse(response)\n",
    "    title = data['GBSet']['GBSeq']['GBSeq_definition']\n",
    "    source = data['GBSet']['GBSeq']['GBSeq_source']\n",
    "    organism = data['GBSet']['GBSeq']['GBSeq_organism']\n",
    "    taxonomy = data['GBSet']['GBSeq']['GBSeq_taxonomy']\n",
    "\n",
    "    res = [title, source,  organism, taxonomy]\n",
    "\n",
    "    return res\n",
    "    \n",
    "    \n",
    "\n",
    "#@task\n",
    "def store_ncbi_data(parsed):\n",
    "    create_script = 'CREATE TABLE IF NOT EXISTS nuccore_table (id INT, title TEXT, source TEXT, organism TEXT, taxonomy TEXT)'\n",
    "    insert_cmd = \"INSERT INTO nuccore_table VALUES (?, ?, ?, ?, ?)\"\n",
    "    select_cmd = \"SELECT * FROM nuccore_table\"\n",
    "\n",
    "\n",
    "    with closing(sqlite3.connect(\"prefect_test.db\")) as conn:\n",
    "        with closing(conn.cursor()) as cursor:\n",
    "            cursor.executescript(create_script)\n",
    "            cursor.executemany(insert_cmd, parsed)\n",
    "            a = cursor.execute(select_cmd)\n",
    "            conn.commit()\n",
    "\n",
    "            \n",
    "            #Delete the duplicate data (deduplicate)\n",
    "            cursor.execute('''DELETE FROM nuccore_table WHERE EXISTS \n",
    "            ( SELECT 1 FROM nuccore_table t2 WHERE nuccore_table.id = t2.id AND nuccore_table.rowid > t2.rowid)''')\n",
    "            conn.commit()\n",
    "\n",
    "\n",
    "            # Display data inserted -- test done for only 4 ids \n",
    "            print(\"Data Inserted in the table: \")\n",
    "            data=cursor.execute('''SELECT * FROM nuccore_table''')\n",
    "            for row in data:\n",
    "                print(row)\n",
    "\n",
    "\n",
    "#query = Parameter('NCBI', default='NCBI')\n",
    "parsed = fetch_data_from_query('NCBI')\n",
    "parsed_id = fetch_id_from_query('NCBI')\n",
    "store_ncbi_data(parsed)\n",
    "print(\"This is the final list of Id's\",parsed_id)\n",
    "\n",
    "   "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "id": "679d6d23",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2240894608,2240894606,2240894605,2240894604,2240894603,2240894602,2240894601,2240894593,2240894592,2240894591,2240894590,2240894589,2240894588,2240894583,2240894582,2240894581,2240894580,2240894574,2240894573,2240894572,2240894571,2240894570,2240894569,2240894568,2240894567,2240894566,2240894565,2240894564,2240894563,2240894562,2240894555,2240894553,2240894552,2240894551,2240894550,2240894549,2240894548,2240894547,2240894546,2240894545,2240894544,2240894509,2240894508,2240894507,2240894506,2240894505,2240894498,2240894485,2240894483,2240894482,2240894481,2240894480,2240894479,2240894478,2240894477,2240894475,2240894467,2240894466,2240894465,2240894464,2240894463,2240894462,2240894461,2240894460,2240894459,2240894458,2240894457,2240894456,2240894455,2240894454,2240894453,2240894452,2240894451,2240894450,2240894449,2240894448,2240893217,2240893216,2240893215,2240893214,2240893213,2240893212,2240893211,2240893210,2240893209,2240893208,2240893207,2240893206,2240893205,2240893204,2240893203,2240893202,2240893201,2240893200,2240893199,2240893198,2240893197,2240893196,2240893195,2240893194\n"
     ]
    }
   ],
   "source": [
    "new_id = [ y for x in list(parsed_id) for y in x]\n",
    "new_id_pr=','.join(map(str,new_id))\n",
    "print(new_id_pr)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "id": "a0b27d0c",
   "metadata": {},
   "outputs": [],
   "source": [
    "#multi fasta files --loading based on the list of ids\n",
    "handle=Entrez.efetch(db='nucleotide',id=list(new_id_pr),rettype='fasta')\n",
    "record=SeqIO.parse(handle,'fasta')\n",
    "outputname='D:/NCBI_Nuccore/Fasta_Files_and_Pangolin/Nuccore_sequence.fasta'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "id": "ca5d294f",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2240894608,2240894606,2240894605,2240894604,2240894603,2240894602,2240894601,2240894593,2240894592,2240894591,2240894590,2240894589,2240894588,2240894583,2240894582,2240894581,2240894580,2240894574,2240894573,2240894572,2240894571,2240894570,2240894569,2240894568,2240894567,2240894566,2240894565,2240894564,2240894563,2240894562,2240894555,2240894553,2240894552,2240894551,2240894550,2240894549,2240894548,2240894547,2240894546,2240894545,2240894544,2240894509,2240894508,2240894507,2240894506,2240894505,2240894498,2240894485,2240894483,2240894482,2240894481,2240894480,2240894479,2240894478,2240894477,2240894475,2240894467,2240894466,2240894465,2240894464,2240894463,2240894462,2240894461,2240894460,2240894459,2240894458,2240894457,2240894456,2240894455,2240894454,2240894453,2240894452,2240894451,2240894450,2240894449,2240894448,2240893217,2240893216,2240893215,2240893214,2240893213,2240893212,2240893211,2240893210,2240893209,2240893208,2240893207,2240893206,2240893205,2240893204,2240893203,2240893202,2240893201,2240893200,2240893199,2240893198,2240893197,2240893196,2240893195,2240893194\n"
     ]
    }
   ],
   "source": [
    "print(new_id_pr)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "id": "24cdcf73",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "691"
      ]
     },
     "execution_count": 40,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "SeqIO.write(record,outputname,'fasta')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "e27a8219",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.7"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
